{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "skip"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'start_slideshow_at': 'selected', 'theme': 'serif', 'transition': 'fade'}"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# setup the presentation settings\n",
    "from notebook.services.config import ConfigManager\n",
    "cm = ConfigManager()\n",
    "cm.update('livereveal', {\n",
    "              'theme': 'serif',\n",
    "              'start_slideshow_at': 'selected',\n",
    "              'transition': 'fade'\n",
    "})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "skip"
    }
   },
   "outputs": [],
   "source": [
    "# some good ol' imports\n",
    "import pandas as pd, numpy as np\n",
    "from emperor import Emperor, nbinstall\n",
    "from skbio import OrdinationResults\n",
    "\n",
    "from emperor.qiime_backports.parse import parse_mapping_file\n",
    "from emperor.qiime_backports.format import format_mapping_file\n",
    "\n",
    "from skbio.io.util import open_file\n",
    "\n",
    "nbinstall()\n",
    "\n",
    "def load_mf(fn):\n",
    "    with open_file(fn) as f:\n",
    "        mapping_data, header, _ = parse_mapping_file(f)\n",
    "        _mapping_file = pd.DataFrame(mapping_data, columns=header)\n",
    "        _mapping_file.set_index('SampleID', inplace=True)\n",
    "    return _mapping_file\n",
    "\n",
    "def write_mf(f, _df):\n",
    "    with open(f, 'w') as fp:\n",
    "        lines = format_mapping_file(['SampleID'] + _df.columns.tolist(),\n",
    "                                    list(_df.itertuples()))\n",
    "        fp.write(lines+'\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "skip"
    }
   },
   "outputs": [],
   "source": [
    "def one_more_thing():\n",
    "    return 'We are hiring, contact robknight@ucsd.edu'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Emperor: interactive $\\beta$-diversity visualization"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "-"
    }
   },
   "outputs": [],
   "source": [
    "# @ElDeveloper on GitHub\n",
    "__presenter__ = 'Yoshiki Vazquez-Baeza'\n",
    "__email__ = 'yoshiki@ucsd.edu'\n",
    "\n",
    "\n",
    "__license__ = 'BSD-3'\n",
    "__url__ = 'https://github.com/biocore/emperor'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<table style='border:none; width:100%;' cellspacing=\"0\" cellpadding=\"0\">\n",
    "<tr style='border:none;'>\n",
    "<td style='border:none;'>\n",
    "<img src=\"https://upload.wikimedia.org/wikipedia/commons/f/f6/UCSD_logo.png\" alt=\"ucsd\" style=\"height: 50px;\"/>\n",
    "</td>\n",
    "<td style='border:none;'>\n",
    "<img src=\"./images/knight-logo.png\" alt=\"knight-lab\" style=\"height: 100px;\"/>\n",
    "</td>\n",
    "</tr>\n",
    "</table>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Outline\n",
    "\n",
    "- Background (why $\\beta$-diversity).\n",
    "\n",
    "- What is Emperor.\n",
    "\n",
    "- How can we use Emperor.\n",
    "    \n",
    "- Analyzing a use case."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Outline\n",
    "\n",
    "- **Background (why $\\beta$-diversity).**\n",
    "\n",
    "- What is Emperor.\n",
    "\n",
    "- How can we use Emperor.\n",
    "    \n",
    "- Analyzing a use case."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Microscopic worlds on and inside our bodies\n",
    "\n",
    "<img src='http://news.nationalgeographic.com/content/dam/news/photos/000/819/81998.ngsversion.1422281191282.adapt.1900.1.jpg' style='height:500px'>\n",
    "\n",
    "Photograph by Martin Oeggerli, Micronaut, Supported by School of Life Sciences."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# How do we make sense out of them?\n",
    "\n",
    "![microbe](./images/microbes.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# How do we make sense out of them?\n",
    "\n",
    "![microbe](./images/microbes-metric.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# What is $\\beta$-diversity\n",
    "\n",
    "- Comparison of two individual communities to determine how similar they are.\n",
    "\n",
    "<img src=\"./images/table-2.png\" alt=\"ucsd\" style=\"height: 500px;\"/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# What is $\\beta$-diversity\n",
    "\n",
    "- Comparison of two individual communities to determine how similar they are.\n",
    "\n",
    "<img src=\"./images/distance-matrix.png\" alt=\"dm\"/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# What is $\\beta$-diversity\n",
    "\n",
    "- Comparison of two individual samples to determine how similar they are.\n",
    "- Elucidate patterns.\n",
    "\n",
    "![costello](./images/costello.png)\n",
    "\n",
    "<p><small>\n",
    "    Bacterial community variation in human body habitats across space and time.\n",
    "    Costello EK et al. 2009.\n",
    "</small></p>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# What is $\\beta$-diversity\n",
    "\n",
    "- Comparison of two individual samples to determine how similar they are.\n",
    "- Elucidate patterns.\n",
    "\n",
    "![costello](./images/costello-colored.png)\n",
    "\n",
    "<p><small>\n",
    "    Bacterial community variation in human body habitats across space and time.\n",
    "    Costello EK et al. 2009.\n",
    "</small></p>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# What is $\\beta$-diversity\n",
    "\n",
    "- Comparison of two individual samples to determine how similar they are.\n",
    "- Elucidate patterns.\n",
    "\n",
    "![costello](./images/costello-all.png)\n",
    "\n",
    "<p><small>\n",
    "    Bacterial community variation in human body habitats across space and time.\n",
    "    Costello EK et al. 2009.\n",
    "</small></p>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "outputs": [],
   "source": [
    "from skbio import OrdinationResults\n",
    "\n",
    "coordinates = OrdinationResults.read('costello/unweighted_unifrac_pc.txt')\n",
    "metadata = load_mf('costello/mapping-file.txt')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAfQAAAHWCAYAAACBsnu3AAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAAPYQAAD2EBqD+naQAAIABJREFUeJzsvXecVOW9x/8+08vOLrtLL1IWBFQEBcGKJSq2RKPXmERN\nMzEaYxKN+SU35aZqft6bn16NJVcxRqPGWGOsGI2CAoKIgjTpZSkLbJ8+Z875/TE849lhet/Z5/16\n5aXB4cwzz5k5n+fblbfeektHIpFIJBJJv8ZU6QVIJBKJRCIpHCnoEolEIpHUAFLQJRKJRCKpAaSg\nSyQSiURSA0hBl0gkEomkBpCCLpFIJBJJDSAFXSKRSCSSGkAKukQikUgkNYAUdIlEIpFIagBLti88\n44wzSrgMiUQikUgkyXj77bezep200CUSiUQiqQGkoEskEolEUgNIQZdIJBKJpAaQgi6RSCQSSQ0g\nBV0ikUgkkhpACrpEIpFIJDWAFHSJRCKRSGoAKegSiUQikdQAUtAlEolEIqkBpKBLJBKJRFIDSEGX\nSCQSiaQGkIIukUgkEkkNIAVdIpFIJJIaQAq6RCKRSCQ1gBR0iUQikUhqACnoEolEIpHUAFLQJRKJ\nRCKpAaSgSyQSiURSA0hBl0gkEomkBpCCLpFIJBJJDSAFXSKRSCSSGkAKukQikUgkNYAUdIlEIpFI\nagAp6BKJRCKR1ABS0CUSiUQiqQGkoEskEolEUgNIQZdIJBKJpAaQgi6RFICmaWiaVullSCQSCZZK\nL0Ai6Y/ouo6qqgSDQVRVxWKxYLFYMJvNmM1mTCYTiqJUepkSiWQAIQVdIskRTdOIRCJ9LHNVVYlE\nIiiKgqIomEymPiIvBV4ikZQaKegSSZbouk40GqW3txdN03C73XHxNplM8dfouo6maQSDQYCkAm82\nm6XASySSoiIFXSLJAqNVHo1G0XU9qSALCx3AbDbHBV7XdUKhEKFQKC7wZrP5MDe9RCKR5IsUdIkk\nDcIqV1UVTdPirnMRQ/f7/ei6ntK1nkzg4dMDQjgcjr8mmQUvrXiJRJItUtAlkhRomoaqqkSjUYA+\nYq5pGj09PXGrOhwOx/+esL6TudaN4i4QFnyiwMs4vEQiyQUp6BJJAkKwI5FI3LUuhFRY67qu43A4\nsFgsaJqGoihxd7x4TSQSAfoKvDHeLjAKuHh/4RmIRqMEg8H4f7darSkPCxKJZGAjBV0iMSAsZWGV\nC7EVMXC/3w/ERNrlcsVFW1GUuMiK6xhF2Sjwxtcmc62nEnhd1/H5fESjUex2e9yCN15LxuElkoGL\nFHSJhL5WuTFWDjHXu8/nIxKJYLfb40KdDqMoW63W+HWi0Wj8n6qqxl+baMWnisOLv2c2m9E0jXA4\n3MeLIOPwEsnARQq6ZMCTaJUbBdVoldfV1WGz2fD5fHm9T6K73WjBR6PRPnF4o7gnc60LK994rWSJ\nduI6Mg4vkdQ+UtAlA5poNJrSKvf7/YTDYaxWK263u+jubKNFDZ+KcjKBF8IusuSTXStZop2qqvGO\nduJAkeimlwIvkdQGUtAlAxKj2EFfqzwcDsetcLfbjc1mO0z0UglrISSLwxvd9CIZD8Dv9x/mpk+8\nVrI4vGx4I5HULlLQJQMO4ZqORqOHiZ7f7ycUCmGxWKirq0tqlScrQysFyQQ+HA4TiUQwmUyHxeET\nM+kz1cOL/4XD4XjDGxmHl0j6L1LQJQMGYZX39PRgsViw2+1xoYpEIvh8PjRNw+Vy9flv1YJRWB0O\nB3B4HF4IPNDHrZ5O4MV1oG/DG/jU1S8Hz0gk1Y8UdMmAwJjBHolE4panrusEAgGCwSAWiwWPx9Mn\nFl3tJIvDC3EXWfACY5JdLg1v5OAZiaR/IAVdUtMIgRNNYowudFVV43XdTqcTh8ORszBVm5BlSrSL\nRCIFN7yRcXiJpDqRgi6pWRLHnBpbt0YiEQKBAGazmfr6+rgAZoO4RiXI58BR7IY3IAfPSCTViBR0\nSc2RaqAKEK81V1UVh8OB0+kcUNZkrg1vjG76XAfPGA8KiS566aaXSIqPFHRJTZFuoIqxSYzNZsPl\nchXlPStlrReLTA1vckm0E68xXitx8Iw4ZNntdhmHl0iKiBR0SU1gtMoTB6pomobX60VVVex2O+Fw\nuGgu4P4u5snIp+FNqth5sjh8JBKJHxLk4BmJpHhIQZf0e9INVAmHw4e1bhWuYEl2ZNPwJjHRzvjP\nxGuJf1oslj5xeCHuxuvIOLxEkj1S0CX9lmwHqgj3uhCEQpPaKpkUJxBeiEqQTaKd2J9kiXaJ10oX\nhzf2pZcNbySS9EhBl/RL0g1UMbZuFVa5pHSkSrRLnA8vXisOI9FoNOs4vBw8I5FkRgq6pF+RzioX\n88JLOVBFkh3GQTDQN9FO9AQIBAJAfpPlEgfPyDi8RCIFXdKPEHFWVVWxWCx9xNzYujXVQBVJ5TC6\nzEVfAJvNVrREO2McXrxGNryRDDSkoEv6BcKyCwQCRCIRBg0aBBw+UCWb1q3VEAMf6KRKtDPWwhej\n4Y0cPCMZSEhBl1Q1RteqiL0KMVZVFa/Xi6ZpebduzQeji1+KQXEwinZiwxtjNn3iazNNlhMkxuFB\nDp6R1B5S0CVVS7Ixp0LQ/X4/wWAQs9lMQ0NDxQaqCMGRQpA92R6EhFgLgc+34Q2kj8PLwTOSWkEK\nuqTqSLTKE8vRINaQJN/WrcVyuYspbUC8WU3iwBNJ8Ug1WU5Y8sWIw8vBM5L+jBR0SVWRbqCKGAAC\n5DxQpdhrhE8PFeJwkKxEK13sV1IYQuAFiR3tkjW8yTRZDtLH4WXDG0k1IwVdUhVkGqji8/ni2e3i\nn5VYYygUipdbiQS8UCh0WOw3VQ12umEnksIo1WQ5cS3o2/BGVVXMZjM2my1uxcvBM5JKIgVdUnHS\nDVQRTWJMJhMej+ew2Gk51yg6z4lDhRCOxId3uhrsXGO/kvzJdbJcohWfruGNONyZzWYZh5dUDVLQ\nJRUj00CVZK1bhegXkmEuJn5lS2LnOQCv15vT+2Ub+5UCX1oyTZbL9V4Y3f5Gj0DiQUHG4SXlQAq6\npCKkGqgC6Vu3lvNBaKxxN3aeMz708yFT7DddcpekuCQ7bGVzL5LFzdM1vAmFQhnL5SSSQpGCLikr\nmQaq+P3+qmjdauw853K5sNvtJTtMpJtmliz2C8TzCKSlV1zS3YvEyXJA3BJPNVlODp6RlBMp6JKy\nkW6gSiQSibux07VuLVZTl1Rla6LHeDY17qVqLJMuuUv0QRcZ/5nGlUoKI9W9UFWVcDiMpmnxyots\nEu0gu8EzMg4vyQcp6JKSIx5aPp+PUCiEx+PpI8zG1q1ut7vkruVUD8ZoNIrX6yUajabtPFfuB6vx\nIa8oCsFgEKfT2SexK9240moUgmpcUzYYLepwOIzdbsdsNhe94Y24lhw8I8kFKeiSkmK0yoXL0miV\nl8utnWmNoVAIv9+PyWSqaI17tgiLPNtSuWoX+P5MqqRH8Z1PTLQzxuKzbXgjBs+I18tEO0kyqvup\nJenXCBexiJWLh5ToxlVI69ZiDVcxZtPb7XZcLle/fDDKUrnKkC7zPVWiXSQSKXrDGxmHl4AUdEkJ\nSNe6FaC3tzejWzsVxXhAJda4w+HZ9P2dXK1GKfClo5QNb8T15OAZCUhBlxSZdANVxMNG1/WKurXF\nA9Dr9RaUTV/JaWu5vne+5VmlcOXWwujaQj5DuoY3qSbLpesumCoOn67hjbieFPjaQgq6pCiks8qN\nrVuBrGaWp8KYTJcPRismXTZ9Nmvoz2QqlUvlFpYiUBqKPVkuWRxeDp6pfaSgSwom00AVkWzmdDoJ\nBAIVS3wT5WjigWe328u+jmolF7dwprivpHCK6VHJFIcPBAJomtanJ71seNM/kYIuyZt0A1WSJZsJ\nK6PcLtfEcjRx0JCkJtEtnEvcdyCJQLkOp7k0H8rUmyAxDi+uIypSEu+rjMP3H6SgS/Ii00AVv98P\nFD/ZLBeXe6pyNDEtTZI96eK+yQad1HqpXKXzALLxqOTamyCxHXGmOLwU+OpDCrokJzJZ5aJ1q3Gg\nSrJrlJpSlqMVGsevFTINOkk1yWyg71spSHXgStWbINP9SBWHl4Nnqhsp6JKsSWWVQ9+BKm63O2l8\nulw/9GzK0aSoFJ90pXKJcV/RYEhaeaUjU2+CxCFD4XA4ZfZ7uoY3cvBM9SAFXZIRY51r4pjTVBPJ\nSkU661jXdXw+X8bhLlI8ykOqxC4R8khWC9/fyqn6yzoh9f0IhUKH9SbIJ9EO5OCZSiMFXZIWkSjT\n29uLpml9+rDn2rq1lK5q41ryLUeTlBZhkYuHvM1mq1gtvKRvbF3XdVwuVx83fWKiXbrxsUZxF8jB\nM+VHCrokKYljTsWficQ3UQJmsVgKqisvxjqrZS2S7BEP92SZ26kEpZpK5WotZJN4P5JVNiRreGO8\nH9k2vJGDZ0qHFHTJYaQbc6qqKj6fL6/WrcWw0I3XyHY6Wrpr5PPgMP59IULSjVg4mQRFlsqVhlS/\nx2SJdqVoeJNu8IwcC5wbUtAlfUgcqGIUL03T6OnpwWw2V3wimXCxV3I6mogHi4OP7IleXPJpkSrj\ntKUlVeKjuC/FbHgjB8/kjhR0CZC5datIiHM4HDidzor9mIT7X8yirsR0NGHRiPp2kUWf+ECTAp+c\nQjw02bZINWbQl3L/a+We5vs5hNgKsml4kypskkykU8XhZcOb5EhBl6QdqCIas4gfi8vlyvt9CnW5\nG8vR7HY7brc777Xki6i1h1gjDrfb3WfGe7oWnVLgi0+mUrlSjY2tpRh6MT9LNg1vsp0sZ3yNca1G\n40O8RrjpB3ocXgr6ACbbgSoie71S7VKNDWusViuRSKSgxLd8Y+iRSASv1xv//8aYvbGcL12LzkIF\nfiA+pHIhVWmWPGBVhnRhk2QdBhOt+Gzj8MbBMyJcaLfbB1zDGynoA5R0A1VE61ZFUfB4PFitVgKB\nQNFO8rlcJ1k5WmdnZ1HWkS2JmfRut5vu7u7450h3MMhF4GWpVvGR+58d5fysmToM5nLoShaHF8Nm\nhBGiKAq33norJ598Mpdeeml5PmSFkII+wMh2oEpi69Zi/OBzuUa6cjRx8CgHyTLpCyGTwMixpaUl\n3f6LJLuBtv/5VnsUi1y8Ktkk2hmvKZ4Ta9eu5dhjjy3TJ6ocUtAHEKmscsiuXSqU58dfSGlcsUg1\n2EX8t2KRSeCTNfco9hoGMrnEfI3NVSotgrVMroeuRDe98d6If/r9/ork3JQbKegDgGwHqpSjXWo6\n6zqdiCZ7bSFrSHeNUg52yWZtmQQGIBAIDBgLspxkKpVLTLQLBoP9vpSq2g8nmX4TiXF4Y4MicU98\nPp8UdEn/J91AFRGf1nW94u1SNU3D6/XGk/DSiWgp15itp6JcJAqMqqoEg8F445WB6CIuN4mlcqL/\neT7d0ySFk+rQJQ5eomLH5/Mxd+5cJk6ciMvlYseOHUQikfjfqUWkoNcoxszPnp6eeHKb+G9ioIpI\n8sqUNZ5vZniy6yRaxsIqB/qss5zkMmSmWHuRD+L9RPZuJhdxYi22pHCEWIsZBuUqlSsl1bqubDFO\nlhOiHQ6H+fznP8/SpUtZvnw577zzDjfffDMnnngip556Kl/72tcYP358pZdeVKSg1yDG1q2J4qmq\nKl6vN+uBKqUk2/npySimy90Ys89nTyqdUJTORSxKEsVrq6Vdan8XEDg8ASub7mnVKPC1lI8hPovJ\nZMLj8fBf//Vf6LpOS0sL9913Hzt27OCdd97hnnvu4dxzz5WCLqleEgeqJCZR+f3+eMyvoaEhp1ru\nQpvCJK6zEHd/sR6Cooe03++vina2xSKZizhdN7VqEPhaIlP3NFkqV34CgQAnn3wyl19+Obfccks8\nMbjW6P9PLwmQeqCKMctT07SKZY0bUVU1J3d/qQgEAlnF7Ps7yQQ+0YoH2Q89F3I52OZayVAJga+V\n+2z0RgmEJ7Curi7+Z7V6eJWC3s9JZpUbrWnR3U3X9YIs0GJY6CLLHij4YFHIOsTDMxqNFpT41l9d\nlcZ4I2Tuhy4FvrjkWipXyjyI/vodzgXRFKuQttX9BSno/Zh0Y06NrVshJqCVcicby9EgltDldDrz\nvl4hhwCR+AaxvvT5inktCVuqGHA6gc83i3sgCEiuFJIHIQ9ZfUlmofv9fpxOZ8U8geVECno/JdlA\nFTi8lruurq5P//F8yddCTyxHU1W1Iu4uY+Kbw+EgGAzWrNutULIVeKjOJK9yUcrPWu4wSS3fN5/P\nNyCsc5CC3u9IN1Clkg1RkpGsHK23t7doiXXZvk6sQyS+KYoSH+QgyUwmga/2LO5SUG5PQ7ZhEsjt\nHtSaxySZhS4EvVa/i0akoPcj0rVuNYpnYly4GL3Pc7HQCylHy2Ut2awj2QFH7F+tPczKRTqB1zQt\nrcBLikMh92AgCFuiy30gdIkDKej9AvFjjUQiSa3yUopnrlRL97lydHyTB4IYuQzXEHFMTdOqvuVo\nfyLXeyCS7BJ7nvd3kv0m/X6/tNAl1UG2A1Xcbjd2uz3pNcphoScbMZosCcVoIRdCunVk2/GtEAbC\nwyFfMpVpAfGpcv3ZeqzmtWa6B8apfkDcWKjFWviB0scdpKBXLekGquQjWqW0Jss9HS3VAaWauuBJ\nPsUoLrqu4/P5sFqtKIoiG62UiVSlckLYRV4O9L0HibPLq51UMfRCqmr6E1LQqxBhlXd3d+NwOPq4\nrYVLOxfRKuZD0SikotNaIBDIutNaMbwFydZkXEemLnjF7HonyQ9jBnd/nAnf38MFIhvebDYTiURw\nOp3xA1ZiLXx/7ygoY+iSipBolQv3pBBBo0vb4/FkXVdZTJe7wFjn7nA44g+EcmD8PMayuHKvAz4V\no3LVA/dnETFi/Bz9oZNarSNKX5PVwot/9oeGQ8kOWiKGPhCQgl4lJDaJMVqR5XZpp0IIqcioVxSl\nYtPRoG8OQbnXIXIBRBY9SLEpFvl0UpN7XhjJ9i3R3Z6u4VCim76S9yGZoEsLXVIW0rVuhZiLvdDh\nIcVyc+u6TjgcJhqN5p1RX4y1iD3zer0lTXzLtIZwOIyiKNjt9viapNgUn8ROatUi8LVwP3PtSV/t\n/QiSfR6ZFCcpC5lat0JM0CvhSk7E2P88XUZ9qREPb9GbOZ/Et0Ji6CL0Idzs9fX18TG14rBSDWJT\ny+Qr8MVM8JL5F7mXyiUrlysHfr+f5ubmsr1fJZGCXgGyGagimsQIS7gQCikVS+x/brfbCxLzfC10\nY+KbeKA7HI6815EPxrwBRVGwWq2YTKb44QvKZ032V0EpxboT9xwO74VeKwle1UymXIhyVDOkiqEf\nccQRRbl+tSMFvcxkO1DFbrcTDocr+sBJjN0LUS83iYlvYg/LiYjXi7wBkUOQiWp1F9c6ciZ8diQr\n8yoWqQTemGRXju++dLlLSoIoyUlmlYfD4cP6nhunLBVCrlZxqnI04wm7EHJZS7LENzEOsRCy3ROj\nh8KYN5Cvp0EKfGWQM+Erj3FvM3338z1oiU6aRmRSnKSoZDtQpZStW7MVn0zlaMUuf0uFaEASDocr\nlvhmbFRTqja2uQq8caqeJH+KMRO+VoS+Up8jU6hEJJmK12Z70JJZ7pKSkWzMqfjCZeo3Xqzs9Gx+\nsEYvQapytGI3qEl1vXIIaaa1GSe0ZWpUU0yyEXiIDeNJbJ1aKwJTCTJlcCcKi7gv/b3BTLUdDBM9\nKZnug/BipfOaSUGXFEwmq1wMVElnfRZT0NNdJ5cBL6W00LPt+FasfUlGviNoS/VgTxT4aDRKIBCI\nC4900ZeGTAIPxFumVlMNdq2R7UELiB+0jIN/RMhMCrokb9INVDG2bq3kNDJBJBLB6/UC6Qe8QGld\nc+XuPJfsUGDci1JNaCsUsSfiISdj8OUhUVi8Xi8WiwVFObwffX8bOFPt6zOSSuCNnTWj0Sher5fr\nr7+eo446irq6urJ52CqNFPQiIrKvg8HgYQ9RY3JVtq1bCyk3S7xOonglrqeuri6rGHWxLHSjRVvp\nznOJbXWz2YtSeglyQSbZVQ6R3AXV2WQlG6rhO1wIQuAFotmU1+slEAhw33330dPTw9lnn82cOXOY\nO3cup512GqeddlpNWu1S0IuESOAIBoP4fD7q6+vjP9x8p4AVWzSEiBrL0Sq9HpH4lktCYLJDQb6I\n03yl2+oWi1QCr6pqvxl8Uu0k+w3k02SlGgW+PyPui8lkorm5mWeffZZIJMKUKVO44YYbWLNmDfPn\nz+e2225j6dKlnHjiiRVecfGRgl4gxoEqxpIJIX7C8it3cpURowDmOh0tkWIJuqqq+P3+ioYeotEo\n3d3dmEymvNvqVjtC4EX4IFFoKiXwtS5iuTRZqaTA9/ekvmQYP4/ZbKajo4Pvfve7NDc3o+s6Gzdu\nZMKECRVcYemovSdYGRFWuXGgitEqDwaDBVl+xbaIC51KVswfvtfrrdghR1itolTQ7XbX3EMtFZmE\nRlrwpSFXgTfuvdz37Ej2rAyFQvF21RC7D5MnTy730sqGFPQ8MLZuFSfcxB+dz+criuVXqKALdyvE\nLNJKTkcTGdpAThnkiRTichfhD13X4/HygYwU+MpQTftey/fR7/djtVqrMsG1FEhBz5HE1q1GMReZ\n2gBWq5W6urqCfiyF/tASx3t6PJ6CDheFeAyM/emBvIaqFEJiuEE8JPOlWAmL1Uaphaa/J2EZKeb3\nN9O+lyq5sZZc7sna2A6kWeggBT1rMg1UMfb6huIIViECaizBcjgcBIPBivxwE2vc7XY7vb29ZV9D\nYrihp6enbE17+jPVZElWC+U4lCTbd1m9kB2Jgj6QQmpS0LMg3UCVxEYkTqeTrq6ukmSnZ/vaxHI0\nTdMIBoNl638uSKy5t9vtcfd/IWsxHqQykawXvPEaxaCcD4tKP5hyEXiZzV08SlmeWCv3JdnzwOfz\nSQtd8impBqpA8tat4ktVCesvVXlcuV3DiXXdxpr7cj08jGuoVC/4gUCuyV65HMYkqclX4BNnwtfi\nfTA+Y4Sg18qhJRNS0FOQb+vWUlh+mSz0TOVoxXqIZmOhGzu+5Zrdv3PnTt5//308Hg+nn3563nPX\njbXludTZ50otPgwLJZtYMNCn+VJ/teCrab2JAg/ZzYSvpe/wQO/jDlLQk5LtQJVU9dPl7CJW7pap\nkPyAYcwjMJlMh2XTd3d3Y7FYcDqd8dcbP8PPf/4L/vznJwiHdRQFhg9v5E9/upu5c+ce9v6pDiiJ\na0hXYVDoPaqmh3k1kyjwopzTarWiaVrV1GPXItnOhBfVOsa9rxUGmsu9du5cERCxclG7KH4Q4uHv\n8/n61E+nsvyKJejpLGtdj00E6+7uRtM0PB5PStdSMS30ZIg8Ap/Ph81mo6GhIf4Qee+99zjnnPMZ\nN24SY8ZM4IorvszWrVv7/P0HH3yQBx74G5r2a+rqNuF0vsvevTP4yle+RVtbW1ZrS7aGWmwU098R\n3yFxuHO73TidznhZUTgcJhAI4PP5CAQChMNhotFoVVmS1bSWXBDi7nA4cLvdcaEzmUxomhavRPH5\nfASDwXiosb983lRZ7tJCH4BkO1AlGxduqQW9XDPUs0Hsja7rhw13WbNmDRdffDmBwDTM5vvRtACv\nv34PK1dezcKF/2L06NEAPPjgo+j65bhc3wLAZKqjru5+enpm8Nxzz3H99dcnfW+xL8ba8kwDZoqN\nOARqmhYf1iEty+zpLx3VahHxzLBYLPH8n1xnwlcbieuSgj7AEF/i3t5eTCZTH7FOl9xVKTLNUE9G\nsS10cR3j3rjdbsxmM6tWreKRRx5hx45dbN++jUBgMHb7yyhKTGQjkfNpa5vB/fffz6233grA3r17\nsFhmHPrvq/F6f0kk8iFwkMcff5wvf/nLNDQ0JF1HIBCI5w6U+/7ouk5vb2/8wRcOhw97+JXqoNVf\nrKZckQJfPhItWkXJbSZ8tQm8zHIf4IJutMpFgpuxdasYYJJrclcpLHRjOVq+WdvFEoFoNIrf7z9s\nb5566imuu+57RKOj0LTjUNUudL0HWIyinHJor93Ayfzv/97NkiVL2bJlOx0dncDLBAKDiES+AejA\nHGAQK1a8xZw5p7Fo0RsMGTKkz2cIBAJEo9G8cgcKzf4XIxsBXC5X0ixj8T7lEPj+Ri6/pWoV+GoQ\nsVKSi8BXy0x4aaEPYMSDIbFJTKEDTIqdFCcOF7lOazOup5gIb4Zxb7xeLzff/BNU9XJstnsBE5HI\nQeBqwuFvAv8LnAG4gJ1o2miWLl0OzASmAa8RiSw89A5/Ai4/9O/r2LHjYi655BK6ukKEQiFOOeUE\nrr32Go4++uh48t3q1av58MMPaWpq4jOf+UzJTuXiYCUeZvX19fEHnNlszurhZ7FYZBOQPKkGga81\n70guh6t0Al9p70mqLPeB1N55QAu6MYNdWGzChVqujPFsEIeLQgaZFHrIEM1poG9b22XLlnH//X9i\nyZLltLd3ATcRDosWrw7ge8BlwFVACzHLeyO6rgA3Ab869LpfAv8NzDj0ehHjm4qmXc6HH95PU9Pv\nUBQPr776NO+9912eeeZRJk2axLXXXs9rry0kHLZgMqkMG1bPgw/ew0knnZT3502GsRzOYrHED4Oi\n4ZCRxIdfOdp4DkSqQeD7K8UIwVWzwENM0IcNG1aW96oGBrygC6HTNC3ePKbQASbFaOYixAOK0xe+\nEETiWygUwmw2xw86zz//PF/72jfR9SNR1Sno+h4g8cDhABRiBRW7gYeJfe104BbD6y4D7gRsgAmz\nOeaa1jQdsKMoDXg83wHA7f4CHR2f5cEH5zN06BBeemk5Ntv9NDScTTD4Iq2td/OVr3yTFSuW9Im9\nG8n1gBMKhfqUw4XD4ZzucboSooEk8KW2bqXAV45kAl/OvU9WTuvz+aTLfSAh+nxrmoaiKDQ0NBT8\n5SrEGk6spQaKMis8nzWJpLPFixdzxx13sWzZ+1itNi677LP89Kc/5Yc//AnR6OewWB4FthGzrh8A\nfmG4yp+BccCvgf8H6AVGAzuArcTc7TowCWgC1gJvEY2eeejv7waewuW6EBBWuxOL5XyWLHkan68X\nk+kazOb6cjmeAAAgAElEQVTR7N9/CtHoTnRdZ9cuH1/60pW88MLzBR3ORLmi6EUv+kIbH075kK/A\nS3JDCnxmSvU5K7H3MoY+gIlEInR3dwOx0o1iTh7KR9ATy9HcbjednZ0VidkJD8GHH37IVVd9g0Dg\nKMzmPxAMdvL44w+wcOFi9u/fj9n8AxTFjKbdDliB/wM+JCbui4FPgPnApcA+4CdAz6F3+QwwmdgB\n4Axigt4GXAFcCAwCXgQO4Hb/4JCYK4COprVRV+di3759mM3jaW//Epo2Gvg/FGU8uv4477zzW+64\n4w5+/OMf57UHxla6yZoIFfO+GAU+UxtPIF4fPFBEp1gUU2T6+96X+7mSz96LA202e53s9yAFfQAh\n6i+dTieBQCCeuFQo+fzQU5WjFTNjPpvriIY1fr8fk8nEAw/MJxicgMPxGopiP/Qj/Bw7dpyArmvo\nunook/154DfAeODnxIT9c8B/AWcduvoxxATZTMySrwMeAa4G6oEg8B3gfuBpwIXVOpVIpIuurudw\nuX4EmAkG/w28yBVXXMdjjz3F2rXziUbbUZTXUJQx6LqKonwJk6mN+fP/yg9/+MOcEhuNe5Aqd8F4\nj4v9YE9s45n44APiQ1AGqlVZLPIRmVpLiqsUuex9tqEoKegDGJPJFL/ZxcxMz+VaxShHKxaJk+Nc\nLhdLl65AUa6L15EDmM1HYjLNxmb7kGDwduD3QAQ4lljSmxX4CnA9MOXQf9OBBYAGfAP4wqF/Px24\nAOgCfgD8DzGr/RhgMZHIeiyWyUQit9PR8TyK4sZkauWcc2Zx1VVX0dTUxHXXfR8YBgxH14NAAIvF\njM02m66uB+jt7aWxsbHPZ011j4x9+sUeVFokjQ8+EQIQJZbSbVxcchEZYz/6/rzX1bLuTHufaVRv\nqjp0KegDkEoIeqrpaKVYV6brGOenGz0EjY2DaG9v7fNaXY9iMu1l3rwzee21NwiFLiRmeb8EHA+c\nSCw2/m3g+8CRwL+BB4EwMItYslyA2Ffwc8CjwH3A54FvAV8nFj/XUdUVOJ1OfvzjS4lEIpxyyilM\nmTIFm83GFVdcwbJly/jTnx5D1z9AUSZht8fCFX7/IkaOHEJ9fX1We2TsOJdt055KYYy/y7hw6Ugm\nMuFwOC4s/Xmvq93TkErgxXc9MRQlQlXC9S6MJVm2NkBIdJsW+wueKsaZWOuerhyt1IKe2A2vrq6u\nj4fgyisv5ze/uYtI5AIslnnoeohw+HZMpt1cd91dRCIar7/+OqoaBGL15zGB/iwxd/vNxFzsUWKW\nuwM499D/V4lZ7qsP/XkHcCMxi30I8BoxF/4TBAK/QdO0eDy8o6Mj/rluv/123nlnGZs3fw+r9RdY\nLC0EAs8Dj3LjjT/LmEyWeD9y7ThX6Qe3TPwqHyIUAuB0OuVelxHj9zxZrommaaiqSjgc5gtf+ALT\npk1j0KBBcdEfCAxoQYfDha4YiUbp/n4lpqOlW4uoq07VDe+GG25g2bIVLFjwJTRtJLruxWTq4Te/\n+Tnz5z/MW29twWq9jmDwaaAZuJtYYxgTMWG+EVhIrN5cIybkPwB+TKzJzBPELPtGYq75fxJLjFtI\nTMx14Mcoyh7uvvt+br755vgDVdw3m83Gs8/+jRtu+D7Lln2TUAjq6pzceOP3uPbaa9PugTHMkMv9\nKMUBsFgkE/hk9cGJXexE7F6SPbVwmKqWdeRKYq6J1+vFZrPR29vL6NGjefnll2ltbWXu3LlMmzaN\n008/ndNPP5158+bh8XgqvfySMOAFXVDML7Wx65zx343JZtnWuhfTQhd104lrSdcNz+Fw8OSTj7No\n0SIWLox1crvsssuw2+38/vd/xG7/I9HoHhSlAV0/C9gMvApMBIYSE/C1gILD8Qia9g7h8F+JCblC\nzAWvE4uha8ALxMraxhKz4AWn0d7+IF6vl/r6+sPu19ixY3nppX+wZcsWOjo6mDx5clpXu9gPUeVQ\n7S72QsjUAES2qc2NdL/H/iTw1XogzQdjX/rm5mbmz5+PruuMHDmS2267jY8++ohXXnmFe+65h3Xr\n1jF16tQKr7g0SEE/RDIRLhTxJUtWjpbLexTzh5cs8S3TWkwmE2eccQZnnHEGHR0duFwuFi1aRCQC\nHs8pqOo6wAvMBu4BHiPWJAZgF3AXYMFmOwdVbSAa3U00+hZ1dVGuvvobjBkzhs7OTlatWsW///0O\nqhoGthDrLAegoevv0NDQnDEe1tLSQktLS9rXiAlpEHuoJoYZku3ZJ598QjAYZMqUKfGZ7pWgGN/N\nahD4/moV5kp/EvhawLhnomX2NddcE0+Ma21tZdSoUZVaXsmRgn4Io6AX61qQ33S0VNcqdE3FsEiF\nx2D06NFYLBCJrMRuPwu7/VhCodvR9enEmsm8Qsxl/hEQxGw+hVDoDkKh3xPLYD8Hv/8NnnzyGd58\ncwFHHXUUbW1t/Od//idPPPE0sVr0PxAT9b8BDzJmzDFxUcnXcyEaCQnR8ng8afd4zZo1/OIXt7Jh\nw26iURg61MWNN36diy++OP4aEcvrr1SDwA8UqlHga+HgkCrDXVGUPuOUxcjmWmXAC7oQhlIIut/v\nJxKJFFSOZnSV54uu66iqGp/ZnckizYapU6dy8snHs2jRz4Hf0dh4B52d3ycYXIjZbEVR9qMobYwc\nOZphw5pZuXIxodBbxBLlfoaimIAuenvP4Pvfv4kpU6by6qsLOXhwD3AUMXf7eYfezYqizGbfvtbk\ni8kS4+HKbrcTCoXSPswOHjzIddfdQmvrRJqbf4bZXE9b2z/49a/vpaGhgdmzZxe0nmollcCrqtpH\n4Gu9TW06innQrpTA9+dDaCqMeyJq0AfSwXPgfNIsKcaX3Nj8w+VyFUVAC1lLT08P0WgURVHweDwF\nrcV4ALr//rs57bQRqOo38XrPxOVazec+dw5Ll75FR8d+/vGP5/B4mtm61YvZ7AGcwI8wmcQDqYlo\n9Hu88847PPPMSjTt19hsFxGLqy/FYlmE3f5PXK5tmM2zqKvLr55UlK94vV4sFkvWQ25efvll9uzR\nGTXqD7jdx2G1jsftvpauruN4+OHH4teudYTAOxwO3G53vMTSZDKhqirBYBCfzxfvp6Cqak3vSyk/\nmxB40fDK7XbjdDrj3rRwOEwgEMDn8xEIBAiHw0Sj0Zre72xINWmtGvpIlJMBb6ELinHTjeVPAG63\nu4+7J9915fNjTex2ZrPZiEQiRf1yDx06lGeffZJ169axa9cuJkyYwJFHHgnArl27uPHGn9DVdQaN\njbeg6w8TCt3J4WdIK6Djdv+CuroLsFiG0dt7Abr+AKp6LXa7m2h0OSbTk3zxi9+K/61s9yWbTP5U\n7Ny5EzgSs9lDT08ve/e2EQ5rqOp43njjZf72t79x/fXXZ7dZNUQhg2YGuvDkSikt+FpsHWz8PF6v\nt2RjlKuVAS/o4gtQqMvdKBzZuHNzWV+ua0qW+BYMBotSj5m4HkVROProozn66KP7vO6FF16gq8tF\nU9OtmExO6usvob39t8CD6PoNh67jR1HuxWRy43bPA8DpPIXGxu/S0fEL4H7C4SHo+mpmz57JTTfd\nlNNahYtdUZS85tqPGjUKXV9MINBFa+sBVNWD3T4ETWvDap3IQw/9iyOPPJIzzzwz88VqmFz60Jci\n+XQgUY0x+GognYU+kJAu90PkK+jCKu/u7kbXdTweT/xLVAlrJBwO093djaqq1NXVxTPqy2kdxaad\n7ULXx2M2OzGbTTgc02hsvB64GV2fh6bdiMl0DFbreurrBxMKrQVi92Ho0Ntobv4hHk8nX/ziRP7y\nl//jtddeyvrHKdqjer1erFZrUjHP5n5feOGFDB+usXPnLYTD27FaIRx+AJNpCS0tNxGJnMhLL72e\n3yYVSLVausa6YIfDgcvl6uOiF+Eov98fb7HbH13G1SKOhbroq+VzFItkMfRa+4zpkIKeQC4PFpEx\n7ff7sdvtNDQ09KktL/dQFSFiIk5cqrrqdOtRVZXu7m6OOOIIFGU9mtYR/29Dh96J230Szc1rmTjx\nba6++kwWL17I9OmT6e7+GcHgSjQtiM/3Orr+Gtdc81UefPBBotEoc+acisdTz7hxk/j973+fMk4r\ncgZCoRAul6ugpJhhw4bxxz/eRnPzcjTte0Qil2K1PsmkSd9kyJDzsNvHsXv3wbyuPVBIFHjxnRST\n62RMuLgkE3iHwxH3nhj3W/yGamG/U1noA6mPO0iXe5xcT3HpytGK3aQm049N1FtGo9G0PeGhcFdn\nqr+bGLO/4oorePzx59i27Vrq6q7HbB5Eb+/TDBrUzv33P8Ds2bMZNGgQAH/601185zs3sWbNV+ju\nBodD57OfPZmf/ew/eeihh7jxxu8DFwHfpa1tFb/73X+zfv0G7rvv3j5rCIVC8Tny+bjYkzFr1iz+\n679+xC9/+ShDh95CY+McrNYGolGNQOBDJk8+Iv75JZkR3x+r1YrNZuuXLuP+dK/TlSSKUIjI+anW\n/c4Xn8834FzuA17QjV/abMQz2+lo5XBxJ4posUQsm/c1kqpZzSOP/Ilf/OI3vP/+TagqTJgwmB/9\n6NeceuqpfR7c48aN46WXnmXFihXs37+fSZMmMXnyZCKRCL/+9W3o+lWYzX82vN9xPP30txkzZjRX\nXnklU6dOxefzEQ6H82rcIwiFQmzbtg2bzcb48ePj15g3bx5PP/0qGze+hsVSh8Xipq3tFQYN+oRL\nL/1FfMBOrT0Qy4GMCZcXo8CLvbbZbP1+v5OFEAbapDWQgt6HTCKczXS0bK+Vy5rgcMs6145vpbLQ\nU01pA5g0aRJPPvlXWltbCQQCjB8/HovFgt/vP2xvTCbTYXXdW7Zs4eDBNhTlK/E/iz2ErgCu4447\n7uLOO/+XSy+9lDvu+P9oaGjIuqogcT9eeeUV7rvvUfbs8WE2w7Rpo/nJT27iyCOPpLGxkf/5n1/y\nxz/+HytW3E44DMcc08jVV3+b8ePHx/dhyZIl7Ny5E7fbzaxZsxg9ejRmsxmLxRLPY5CkJ1eBt1gs\nsg99nojvfrqmQv1J4BPXI13uA5xUImycSJZpOlo5KLT7XKGITOZ0U9qMJHZnyvZBEOviBrq+O/5n\nsaSqtkP/71F03c9zz93AtGnHxCex5cqSJUv41a/uJxg8n8GDL0BVe1m69FFuuukXPPbYn2hsbGTC\nhAnceefttLW1EQqF8Hg86LqO1WolEAjw+9/fxccfe9G0cWjaLv75z/f45jc/x5w5cwiHw7LLWp4k\nE3ij4IRCocNeVw6BrzYxKxb9VeBllnuMAS/oiV/CxC+GsRytEtO4ErOxs3H3Z3OdQtajaVq8WU2u\ntd25MGrUKE477XTeffe36PocdH0CsSEuNwODUJTPoihuNO095s9/JG9B//vfn8Pnm8G4cd+Lfw6X\naxLbt1/F66+/zhVXXBF/7eDBg+PfB4j1Gnj00cdYudLMxIm/wuFoRtOibNv2DH/968scf/zxeDwe\n2Ua1SFRDm9r+FEPPRKbfbbYCL5IfKynwie8nXe4DHOMXIpeJZKmuVUxBV1WVQCCQNvEtGwpdk/gh\nF5J4lsve3H//PcybdyGtrUcT6+u+EzBjMj2Foogf6zHs3/94zmuA2H5s2tSKy3VFn/20WDzo+kR2\n7doV/zNj0p3L5cLv9xMIBHj33XU0Nf0HDkczACaTmbFjL2bz5sWsXr2a008/PaMADbQ2qsVsnVpp\nge+v5BN+y0Xgy+kxSfY8CQQCNDY2luw9qxEp6AaE0OQzkSwZxRB0cQ2v11uQu7/QH5MoixMtZBsa\nGsoiOuPHj+ejj1bwzDPP8Nhjj/HOO5uAxSjK7Pi6TKYXOfbY6Xm/xxFHDGPnzvV9/iwaDQDbGTbs\n6D6JkCLpTgiFqqqEw1ECgQO0t39EXd0R2O1NmM02dN3Sx0UJhz8Qs+my1tbWxttvv822bbsZOrSR\nE06YxaRJk1i5ciV79+6lsbGRWbNmZZxENxCQAl9e8t3vUgi8tNCloPdBUZR4HTUUFp8uxpdV07R4\nSUkh2dvG9eRzyDAmA4rs2GIk1onrdHd388QTT7BgwTuoapTPfOZErrzySoYOHQrEZrJfcsklh0a4\nns3+/V8hGv0JijIcXX8YeJuf/OSpvNdz2WUXsWzZ3ezZ8yhDhlyEqvbS2vp/OBx7WL16K7feeidj\nxw7BYrEQiUQ47rjjmDx5MgD79++nvX03Gzc+R13dJOx2GDNmJk7nMOrrg/FWuKnI1EZ17dq1/Pa3\n97J5s41otAWTaQPNzW8ybJiV3t4mVHUIitLBhAmv8MMffotJkyblvQ+1SLoD1ED2kAhKIaqVEHhZ\nhx5jwAu6UVzERLJCpqMZr1vIlDRj4huQt4u9EIy96YV3IBQKHWZ1FoLP5+P667/P8uUdmM3zUBQr\n69Yt4M03l/Dww/cxaNCgeBb9iBEj+Ne/XuXGG3/A22/H+rqPHn0Ev/3tQ1x44YV5r+Hss8/m5pv3\n8dBDf2fPnifQ9QiK0smQIWewZ89JLF68gW3b/o6m7cdqHYPH8wxnndXCddd9g0ce+Qc227k0NAwn\nGBxBJHKAVateYOTILq65Zh5jx47NaS1Ggdc0jQceeJSPPmrGZPoqZnMT0aiPDRseZ+PGtznnnN9T\nXz+BSMTL5s0Pcu+9D/OHP/ymLKWL+VLp+HMhfeiN1ILgl+NelFPgZZa7FHQgVnLk8/nQNA1FUair\nqyv4B5tvDD2xzt3hcNDb21vQWsR6xPWzwTg3PJdkwFx5+eWXWb58D0OH/gW7PSZ+kcgXWb/+Kzzx\nxBNcffXVfbLoW1paeOWVF9m+fTvd3d0cc8wxBYUghJfg6quv5qKLLmLt2rW89dZCPviggcmTv86B\nA+3s2eMlGPwPLJbVNDdfTWfnezz//Ku0td2Grk/l+OOvYeLEAFu3buPgwQYCgSOZMmUbX/3q1QXt\nTVtbG0uWbELXf0Bj40wUxYSqBunuPg1VXUdPTysu11gUxcmoUf/Bxo2/Y926dUybNq0mBKcc5CPw\noueAJHdKJfDJvIZS0AcgkUiE3t7eeLtEVVUr9jBMVucuBLiclo3RO+DxePq0sy1Gsp9RTJcv/wBN\nOyEu5gBWaxMm01wWLXqfa6+9NmkW/ZAhQ2hsbCzqg7WxsZETTzyRv//9XwwaNB2n083Wre/S2bkc\nRXETDK5j69avExv7GmLBgg8ZMcJDY2P3oTj28QDs3VuHzfaPgr9HHR0d+HxhHI5hxObHg65HMZkc\nqKqFQMCPxWJG03RAoaPjIOvXr2fChAnx+mwZH86NbAfNqKpKKBTq1y76ahiQU0yBlzF0KejYbDbq\n6uqwWq1Fm0gGuQlfMtd2olAVU0TTrUPUlhcj7JANDocVRfnUA6HrGtGohqZ58XhcOJ3OpH+vWFUE\nRkQWu81mwWTSUFUfO3b8i2i0CZNpOqr6T+BcLJar0PVGIpHbaW1dzcKFSxk5cjRHH91CU1MjXV2b\nOeGE5oL3zmq1oijtdHb+lkjkPFyuk7FapwPvoSgd1NVNQdejbNr0EBs3Po+qdnPPPc+xfPlH3Hjj\ntxk8eDAgE8DyRYhGosCLvJZIJBJ/XlRTTXZ/Jl+BT/SaiPs00BJFB7ygA/HEt2KKRLbXKpdrOxPG\nevt0ZXHFttDPPPNMnnnmVrq738LjmYum6fj9qzCZ3uX8868p6H2yQTykjVnsp58+g8cf/4Dt23uB\nRhTlfKLR59H1kVgstxGNdqDrQazWG1CUm+npeQ2b7RJWrWpj5MgIbvdazjrrs/E9NU6/SkZvby/b\ntm1D13XGjh3LoEGD2Lt3L3/84yMoyjRU9Qi6ujbT3f1v7HYrVus+LJaDbN/+d7ZsOci+fWsxmT7H\n2LGnUF+vs2jRnwkE/sjdd98OEH8QDvQEsEIRAi/ExGaz9fs2tdW6LkG2Ag/E+9Pv3buXUaNGycYy\nAxUhUolx1ULJJHzpXNvGtWVzrUykuo6YwJTrUJNi7dGZZ57J5Zcv4umnf05n50QUxY7Vup7zzpvO\nJZdcUvD1k6HrOh9//DFLly7FYrHQ0tLCtGnT4hPqPvOZs9iypZVnnnkCmI3VupVAYDUwCbCgaaAo\nXtzuwSjKRVgsT+B2N9HevpkpU0by1a9eis/n43e/u4u2tgD19Vbmzp3G2Wd/5jBh/+CDD3jxxaXs\n328GFJqb3+G8845j9eq1bNzYyCmn3MLKlZvo7AwSDq8lGv0b9fV+HI5murtfpqNjL2bz1UyceBFH\nHDEWRQGb7WY+/vhHfPTRR8yaNQuLxYLdbs87AUySHKOVCP2rD32lkxPzJZXAB4NBdF2no6ODGTNm\nxENyr776KvX19bS0tFR8z8uB+Wtf+9qvsnnhuHHjSrqQSiK6fokTXjE6n6W7lrAIA4EAVqsVj8eT\nVkQDgQAWiyWp4OeCaNNqTADy+/0Eg0FsNhsejydjTFp8rkI8CZqmEQ6HcTgcqKrKccfN4NhjJzB8\nuJ8ZM+r49re/zLe//a20Vq2wOFO55FOh6zp//vPD3HPPS6xYYWbVqjBLliymu7uVk08+EYvFgs1m\nY+bM6fT07KK9PcLRR8+gp2czfv96dP0sIIjTaaKhYQyh0GNMnjyJs8++GYullW996zzC4TB//vO7\ndHXNwuM5k9bWIAsW/JOlSxeh6yput5OGhgZaW1t55JE38PtPYPz4i6mrO5Y1aw7w4otP8d57y3E6\nP8uYMcczYsRQPB4rHs8o/P4PsNnqmT79XlpavsaePcuBCwE3w4YNRlEUrNZGDhx4kVmzxvQpmxMC\nJL4DVqs17n43inzivOxiPgiF2JW7XXExCYfDmEymw36zxm5pYn8tFgsmkyleRVOOPc6WSCQSz7Xo\nr4g9D4fD2O127HY7xx9/PG63m/fee49XX32Vu+++m4ceeogPP/yQ2bNnU19fX+ll58z27duzep20\n0A0U80eV6lrGxDe32x2fC12udSWuQ9d13G531kNNioH4PH6/P37omTdvHuedd17J3/vDDz/kmWdW\n4HB8ncmTj8dkUggEdvDmm//LzJlvc+655wKxMMwll1zMzp0vYDKZmDr1lzzxxDV0dNwKzMNiaaa3\n9yns9vXMmPHfBIOdDB3qYNiwYcyf/ywm08mMH38O+/atpK1tP729p/PRR71Eo92sWPEiV155Fjt2\n7KC9fTDTpp2Kz+djyZKVtLUNx+c7Dq93F7297ZjNm5gy5UhaWloIhYJs22aiqekUPJ4WdD2K0+lB\n07rx+VR6enoYNGgQnZ2fEAi0sWzZB0SjGrNmzWTkyJGH3YPE+HB/sS77C6kseFVVK9pVLXGN/R2j\nt8Fut3P++edz7rnncu+997Jjxw4+/vhj3n77bRYuXNgvxTwXpKCT3OVejGuKa4nrZ0p8y7S+YqxJ\nNKsR68jGKk+8BhTmchcekUgkkvWhJhn57Mny5e/T23sEI0fGMtJNJjP19ZPYt+94liz5IC7oABMn\nTuSyy2bx0ktvs2mTzsiRRxCNvkcotJRo1ITDUcfs2V8hEvHR1bWIc84Zg8vl4sCBMA0NY9m06UVW\nrHiKUGg2gwfPJhrdyYgR0+nqWsOCBUsYPryeQMDKsmUr2LJlDwcORBg+fAYNDdNwOlfT3b2T3bvb\nGTasg6amZvbsWYjZvJ/Bg+eg62A22xg79hzWrXuZUKiLQMBBILCWzZvvw+OJsn378WzevIe3336U\nb3zjIo455piU+5Kr+1hOOcudZHtcqa5q/dXlng7jHgUCAXRdZ/jw4YwZM4YLLriggisrH1LQDRRT\n0I1Eo1F8Pl/FE990XScSiaBpWkXWYeyPD8ST7/Ihn3WHQiF6enpQFBdWq5VIJIK4jNnsJBA4vGHO\n6aefjtls5i9/+SdDhgynpeUWotFduFxtjBw5DJ+vi4aGj5g1ayqnnHLKoQoBldWr/8bBgwo+3xHY\nbOeyf38XJlM7qhplxIhjWbVqMUuWLODdd82YTFcTjdahKI3s39+N2fwuxx57Fnv3rmPnzgfZsGEa\njY0KLtcWpkxpwOf7tL/8uHFfoqtrA3v2zKe393V6eztobBzCvHnzCQY7iUSGsWfPJp599nUmT56c\nddgmk3WZaspZNhn0tXAAKMZnyDejW1Yp9CVVlzin09mvwwn5IAXdQCks9HA4TCAQQFGUlIlv2Vyr\n0DWFw+F4Rne+6xBrgdz3SPSCD4fD2Gw2wuFw2X5sxiz2Y445mgULXiUQ2IfFEhumEg73EAp9wMyZ\nMw/7u7HhK2vxeC7iuONOR1EUIpEg69Y9x+jRYb70pcux2WxEIpG4+3r0aDtvvrmdxsavEwh8iNk8\nGFV1Eo3uY9euXWzb1sOGDRvp6VEIhRRMpk3AeDTNhK5vwePZQVPTuUyceD5LlvyMlpa1zJ59PCee\n+FXa2tq4445n2bJFpbn5ePz+XbjdXVx//aXMnDmDRx75Ny7XZ1m8+C46O3uJRsFmC9PW5uWLX9wa\nb1lrJBQKsX79eoLBIKNHj2bUqFFJa3xztS7FHPhao1TWbSUEvpbuj/Gz+Hy+AZfhDlLQk1LMH6zf\n78dms+FyufL+4RUi6EYxg1hsuNDkulxJzBuwWCzxA0ahZHL9J5bjnXnmmSxdupJ3370dk2kWVqud\nQOB9pk83cc455xz297dv387u3RoTJ54Yfx+r1cGIETPZsOFFgsHgYV6GiRMnMGSIQjQaRFX34PU+\ng8VyEibTYJYsWUA4vBKrNYjFMp2hQz9LKLSN3t5/oSgWdH0UdvtxRKNhentbOfLIEfzoR5czceLE\n+OdVVZWXX36LvXuX0Nxs5Utfms2MGTPYuHEjfn8v69Y9gt8/nfr6KzGbm/B632b79jt59913DxP0\nTZs28fDDz7FjRwRVtVJfH2bu3Ba++MXL035P0omPSP4KhUKHZdBLsiebPTa+LpcqhVpyuSf7LELQ\na+nAkg1S0Pn0ZFesmy8EDKiq2nIh6oWQi4VudLEb8wYK6XGfuI50GMedGsvxfvKTm3jttdf497+X\nARgCJ24AACAASURBVCZOOeVYzjvvvKSjFmPZyCY6O7vYuXMfnZ0+PB479fVR7PZP8wGMNDQ0MHHi\nUEaOPI6tW6OsXPkmkUgQv38Z4XAbcCThsB2TaTPR6BsMHXobmtZMILCWYNDN3r3rWbToPWbMGM6X\nv3wKLS0tfT733LlzOeOMM+LZ0s8++xL33rsAr9fO5s1dtLXtY/Ton2K1jjz0d47GZpvL8uVr+fKX\nA/HKAJ/Px/z5z7BjRwstLZ/DZvPQ3r6eF1/8O0OGvJlTkqJRfNKVyInDqaqqskQuR7Ld44Fahmj8\nnAOx7StIQe9DoS73xMS3aDR6qNtXefvCp5rlXiyrOBuKNYI2H5KNOzW+t9vt5tJLL+XMM8/MmOE/\nZswYNG0fr7/+MmbzCTgcR9De3k1Pzz8566xu6uvriUaj9PT04HK5sNlsTJ06laamtfj9m2hqamH4\n8Dp6el7D622nru4arNbT8PlsaNpSfL6H2bv3vwkE7CjKUdjtrdTX+6mrs2E29zBnzglJ903c16ef\nfo5nn91BIDAdTWsgGnWiquvZvfsvuFz/gdlsweE4gMm0kQULVnLZZd9mzpzJXHXVFzl48CA7d+pM\nmnQpVquLtraP2LTpVXbt2sAvf7kUn8/HRRddlFeeQ6oWqiK5LhgMAv03g74a1lmMQTPV8DkKJVUM\nXVroEiA/QU9MfLPb7XR3dxc1Oz0b0glpsRJ5IP0eiWE3uq4nHUFbzFyFRJd7th3vsiU2FMZHMPgJ\nHs9wdD2Arm+nrs6Hz6ezZMkS1q/fzYEDQerqTBx//AROOGEWn//8CbzwwkLWreuhq8tCZ+db6PqJ\n6PoYAoGtWCzjCYWOIBodg9f7FLo+E5PJj9PZzZw5NzJ06HQ2bPgt7733HhdffHHStfn9fp566t/s\n3j2H+vqZ2O0NRCI+otEAut6KybQHq9VJMPgg4bCLQYO+i65P5JVXFrB27f/LZZedja57sFpd7Nu3\nkqVL/0IodAwm07V0d3/MAw+soLV1H9/73ncK2kNjiZymafE+Bv2xRK6aXdW5CHypEoArSWIMXVro\nAxSj4OUTrxbWsDHxrRJDVbIR0mK4u1Nh9FAYJ6SVgmQP+lQu9nRkuj8HDx7EZhvFqafOpKfnAH7/\nBhobhzJs2DdZvfpOnnzyA5qbT6WubiidnR385S//4o033mLq1KP4zGemMmLEOh599A00zYXVeiRW\n6zGEQttQ1TWYTA50fSgQxWptoLn5eFyuJiKRoZjNVszmCezdu5/29nbWrl3HwYM9NDd7GDv2CEaM\nGMG+ffvYvr0Lt3s6gwa1EAx2oetuLJYhaNr71Nf3oqpbaG/343Z/m4kTZzB4cDNNTSezefPNfPLJ\nJ9jtCj09rWzc+Aqh0LE0NX2brq7NjBw5nBEjTCxadB8XXLC5qHPWZYlcecg0aAZiSZ/CghdNcPrb\nHqeLoQ80pKAnIZehKsbM7UIS39KR6ZCRKKRut7tkCUipTvb59KQv1mEnk4s9Gdk+tGJiAh7PEbS0\nnB5/v61b32HDhnV0dDiYOrUVp7OJzs69bN6ssW6di927bdhse1CUPZhMY2homEFX10YURcHhmITX\n24Om7cVm24jDMQqTyc2YMWcRifRw8OBexoyJEo3uRNfreeSRl9m3rwFVtdPZ+RENDYv42tcuOpRF\nrqLrBwHw+9tQVRf19ZPx+Z4EXiAU2oHJdBIeTyNNTY0Eg0HMZjN2+yz27/+QESPsvPnmD9i7twOb\n7Tu0t6/D6fQyevR4Bg1qoq2tjq1btxZV0BPJlEFfSImcJEZiI6FwOBx/bonuj7U0aEbG0CVA9g96\n4wz1ZM1RyuXSMrr6nU5n2ra1xWpQk0g2PekT11EsVFUlEAgUzcWeSHNzM0cdNZjFi5fg8YzAanWw\nZctC3n77OUKhE7BYzmHz5g62bfs7kYiJ5uYr8PsPMGLEEdTV2Vmw4BZstiMZPfpU1q17EJ/vV8Bx\nKEoHirKQsWNnMmrUPFatepBdu56gufkUFKWDjRufYNiwDgKBQbS1jSESCbFly1b8fjtr1wbZufNP\nfP/7X2L06MHs3r2M9nYIBDTC4W2YzV4GDz6Fc8+9knXr/sy6de3oOmzYsIVQSMdk0gmHP6Gjo42e\nnnHYbNOJRpfQ27sHaGDKlCNpbm4mEvGhKIGyPxhzLd8yWvCS7Pi0YsOKzWbr150CjS10BQNxMAtI\nQQc4TIhzsYbTdVorZoe3ZNcRQlpIjXs+a4FPp5SVe9xq4jq8Xm9OLvZ8uOCCszl48B9s3Pgg4XAd\na9YswuM5geHDW3C7j6a+fhjr1t2O3x9kyJDhqGo7LpcLj6ceh2MMO3ZsweOJMHz4qXi9y+noeAWT\nyYvb7WXUqAsYMmQWkyd3sG3bkxw8+BIjRtiZMGEUF110LosX7yYajbB27Q7c7gsYMWIy9fX72LPn\nMV59dSnTp49i69b3aW9/mWAQotEIVusEBg8+l4aG4bS0XMiaNd+jt3ccZvOV2O2D8PnewO9/j40b\ndaZPv5558+ayatUjfPzxehRlOj09XlQ1wLZtTzF6tMKMGTNKsq/ZkkrgRZObSkyRqzZRK5RcwyDV\nJvCJa5AWugRIL8K5WMOCUgi60cWci5AW00LXNI3e3t6c9qJYaxGHKohZGHV1dXk9VLJdw7Bhw7jh\nhqv5+OOPWb58OV1dwzjhhC+wa9dutmxpxW534naPo739Iw4c+ISxY200NNSza1crO3ceoKtrA11d\nf0VRxmIyHYOuN+JybeCoo2agae+xc+cONM2KzWZh8GAvU6aMZObMo2hqagJ2s2PHJ5jNJ+DxTAHA\nYvEwaNAcdux4nX37FrN3r59QaC4QQdeXYDa3c/DgVpYv/wMej53hw3U6O18lHF5LOGzCbvcxadKx\n7NmzH6czNrxl6tTL8HrvZceOv7J9u4rJVM+4cSZuuOHKqpspnSjw5SzfqpUksmRWrZF8BF40Eiq3\nwKeKoUtBl6R8yCdLfMvmWsVGVVV8Pl/JXMzZIpJpSmkZJ8OYxQ4UZTJeNtjtdqZMmYKiKGza9D51\ndU7GjRuLpm1n3771BAKd6PoneDw7mTr1HHw+PytWLMXnMzFs2Ew6O0HXx6CqoOv1OJ2zcbubmDLl\nWDo6drN8+ZvYbJ0cc8x38XiG8+67m9iyZTFut5kDB1ppaIjVhGuayr5971Ff30lr6yZ27TLT0PAQ\n0ehYzGYHqrqKYPB7NDSAqq7jpJOOoaOjGat1GHa7laFDj2XSpPMIBDrZs+cufL5eYDg2Wx0nnXQL\nTU3Ps3v3Q8yePYQTTjiBcf1gymK65K9aiw1Xikx5DiImn5jnUC6BT2ahDx48uOTvW21IQc+AGDGa\nT+JbsV3uwWAw3qQlHyEtdD3CMwDEB7uUs/udMcTgdrvj/15qwuEw27dvp729nbq6OkaPNrN9+xJG\njpzDtGlHM2bMQT7+eAUzZ47EZmtlx45/097ew+7dC2hoOAa3eyhu91FAM35/B5pmob5eYffubYwY\nsYtIRMdsVjnttOsZN+40IpEIzc2T2bjxHxx11D6GD+9l8+Y36OlpZ//+N1DVED6fjb17P8FiOQkY\nisNRh8lkwW6fQzh8HEOHWmhsHMtLLy1jw4bBhMMTMZs7aW19l7q6MYwZMxOXq5uOjvcZN24iiqLg\n8/Xy8ceLgQCLF5tYvPgNnn32DX70o2s59thji7afpXaFG5O/+pvruJwUWopYLX3oU9WhSwt9gJIY\nQxelXcYysHxGjBY7Cc3v9xfcpCXf9SRaxsZZ2qUmWRZ7sg5tpXjfnp4eXn/9TVatOkAw6MRqjeJ0\nBrDZ3mfjxs1YrUMxmQ5w0kkWvvCFH9Pe3s6GDZtZu3YbnZ0uFGUyPT1dQBQ4AOxHUXoYPXoIqhrG\nZFpJc7OZUaNGo2mjWbhwBaqq09Bgw+0egs93gO9854vcddczrFv3PibTqQwefBKa1oPFEiEc3ovJ\ntIZodA5Wq+OQ50gjHA6wevUaenrmYLVeSDjsAgbT3f0Cb7zxEDNnbubkk8ehqmtYs+ZuzOahbNny\nbwKBHUyf/gcGD56NqvrYunU+d945n/vuuz3j7PlCJvCVilxcx4ntU6vtsxSTYocOKi3wifdKutwl\nwKeC7vf7CQaDJS8Dy0QkEiEQCAAUPLc83wdUYn13b29v2eKRqRrFFKOKIFO+hNfrZdmyZSxd2sOI\nEeczfvxYwmE/W7e+y7Bh2zj11BGYzWaGDj2OlpaWuPdm7942TCYPkUiYYHAJqtrEwYPbMJtPJhSy\nY7H46O3dz3HHubnxxqvYvn07y5a9xr59nbjdLVitDtraDuL3v8+wYb3MnTuXnTt3cuDAaqzWKVgs\nPYwY0cDgweeyZMmzaNo7mExTMJujKMpHwCqi0WPp7g4TDI5D00KYzQF0/SBm87EEAi/h8/2Ln/70\nXjRNY+XKlRw40M7u3dsYP/4amptnAWCxuBk79mp27LiZjz76iJNOOumwfVJVlZdeeolXX13EgQPd\nTJ06ls9//gJmzZqV930pJZlcx9kKTy2LfTEop8BLC/3/Z++9A+Oo7/T/18xsL9pV781q7gVXbGxs\nA6EaQmhHCvkGkpDc5ZIcyaXf/UKSSy69cSGBAIGEHqopAYMx7lW2XOQiq1urutreZnZmfn/IK2Qh\n27IlG+z4+Qu82p33zGdmns+7Pe/3cIHQhyG1g4/H46dV7DUUYy38SlWQD5WRHQuGkuBozml4n/3Q\n/u6x7vBHc/yhIfazmatPHVfXdQ4c6MTpnEFmZhkAZrOdioolNDd7uOSSbGbOnDk4Pc7n8/HSS2to\nbXXidl9BUVEB27evIRDYTTK5CF2PYzSayMycRDy+B6s1gtVqxe/309u7m2TSjixPwuHIxWx24fN1\noqoD1zkjI4Pq6vlUVCxA01SMRhMHDhzEYkknkXgZQQiRSITR9U3k5pqYNMnIvn0CspyNzTYXUFGU\nFgShE5MpjexsF2azmaysLOx2O88++zy9vSCKO4nHfRQULMHtno7J5EZVjYNtiSlomsbevXv53e9+\nz9atHTid1+F2z2Dt2h3s2nU/3/3uXSxcuPCsrNdYcKrEcz71vp/NTcmZ7lS4UOU+gAuEznvEmyrs\nAMaFQE5XmW14Nb0oioMEc7YwfELa0MjAmX4RDA+xnynBnpGOO7QNz2g0Eoup2GzuY/7OZLKiaZbB\nSvsU6uv309pqo7r6I0iSgZycKpxOB//4xxMYDKDrh7BY7BQV5TBhwkU0Nr7CPff8mD17DtPVZcZs\nTiDLQfr6mrHZWqioKMZgMJFIJEhPT6evbxs+Xz6iaCQvLx+/X6GsrASXawGCkCCZ1MjP/wQZGRqZ\nmQmMRheJRDtgATQMhgoikX9gt7cB0+nv7yctLY2//OVpamtNOJ234PMV4vNpRKOvUl1tRlECOJ0y\nEyZMoKOjY7DY6KGHHuXVV+s4eFAFJhCN7sXhKKey8j9pbLyPJ554kQULFrxv3T7sVeKjJZ5U9O5c\nHYDyQa/DeHYqjOSgXOhD/ydGSv88NaNbVdVx8wbHQ0Y2dXOPFaPx0I83IW2kvxurLSP9xmi12Mc7\n5D5U6W5oZCY/38m+fW1kZ7838SwU6sViiZKRkXHMtWxp6UWWLezZ8xaxWByn04GiBEkmraSlVTBh\nwgLsdhuRSCe1tevo6ekmI+MiIpEMdL0GVVVxuczk5y8kFtuLydRNWppAMpnk+edfZvfuWmKxJJI0\nHaPxICZTK2VlOldd9WVyciqOnodKQ8PThEKNTJ58ObW1OwkEvocgVKBp+9D1d4nF7DQ2RvnFLx5n\n4kQ3e/eqVFd/Grc7wLvv7kJRCggE2tm//3ekp8tcfnkZTz/9InV1XSgKxOOdeDxB3O5vYDTacTjm\nE4+/RmPjc6SlTSErazHNzZvp7+8/5yuNhxOPoiiDY2EvTDgbP4x10MzQ651yCM51D3379u385S9/\nYc2aNTQ2NuJyuZg0aRJ33XUXRUVFI37nAqEfRcoTTd0M41HgcyrfP5GM7NlSnRvthLQz9bL6oELs\niqIMjrsd3pJ48cUzaWlZz6FDEtnZE4jHg3i9u1iwIJ3CwsLBvxMEAY+nhdpaBbt9NsGgF49nDcFg\nH7GYl2SyDoNhOhkZGlarkYMH92CxQCzmQFEcGI01JBI9eL0eamoK0LQAvb1rmDv3Bl544UXeecdH\nQcF/oCg+gsEOotFOIpHdOJ0LBskcoLe3ifR0nZycHBobTbhcs5BlD8nkWqARWISmTSY9fTqJhMbj\nj/8WWfbT12emoGAGS5ZM5cCBZjyeOEbjQb7whU+yY8cB9u61UFj4RczmTN5993ECgU2kpQUQBCu6\nrmG1riAYXIvXuwmbrQSjURhTvceHFalnMrXRPJdb5D6MNqVwMh36oQSvadrgWqTO6Xwg9J/+9Kds\n3LiRW265hWuuuYb+/n5eeOEFPv/5z/OHP/xhxJbSC4TOe6MogXGZGZ7CaHPow8Pbw2VkUxivvPVI\nv5Oy4XiDXcbblqG/cbpCOeNx/GQySSKROO4wmaqqKm69VWXjxl10djZgNsM115SwaNHCY0Lufr+f\nvj4VUaxEUSAQMJFM3oIs92AyHUQUowQCL9Dbm46utxCLdaHrk/D5UlrvlXi9EsHgfhobVwIHycjo\np6Ghi+eeexFNW0R+/g0Igkg02omqxmlvf4xQqJGDB9fidhcQjfrQtMPMm5fJ6tX72LNnP37/HETR\ngSha0bQpSNKnEIQEBw50Egym09lZQzJZh6LY2LPnKcxmI+np1+Ny6VxyyUWkpaWxeXMDmZm3oKoy\nBoMNs/kiBMFIOFyL1Xot4XAdRmOQZLKXYHA/sVgtN988FafTecbX8Gxj6H1/LrfIfdAh91PBSNd5\naCoE3ovs3XXXXcyYMQOXy3XSd9iHHV/72td48sknMRgMrFmzBoBly5Zx55138sQTT/Cd73znfd+5\nQOhHMXyHN14e+slkZEcT3j6TD/9QKdtUb/nJKvrHw57Ub5zuuNOxRi00TRusbzjZMJmJEydSXV1N\nJBLBZDINep5DCb2zsxNRzGPq1ErWrn2dSGQuspyO1SogCCq5uRMJBP4M9GCzVRGJpGM0XovRGCAW\new2TqQGXq5hw2ITL1Ug0mqSk5Ca6u4tpahIJhQ4SjT6OKIrY7blkZNRgNmdTUtLHjBl+Ojvbycmx\nMH36ZDZt2k5TUy7FxUaCwa0oSh6QiSRZcblmoKoJotG3aGsL4XItJxIJIMtLicfnEIn8CV3fi643\nsmVLK3V17XR2FhMIeIlGf4Yo6uh6JokERCIKBQV97N37OKGQgKZ5aW+vY9q0dBYvvok//OF+Ghs7\nyMvL4Iorlo8oIevxeHjttdfZunU3fn8POTmZzJs3j8svv5yCgoLTWtsPCqNtkfugxFfOFwwn+HA4\njMFgIBQKAfDHP/6RQCDAxRdfzOLFi1m2bBnLli1jzpw5Z1UEa6xYsGDB+/6tsLCQsrIy2traRvzO\nuXN2ZxjDyffDEt5O2TYeNg3/naE2jHZC2nhC0zSCweBZD7GnohEw0Hs8muIZURRH9DiHSmiKokB+\nfg65uZk4nfn09kqIYjZ+fyORSD/RaAKbbR66nofRWIcsd2K3z8dg2Ek8vgpZ1jGbDxONWpg69aPM\nnXsJTU1bEMVKIhE/iUQSScqlv7+d3t5WbLY9LFkyh2XLFtHQ0EA8HicYDLJ9eyu5ubeTkRGmsfEA\nyeRsDIZS4vE1aFoMXVcBCVnOQNcbyc0tobvbgyCkAwVo2nqmTbuZhgYJo7EYkymLcLgdRbkUXXdh\ntQokk2/j860nkejCYLgEg2EmDodAcbFMNPoQ99zzI5LJKZjN05HlJlat+g1f+cotXHbZZYPXrqmp\nie9856ccOKDQ23uERMKOIBh4/fVXeeqpf/CDH9zzoW1/Gw3Gq0XuTEDX9fOqYl+SJLKzs3nmmWeQ\nZZmysjK+/vWvs3XrVn784x/zne98h5aWFkpLSz9oU8cMn89HeXn5iJ9dIPRhGE9CO56HPjRnO5rw\n9njaAwMP81hsGA/FOVVV0TTtrIfYh0ZExvryHHqvFBUVkZOzA6+3CafTgtEoEg6L9PQ0IooGQqF+\nIpE+wuEGXK5+srJK8Xq7CQTWoushXK4+CgpE0tPj5OTMYsGCSwmFemhu7sRkWoogHEJRWhggXJDl\n7aSl1VNRcTOPPPIyhw/H8Xja6e7eS39/iJkzr6G0dDJpaWaCQQ+ieCWCsJZY7D6MxtkIQjeqWofZ\nnCAn52rCYSsWSwWBQDrFxTNxu0swGr1YLJUoyhoikSwcjk+iqn3I8iEyM5eiKI0kEj4yMuZTWJhH\naWkpRqOBN9+8D1mewqWX/gJBkFBVjf37/8T3vvdLtm7dysKFC7nssst4/PGnaW7OR5b70fVLSU//\nL2Q5gqI00tHxKr/4xR/561/vPytDh84GTrVF7sIUuZPjRIW13/zmN7FYLCiKQl1d3XlB5qtWraKv\nr48777xzxM8vEPowjGcB2vDw/dC2qOPlbM+0TTBQJyDL8inZMF5IPWyapiGK4pgGq8Dpza5PRURS\nIbqxIhAI0N7eTlVVOnV1B9D1Hnp6eohE0kgmBdzuaajqflRVIhbrQtMUQCYjIwNFsQCdLF48k8zM\nNLKyOunvzwQgGOymu1vB77dgNC5AEHYjST1oWi8WCzidpbz++jqOHEln9+4ddHVJyHIVirILr/dH\nzJ79HYqLL6Gj42lCoQGysFg2IQhrEYR+7HaB8vJ7sdunIIrNKEo7yeRWwuEC9u9/jXDYgMNRRWam\njXi8imSyA0XxYTJ5mTt3Oi0t84hEmpk/fwGCMLAm4XAb8biO2bwIQZBIJPrZseNP9PfXo2l9/O1v\nB3nllT1ceeUGdu9uxWZbQWfnc5hMX0FRJHTdgSybsFiuo63th+zdu5dZs2aNyzqNJ8Yr7fRBTpE7\nnzYKQ88lGo0OjoWFAVXLcznSk0JbWxu//e1vmTp1KldeeeWIf3OB0IfhTFWUD80Vn65gzVhtSn1f\nluUxieacbn/90Cp2o9E4WJ16pjH02o9VbW8odF1n06ZNrF/fQE+PhijquFwxrr66gCNHunnnnS04\nHEUYjT5EMUI0mkTX3fj9OUSjEpJ0GIuljZwcM7m5cRYtKmHy5Hn8/e/r6OjYS2trGwcPNuD3OxGE\nBCbTVOz2UtLT+zEYtpNMbuLwYS8tLa10dU1D024kLS2PaHQHsdh97Nz5Y0wmMwZDGclkGwOSsMWA\nQGVlGZmZQYLB54hE2ojH2wiFdpGWlo/bfTfBYD2h0JNkZZWRl5eJolix2wvp6wtQUzOFmppq+vo0\nkskgstyLxZJ7lJAgmQzhdhsIBA6wZcvniEZjwFR03UEw2ITT+QWef/5ZLJYEdnsEVdWIxfSj6QDQ\nNJmenhCZmUkaGhrw+/2UlpYyYcKEcVm3seBMpuLGszf7ZDiXiuJOhONNWjvfetD7+/v59re/jdPp\n5Pvf//5x1/wCoR/F8H7G8fTQU2He051ONh6klyJTAKvVelJd7vHESFXssVjstDYFp4qUbO1IRYdj\nTR0cPHiQp5/egdE4n9zcydjtVlpbt3DkyEHuvvtOzObn6ekpp7h4Jm+88Xui0amI4s0IQghBCCGK\nbkQxRHGxlRtumDfoiV522VRefnkLGzbUkkwaMRprgWpstmUEg83I8mbc7iM4nSHCYY3eXhVdvwqb\nbQKSZMBmW4CmNSDLP0JVl5OW9i+I4j9IJssxGKowGg+yaNESkskG0tJ2Y7P52bz5IA0NWVit1xIM\ntiOKImVl5SjKTmQ5n3D4LWIxldzcAkpKCmhuXkVVlYgoVrB3729JT/8IRqOLnp71ZGTEkKSt1Nc/\nTSyWh67fhyhaEYQ+ksnf0db2GEbjDAyGN+npeRtZVkgm38Bo/Dq63oMgSIRCG4jF9vPDH96Pohiw\nWnWWL5/Nvff+14dunOuZwvFat1Lkfi61yJ1pDPfQh6panusIBoN885vfJBKJ8Lvf/e7oWOWRcYHQ\nj4PxHqoyXDr1VHG65DNc/UxRlDGH2E/FltOtYh+rHSMNdBnPBzwWi/HEEy9SX19IRkY6HR3tZGeb\nqKqaT3NzOwcPHmT27Br+8Q8/0aiflpZejMYb0bRcjEY3LlcestxIItFMJNJGSUnJ4G9PnTqVxsZG\nDIZO8vIm0N29nUhkJ4HAVlRVIZFox2g0k5ubhqr6SSSsiKJ78HokkzFsthJCIRcmUw0GA5hMVaSl\nfQpNkwiF/Bw82MiSJbNRlH4+//kbEAQ306cvQFWdxGIxXC4XOTkfYeXKe+jtfYVg0IfBsAq3exFH\njhSTm6tw++1XU1lZyTPP/J21a59GUXQWLcpl9uyv8+CDT7Np0y7gl4CGrvsRBBOq+mlgJZq2Dkmq\nIZk8hCxrCMIzKMoBBKESk6kHRVmFLKdhNv+azMzpRCJbeOWVn+Nw/Ip77/3vcVvHcwVnokXufCC8\n891DTyQSXHfddXR0dPDLX/7ymPfESLhA6MMwXjd5am45DLRFjccNdqqEPpxMTSYTfr9/zHaMFicS\nihmrd3winMlNRAobN27hwAEZh2MGOTmziMfDeDwtiGIrkpROJBJh3rx5HD78OitXPoDPFyYeD2Aw\ntKHrZlTVTTKpoao9ZGTY8fv9mEymwUr6vr4+gkEBm202bvdS4vF/IMvbkaR8JKkSh8NBdrYdQfAj\niptIJLaQSNgRRRDFOFCHJKloWhCnMw+/v51ksptYbB+y3EBTk50FC6aiaQP3ldNpIhSKUV7+3pjU\nnTufY+vWVxHFaRgMNxCPb+XQoTe47LI7+Ld/+zfc7gFJ3Lvv/hyf/nQcRVEGayKysrK4/fYvoSh2\n4vEIYEPX04ABr1LT7CiKi7S0CSjKZozGQmy2VozGfqzWdDyeJPBZfL4EXm8nBkM1ovhJXnvtfr78\nZS+ZmZnjup7nGsbaIne+hNxTGPp8pwj9XN+waJrGrbfeypYtW/jhD3/IpEmTTvqdC4R+FOMVmxSh\n6gAAIABJREFUch/a1y2KIrquj0sV+6nenEMlZIeT6ZmSbR36+x+EUAycutrc6VyLcDjMvn3dFBbW\n0NHRg6apSJIZp7OQzs69ZGW1kJExn7S0NIqKMnE6RRwOEVU9iMk0AaPRiij2IAg7sNl6MJmqeeON\n/Vith6iocDFjxhSamnoRxVmYTMsxGGzYbBno+mEEYReFhZcyZ84CwuGN5Ob6mDPnINu2PYIsH0aS\nChHFBiSpjrw8O37/TnT9CqLRtwmH3wXKAQOtra383//9hNtvz8XhcDBrViVNTfV4vXlkZpYSjfp5\n6aXvo2kfJTPz9wiCgKpqhMPf45lnXuarX/3qMdfEYrFgsVgG/3/+/PlUVRXS2PguslyNqlrR9QTw\nMmAEfoOuh4jF3jk6x/0GqqsXUVpayoEDv6KtbRu63kV/vx9JMiMIF6NpuQSDfbS1tX3ghP5hI4tT\nbZFL/c25jvN50to999zDypUruf766wkEAqxateqYz6+44or3fecCoY+A093BDtUDt1gsmEwmgsHg\nuOXjR/M7oxlsciYf5FPRYh/PjcXw1MJoNhGna0MsFiOREKisnIffv4H29ldxuaaiKBE6O99hyhSB\n0tJSPB4Phw+HWbTo85hMa9mxo5VgcC2Kkoaq9iJJ9aSnWygq+gj5+fOIxyPU1u6nre0NgkEL06bN\nor5+C7GYkUTChiBMw2g8QGVlKQ5HNolEMU1N65g4cQJudw/7979FKCRhtUpMnVrCihW38cADz9PY\n+BPC4Y3A7cBcIAfYQij0O9avL+ftt1dz+eWX4fX6WL/+JTZtaqKl5SChkB+r9Q6CwRCyrKDrIAi3\n0Nn5V/7whz8xY8YMysqKqaioGCQJWZY5fPgwuq7zb/92J9/61v8iCG0YDEtRlJ3AagThBiyWBWia\nB4MhE0XZgaY9S3d3kETCSWvrLjRtEXANBsMsZHk1ur4OsJBMhvnKV77BI4/8kYqKiuMt0RnDuUKC\nJ2qRS5G7oigkk8lzeshMCiN56Oc66urqEASBlStXsnLlyvd9foHQR4nTedEPLTpL6YGnir7O1ktg\nNGQ6Xu02I53TB6XFfrzBKmcCR44cYdu2bTQ1bcNoFJk3bylNTbV4va8iywEmT+7n1ls/SX9/P62t\nrXi9MtXVRcyatRBV1WhoOEJ/fzO63k5pqYlZsz7F/PnLEEUBi8WByWSloeEFgsEAZrOdvDwr7e3d\nQAyLJYzL5cDpdJBMxqivfxSfrxW/fy42Wxp5eTY+9anJXH755RQXF2OxWAiFEvziFw8iipPRtH9H\nEBqBw4hiNbq+iJ6ePaxff5BFixaybNmlbN68k1jMRU7OlbS2Hjpaoa5gMDgRBAlFaUPXFdau7SAW\nu5ht2+qYO7edyy5byq5du3j00Rdob0+g61BYaOQzn7mRRx99kXC4HlGMo+v/gqpei6rG0PUkqpqJ\n1ToQbu/r+x3t7QqqmgsIwF4SiQnArUAEeA5JqqKpSeHzn/8SP/rRfzNnzpzzpk/9TGIowZtMJiKR\nyODzeS4PmTmfPfR33nln8L9T0q8nwwVCPw5Opb/5bISXT1YAliJTSZJOSKZnIn92Ot5x6ntjhaqq\nBAIB4P2DVcYbr7zyCo899iZ9fQ6CQZVI5Cmqqmq48sp/xettJxo9xMSJBWzefIjeXo1wOEhraytu\n9zTy82tYvjyHadNa8HgasNtbyM0twe2eiyi+99K02dIwmbKIx9fj8dRTVXUHBQUq+/YdpKvrFUKh\ng+zbt49E4iF6e3uprPwPZs5ciiiKdHS8xdq1L3P11VcPhr8/9rHreOKJR2lrcyOKVYjilKPV0xLJ\n5FaSyVoCAZ3+/n48Hg9NTRKzZ38Fk8lBff3f8PsfRBAeAEzouoquP4Qo2gmHo1RVXUIo1Mu2bW9h\ntW7kz39+GZ9vLkVFVyIIIkeOvIXfv4EVKy6nrs5OZ2cPinIJgUCceHwXZrMDm62HRKKHUKgPp/MT\nJJNvo2kF6PpHgAxgI/ATIA+jcTJwhGjUT21tkk9/+ltUV+dw773fYtq0af/UVd6nA0mSRj3dzGAw\nnDPX9nwh9NPBBUI/iqE36mhv2pN5xOPdAjfS74wkmHKmH7qhtoxVi/10kWrjURTltAVyTmVz09DQ\nwCOPvEkyeR2VlUsBkdbWjTQ3/4l9+/7ItGlTyM9P4+DBCLpeTWZmEenpSdrb32TdupVceaWLrKxi\nNE0hkTjCJZfMo6MjQF9fF3Z7OqI4QEayHCUe76WwcBKC4MPjeQRVzSIa3YkoNmK1WtC0dvr6NmO1\nfoTJky8eDHcXFl7O/v3r2bhxIz6fj9raWhRFYebMGWzb9jS63owkTUIUJXQ9ArxEYeFULJYBtcD6\n+gPEYoWYzS4EQWDixGvZsuXPwBUoymwEYSei2EVOzr8elX0Nk5aWg8eTy9tvr6a7O5vJkz+BIAys\nQ0HBcjZvfgGPJ4CqJlFVH+FwD1br5bjdeRiNzSQSq7BYWtD1K3G7r6CvbxOS9B+I4iSSyRCwFPgV\norgFQUhHUXwYjd9F16uw2w0cOvQg3/jGvTz++AOD7Wwp79JgMAwWgY0nzgVSGw2Gnse52iI3VHo5\nhfMl5H46uEDoI2A0RV8pj/hEveVnSqQmheFT2kYjmDKeHvoHGWKPRCLouj44UOZMv1i2bdtGX18W\nNTXLkCQDRqOB8vJLUNUm0tJauOOOFaxdu4HeXhFZVvF6G5BlmUTCQXt7Hy+99D+UlVUzYUIeF19c\nwIIF82lqauLNN5vp7bWTlpZDIhGhu7set1tGkipxu43s3r0Fj2c7ZrNARcXVWK1eqqomsn59NYFA\nIaFQePDlGo2209q6lXvvfY1AII4oShgMuUA3BoNGInE9mvb/EEU38AQmUy9FRddSVeXk3Xc3sXZt\nHc3N6cA+ysoKKClZwoED25HlciSpHav1Iszm2xGEZqxWM0bjgJaBruv4fEGMxkmDZB6L9bJt2//S\n0+NElhdQVJSFy7WanJwmHI4NmExZmM0Kc+dO4emnm9C0y4nHGxDFSnQ9B02LIggxdD0KlKJpz5JM\ntiJJ/w9BuBJJ6sblqiIt7Yd4PDeyefNmVqxYMUhCsiwjy/L7qrzHGj07V3LoJ8LJzuFMtMidSQw/\n5gUP/QKOwYlIT9M0otHo4Nzyk/U3jxeBDlVnG+2UtjONcDh82mmG4bK4o8XQMa+iKA56YWcSqqri\n9XoBF0aj8ei1HjimyZROItGE0WjE4/HS1JREECpwOCbQ3d1Nb68RXS8lN1fH6Rzw2ufPn43JZKK6\nuhpZVtiz5xBe70FMJpg2zUxJycXcf/+L9PRUkZFxO4lECFUN4/Xuo7LSQFbWVCoqDrFhwy4aGyeR\nlpZLMtlHS8vPCIdz0bSrkaRSFKUWRWnD4fgiRuOvyM7W6ez8LaCQn1/B7NkfY+HCXGRZYPNmgaKi\nW2hpeYMjR5qIRpPU1EzB6bTi8wlUV/8EiyWHgwffIhp9DF3P5YUX/kFZmRuXy0N19QQOHTqMpqmI\nokRb22v09xswmz9PYWEmFRWVhMNL8Hh+wOc+9xEmTpxIRkYGOTk5bNy4jYMH6zGbqxCEECZTOrIc\nQNNSG5GDCEKMgds/D1HsIjs7A4PBCGQC+XR3dw8Si8lkGrHK2+v1EovFKCgowGaznRM54g8DTrdF\n7mzNZxiOaDR6Xo7uHQ0uEPpRDA+Vj6RipijKoGc4nhKio7Uv9SCNdkrbiX7ndKGq6uDY0DNdgDYU\nwzcxTqdzcC3OJFJDbMrLyzGZ3kaWfVit2QCoqkwotJ30dBNr1qzh0KE6urtzmDFjEoFAgETCSX5+\nFR7PJlyuTObNu4GmprdoaWlh8uTJiKLI9OnTqKqqJBgMYjQaycjIwOv1IghGBCEPgyETk0kjkUgS\njxvZs6cRn68WXZeQ5Tp6e3VE8Qr6+l7H75ex2T6NLGdhsVxMLHYFyeR3EcUQknQNmZmH+PjH7yca\nfY3rr7+YgoICfD4fTzyxh+rqqzAYzChKgp07t+DxbCAel5gyJY/u7oPEYt+ipcWH3+8B3PT2Lqa/\nfzf79jVx/fU53HHHXezc+X8cOvQQBQVX0NGxFUWZhMkUx+Ppor29i4wMJ6paTGdnJx/96EcHr/Gt\nt67ghz/8C8lkJgZDDFl+GE1bgsWiYTS2oWk7MRgWkUgcRNN2YbEsJicnBwBZPgJ0vG/wxtAiMK/X\ny2OPPcGWLY3E45CZaWDFiku47LLLjiGgfxaCHylMfSo41Ra5M03ww88jFouRl5d3Ro71YccFQj8O\nho9SPdWZ4SmMZ4hb1/XBNrizOaUthaGV/MCYBFtOxUM/E3UCuq5z4MABamv3EY3qFBWlM3v2dMrK\nygY/T6250Whk2bJlbNhQy5YtvyItbTGSZKGz8x2gjiNHlvLKK3EaGiL4fE20tLwFFJJMioTDDTgc\nIgaDE4PBiMGQS0+Pj8mT37NluBRvIBAgO7ucgoJiurrasdmCxGJ1xGJZaFoRPl8a/f3NmExTmD7d\nhSyvJRw+RHr65ei6m3jcxMAtZ0IQZiDLLRiNecRiUVyuAuJxF8XFxeTn59PU1ISuZ2E0Wujr6yOR\nyKa6ejFFRQdwu5v50pe+RFFREY8++ig///mfsduXYDRORJa70PU4sZiD1tYBr+ieez7NX//6PI2N\nv0ZRdqNpaUSj1cTjRUiSDb+/BzhMV1fRMWtxyy230NXVw5NP/g2LxY8s70DXn0KWbcRiEbKzF7Bw\n4Q/o61tNbe2viccfp7PzSlS1l0jkYcrKVKZNmzbiOieTSX7zmz+yZYtATs5nyc7Opq+vlkceeRO7\n3c7ixYuPKQIbTQj5n4H0TwUnapEbTvDj3SJ3PKW4CyH3CxjE0BtteG/5qc4MHw9CHzpu1GAwYLfb\nTzvEfjr2DK9iT7W9nA0MrRMYaRNzsnOJRCJ0dXVhNpspKCgY9BI2b97M88/XIcvlZGQU0tZ2hPr6\nt7n11kuoqqoacU78t771FVauXMmaNW+gKCpVVWE07SqmT78LSTJhMqXT0/Mu4fAuLJYG4vEEGRlp\nAFgspqPnE8VqPXElvsViwW4Xyc7OpqyslL1797BnTwaSpKHrHlTVgiCEcDonkZ8/lSVL5rFq1R/Z\nvDmCotjR9b6jBWUmdL0DUbShKO9QXj6NUKgHu10fDEna7XY0zcPf//53tm2rI5kUEAQdq9XDF75Q\nOdjr3d3tJZHIQ1GySSQqgSUIQi+KsoaGhm46Ojq4/PLL+fGPJ9HW1sZTTz3Fb36zDoPhY9jt04+m\nSDxEox4OHpSP2ciJoshXvvLv3H77bbz11ls8/PBLeL1Z+HzNJJO3oChXsGfPYebMWYGmxdix43/p\n7f0bkmQlPb0Ah2M+v/zln/na1z5LYWHhMddy79691NV5KSv7Dnb7wGcORwmHD0d54431XHbZZYMR\nuePliM/XMaZn6nyGE/yZHDKTOt5QXMihX8AxSJHeSL3lZxtDNxQpO8bDOx0tVFUlEokc0+OdsmWs\nG5VkMsmuXbsIBAIUFRVRWVl5zLmdaLAKnPiFpOs6GzZs4J139uL1ahiNUFnp4Prrr8DpdPLuu/WY\nTPMpKZmM1WqlsHA6hw6tYfXqrWRnZyMIwvs2EC6Xi09+8pN84hOfIBgM8rOf/Q2j8XJMJiuqqpKT\nU0Np6WG6u6Pk5Njx+0P09spYLAoej48dO14jN1empGT2Ca9LXl4eBQUiW7a8TkbGJA4fbkeS3JjN\n7ajqgGKb2ZxDMinh90eQJJGpU5eyfv2PCQbTEIRqIpFNwGFgHYmEgezsJBUVi+jr28KyZXls2rSJ\n+vp6RFFk//4tbNxoxmD4HFbrApLJ7YTDv+Ivf3mUu+/+DJmZmXi9fmIxBVWdhNn8UQRBPFqxH6O3\n98+DKaqBoS5llJeXYzA8h67fRzj8DoIQR5I6yMqaz5EjA2ueko5NIScnB58vgKrOYubML7Bu3ecI\nhwNIUojOzihHjnSQn/8xrNZXyc93M3fut3E6S1DVBPX1f+Dll1/ji1/83DG/2dnZiSy7Bsk8Bbd7\nEh7PemKx2OAGeaQccTKZJJFIvO9+S43/PRdxtgv7hlbQw/i2yJ3PfeingwuEfhTDb54UkY61t3ws\nHnoqfwtgMplQFGXMZH4q9gytYh/vDU1HRwc/+9l9NDZGkWUDDofKwoXlfPnLX8Rut495sEpdXR3P\nPbcbi+ViSksnIsth6urWE4m8wpVXLqSvD0pKao65FllZlTQ37yYQCFBSUnLcKMiADKqKqoLV+l4d\nhcORxfTpF7N69U/xeDKw2ydjs5lIT68kFlNobt7JkiVzSUtLw+v1Dmr8Dz+3vr4+wuEE3d372LWr\nFo+ni3BYwGabSE7OZ9F1mUjkLXy+TRw54kXXl5NMmrFaXcjyaziddYTDXuLxRgyGABMm1DB16mLM\n5r1Mm+biuedeZufObjStGl330tlZi6blIUnNqGozkiTgdn+ZUOhu7r//fm666SZMJh1VTSIIHpLJ\nl5GkCqACXTeg6xxDzm1tbTz22JNEIkFEcQeStIvMzCUUFv47sVgbklT3vo6IYDBIOBymvr4Nk+ki\ntm37DqGQgqY1oGmXoKoW9u6tx+9fjSh2MWvWD3A6BwZVSJKZnJxLqK19mng8fowEbUZGBpIUJJHo\nx2x+b0pVONxKQcGxcrVD1/d4BXYp8klNT/xny7+PB0ZqkUvNgD+dFrkLHvp7uEDoQyAIwjE78vEY\n7DHWEHeqxzrVhjNWjMaekwnFjLUdT1VVfvvbB9i9O50JE/4Tu70Qv7+eN954GJfrb3zyk7ePenb5\n8WzYsmU3uj6J4uIBb9hstlNdfQ2NjY/S1taGwQCqmkCSBnLXsiwTjYawWAykp6efNKXhdrspLLRy\n6NABcnLKgYENWHt7K6IYw2TKZcqUGsrL3yvW6urSqK3dw8svb8HjCWOxaMydW8aKFVcOFnVpmsb6\n9TtoajLjdE4hHteIx3X6+70IwkXIcjfxeD+KUgEcwOvdw+rV9xOLBSgpqWL27G/g87WRTMpYLE46\nOjayYkUhc+fOxeFw8MQTT7BtW5icnN9js5Wj6zrt7fcCb2KxVGMwFCAIeSjKThSlgHfe8RGP19HY\n6EXXA6hqK4Kgoihr0PUYJlMeGRm2QW31aDTKF794D/X1TgyGH6FppWjaOrzeV0hPbyAUeo0bbpg+\n2DMeDAZ5+OFHWb16J4kEeL3N+P1HiMWMuFxPEI0+RSLxMppmIBI5TDQapbCwAJer/MQ32VHMnDmT\n6uqV1Nc/TEnJrVgsOfT17SCReIerrlo8qtTV0BByitxNJtMZCyGfLXwY7Eu1yKUiYaNpkRuqLzBS\nDc6FHPoFDBZBpTxSGFvRVwrHq5g/HkYKcQ+14VTbvE4VJzv+eKC+vp76ej9FRV/Abi9AEATS06cQ\ni13HqlWPccMN15GXl3fSvvaRNieaptHV1UVzcydO55xjPjOZrGhaGlarlZISIw0NWykpWXhUmz2M\n17uXSy/Nf18oeCSIosjixRfR3b2efftkbLZ86ut34fc3kp09GUG4iO5uN7Lcwdy5k7DbbWzd2kBr\nqw+zeQEORznBYBevv76F3t6V3HnnCkpLS+nr66OlJYrfr5BITKaycg4221q6u58iFqunq0vGap2D\nySRQXX0jGRldpKcncDj6sNlmMtBOJ3Do0FZ8vgShUA/h8D7y8vK49NJLefvtrVitN2K3VwxeO5vt\nesLhjSSTjZjNi1GUA8RiO9H1GqZM+QQlJTWsWbMVo/EyRPFSNC0XVe1Ckt7Bbm+kpiZtsJhw1apV\nHDrUT37+C0QiNrq6+kgmP4WitNHU9A0WL57DXXd9bXCtfvKTX7JmTRC3+05MJgd+///Q27sNQbgD\nTdMRxQrgMCZTD2azDZvNTzjcx7p1f2HWrFtJS0sjmYzT07OOa64pe5/HbTabufXWa3nggcfweL6P\nrttwu3Vuu20WV1999UnX+XhrbzQaRwwhf5hFWFL4MPfSn6hFbiR9geHnklLuTG0Y/9lwgdCPIuWV\nms1mJEkiGo2edRtOFOIer5fBiTz00YbYx+qhD7R0gc1WcPRfdJJJFZMpj3B4wL7TEanZtWsXTz75\nEo2Nftrb2zEYWrj66m+TkVEGQCIRRpICZGVN5dpry3nyyX/Q0NCGIKRjsYSYOtXI8uXXjPp4lZWV\nfPzjJg4damTHjjXYbL0sWPAxTCYbdXUdZGdX0d19gPb2DtxukZaWbszmiyksXIbV6gSm0dVloaFh\nPbt27ae0tJRkMkkg4CMYTCMvb2CUqSAYsNlMQAfgprDQRlZWPoKgYrUmyMhwU1e3jba2HbS0+Ghq\nehtYSF7eTeh6O4rSyEMPrSIzM5NYLIHB4Br0cBRFJjd34LrHYk9iMEwhFltJItFNbm4JU6dOo7d3\nP2bzBDIyqonFOkgmNwH9qGo/sdhB5s+/HUmSaGpq4u2330ZV8zEa83G7wWIxEwqFCIfnY7Vu4L77\nforL5QJg3759bN3aTkHBf2O3V3DgwG/Q9UWIYh+q2kI0+mV03YcoViBJEAw20tyskpExm+bmZ2hv\n38bEibOxWjupqZG54YbPHrM+fX19fO9732ft2loUBQyGJMuXz+Pb3/4WRUXHVtqPBcNDyKMtsPuw\nEPyHGSdKf6iqOhiuj0Qi/M///A/Tpk3DYDBcUIr7Z4coirhcAy+61EM4Ht7waEPcJ9ODP10hltHg\ndLXYTxdFRUWkpYHPt5vc3Dkkk0k0TScY3EdBgX2wx3i0tgMcPnyYX/ziMfr6ppGffwd5ed3s2fM8\nzz//fT760f9CkowcObKRyZONVFVVoWkaH//41TQ2NiIIApmZNVRVVY2YU00hHo/j9/txuVyDbWZF\nRUVMmjSJ3Nws3norQkXFXGQ5Rm7uEbq61qMoFhobe3A4ujCbTdhsJUfJfAA2WyHRqI0jRwLIskxm\nZiZpaRAO+1BVBY9nM/39PmTZTiLhw2qNkZdXjCgaCAR2M2NGPm1ttSjKFIqLJ9DQcIBYrByjcRm9\nva2Ul7uZNOlWDh0Ksn79ZhYsmMbLL7+Jy/UR2to8eL0BkskDSFI7ZnMARbkZTRMpLb2F22//FKIo\nEotFkWWV9PRMVPUFZDkTg+E6TCYJk2kLmzcf4YYbbqSu7hDR6EChnKp+hrS0q4AkDscMFMVDTc2E\nYwQ/2traiMftOBw1+P078fn6EMU5GI1lqOoadH0OovgrzOZSEole4EEEYSOVld9FlntoaLiXROIp\n/v3f/5WFCxficDgGc+i6rvPVr/4nGzb4sNt/isFQQiCwhmefvY+enq/xv//749Oa1jYalbWRPMxU\njnhogd3ZFmEZydZzDcMr6MPhMAaDgXA4zKpVq7jvvvsAuPnmm7nqqqtYvnw5S5cuHVXU7XzABUIf\nAkmS0DRtzB7oUJyM0E9XC3287DmdSWVjvT6lpaUsWVLNiy8+RiLhxWYrIhw+gKa9zY03XjlqwZ6h\ndr711jt0dxcxZcpnEQQBt7sKq7WIXbu+R13dL6ipmcLFF2dw1VXXIcsyiqKQnp7OjBkzyMjIOMFR\nBqrxX3/9H7z99k78/iRpaRJLl05n6dIlgy+WAe/Bi67rmExWpk1bRHr6fvbs2YbT6WfBgkqi0She\nbwxVTSJJA99LJPoxGJLY7abB6t7ly2ezY8eL1NU9gqLkYzBMprS0hO7utYRCHRw8+CdKS2cxfXoe\nBQUZ7NzZx6xZV1NQUEYgsI3e3kwsFhMOh8iECcVHPZYSurp28ulPf4Lt279LXd0dxGJzEUUQhHex\nWmuw2Xzcdtt8ZsyYxY4dZvLyconHQ/T0HKa/v55w2I+mSeTmfh6ns4hwuJ38/BJ27HiIYLCRtLT7\ncLlm0Nv7G3y+R/H7NyNJWeh6P2ZzgJtu+u4x1zU9PR1RDNPd/QZdXW8QCu0BBFT1WgRhL7o+B12P\nkkjUA1Eslo+j6834fOspKLiVkpK76e39/3j++TXcd99fCQb9uFyZLFgwhTlzZrJt20HS0v5EIlFD\nIBBHED6BrmusXfsbfvCDP/Dtb3+OiRMnjv7GPQ2MVoTlbObfP8wh99OBKIpkZWWxbds2Ojs7Wbx4\nMXPnzuWVV17h97//PaIo0tTU9D7xofMRFwj9BDjTN36qLetEevApnAld+DNZxX4yfOYzn8JqfZp3\n332eREKgqMjKjTdezTXXjD7kPRSHD3twOOYd8yLMzy+jr28hCxYE+MxnbsXtdg9uXhwOB5qmDeY7\nT4SXX36FJ5/cg92+HLe7nECglccfX00gEOCmm24EoLi4mIyMw7S376WoaDIWi4OsrCKmTm3nuusW\nU1FRgdf7d15/fSednTZycycTjbYTiWynuFhj+vTSQS9t1qxZfOpTXfz6168Qj6fhcoWwWpNUVs5A\nEMppbNxERUWc0lIn7e0bcLsFSksrsNkczJw5j56e7eTk5CPLwtFBLDqRSAOlpdlUV1dzzz2f5bOf\n/SpmcwCTqRC3+3YyMm6jv/8JVq9+hLvuuouuro3U17+Gx9OOx2PFYikhHq8nGi0nFNJJJhtwuRI4\nHG4CgTREcT5m8zVomhdB2Aj8C7p+K5qWdXSwyq/fl8YqKSmhr283e/d+BV23oGlhwITReBVGYxmC\nMIVkUkKSfEiSFZutmnjciqbFicXa6eh4jUhkMps2FREKFWMw9BKNirzySpzVq+8jFtOw22cSCvmQ\nJDcGgxVRXIyqPkh7ezYvvvg6d9+dy0svvcSGDTswGg0sW7aQFStWnDBSMxacao/2uTTl7IPC0GuT\nlZVFT08PDzzwAHa7nebmZtavX09JSckHaOHZwwVCHwHj+fCM5KEPVT4bbVvWeBF6yp5oNHraIfax\n2JKSrm1ra8Nms3LZZTMpLi5m2bJlp5X3StmQl+dm7972YcdSEYQuJkyYgtPpJBgMHqP0l5KwPVEa\nIxgM8vbbdaSlXU1h4cUApKUV09lpYd26v7NsWT9OpxO3283SpVNYt24/hw+3oOsG7PZYNBPbAAAg\nAElEQVQoCxYMkKgkSSxbNpfe3pXs2PEEDQ0mTKYkEyZYuO66i9+ndLZs2TLq6/vweouORhxcFBUV\n4nDY2bixl5oamYKCXmbPrmDTJohE+rDZHFRWLqCubjNtbX8kK+si4nGJ1taNZGcf4dJLP0MsFjta\n5Z9DaelKTKb3FOrs9vmEQg+gKAo337yclStfp66ukaysq7joorl4PKvYtGk3otiOzSYxc+ZM+vq8\naJofk2kCALHYq+i6hCTdi653kpeXidv9cfx+H88+u5I77rhj8Jr/9Ke/xOezIoo3oWnzgSPAm2ja\nz3C7JxMMbsBgmIfFoiMISWKxVQjCESKRRrzeDQQCJtLTl5NIuMnNvRpV7UaWH6K4+HpaW5vQtC4i\nke1oWiVm88B5qupODAYjBQXL2b37Sb7whf+gvj6OJF2KrifYsOEp1q/fws9//uPjKjGO5/vhg5py\ndj5sEI6nEieK4mCUr7y8nPLy0XVEnA+4QOhDkLrJz2TIffiENJPJdFYfrpQt8Xj8rGqxw3ta+K+/\n/jrPPLMBrzcdg8GN2byK2tq93HPPl06p3WSo3cuWXcKGDY/T2vo6hYVLUdUELS0vk5vrZ/r06UQi\nkffJxo7mvHt7e/H7NQoKao7594yMGg4dUqmrqwMGPIPKykry8/PxeDwkk0kEQaC/38+zz75CW1sz\nsZgdgyGfmTMtZGUJzJkzm7KyshFrBsxmM5MmFXL4sJOKiveEaPz+LoqLnVxxxdxBVbRkUuXdd3ci\ny1Hs9gxmzLgYVX0WSTpEKORi8mQnN998M4Ig8PDDz1Fb240sJ/B4tpGePgOXKw0QiMXqsVgGwpfp\n6enMm3cRDQ12Jk0amDBXUVGC1/tzursbcDhmYrVaSSabEITtGAw3I4oiut4P5AIDpONyuTAYDBiN\n5fh8QVRVRZIkuru7eeedHUjStWRmfpdo1EcwWISqLkDTPk8yCaJYi6r+F7FYDSZTgHj8dXS9k5aW\nBJrWgyhehShKCEIeomhCFIuJxbJJJDykpX0ETTtEIPADFOWLaNpUVHUDuv5bKioG1OE6O9vp7ZUo\nLHwQkykXgGj0Rtas+TKrV6/mqquuet+6nMkuk1Sh3MkK7Maafz/fQu5D1yPVg36uiv6MFRcIfQSM\nN6HDgGc60Ot8ehPSxsMmWZaJxWLA+CjfjdaWoYNVPB4Pzz23CV2/kYkTl2A0mojFOnn33d8wefI/\nuOmmm07Zjo6ODhKJBMuXl7N58ys0NLyOJEFRkYGPf/x6MjMzT9rT3tvby7599XR1BUhPtzJxYjWl\npaVHC+AgHPZgtb6Xa29r20N3t4c1a9qpr1fIztZZvHg65eXl1NTU0Nrayhtv7KK/P4PGxm5aWx1k\nZ5czfXoVFksCr3cXRqPxGDL3+XzU1e3m0CEPTqeFgoIMrNZ2Ghs13O584vEI0WgT06enH5P3nzt3\nDlbrXnbvPkA4rDJpkpHbbvsKGRkZqKpKfn4+0WiUhx9+kd7eSubP/xcOHGiiufn3eL3/iihOQ9fr\niEYf5MYbl5Ceng4MzEi3WGRkOYrVmobVmsayZZ/g1Vd/QySyjsOHC3C7dSZN0jl8+AmCQdB1AU07\ngCBsISNjCiaT6eiEwtXMmTOQVkgmk/T39xMMxjCZ5iMIAnZ7OrLsQ5ZNJJNFpKcbiEavxe9/BoNh\nB6AjCH5stp9itS4lHn8QRcmjr8+H0XiEUKiVZNKLotQSj9tIJNq47rqPsHXrDmpr/xVZtiAIYDTa\n8HoPYzT+HU0LYDTePEjmADbbRJLJGWzbtm1EQj+bGG3+fejfjXb64PnqoUej0XGZ9XCu4gKhj4Az\nka8ey4S0oTgdm4ZWsUuSNOglnS5Oxfah0+EsFgsHDhzA50tn8uQrBj0Oh6MIq3UR7767+ZQIXdM0\nnn/+RbZt6yQQMCFJSbKz07n++kmUl5dTVFSEzWbD4XCcsD6hvb2d559fS3e3G6u1iHjcy86d67j2\n2iDTpk1j3rxSXnvtNWKxJIKQQU/PIZqaXqK6upSpUz+NxWKmubmWN9+s5bbb0nE6nWzZso9wuJSc\nnDz27++ipuYawuEw7e2dVFVl4PMZWbVqM5MnT8ZsNtPb28uDDz7L3r1mIAdI4HLt5pJLsigu9tHd\n7cHtFlm4sPh9xT2SJDFjxgymTp1KIpHAYrG8z0Npamqis9NAdfVCJMnALbf8hOee+y7NzV+nq0sj\nM9PCihVz+NrX/mPwO6WlpVRU1FJfv5ri4gVYLGlomsrChbOZNctGaWkphYWF3H+/SFPT8wSDfwAU\nBCGIyfRf2Gz3EIkUEAy+hs22kbvv/v7gvZOZmYnDYcDv34fNdh0ATmc+/f17gAaCwS50PYvq6iup\nqfkXtm69l2SyBqPxUoqKCgiHF9HbW0c06iGReJBw2Imum4B29u1bh8PhRBSXYDSm4XZ/jEhkIlCB\n0ZhLf/+vyMraRVlZGXv2qO+7HwQhiSSdmRz6WDA8/z6U4JPJ5KAo1skK7M5nDz0cDv/TtqzBBUI/\nIcbjxk+JyiiKMqYJaae7ARhexS6K4rgNVjnZ9Rk6uzx17gM5wfdvaAwGG7HY+5XwdF2ntbUVv99P\nRkbGMcUtmzZt4o03jlBQcAtFRZNIJuM0Nb3B5s11zJs3D6fTecLwWyodsm7ddnp7i5g0admgXa2t\nO1mzZjeVlZXcdttN7N//YzZs+DWxmI1EIoTBkKC09BIgJV2p8dZb++joaOWiiyppagpTXLwEv78T\nRTFgsQx4y/v3b6SrK51k0khjYw95eS+zYsXlvPHGW+zYYaOs7EYcjiwUJc6RIzvYuHEd3/veUrKy\nsgbDq6koy3BIknTcl9lAvYBjsLo+Pb2IO+98hLq6l1CUdXz1q5/FYDDwzDPPEI/HmTNnDnPmzOH6\n6y/Hal1LQ8Nr9PYKuFw6118/kfnz5yGKIvfddx9///t28vJ+hdE4l2j0MJHI73E4DmA0/hJF0Skp\ncXDlldficDiIxWI4nU5ycnL42Mcu46GHnsPvL+T/Z++846Oq0jf+vXd6yUwqpBFSCCUh9CKggEoV\nsVfsKO66rrru6q7oWtf6w7Jrx1Wxu4KKICxNQEDpvZeQQgqBtKmZeu/9/THMkIQEEgis7ub5fPgD\nmDnn3DLnPW97HoNhHH5/GYLwAUlJXiwWI4FANE7nEdate52qql1AZwKB0LVYLBfi8ezDbp+NIFyO\nLN8KWBGEMmT5GfT6BFascFBQcBC1+nH0+kQkqY5AINTLX1KyluzsFPz+JXg8V2EwhFjzjh79ikBg\nBQkJtxMMBps8CP5SvL/6Bl6n0520wC7c//7fFIo+mYf+v4p2g14Prcmtngr15TfhzEPcpxM1aEpc\nJvwjb4viupOhvrBKfbnZbt26odevxeEoQKdLARQkyY/dvo7x47s2GMNut/P557PYtq0KtxvMZujX\nryM33ngNUVFRrFmzA7W6DwkJIS1StVpPp06j2Lt3FwcPHiQ9Pf2U63Q4HBw65CYxcViDzyYn51BY\nuIPy8nK0Wi3JyQO56qrOCIKKAwcOU1eXSUXFHhIS9lFbe4RDh7x4PANwOFSsWeOitHQ/RmMVer0R\ntTqAx1PN4cNrcTispKdfQiDgQa9PpKDAy/Llq9mwoQiLZSwWS0hrXacz0LnzIPbu3czevXu58MIL\nW/uIIlAUhdjYWDSa/dTV1WI0hkLqoiii1+sYOXI4a9as4dVXP8TttgIGtNqZjB8/iOeff5Zrr72M\nyspKvF5vJH0BId3pmTP/jVZ7KzExoWp/szkVrzcZu/02Hn/892zbto+iIoFdu+LZvXsVqak/cvvt\nV5Kdnc1TTz2Fy/Uw8+a9TF3dP9BoFLp3NzN16tM88sg0amrUmEzXoVZ3QBST8HoXIwh9EITLEEU9\noqhHEMzExv4Wj0cHxKNWpyNJdxAIzCAYvAivdx1xcSloNJ2w25cQCLyPovipqenJ/PmlGI0VlJXd\njiAMxGZbg9frQK2O5sUXv+b775fy+usvRljwfuk4GUd6fYKb8HsuSdJ/RQV9Uzn0X/s1nS7aDXoz\nOB0O9jDqh5nDRvRcnozPFVFMU/fnVNrlvXv3ZuTIDJYseRPoh04Xjcu1iawsG5dccmuDsWbOnM2q\nVUE6d76Fzp07YbMVsXz59+h0c7nttpuw2eowGEIc4uEaBUFQodHEIUlSi3OJghCqiK+PUIV8aJMM\n5ejj6d59MAB+/1527QogCCkUFGzH6zUTG3sRolhERkZHEhMTKSsrYOfOnxkx4mri43Xk58+nrKwM\nq/UC3G4ndvs+OnXS4PVa2bx5M06nC5eriN27K/D7vURFxdGhQw9kWTmhta6l76bb7WbJkh/46acd\neL1+gkE7VVV1pKVdiFZroqpqP7Gx5URFpfLkkx8SDN5FUtIdgBq3+ye+/34qffrM5Oabb26ycM9m\ns+Fw+DAYGlbo6/VdqaoysHTpj9TW9qJLl+vR62MoLS1k+fLP2b79aX772xsZNmwYf//7azz0UCk7\nduwgOjqa/v37s3//fmTZiko1HoNhKIKgJj7+XkpLj+L3v4zXa8LrdWGzTUejiSYhoTPl5bXodNEI\ngojPl4okBVCpEhAEA37/HCTpGurqPkVRegIXo1YrGAyx1NU9S7duRwgGN3D0aBSCMBlF6Y/DcYSf\nfvqQK6+8mRUr/o3FEpLB/bWEq0/GkR427h6P5xdBcHO6aFdaOxG/nqd3jnG6Bj0QCGC32yO9zmFG\nsbbyiE81jizLOJ3OSBW72WxuU2GVxuPUhyRJOBwO/H4/JpOpyZOySqXi/vvv4f77R9C9+3ZSUpZz\n883JPP74fRGBD4CKigq2bKkgNXUcVmsaIb73DJKSRrNxYwnV1dV06dIRm21fJHcYMsxOtNoqEhMT\nW3QdFouFrCwLFRU7kaTjsrAlJdtITlaRkpJy7F4dv4dJSYno9bXU1JRgs5VTV6fBZjtCQoJChw4d\nUKvV5OaOQKU6SEnJj0RHWzEYtuP3b8du38XOnYs5erSMykoThw4Z2bnzKDbbQQ4c2EJlZRw+Xx9K\nSxXWr/8MtfoQmZmZrX4+gUCAd9+dwSef7KOycjhe7+UcPtyDqqot+HwLkKQf6N3bxnXXjWDbtm24\n3UnExd2FoshIkg2TaRgwhjlzFjU7R2xsLNHRBurqNjf4d49nNyqVm6oqicTEkRgMMezevZt164px\nOM5n//4YXn99BdOmvUtVVRWpqamMHz+eIUOGoNVqKSkpITY2m/j4BJzOjdjtu/F6C4iN7UlMjIIg\nPIte/xbjx+eSkCARCGxHFBUkyUsw6MXtno3fX01JyQeoVDUoylwcjj+gKOVAbwTBh1abhtGYg0p1\nOWVlXiorJWAcRuMkDIY8RLEzsnwlBQVBJky4gTlz5vxqjHlTCBturVYbMdx6vR61Wo0sy5GiVbfb\njdfrJRgM/mqut/4e43a720Pu7Qih/ovRWoNeP8QeVkgTRRFJkiL/3xbrO9k49eVWzwVRTP21+P1+\nXC5Xi0hydDodl156KcOGDcPn8zFz5ix+97vH8Psl8vLSufXW6zGZTHg80LFjxwbfNZs7Ul4eCpUP\nHz6ErVtnsWPHlyQm9kaWfVRXr2bw4Bhyc3NPuf7w8x42bCDV1SvYt+8bVKpEJKmG+HgHAwfm4PP5\nSExMRKPZjNNZhcEQfcxgJ1NX9zM63VFsNi2ZmZlkZXWPeER6vYkhQ3Lp2zcFp9NFt2698PkCVFfr\n0el6YTTG4nQeAarQaMzU1sYRFZVOICCjKBKSFIvTCX370iKDvn79er755huOHq2iZ88cevbsyYYN\n1WRk3IfJFDrcJCYOZPfut+nc2cANN1wXOWyG3plYamo+x+ncgizLaLUWBMFLba3zpM9x0qTLeO21\nz6muNhMVNQKv9yB2+xvk5SUSE9MJnS6K2lob+/cfQafLIyrKit2+mk6dxrBr148sXryESZNubDBu\nqLMgSFZWOjabD7e7Dq02Dp8vBo8nlmCwDofDQSAgkZubxLZtT6Io43G5EggEVgKrMJmuJxAwAxIG\ng55gcCfBoAFR9CKKCRgM8cdmi8Vur8LlEggGrcjyJkBCls0IwggU5V+UlSXz5psL0Wg0jBgxosl7\nYbfbURTlV0MxGs6rNy6wC4fo/xMMdq1Fc33o/8seertBPwlaaoTrF57p9XoMBsNZ6Wk/2TpbGmJv\naw+9Pg99a7XL/X4/Tz31AuvX+4mKmoRWa+XHH39k587/46mnfo/FAjU1+SQm9ol8p6bmIFarQExM\nDDExMdx55yWsWrWBQ4fmYjAIXHNNNmPHjmmVuEvHjh256aZL2bdvH4cPH6G6uha3G5Yvz2fVqnyy\nsqx07aplx44lBIMd0Wh0CEIlV1yRTbduo1i5ci/BoBgxkA5HJV5vAX37diUvLw+fz0d+fhV5eSNZ\nteogguBAr4/D4bBTU7Oa/v3Tsdn8DBzYj5oaD3Z7MVqtiry8fiQnF+PxeE66SX300Uc89dQ0/P4s\nFKUrixd/g8Ewg6ys6+na9XikQhTVWK157NmzIrJWgJ49e1JX9zUuVywazWWIYgc8ns34/e/QqdPJ\nGbYmT55MXZ2HL798l9rat9Bo4OKLc3j88Uf44INZHDiwmrKyIkpLd2IyZaDRJGCxSMTEZCNJHtat\nW8iNNzbs7e7bty8ZGXPJz/+CjIybSElJpqpqKzt2fIzXW4BO9xvU6iSWLp2PyXSAiRNHsG7dHA4e\nLCYY1GIy3YzROJz4+HgqKzOprHwBOAIEkeVHURQ9LlcXBOEv+P3vEQwKSJIJqEKShqAoxYiiFUEo\nAUSioy8DKpk9+weGDRvW4KB84MAB3nnnfTZs2AtAv37Z3HPPXWedUvZM0FQvfUsZ7H6pCnJhtBfF\ntaNJtMYoNS48O5trakoutDVc7G15wFAUBYfDcdo89Bs2bGDTpqN06vQqRmMnAOLiLmD//sdZsGAJ\nw4f3YvbsxQSDXiyWUA7d5fqJq67qHll/dnY2gwYNwu12o9VqT7uLQK1WU11dy+rV+9m6tRSNJoZe\nvYaRlNSV9et3kJpq56KL0qmpcSKKATp3zogodpnNZhYv3sKBAweprXVTW3uAhIQAR44YSElJOaag\nBrm5Iykt9VFRUYSiHCYmRkCni6FDh0xUqiI6dIinb99OeDxeamtr2b17HZs3b2LuXB1DhpxHamrq\nCe2GR44c4bnnXiUYnEJ09KMIgoAsO6muvpoDB/7NoEFPIwjHD3c+nwOzuWE/fmZmJnp9Ik5nV0Sx\nI7KsRZI6otH0x2AInpRMRa1W84c/PMCtt95CYWEhcXFxkSKywYN7MHPmS1RWJhAIDMbn8wDfoNdn\noNNFNzuuwWDgwQfv5PXXP6Sg4HH8fjWSdBSPZzvR0V9iNIa8ZEW5haqqmzl6tIZly+ZyySVXs2+f\nAmzF59tBcXEKPt8oFCUbSSoDRgNXoigGfL7P8fmuR6WS0OsHoyg+fL7ZKMpaYDCyHAD2o1Ilkpyc\nh9FYxZEjS7Db7ZHD0OHDh3nwwScoKkohJuYRBEFg2bLZ7Nv3BO+//2qbKrqda7RGQa6xRvm5QngP\naFwU127Q23ECThXebqlXfDZZ5851iL0+FEXB7/e3KMTeHAoKCpCkzhFjDiAIIhbLELZu/ZwnnngU\nvX4JK1Ysx2aTsVhExo3rxpAhQ46xj2ki/OBnon+sKArz5i1mzZo6amp6AEPQaNTs3r0Dkyma1NTz\nKCr6nqFDrZx//vmR74Wff7du3YiLi2Pu3LkcPLgXWc7C6+3CN98UsWnT11x55Ui0WoVAwENOTm9k\n2UOHDt0JBBwEAqsRBJEOHdw4nUUIQhq1tbX8+ON6Skt3YTAYee21rXzyyQKmTLmcMWPGROaXJIll\ny5bhdgtYrQ9E3jVRjMJg+C0Ox2/Zv/8bsrOvQhBEbLaDBIObGDZsSIPrr66uJjt7ED5fb4qKdiPL\nMqmpCaSmXo3LNQuXy9VAJa0pxMbGniByE1KyyyYx8QGOHAmgUlnQakdgs71DefnP2GxrGT++R5NG\nIDs7m1deeZodO3bgdrtZunQp+fk1EWMOIAgq9PrL2bDhKb788l/k5wcIBq8nKmoULtcuXK7PEITv\nUZTtCEIOavX7BINHEAQJRbkfRVmNKIrIcgZRUdcgSdUEgz8D3wHlCMIg0tKmkJiYSHn5RmJjtQ0i\nJfPmzaO4WEt6+jRUqpARsVqHUlR0B3PmzOHee+896T37T6I1hrcpgpuTaZSfywK7xtfRHnJvRwSN\nc+jhHvLGqK+Q1lL61LYMudc/TNTP17cEZ3rACM8dVqWzWCyn/cM1m80oShWyHEAUjx9GfL4jdO4c\nOqBMmHAJF110ITabLeIJhO95/Ra80/EMwt8pLy9nx45qUlMnYLMdJjo6Bas1kYqKAHv2bGX48K5o\ntYkniIvUhyzL7N9vp0OHa+nUaQAAkhRkz55v2bx5OzExKlau/DdGYxYaTR2lpdW43eV07FiN2Rxk\n0qTzKSgoY+/eWaxZc4CamgCgx+MZj1rdiQMH5vLii59gMBgYPnx45Dl4PB5Cj1KFokD4NqhUBvR6\nDTrdSvbs2QfoMBprGD06neHDh5/wHPT6IN26dWXAgAHY7fkUFs7m4MFtpKa6T/v5rly5ldjYS0hJ\n6UdMzGHKy20EAl2oq4ti587/Y9y4gYweParZ72u1Wvr3D9He7tu3D0FYgSx7EcXjpC+SVIXJpOW7\n75YTE3MTNTWDkOVEPB4roEVRXgJqUZRLkWUHKpUJnW4RgrAatzuJQMCHolyGVtuHDh101NWNwG6P\nQlHeIytrFHl5/ampWUdd3Xxuuum8BmyDu3fno1INiBhzAFHUoVYPZNeufad1z84F2iLddjKN8voM\nduH+97ORf2/qOjweT4Tp8H8R7Qa9GTTnoddXKGuJZ9rWQi/hKvbWyJ22FeqH98MhuTM5hQ8bNowv\nvlhEcfEM0tJuQRT12GwbCQaXNVBd02g0GAyGCEFNW0ciampqcLs1dO6chMlUjc3mJhgMotMlU1e3\nCwgATozG5ivnDx06RE2Nhq5dj+f7VSo1HTr0ZtWqj0lNzcDlqqWszI3P50WnO0KfPlZGjhxMjx7d\nSUpKwuFwsHLlSnbtOowkDaampgcJCUOORUFiqKoq4bPPFtK3b1+ioqLQ6/WMGDECne5lXK4ZmM2/\nRVEEwI/HM4P+/XOZNu3P7Ny5E7/fT2ZmJtnZ2Sc8s169epGZOZ/8/C8JBCQ2bvw/JMmCokRTWnqQ\na665kS+//KRVBV/BYJCjR8spL5+Fz3eEpKSrMZtTqK6u4fBhO6mpEhkZCRQXFxMdHX3K9+iCCy5A\nEF6houJaTKYrMRpHIctOgsFPGTFiEGvXFhIT0wmPJ8DRo1uQpDDjnoQgpKEoZZEwus+3HK32SkQx\nAUHYi6L4cbmqUak6YDCo8Xo7IAg6rNZlFBb+hE7nYcSITK644ooGe0JcnBVJOnTCWiWphISEGCRJ\nYvv27VRXV5OZmXla3Qq/BjTHYBdmr2usINeWBr7dQ28I1e233/5USz74ayFXOFOEq9IDgQCyLEdk\nFMPFXx6PB41G04As5VQIe9Jnaog8Hg+SJKEoClFRUaelnS4IQuQaWhMmDwQCOJ3OyNxhD72l2uVN\nQaPRkJoay44dcygtnU919b+BZVx6aU/uuutORFHE5/PhcrlQqVQnHKDC+bzTPdSE23WCwSDbtxdi\nMGSh1eooKSnF5wt5fwZDOWp1kJQUJyNGDG6Qow9vVgaDgcOHD7NpUwlxcf0RxePvRW3tIQ4eXEtm\n5kQGDryCtLRUOnfujFYrMnJkF0aNuoioqCgCgQAFBQXs3XuAbdsOU1mZhcnUH53OcuxaHcABBMFN\njx7xpKSkoNPpiI2NJRCoY926t/B4NuD378Tr/RtRUft5/vknyczMJD09nezsbOLj45s0nBqNhqys\nVLZsmcvSpe8gSXegUn2IwXAnev0ESktn4PNVctFFFzX4nsvlYt++UNtguE8bQv3pl19+NZs3b8bl\n8lNVtYzi4neIje2Gx7MSu/0nZDmbXbt8LF/+I2Vlexk0aECz72OoTuA1Kip02O0W6urW4nK9QzA4\ng/R0I6Wl1eTnH6K8fBl1dRuAFGQ5AXAA29FobkFR5qIoASAflSoGGIgoHiQqyo9K1RW/v4hAoARF\nKUSn283IkVFMnnwpW7YsZ+/e3Wzdupevv/4Gk0lHXl7eMVY+Az/88B0uV4gDXlEkjh79GrV6Mddc\nM4Zp095gxowFLFiwkXnzFnPo0D4GDx54TlNjTSEQCDQIobc1wgIzarUarVYbkYAFGhTZhfey8Hda\n+xsOt9bVv5+zZs2ie/fuDBw4sO0u6BeAoqKiFn2u3UOvh+ba1uqH2E+n+OtMSGrgxDC31Wo9ZwQQ\nzbXj+Xy+ZlMSLYUgCAwfPpxBgwaxdu1avF4vOTk59OjRA+CkBDXh74fXeLrzQ0jLvEsXAzt2LCMp\naSDdusWwf/8mqqs3kpOjoUuXOC68cOgJJ//w9ysqKvD5fKhUFRQXryUz8wIAgkE/hYU/IAgaRFHE\nbq8gOjqZmJgUdDojhYXrjvXPC8ydu5BNmxxIUmfq6rZQVbUbSep6rOVMweVaTXS0EZ3uRO/joYce\nIicnh3/9axaHD6+gb9+e3HHHy2RnZyNJUpMynI2LmLKzs8nN7cz338ej0/0Vtdp4bMPvjd9/BzNn\nTueZZ56JRIneeust3n33I+z2OlQqGDHiPF566XmSkpJ49tln2br1EFrtDwQCfZAkJ5I0lS1bpqDX\np5KQMIVeve5FFNXY7btZtOhVunefz1VXXXXCtR08eJDHH3+KDRv0pKW9RteuyRw5Uk5V1UxMpjlU\nVfkIBi/BaLwYm60GQZgNvIMgPISirAKiCQZHIIqVSNIbgB9FuQ6drpjeva/gyJGPsdsPolJ1JSoq\nSHJyALW6hquuGsWrr75Ffr4Wne4fiGISJSWzmDr1ObZt28att95Kr169uO++a2KTgs4AACAASURB\nVHnvvc8oLv4KAKvVz223TeTdd2ewZ48bk2kAsbEXo1JFMWfO28TGfsADD9x3Wu9rW+Fc95e3psAu\nHKJvqYFv/Jn/dWKZdoPeDMJGOExheibFX2fKOhcOc4dP1GdqzFu6nsbCKvXb8doS8fHxXHrppZG/\n1z9AnUolrS0giiLjxg3H41lIcfEcZFlLXl6QzMxsRow4n7S0tCafeyAQ4IcffqSgwIvTqcbns1JY\n+DnV1TuwWLJwOnfg8+3F40ljy5ZStNp8UlOj6dFjGHD8Pu7Zs4dNm5ykpl6OyRSL1ZrEF1+8QWXl\nhwjCftRqO2azHZMpgY4dD9OtW0MpV0EQmDBhAhMmTDhhjc1too2LmNRq9THxnlh0uoYFcKE2tjpk\nWUalUvH+++/z4ovvIAj3oNdPIBg8yJIlL3H48F18//03zJw5G0V5ELX6PFQqhWBQhSS9iCTNxmi0\nMHDg7xDF0P20WnOorR3O0qVrTzDo8+bN4623vmbr1kLgPg4ccBMTU0yXLpnAlezb9z4wGIvlN+j1\natRqB7LcDVm+HngYrXYEKtWDSJITs7kn0JHBg5M5eNBDv34jMRgsJCTcyc6dH1JRMf/Y3xMYPbon\nsixz8GAFFssyIIWqqmr8/ngUReKDD77hk0++o1evbF555UU+/vjvbN68GUEQ6N+/P88//xKbNlUj\niuPx+w3YbJ8SHd0Zq/Uq5s37kjvvvOOMijjbAv+pdrNTFdj5fL4TPtdcgV07U9yJaDfojVDf2IVp\nTFvbX91WaFzF3hZecRinI6xSH2cadWhujNbWKLQFwqx6t912HQUFBfh8PrKyshow1zWFrVu3sn69\nm/T00aSkJNK5s4vt238A1pOXZ6C4WCIt7QaOHDEgSRlUVq5gzZqPUaleJj4+luuuG4pOpyM//xCC\nkIHJFKoST0npxTXXPMDMmc8TDP6TmJjBmExmoqP3cv31FxMdHd2q96CpTbRxEZPP56N3797AhwQC\nq9Fohh77bJBgcBZDhgxApVIRDAZ5771PUJSbsVgeAUCjyUOtzmT37ktZtmwZXq8XUUyLzB1K78Th\n9cagUukQhIahXo0mBoejoeBMWVkZ//znd/h84zAYFiIIuajVeVRX78bp3EYgYESWVahUQ/F6dXg8\nTtRqDVptKh7PIPT6n5DlUoLBGYiiG5PpIKNHD+bPf/4jzzzzD/Lz36JDh4uB0HuWnh7NHXdcTd++\nfUlOTuaFF15AELLw+xOoqjqMoswDpgP3ABchSQ62bfs7d999PwsWzGbs2LEoisLKlStZubIIUXwY\ns/kmACSpgNrav6DXJ6FSBbHb7f8xg/5LY39raYFdc/n39hx6Q7Qb9CYQDAaPqVOB0WiM5NFPF6fD\nOtdUFbvf72+z9reTzR2mgWwsrHI20bgN0Gw2n/IAdaYh93C9RDAYpKysjH/9azY7dpQhCNCnTxq3\n3noDWVlZzX5327ZiTKbuxMQkH2NXM9CnzxiKilykpydQXW2hc+fhxMUd4euvXyM/fy+yfAFqdQxu\n9y4+//zf9OqV2+BawsjI6MmYMdcTG7uJ6Oh4YmMtDB58BV27do28m6eL5khERo4cyXnn9WXNmlvx\n+W5GFJORpNkYDDv5059mAKH8eGVlNTrdyAZjajS98HhiOHToEDk5Pdix4yvU6hsiPfCStAo4QlRU\nKnV1hzCb04/NHcDpXMOYMdkNxtu4cSM1NWaysm6ipmYnVVU/oNUOBEzY7Q4sliNoNFoU5SA6XTxe\nr4yi2AkGaxHFArp2nYxOF8PBgx/QvbuF3/3uAUaNGoVGo2Hq1N8xc+ZsNm/+EICxY9O57rrHycjI\niMyflJSEJJVQXV2MoliBWcAE4EGgDkGIRVEyKC29jMWLFzNp0iQURWHLlq0IQn/U6iwCARsqlQVI\nRFEGUV39A506GYiPj6cdTaOlBDehdkMZURQjXS7hOqf/dPTjP4l2g14P4XxxOMSuKEqbhHtbY9BP\nRhTTFl7xycY5lbBKS8ZoLcIht9aQ47QF6hMC1dTU8Pzz71BSkknHjr9DUWSWLl1EYeHfefnlx5sU\nJgkGg3g8EgaDBTh+HzQaPbKsw+12I8taNBodRqNAdfU+zOZ70GguQKVykpZ2F2Vl03j//c+4774p\nbNiwm7q6XhiNoUpyp7MSs7ma66+/9qyzjtXPcX766Qxef/11vvpqFg6HgwED+nDvve/Sq1evevoA\nRuz2bej1o+vdj2IUxUZycjKPPPIQt956Jz7fFahU1yLLRcB0zjtvAD165LFu3YuYTKPQaCzYbKvo\n1KmCyy+/ucGaQrlVPaKoJjNzEk7nyzidj+L3JyJJh5DlUjp1GkV5+Tq83jeAEcjyLILBxahUhQQC\nXTEaJW688WIeeui+BkV7nTt35uGH/4DTGaK1lSSJn376idWrV5Odnc2AAQOYOHEiTz75IrL8JPBn\noAi4BXADJmRZQBQTgAwKCwsjOV9BENDrTcTGmqmo2I0kCSgKKEoZweA2Ro16gLAK2n+SSvWXxvDW\nHOq/m9DQwMuyHEkL/vGPf6Rr167HWjB/eVr25wrtBr0RvF4vOp0OtVqN2+0+7R7n08F/kijmXOet\n4Xgbnt1uB84d/3z9Ir9gMMiPP66gpCSa7t0fiuR2Y2J6sW/fX1i2bBk33HDDCePodDqSk01s315C\np07HDa7DcQSDwUvnzjkUFRVgtx+hpGQ7fr+eDh0m4vG4MJnM6HQ64uLGkp//FPHx8fTrZ2Lz5tkI\nQjogIwiHGDQoli5dupzV+9EYZrOZRx99lEcffRQ4MTyvKArXXXcZ7747nbq6NHS6CchyAW73o6Sm\nxjJ69GgMBgOffvohL730Krt2/R6z2cikSdcydepU6urqWLBgAcuXL8Lj8TNiRDeuvPKPJ7R05eTk\nYDQuxmbbSkxMP/r2fYySktkUFb2PXi/Rs+ffSEmZSGHhx+zc+QZu91OABVE0oShqjhxZzB13PMht\nt92G3W6nuLiYpKSkBqmjqKgo1q1bx1NPvUp5uQxY0Wi+ZsiQDJ577ikmTLiIzz6bD6wEPMA64ErA\njKJ4gFoEoZhOnY7n/vv378ecOR+j0YwFJBQlHkXxA3vR6VLYuHEnt9ziibzn55or/ZcWcm8t6ht4\nl8uFWq3G5/Nx9OhR5syZg8fjYfTo0YwaNYpRo0Zx8cUXk5WV9as5wJwp2g16PQiCQHR0iJKyrXTD\nw+OeLOfZXCV5U+OcDQ+9fuGf1WptUYi9tWtxOp0sWLCAn3/eAsCQIb254IILMBgMrSbHqb8GaB3n\nfrjIz2AwoNVqsdvt7N9/CJ0uL2LMQ/3KCmp1LgUFJc2O169fLvn5P7N//ypiYtKoq7PjcOzhvPNi\n6NmzJzU1dtatW4fNVoUs+3C5StFq9RFvMRh0oFKFDMsVV1xCt257KCwsRRAEsrL60L179zavIXA6\nnSxZsoSjR4/SrVs3hg8fftLnXT8EumnTJhYsWEBtrZ1evZLYvfuPOBwPoVJBZmYSb731JjqdDkVR\nGDt2LGPHjsXn86HRaCLPVhAEJk2axO23337Sw3JOTg6jR+cyb97r1NYOQ6uNRa12MmxYLrIMLtcR\nbLbtVFVtpq6uHEgHSpDlWmRZ4cgRB3/60yO89dZ0qqtduFxODAY9V145jldeeQWtVovT6eTpp1+l\nvHwAqal/QK224HbvYsWKp/nggw+56KKL+PzzhSjKhcAK4EtCRj0d6A1spkMHHRMnToyse+TIkQwf\nvoIvv3wYRbkAvT4JRVmNxdKZzp2nsWvXQ2zbto3hw4f/KrnSf0kIt8dZrVa+++47vF4vubm5TJo0\niQ0bNnDvvfciSRJFRUV07tz5P73cc4J2g94IYUN1Nilb66M14ea2MuhhnImwSmvgdrt58skXWLvW\njkYToh1du3YlK1as47HH/kRKSkqbzRsMBiOFNo3/vXGRX/iQlZBgJRAow+utoqjoG44c2UFIRrQA\nSTqv2bkyMzOZONHD/v2HKC0txGRSGDYsHavVwhdffIHf7yclRUtMjIZVq5zU1X1McvIf0OsN+P2V\n1NbOZOLEnhHClt69ex8rTDs72LRpE7/97YNUVLgRhFgEoYq+fbvw/vvvnDKv+/HHH/Paa59QV5eF\noqQgimqysjK46aaryc7Opn///giCQF1d3QmFTid7n5uDIAg88MC9dO++iGXL1mC319GnTxYTJ95B\nTU0Nzz33OsuXP3Os8twIHABuBx4GNMAryPKb5OcXIAgXI4oTcDr38umnX3D4cAVffz2Ln3/+mfJy\niZSU+1GrQ4cskykXk+la/v3vj3j11SfRaiV8vl2EuhIygRFABfAuanUdl112FxUVFcTExER4GZ54\nYiqLF4/F4ShAp/MRHX0Z8fGXolZbqKpKoqioiJEjR56yletsUqn+NxwUGu+FarWayspKnn76aaxW\nKw6Hg7Vr1/7PGHNoN+jN4lwY9NMNsZ9pGiAcMTgTYZXWHC6WLVvGunU1pKU9i17f8Vj++WI2b36C\nVatWceONN556kJNAURQKCwtZuPAHtm8vQasVGTYsl3HjxmKxWCIRiMZFfuHrHT58GD/88B4//3wf\nPl8mWu1V+Hw2JGkRmzYVUlxc3OymkJ6eTp8+fY7lzGU+//wLvvhiLTabBVHUYLFUc+GFXXniiXuZ\nNm06BQUrEMUOaLU2evaM53e/+/MZXXtL4fV6+f3vH6KiIherdRpqdUd8vu1s3HgPzzzzHK+//lqz\n3y0pKeH11z9Bku4iJWUygiAQDNZSVvYbDh4sZMqUKcDxgsoZM2bw+edfU1lZRc+e3fnNb+5kxIgR\nrS6u1Gg0XHrppQ1aGiF0z5OTLYhiV6Ki7sPhuBfoDvzj2CdE4CVgFVCGonyLWh1KIQUCA1m27D52\n7dqFw+FAUQyo1dYG42u1SXi9QWJjY0lLS6ewsJJgsAvwBqAnZNyX4Pc/zWefbWDu3J+55ZYJ/PGP\nDyKKImazmaysLpSUXERKym8i4waDDqDqhO6JlnQhhD93plSqv/aQexhNCbN4PKFOibA4i8ViaaB9\n8L+Ac8NO8ivCuTi5hiu6nU4nKpUKq9XaImPeVmuTZTnCsmSxWM56EdrmzTuAPuh0HQgE/IBCVFQq\notifrVt3n/a44TWXlpby2mufsGgRBAJXYLeP5osvinnrrQ+w2WwNWg8LCwtZvXo1W7ZsiRymevfu\nzfnnp+Px1AJDCQbVmM3xDBr0CHZ7FosWLT7p/BBSCFuzZg0ffbQDr/c6MjP/j9TUvyDLl7JgQT4/\n/LCaPn0m0q3bWFJTU+nXrxsPPng3ycnJp33drcHKlSspK7NhtT6LWh3SmNfpeqHX38eiRSuoqalp\n9rurVq3C5TISF3dLZG61Ogaz+QaWLVsfqboXBIG//vUJnn32HQoLR+D1PsLq1UamTPkDCxYsiHwu\nEAjg9/sbMIW1FqtXb0anu4aoqAsAmVAIXCRkbBUgCAwEQtGP8DyieA2yrGflypV0794dvd6F07kx\nMq6iKNhsP5KVlUhycjLXXnsJsuwCrgBigEQgHhiHoiSi041HFKcyY8Z8fvzxx8h9uPrqsQSD86ip\n+QFZDuLzVVBaOo3UVPEELv3GCBvucGGqyWRCr9dH2gbDhbvh6NqZ3Mf/JtTV1WEwGM5JV84vFe0e\nejM4Wx56W1R0n66HHj5ISJIU6fM+3TBe/ftzqrVoNGokyUMwGEAUVfXywj40mjP/8f3440oOHUog\nN3dKhHY1NrYH69e/wsCBGzn//PORJIkvv/yGbdts+P1WBKGO5OQNjB49gNzcXJKTk+nS5WISEvoA\nAnFxceh0OoqL+7Jjx8+nXIPP52PhwjV4vTlkZl5y7DpNxMWNYf/+RaxZIzN58h30729BURSKitax\nfPl2unXrdkols7ZAbW3tsb7tFABk2Ynb/R0+33oEwY3NZjtBLS2MUGpCdUL/uCBokGUlkrrYu3cv\ns2bNQ6OZhtF4HQBG453Y7Xfz97+/w4QJEyLEIWcaVjYY9MhyDSpVEhpNdwKBHwlRvRoACfABS4EO\nDVjHFKUWQQgSHR1NXl4eI0fmsGDB89TVXYVOl4rdvgqTaTW33XY/giBw332/Z9q0t45dY/jwJQBG\nIIDHs4OkpKmUlS1kwYLFEXrc66+/ntLScr7//mWKil5Fo1HIyDDy5JN/wWptGBE4FU7WytUUC2BL\n8u//DSF3aHgdbrf7f1o6FdoNerM4Gwb9TKvYz+RH2JhxTlGUc0IfK8syvXvnMm/eTFyu/cTG5gFg\nt+9BpdrGwIGXn/Ecu3eXYrWOiBhzRZFRqy0EAilUV1ej1+tZtGgJ69YFyMi4FrM5nmDQT37+CubP\n/5nMzEzMZjNqdQlJSckN7rPXW0ls7MmJKjweD7W1tTgcQUQxGllWCFWqC6hUUXg8Cnp9ZzSa0GYj\nCAJpaQPYv38PxcXF9OzZ84zvwamQl5eHTifh8SxGEEwcOXI9smwDTICDKVPu4euv/9WkUtXgwYMx\nGGZgs80nJuYyAGTZh9P5LWPH9opsohs2bMDv1xEdfbzqO9TGdQOFhZOprKzEYrGg0WjQaDStIhBp\njKuvnsArr/wLv/9K4uLeo6LifOAG4D5C29q7QCEQRBAOAVlIkgtZfgKzWWTcuHEIgsBTT/2VjIyP\nmTv3a1wuLwMGJHPrrX9gxIiQTKvJZMJs1lJbOwdRvB1FiSEkgPMlYKsX+u2IzVYcWZ9areYvf3mY\n668vYu/evZjNZgYMGNAmLVUtoVJt7qD03+LJN3UdYYP+33JYOR20G/RGaPwytFWvNYQqjE+3orv+\n2lq7psYHCb/fH9lATxct8dDD8w4aNIiJE3excOE0jhzJRhAEtNr9jB3bhaFDh57xGqKidJSU1AIh\nydJAIAgoaDR1REWlEwgE2Ly5kNjYoZjNoeIvtVpLZub57N69i6KiomP88bNYu/avJCUNJTY2B6+3\nElHczIgRlzU5f/g5+Hw+LBYLGRkd2batErf7MGZziGzG4ykjELDRoUNs5CAVqs5VoShii56DzWZj\n0aJFVFZWkpWVxbBhw5qsfj906BBr1qzB5XKRlpbG0KHH+edzcnIYN+4C5sx5GKezDEUZDLwHJCGK\nK9my5RYeeWQq06e/e8K42dnZ3HTTeD766AXKylaiUqUSCKwgKamW3/3u5cjnQtTAQWTZjkp1PE8s\ny5WIIg2MWUsJRJrzOqdMmcLPP69j7dorUZSeGI2d8HpXoCjLURSO5bKj8Pur8fvPIxDIBYoxGJy8\n8cZrkYOLwWDgt7/9LVOmTMHn8zVJb5yVlcXmzYdQlHEoyhCgDNhLqPhOxustQlFW07//uBPuXXp6\nOulnUdiqpfn38EHpvw31n9X/Ou0rtBv0ZtFWHnpY0Qs4q3zoTaG5drhAIHBWT+qN57Varfz5z39i\nxIh1bN26FUVR6Nv3LvLy8s74YAEwdGgfdu5cTUVFFlZrNoKgUF6+goQEB7179z6Ws1XQ6RoySGk0\neiRJw+HDhzlw4ChxcedRWVnHzp3LUKk+JS0tyM03j2XkyJEnzClJUkQf3WQyoVKpGDVqIFu2zKOw\n8Dtcrj6AhN2+jKQkG/HxQUpKdqLRaImNTae2thCLxUdKSspJr23Tpk3cd98jVFQEEIRE4Eu6du3A\nG29Ma9CjvmLFCt5++xuqquIQhAREcSG5uSt49NEHSEhIAODll1/C4bibOXMOAG8jCCnHvL1RSNKf\n+fbbx3n55WlNpgD+9Kc/kpubw/ffL6Sqqpj+/XsxZsxoNm7cyLJlyxg8eDAXX3wxVuvz2O3PYLW+\nhCDoCQZL8PvfYty4YcTGxuJyuZp8/1vrdRqNRr744hMWLVrEmjVrMBhyGTduHEajkfLycmJjY+na\ntSuSJPHuu++yZ88eMjIGcvvttzdpYMNjNoVbbrmBvXunIcv9kCQHPl8s0B/4DpttOQ7HQDp37sDV\nV1990md5LtD4oBQmsQkb9/DvPsxC+Wttj2uOx73dQ29HszjTF6O+ZwyccfFZaw4ZZ1tYpbm1KIqC\ny+Vqct6hQ4c28Mi9Xm/EEzsTDB48mPz8Qlas+JTKyhhEMUjHjn5uvnksKSkpKIpCWpqFbdv2Ex9/\nnMCkurqQqCgfRUXlFBV1ZPToWzj/fC8lJSVUVOwgJaWIa665+oRoit/vx+VyRdgEw5tnbm4uDzzg\n59tvV7Bv31cEg16GDo0lLe1Svv56IZs2rUSjMWA2K/To0YkrruiNyWTC7/c3ubH6fD4eeuhxDh/O\no0OHF1CpYvD7i9iz5z5eeGEaH3wwHQjlx//5z29xuy+mR49rEAQRv9/Gtm2v8NVXX/P7398DhLzR\nwYMHM3/+CiCzwVyC0J1gMIDdbm/SoIuiyCWXXBLRqf/++++57LKr8XiCCIIecDF69MW88MITPPzw\nE9jtyxCEzijKbrKyOvD000+0+Hm2xuscM2YM48ePbxCez83NbTDeI4880uK5m8Itt9zCpk1bmDNn\nARAL2AAngnA3ong7KtVXVFXNYOPGjYwfP/6M5mprCIIQSXEoioLf7ycQCCAIwhkrnf0S0O6hN0S7\nQW+Ehpvc6fV9N/ZQdTpdhHWuLdb2SxBWaQqSJOF0OpFl+YR5FUVh48aNLFmygsLCo2RkdGD48PMi\nUqmng/D9CAQCjB59EXl5ObhcLvR6PXl5eZH2IEEQuOCCgRQVLWX37vnExmZQV2fD59tNnz5xHDpU\nR8eOPVGpQlGMHj160LVrNgcOfEJZWVmECKYx37xOp8PlciHLcqQmoX///uTk5OBwODAajRQXF/Pq\nq98QHT2JuLhM7PZanM6f8HrLGTjwzhM21vptSWvXrqWkxE5c3NRjGt6g1aZjNt/DmjVTqaioIDEx\nkW3btnHkiJrs7MsIc6drtdHExY1mzZrPufNODwaDAYC+ffsiy24EYTGCMDYyryx/S8eOiSQmJp7y\nvpeVlXH33ffi91+GRvMKEIMkzWPJkjvJze3O0qXz+O6776isrCQn5womTpx4RvzaZxqeBygvLycY\nDNKpU6dWGyu1Ws0bb/yDW25Zz9/+9jfWrq0mOvonNJrwu9sTm+0gH3302S/OoNdHmIgFjqc/TqV0\nFpbZ/aXhZDn0/2W0G/ST4HQMX/3is7CH2hZh5ZbgXAqrND5c1O/1bopt7ocffuCZZ6ZTVhZAkjSI\n4j7mz1/Ngw9O4uqrrz6tSn+v10tFRQXfffdv9u+3IUmQlmbiqqvGnNDr26VLF267TWTt2s0UFq6g\nUycdAwbkkJyczEcfLWhqhhPmC0cewt0JYUMSLjQMk9WYzWaMRiOKorBs2U84nd3o0eOqiPHeuzea\ntWuf4557fseUKXcxatSoyDj1PdCqqiqCQVCpOjZYi1qdhMcTqslITEw8tg41otjw4KZSGQgGafD+\nDR06lPPOG8L69bciSQ8hCDnI8mwE4VMeeuiFFjHTff311wQCWjSatxAE07E1XYbfP5lPPvmSv/71\nr9x///2nHOd00Zrw/O7du5k69XG2bNmKokB2dheeffbJSDU6hJ7t7Nmz+de/ZlJdbeP88wdz9913\nN0iHCEJIGjUlJQW1ulM9Yx6CStWfoqJ/nbVrbis07t8+ldKZz+c75/S0rUG7h94Q7Qb9JGitQW+u\nir2t8vEnG+c/IaxSf96Tsc15PB6mTXuH/Hw1BsONGAzd8Pv3U1DwDW+++THjx49v1Q8xPKfNZuO9\n974kP78jqam3oNGY2LdvHW++OZs//9l4gvefmZlJZmZmRKUJQuHq7Ow41q/fQUxMGipV6CdRWrqd\nhASZTp06RSIP9SMeiqJE7qHH44mENtVqdcTQKIpCebmNqKj+gHKMx3wxNTUeJCmVlSs3s2HDn7np\npkt57rlnI/z54bxnXl4eRqOM0zkPiyVUOS4IAk7nfJKTLaSlhSRKu3fvjtX6PUePrqFjx2EAyLJE\nZeVKRo5MaeAdC4LAV199zmOP/ZWZM/9GIOCjY8dEHn74Re66664W3f+qqipEMTlizI+P3ZXa2poG\n97fxc2trnCw8X1JSwjXX3IjdnolK9SGCYGDfvuncdNMdzJ8/m379+qEoCg899BAzZnwKnI+idGPr\n1i/57LOvWLjwe7KzG6rAZWRkIIqzkWUXomiuN+dPdOuW0Xh5vyq0RSTkXOFkOfT/ZbQTy5wCLdmE\nTkUUc7YNejAY0lj2+/2YTKYWU7ieyXrC44eNeZgAo6l59+zZw969lej1t2K13oLBMAir9Wb0+ls5\neLCGPXv2tHheSZJwOBz4/X72799Pfr5CdvZtxMfnYLV2plu3a6mtzWTp0lXNjtHY2AwdOpCMjBr2\n7v2K/ftXsHv3d6jVWxg9um+E7x1CzFP1PRlBEIiKisJoNKLRaAgGg9TV1eFwOKirqyMQCNCpUxx1\ndUWIooodO3ZSUxNAo7kEjSYGq/Vp1OppfPHFPH766adIP3f4cNC1a1euu24cPt9LVFVNw+mcS0XF\nX1CpZnHXXZMirZCpqalMnDgQl+tTDhx4n0OH5rJ37/MkJR3g2msvP+GZWK1W3nzzDQoLD7B79072\n7NnOlClTWrwx9+nTh2DwALK8M/JvoQPMd+Tl5TXbweH3+5k+fTrDh4+me/c+3HTTbaxbt65Fc7YU\n9UlZZs2ahcMhotXOQ62+CrV6PFrtLAKBDF5//U38fj8bN248Zsz/gVq9GI3mA0RxJzZbDM8882yD\nsRVF4aqrriI6OoDDcQc+32oCgR04HH9Cq93EnXfe3qbXcrbQ0uccjoLo9XqMRmNE/wBCz9Lj8eB2\nuyO1MCfTqzgXaPfQ2z30E9A4h34qNBViP5cn1lOFuptCW6wvHMZVFOWUPfWVlZUEg3qMxj4N/l2r\n7YvXa+Do0aMtmjMseSoIAlar9Ri7WTJabUMP1GrtRkHB0haNKQgCCQkJ3HbbFezatYvDh6uxWIx0\n6zaKhIQEXC5XRJ/d5XJRUVGB2WwmLi4u4p00lncMBAKRcOXgwX1Zt24usvkoHgAAIABJREFU+flz\nyM+vApKRpG/QaBR0usEIgoXa2rdZunQpgwcPBo6HlEVR5K9/fYyUlGS++GI21dVf0qtXKrfe+hBj\nxoyJMP75fD6uvvpKkpMTWbVqA7W1e8nJSWPs2GtOUDGrj/Dhr7WYMGECXbv+nfz8K5GkPyIIKUjS\nF4jijzz88IdNfkdRFO69937mzfsJuB61Op2lS+exevXtfPzx9FOyp50OduzYiSyfjygeJ3IRBC2K\nMo7t2+fg9/uZN2/esa6AyfU+E4ui3M2iRY8SDAYbpCGSk5P5+OPpPPzwYxw8eAOKAh07RvPYY3+L\n9K7/knG6h/imIiHh968xPW3jAruzgaaoX91ud7tB/08v4JcMQTi5SlpLiWLOhod+JsIqrWF5a4z6\nBX8Qqpw+FUFOSKdYwOXahU7XCVFUI8tB3O6dmEwKXbt2PeWc9YvRzGZzxKgryk4kKYBGc/xVdrlK\n6dnzRIKUk8FqtUYq8MP5co/HE7m+b7+dzaJFm6itlTEYFAYNymDSpGtPYP0SRRGdThdRHTvvvPPw\neDzMnv0jgcAqFCUZrXYgUVGTEcVoBCFkZCAUygxrPNf31idPnszkyZORJAmNRhPhETAajQ3CokOG\nDGHIkCGRsdRqdbPh7zOBXq9nzpyvmTr1MebPn4osS2RmpvP44+82WxS2ceNGFixYjlY7HZ3uUgQB\nFOVunM4beOmlV8+KQU9KSkSlWoeiyJFiQQBB2EFKSiImk+mUNQNN/WYHDBjAsmUL2bNnDz6fj9zc\n3HMiN/xLwslSHcFgsNVEQae7hvqoq6s7pcjQfzvaDfpJ0Fyuubn+7pONE/5eWyAcdj5dYZXTRf1W\nOJ1OFymYORXS09MZNSqHuXO/pbZWiyimIsulwLcMH971pMQbjSVP67f+hYqUlnLgwFd06TIRtdrI\n4cPr0Wh2NEsGcyo0lS+fO/d7Pv10M2bzeBITu+PxHGHBgnl4PJ/y4IP3nrRWQaPRMGbMGC644AIk\n6QHmzNmNxfI+KlUciiLj8axCEPYxbNgUVCpVJP8e5ucOG+z666v/HjVXINa4avlMRT0ao2PHjnz4\n4fu4XC7cbjcJCQknfRfWrVuHLMdgMBw3+IKgQqu9gR077sfhcES6CU4X4dRTmI3upptu4uOPv8Dt\n7gM4AS2imIIobuSOO95EEAQuueQSXnvtDSRpBmp1qIYgRA/7HhdffBE+n68Bu2Lo/xVWrVrFwoUL\nCQaDXHTRRYwePbrNpW7PFs7GXlE//67T6c56/r25HHq7h96OBjhV29qZ9He3lUEPG1KLxXJam8jp\nHDAat8Kp1Wp8Pl+LxhAEgccee5hA4P/YtGk6Pp8Znc5Jv34defDB3zR7/07VfhcbG8udd17JV18t\noLT0ZYJBiI8XuOGGYQwcOLBF11X/GdcP6VssFlQqFV6vl0WL1qPXX0hq6gWAgMnUAY3GxKZN0yko\nKCArK+uU8xgMBp544gm2bp1ESckEBGHsMcOxhDFjBjNo0KAI6UpTxXX1PfewUQ8EAg3C/vW9pvq5\n/rDHFN5URVFsM1IRs9ncopa0UNW/F0WpQxCO97nLchVarfqMPFxZlnn77bd5550Pqa6uwmKxMHny\nzVxzzTUYjSZsNj8haVU3kvQ58fHRjB49GoB+/fpxxx23MGPG/QSDM1GUTojiQqxWeOqpj9BqtQ2q\n52VZ5oEH/sDMmXMJyanq+PjjWYwePZwPP/znL95TP1fUr2dCT9tSNH5v20Pu7Qb9lKj/AzhdLva2\nOBGHw85AxJifCy52aJinD7fCtXZjSElJ4a23prFx40aOHj1Khw4d6N27d8SLbIzq6mrWrl1LbW0t\niYmJDBw48ASDDqEK7yee6EZ5eTl+v5+MjIxWi1+E763H44mE9CG0eVdXV2OzBYiJySIkyhGCxZJB\naalIVVVViww6QGpqKnPmfMVHH33EihVrMZuNXHbZn7jhhhsiRXXhP+FNLxw6D//xeDwoihJ59+p7\n72HjXN/AN65aDuc8z4Xmdn2MHz+ev/1tGm73c5jNf0MQNASDBwkGp3PVVWPOyBC+9NJLvPrqu8Dt\nqNUjqa3dwCuvTOe77+bg9cZhsawiGAxt9IIwGZttBLNmzWLy5JAU7LRp0zj//PP58suvqK7ex/nn\n38Dtt9/Oxo0bmT59OjExMVx//fWkpaUxb948vvpqDirV22g01yIIAoHAMpYsuYlPP/20xZ0C/0to\nKVFQayJJ7R5602g36E0g7LXV92RbE2I/2Zing/qFd2EP7kw23pZ66C3J07fmmrRabQOmuHBYuP4Y\niqKQn5/P229/SkGBgErVCVhH586ruffeSQ3oTsPQaDTk5OS0eB2N1x82oOGQPhDxhi0WCxaLGofj\nEFbr8QIzp7MEo1FuVqGsOSQlJTF16lSmTj3x/8KMXuH56xfX1T/4aLVadDodoihGPJ8wuU39mo/6\nxXXhv4cPRc0VNZ2tnGdSUhLPP/8Ef/nLk7jd/wZSUJRddOuWwmOPNXEzWgibzcY773yAIPwJvf5R\nADSaifh8ndiz5w9oNA8giv/P3nnHWVGd//99y97tu7B0QUBEIQoiqKioKCBlWcCGiB2NsUSNRqIm\nUTF8408sMcZeMXZjNBoWEASkSBMFjYooKEhT6lK23j6/P9YznB1m7p3b5y7zfr18xeBy75nZmfOc\np32eMg6cBXsSCAxg2bJlXH11YyGc0+nkvPPO47zzzgMaD5MjR47h22+/xu0+hnD4Jx5++GEmTJjA\n3r37gBPxeMapa8jJGUwgUM6///0+l156acZbuqKR6TVpD5qRIkniQKt3L/Vy6LZBtzFEGGEjKdN4\nPitWtFEBoR+eakKhELW1tYZ5+lRsCuFwmJqaGv71r/dZv/4wfvWrCXg8hQSDPtate4tXX/0P9977\nh6iV/Hv27KG2tpY2bdqoCml6CEMINOkvl/PUBQUFDBnSl1dfncf27YWUlf2Kurqf2bp1Gqed1tq0\ndx4rcnGdaBESf+73+/H7/WroPCcnp0lvvFi/OKwATbx2bXgeDtb81uY8hWJYIr/38ePHc8wxxzBr\n1iz2799Pnz5jGTNmTEK9w2vXrqWhwY/Hc36TP8/JOQ+v9xbC4d1N/rxR0W8PBQXGv7d77/0L69b9\nBKwgGHwZmArk8PLLr/0SQh6DHK1pFCEqo75+DYCh8l+6ImqRSFfIPRaM+t/Fc6j3LIbDYdug62Ab\n9AiIzT4YDOrmcFOJUVQgGaIw0Tx0cYgQueRIefpk9LKLfHBtbS07d+7khx+q6dx5LB6PUCHLpXPn\nEaxf/wQbNmxoIvYh34/q6moqKz9g1aqteL1QVuZk8OA+DBo0SFePva6uDqCJQRRGXvYIRo2qoKHB\ny/z577Jxo4P8fIVBgzpw5ZUXp3STFsp/Pp8Pt9utigUJz1pseMKAyKF5oTNvVFyn9dxF5Mco5+n3\n+5PSknTkkUfyhz/8IWkFZGVlZTidEA6vx+Xqqf55OLwBlwuczmkEg5fhdg+kUcv8FRyO1Zxzzm26\nnxcKhXj77X8TCv0R+Bh4GXgUuBzYTCBwITCHYHA9bveRv3xXFQ7HDIYMGUV+fn7Elq5kFyc2R6Ll\n3+V0USAQoKGhQXV2bINuo4tok4JGzzjRDSgWQxypsjuZaNcTb/V+ovj9ftVo5eXlEQ47yclp+mJ6\nPIUEAk29H+3a33zzXZYtC9G+/Tm0aNGG3bvX8dZbS8jJyVHbouRrzMnJUU/6ctGZNryXk5PDJZeM\nZ/jwKrZv305paSkdO3ZM6YYs8vqio0COkIjQuTiECOMRCATUZzZacV2s3nukliSjkGg6OOqoozjh\nhL589tm9OJ3dcLl+RSj0I4HAHXTvfjRdunRi4cLzCAZ/hcPhBX7kqqsuaSL9KrNs2TJ8vgagA/Ag\ncAXwWxq98O7AdKAXDQ1DcbsvB3KBf9G2LVx77bWAcc44lcWJZmmMUGQ+UmAWo3vp9XrVd7lfv360\natWKoqIivvzyS0466aRD1rDbBl2DoihUV1cTCATIyclRK4kTxaxBlyu79Qrvkumhy8RbvZ/IWsTf\n9fl8qlxtoxb9FubPf5127QbRoUM7Dj+8E9u2fUbbti5V7lTLjz/+yJdf7qFLl8spKTkMgM6dT+XH\nH/0sXPg5AwYMwOVyHaTHLlrUhPcaaWNt1arVQRrxqUD8LsLhsKpCZ4S84Yne92jFdfJBQK6cF0Qq\nrpNbkvRCorKBShdPP/0YF110GRs2nIbT2ZZQaCcdOrTm5Zdf5+ijj2bu3LnMnz8fj8dDRcX9nHrq\nqbq/41AoxPXX30SjkX4J2Ar0++W/Nj6rbndXHI7uHHOMi3373iYUCjNixFnccssthqNw5fsnp0Lk\n4sTGz7ZWeN6KiHspDpMOh4O//vWvLFiwgJkzZ3Lrrbdy++23M2DAAIYOHcoNN9wQc51LNmMbdA3C\nsxEeUbJmh0czxCK8Wl9fH9E7jiZ2Y3Yt4jshentYtM+Jh1AopIa8c3NzKSwspKqqigcffJKffnKx\nf//X7N7tZ926Elq3rqNHjxrOOWfgQaM9xX3ds2cP9fU5qjEXtGjRhT17PmH//v2qcIucLxfXL0Rr\nMt1LLORjoTGvH+twHTl0DvrFdbKBEZtjrMV18fS+pyp/27VrVxYvns+HH37IDz/8QOfOnRk5cqRa\nP1FeXm5qCtrKlSvZsmUrUAh8SuP2+HvgTmAIcDcORwtCoR+4+uoHueqqq2JeqzgoRStOTGV4vjmE\n+sWz5PF4uPjiixk/fjwdO3bko48+4ptvvmHu3Lk88sgj3HTTTRleaXqxDboOQoFLljdNJXLhXbTB\nKslGryUt1cg5ekA1PnPmzOWrr9ycdNLjVFf/yNatS9m1ay2BwBpGjbpCnUqmR2lpKXl5AWprd1JU\n1Fb98+rqrbRo4SAUCjWRxxVeeU5ODg6Ho0koVGvs0vW7EMVvLpeLgoKCpHhpWuU6OfQrF9olo7gu\nUu+77HWm4n3yeDyMHj06oc9Yu3YtEAIeAf4fsAe4BigFXgEGEg63p127towdOzbBFTeS7vC8FYvi\n4kW+ByJFdfzxxzN48GBuvvlm9Z0/lLANug7iQUmmwpuRZx2rd5xMT0eoYMUqHRvvWrT564KCAvbv\n369+xqeffktx8al4PMW0bn0crVsfB8A33/wtaoX1kUceybHHlrJy5XQOP3wohYVt2LXrO6qrlzFk\nSDfy8/MpLCxUvVHxj2zE5GIzORctjF2qZkOL++L3+/F4PCmrmTDqS4+3uG7Pnj2sXLmSgoICBgwY\noK7bKLwsjJPP51M3WyuFl7ds2UJj9frPwGbgC+AYGsPtNwLH0aLFPmbMmH1QpChZ6N0/YeDt8PwB\ntPuOeKbk3PmhZszBNugRSbZB13sI0+0dQ9Pq/YKCArX3OpXotf9pcTodhMOhg/6eogQNNy1xUHI6\nnVx66fnk5Ezjm2/eYscOyM/3M3z4EQwZMkQVi5Fb1WRPJ1ouWpsnTtbwiXA4TENDgzrcJ51KY/EW\n14XDYf7+97/zyCOP4/X6cTigdetWPPHE3xk8eLAakteGl6uqqpg6dSpLlnxCQUEeo0aNZPTo0eTk\n5Fhi3najIJEC/Bc4DThW+q8lwKW4XK8fNFI1Vch1DPK9TzQ8n+0hd7GPytdRX1+Px+MxLfbVXLEN\negZQFPOzy7Uk6qHLfe25ubkJGXOzaxH66HL+Gg68mOJ/TzvtOFavXkFDwwDy8xuHLOzatYri4h0c\nd1z0cGqrVq244Yar2bx5Mzt37qRVq1Z07NhRt788mlejl4sWxi5SLjoWQqEQ9fX1KIpialBIKoml\nuK6yspL77/8bcBs5Ob8B9rF7971cddW1LFo0Ty1clMPyu3fvZuTIc9m4cReKMgSHYy/z5v2RhQs/\n5vHH/5Gy3vdYOOuss375t69plHUVz7b4310ZrZ5ORni+OYXcZerq6g75WehgG3RdUhVyFy+gEGwp\nLCyM2SOL16Brw93Jqt6Pht/vp7a2FqfTedB4V+1GPXToUL78ci2ffvoAitIDRamjoGATF1zQlx49\nepj+vuLiYkpKSprI1Or1l8eC1pvVq/IWnpQZ7z0QCFBfX4/T6YxLeTDVRCque/75l4BhuN1/JBSq\nRFFWA6fj9S7jnXfe4c477zwoPP+3v/2NTZvqyMn5BKezCw4HBAL/Zdq0CYwfP44hQ4Y0Ka5LZu+7\nWXr16sXQocOYO/cjYC3wGHAT4AQ+xuX6FxdeaJ0iq0iCLHJ4Xr5/zQmth57O2iOrYhv0CKQih75/\n//6EBqvEg15fu5y7jpdIhwtRpKIdeRqJ4uJi/vjHW1m+fDlr1/6Ax1PICSdcRp8+fQz/rliD9sCi\nly9PVoGbvJE29s2Hm3iyotJbzr3LBWF+vx+v19tELMbqyMV1mzdvRVHOwusdjKJsBo4EfgLqWbZs\nmXp/5BzwzJnzUJRLcDo7AwqKAi7XOYTDPZg1axZnn322bnGdXu97Knu3X3nlJf7857t49dXXCIcn\nAn/H6SxFUb6lf/8B3HDDDUn9vmRipvsAGlNtcttXtqG359geeiO2QY9CMorQhBIaEHcBWrzrMSq6\nS2Ubkaw9H00YR7uO/Px8Bg8ebCj8oYdRfl72EFNZrW7kvetVkgsjpRWLySZ69OjO4sWvoigdgEU4\nHN1RlP3Ab/j884Xs3LlTTSEI49tYtyA8RAeg0HjpLvVeme19T2VxWFFREY8//hj/7//dx0cffcTi\nxYsJBoMMGfInBg8enDVerl73gTDs4XBYrZHIhLhNspDXKgx6Nq0/FdgGPQqJGj45xA4k/NDJUYNo\nn5Pqoju9yv1owjjJRhZIMdJjT2ceVhsGFRupXEUOqKIimVRZi5crrriUjz++CfgL0BVF8dPY7nUP\nweBiPvnkEyoqKpoU1w0bdiZvvvkm4fD1uFztAQfB4BwUZQ1Dh/6uSe+7bNgj9b6ncrBMcXEx5557\nLueee676Z+Jwlo2I51KIOLlcrqjheaulgQR6+7Et+9qIbdB1SNbmKs/XzsvLw+v1pmXjVpQDU9KM\niu5S4aHHc4BIZB2yrrO2vzyZIfZEEMZJGB2h9S0bO+EliX8yueZwOMzHH3/MZ599RmlpKaNHj6ZD\nhw5NfubUU08lL68Ar7cVUAU0GoKSkm54vR5qamrIzc0lJydHVby75ZZbmD9/Mdu398fvH4XDsRf4\nkOHDB1FRUdGkPU5Ok0Dig2Wy0ftMJfJhKZo4ULrqF+JBz0M/1LENehTiMTja/HFhYWETkZpUeuiJ\nFt3FuhaRJ402ZjWZyPlyYQhEWDfZ+fJEMRKLiVZJLufe03UdtbW1XHbZBJYsWQG0RVH2M3nyA9x/\n/720aNGCHTt2cPrpp3P00UfTtWsnNm2aQ15euXog8XrfJScnwEknndRECbCoqIjS0lLmz/+Q559/\nnnnzFlFQkM+YMXdx4YUXquqIesp12uI6OFi5Ti7gk3/eaO57pg9NmcJoHzMSB7Kidj/YHnokbIMe\nhVgNeiz540TQW5McEYhWdJcsD11RFGpqatSe9lTnhbX5clFklq58eSzrFGIxOTk5utr42kpyeQPV\nCtvE2xoXCw899BCLF68mJ+dd3O4zUZRqampG8Nvf3kpj/lvB4VA49tieTJhwCfff/w98vmrc7hE0\nNKwB3uDCC8s56qij1M4GUZwI0L59eyZNmsSkSQfuUTTlOm1xXSzKdfJ3yAbK5/PFbZwy/Vylg0j1\nC1bQ7pfXKbANeiO2QddBflBiMXza2eVy/jhZFfN6G4peRCAdL5jwhhRFibtq3+FwUF9fz4IFC1ix\n4iuCwRAnnXQsgwcP/kXo4wBy9EHky4XuuVxUlelNV0QshFiMx+MxtSZtH7hQV0ulsI285jfffBeH\n4xpycs4CwO//kFDoGxr1zK8BGlCUyaxe/QHPPPMiTz75EM8+O5Vvv72Dtm1bc+WV13LttdeqHne0\nehG9tiuzynXxDJYR3yGMe6zGKdHomlWI9RqsFp63q9yNsQ16FPQKv7TIIWCjwSrJNujic+IdtWrm\nuoxQlMZBMnL+Ot4DhM/n48knn+eTT2rIyTkRh8PNZ5+tZMWKr7jrrttUiU05+iDny8V6amtrm2z8\nmapGTpZYjJGxS6awjXbdNTU1OJ1d1D/zel8ABgOTAKHs9yrwK7Zu3UooFGLu3A8IhUI4nc6E5Wvj\nVa4T60/lYJnmQLLab60Ynrc99EZsg54gZseOJrOnXXxOuivKxfcKlTtRKZuIIVm5ciXLl++ma9c/\nU1jYOCnN5zubzz+fwoIFCxg9enTE/nKXy0VhYaHqcWWy0EwWiyksLEzqocJsa5xZYRstbrebPn16\ns3Lle3g8l+FwOAmHfwSu5EC7GTQa9j7AEtatW6d+r4hI5Ofnm57WFwnZcMQ6Fjae4rpog2XEAbI5\nKK0l810wCs9HioAkGkUzkn5Nlb5+NmEb9ChECrnHO3Y00fUAqoBJvC1p8eTQtQV3cj9rvHz33VrC\n4R6qMQfIzW2J2308n3/+LYMHD8bv9xv2l4sNQmwo8sYfKWybzE0t3WIxkULVWmEb4cmaOXTdfvvv\nueyyX1NfPw63exyNc8EXA3+QfqoaWAX4OPzww2Oa3Z4I2nqDWMfCmi2uMxosI56jUCiE1+vNysEo\n6TiMpCs8r/079fX1tG/fPinXkM3YBl2HaDl0EXKur683bVCT7aFHakkzSyxr0Su483q9CV9P4+YZ\nOOjPw2Ef4XCjgYqlv1ze+LUqbnph20S9d1G/IEbfZkIsRi9ULYyd1+vF6/WailgMHTqUV199gQce\neIQ1a35LYWGYurofUZTfA9cCtcD9wH5atWrBqFGj1JqReGa3J3rNsYyFlY208NqjFdc5HE3nltfX\n16upqlT0vjc3UhWeN8qh2yF326AbIgy51qDLIedYDGoyDLrIc0Jj21MiD7DZjcdIUjVZ9OlzHLNm\nvcuePaspK+sFwN69PxAIrOSEEwYf1F++adMm3nrrLZYv/x/FxQWMGTOC888/X9czNBu2jdd7D4fD\n1NfXEwqFkhZqThSjHm0R/ox2zcOHD2f48OHU1taSm5vLa6+9xp133kN9/asIdbcOHdrz6qtT1b+b\nrNnt8RJPcZ0w8LEU1wHqsySHlq0wWCYWMrWmWMLz0e6hnoduG3TboJtCLrxJV4+3FtlDBhLWgTcT\nco9WHyAfUuLdJI4//nhGjPiaefOe4+efuxIOO3C7v2fw4MMZPnx4E2O+fv16Jky4kZ9+KsTtHkIo\ntItPPnmaTz9dycMPPxjTBDW56Coe7z0YDKoV9pmelBaJeK9ZjJu9+uqrueSSS5g7dy7r16+nd+/e\nnHzyyYRCIcN2vEwTT3Gd8LwjFddpv0MvtCzC8+keLGMGq+X/472Hdh+6MdbchSyEeAmFCpre1LBY\nPidRkZqCgoKkDFaJRrrqA1wuF1deeRmnn76ezz77jEAgQO/eFzBgwABycnKaeE4vvDCVrVtb0b79\na7hcjS9vbe0iZsyYyIUXfsopp5xi+ntjKbrSirwYicVYnVgLzUTuPS8vj9GjR1sivRArySquk/Pw\nYlKhXnEdNO19T+dgmWwmlnsoR07FPbQNeiO2QTdJXV1d2gerpFKkJtJaxMjTaPUByfDQxed0796d\nI444ImK+fMGCT8jLu0415gCFhQOpqenE8uXLYzLoemswI/IiPLlsmpRmRCyFZi6XSxXwsUp6IR7i\nKa4TqRXx54BucZ34d6PQcqoHy8RyD6xOpPB8MBhUU58PPPAAHTp0UJ2dQx3boBvgcDiabObJMKix\nGPRILWnxVKgbIRtjORqQDglXOFCxLo+UlXObsifT+N98mk9QAH/SQ956Xl1DQ4Mahg0Gg9TV1WVE\nojVV6BWaaYVtROV4MBi0RBg5UcwU10HjO5eXl6eGh80W14nviDZYRj44peq+Wi3kHgvyPRSHK6fT\nyerVq3n22Wfx+/1cfvnljBo1imHDhjFkyBBatWqV6WWnHdugG+D3+9UCNMC02leiaCvoS0pKdE/v\nyVack6MBZiVcEyn0E8V2YkMT1yl75lojWVExiBdf/A9+/yg8no4oisK+ff8mL28ngwYNinkNZhHt\neYqiUFBQgMvlalJFDgc2ZGHgs93Qyd6oyGXm5uaq+c1kC9tYAW1xndfrVa8TGqetCfGoRIrr9ELL\nRr3vdnheH/E8/ve//6WmpoaBAwdy6qmnsnz5cqZOnYrD4WDt2rUcddRRmV5qWrENug6iRUWEe4Sn\nnCjRPGuzFfTJfrkzMfJUFqcRUQI5V6m3iV177W/45JMv+OabsYTDJwG7yMv7nuuuu4BevXqlZK2i\n+E0UiomNOJonq81DZ+OGLGoFtOkFM21i2ey9i0O1GDaUl5cHYFhcJ67ZbHFdrL3v8iEg0cNitv5O\ntMjXUVRURFVVFZMnT6Zbt25s3bqVhQsX0r179wyuMDPYBl0Hh6NRXlR72k7G5xoZ9Fgq6JMRchcv\nhGhJixQNiEYsa9HqsYu8opwvNzKAbdq04c03X2L69Ol8/vnnFBV1YfjwaxPKnRthVixGr2UqUk5W\nbPpWRkRPjGRcY20TMytsYwXkwj+tDn+k4jo9rX1tcZ1ssMFc73syB8tkc8hdJlqVe6dOnbjsssvS\nvSxLYBt0A+RqSkjtyxDLlLRk09DQELdATaw/L4bXyNcpClyCwaAp76O4uJhLLrmESy65JKbvjgV5\nU49VlzxaHlocnmTNeSsZeBGdimWwTLKEbTKNiByZ0RWIp7jOrHKd+He9g1O8g2WaG/LzIw6RotXy\nUMY26FFIpkHXetbalrSioiJTG12iHrqo2oXGYj8hqRov0dZiJE4j/p4wIolokSeLZIrFGHmyqRqw\nkijytccr46qXI063FG88yBK28egKxKJcJ6IW0YrrhOeezMEymb7PyUC734i9I526IFbFNuhRSLZB\nF/m0RFrSEjHo8ohXIKF8uZn1yvlyrR67aP8qLi421CKXq8hTTSixcD3QAAAgAElEQVQUoq6uDkiN\nWIzRgBU9byvdhxr52pMp4yp7spGkeDNZUJjsa481JWFUXJeMwTJy73u80xWthNzGKhDh9kMlOhEJ\n26AbkIpNRRjiSHPTU4XsJbvdbvLy8poY9kQ/Ww+9+eWyWIe8OaVykpgZ0i0WI2/6sqHLhPeunRKX\nqu+RjZCZPHQ6CgpF+2Eqr92scl204jqzg2WAJt673PsOjb/v5hSer6+vz3pNiGRhG/QoJNNDFy9z\nTU0Nbrf+3HSza4rltK3nJYu/n8h1RXqB9PLlRv3l2s808m5S0S4lVzRnUso0E9672cK/VBFvHjpZ\nGFXxp5JYDzXxFNeBfnhetF6KCFg2D5aR11pbW2uLyvyCbdBNkIyqcvECKoqS8JS0WNaj5yUnG21d\ngFG+XHgbsXhdRoZO9t7jNXTxFIClAyPvPZmHmmiV7JnAKA+d7EONVQ5xoH+oMROpiVZcp9f7Lp6R\nvLy8JsI22TRYxqjC3TbojdgG3QBtm04iBl0YVeEVp0tzOFL1fDIiD9oXPlq+PFZjrvd9Rt57rB5d\nuuZ4JwMzh5pYUhLyQcaqMq5mDjUQe52FfJCxoh692d91tOI6Ofcue+7i/2sH9lh9sIwWvRy61daY\nCWyDboJEDLrQRRfehzaflYr1aKvn9XKDyX74I+XLo/WXx0u8YWojsZhswExKAmgyPU3+3SdazZ0p\nYjF0RsI22XCQkUmkuE6OiMm5eIfDQSgU0i2ug8hDUeLpfU820XrQD3Wy423OQmSjKnTRfT5f0vrZ\njT5HHnlqpno+GR66bCD18uWyh5AqzBaZifqDVBeApYtYvHeHw6H2hGfbQUYmHkPncDgSbsnLNEbF\ndUYdA+J3HgqF1Py5y+UyXVwXbWZ5Jnvf5T2trq7ODrn/gm3QDUgk5G6kiy6HuRM54Rr9XVnC1Wy+\nPBGDLv6uz+drUuQXb748mWg3P7HpyW2DQt43m+VZZaKlJAROp7OJl5btmKkiF+Tl5WVNVCISsRTX\niXsiD5dJpLgu1t73ZKK3X9XV1dke+i9k/5OdBmIx6JFa0pJp0LXrEfPao4081a4lXkS+HFB7ySF5\n+fJkIoqgxOhPt9ttWHjUXIarwAHPXBxihFfVHPXXBVpDJ1ryxH8TynV6c+6zGaOOAflA4/P5CIVC\nBxXXxTNYJlLvezoGy2hz6LaH3oht0E1gpk1MGA3h9cXbkhYLwqjX19ergyRiKQ6JtzZAzpcDTTYR\ncZ+sskkaicVow9R6w1WyfcOXJWy1BWCRwtTpFPNJJcKYC20B8R5bTdgmVYhnWVy7meI6QHewjGzY\n4x0sk4zwvJ1Dj4xt0E1gpghNW91t1GMtfj7R9cCBfHksI08TRdtfLkL8WrEYK2BGMEUvTK03GlUu\nMsuGDT+ahG0k/fXm4L37fD7d/vpYe8Cz7brhwL4gUm8iWpeM4rpIynUOR+TBMpCa3nc75H4A26Ab\nYPZBC4VC1NTUEA6Ho+atk23Qq6urgfjV5mLx0LVKc3IEQry0Vtn8Eukz1vZCyxu+2Pis7r2HQiHq\n6+tRFMVUJbtepbMcscim6Wny7z5af328PeBWRhuR0ku9xVtcZ2awTDTlukR73/WkXxsaGigrK4v3\nljUrbINuAiPDJ+etS0tL01I1LDZbaHyJiouL0xLaN+ovdzgc6uZvhbCl3JqUaJ+xdsOX84VW9d5l\nKdNElAizcXqanGLIy8uLeVhHsvv9043cbWK2gyOW4jo9791osIzZ4rp4et+1f2576AewDXoEhCHX\nGnRhNGLNWyfqocuGFUi47cqMh643p11+kbdu3cpbb/2LNWvW065dGWPGVHDyyScn5MVu2rSJHTt2\ncMQRR9CmTRvTfy/VYjGRNj69sGW628JSIWWq573Lvf5W8d7Fu5GstjSz/f5WqTmQ6wUSEVmJJ2oh\nCi/jKa6DyL3vcmuc0X5l59APYBt0E8gPklFLmtnPgfgMumxY8/LyDmrHiZdIazHSYxc5tU8//ZQb\nb/wje/a0w+U6hVBoLfPmTeb2269m/PjxMXuxu3bt4u67/8Lixf/D74eCAicXXjiCO++8PWoLnvBO\nILnTwoyQNz6jsGW6vNh0yrgaXbeoqM6E9y7njFMllhOrgls6oxap1KSPR7kOaLJPyMV14ufN9L5r\nB8uI/UeLbdAPYBt0E4gXREipQnx563gNutawAkkx6EYvvlHFvrb47aGHHmPPnr506PAEDkfjvdi1\n6wmeeOJlzj33XMrKyiIW38jenKIo3Hrr7SxdWk1x8UO0bNmT2trFvPTSY+Tlebj99tsNryPdk9K0\nRApbptqLlVMM6dajj/W6UxG1MJMzTjZa7z1a1CKVtRai+C8dmvSRohbJKq4D/fC8qJ4Xf6++vp73\n33+f1q1b4/V6bYP+C7ZBj4DwzIUBrq2tTVtLGhgXoolwVjKK67SfESlfLveXb9myhe++20Jp6e2q\nMQcoK5vAzp0v88knn1BRUaE7C1vPm1u9ejUrV35PixbPUlBwEgAtW46nqqqef/3rea6//nq11117\nf6w0YASMZ4An24u1mh59tOuG5Hqxcr1AJlX/zERrkl1rEUvxX6pIZXGd+HdteL6hoUHdg55++mm+\n/vprPB4PNTU1bNy4keHDh3PMMcdYYh/IBLZBj4J4cYCkTEkTn2nme2tra9UCH73Td6IGPRwOs3bt\nWsLhMMcccwyFhYUR8+VAk9O0wwGKou3PP3DyljHy5oRX88MPP9DQoNCiRV8UJfzL33dQUHAidXVP\nsWPHjiYGXW7LiqcAKl1Eu+54vXfZM7WijGs83nssXmwmRp+aIVqRWTI6JeSDrFWe/XQV10HjHpSX\nl8fixYtZs2YNN9xwA263m7vuuouJEyfSsWNH5s2bR8+ePTNyLzKJbdAjEAwGqa6uVo1ZMk7BZgvR\nIrXCJWPz+vbbb5k48U98++1WFMVBaamHq666kCuvvPKgfLlYr/xSdezYkWOPPYLPPnuZgoL+OJ2N\nL3FV1fOUlbk59dRTI36/1qs58sgjyc0Fr/d/5OX1BRQcDqiv/5zCQneT4rhY27KsRCw5aKP+bzP9\n9VbDyHuP1YtNpCUxE5jtlDDbGienWKw8YCZVxXWyyqbT6eTYY49l7969/Pe//6Vnz54sXryYuXPn\ncsQRR2TkujNN9uyEaUZ4yKIFRBTdpBp5OptRK1yi1fLV1dVcc83v+PHHwygtvRe3uy379k3j0Udf\nomPHjlx44YVNcl56HoTD4eDPf57Iddfdxvbto3E4TkZR1lJQsJ477vgdLVu2NL0eh8PBiSeeSN++\nR/Lpp5NRlNvJzW3Moft8L3DRRWfhcDjUlhy/359QW5ZViMV7F8ZdGAKreaaxYNaL1TNyVh59agbt\ndRupFOoJ2yS7kj+dxFNcJ4fntSk/sQeIfaGwsJD8/HyGDRvGsGHDMny1mcM26AaIAjTx8EHiIW7x\nuXqfI3poRYFLUVFRyjarDz74gM2ba2nT5h+4XK0AaN36Znbu3MbLL/+LCy64gHA4zK5du1i4cCF1\ndXUcd9xx9OvXr8ma+vbtyzvvvMy///1vvvlmHZ06deH882+lX79+Ma/J6XTy2GMP8cc/3sOKFb+n\nrg7y8+GSSwZzxx2343K5CAQCTYQlAoGAZcVd4kHPe9eqt8GBzbG5YDYHDY3viVXCzIkSqchM68W6\nXC78fn/Wjb3VI57iOofDod4PYewVRaGmpoZ9+/apYfpDnex9KtKAGDWYTGOhZ9BjHXmaqIe+ZcsW\noKtqzIVBzM8/kQ0bZhMMBpk3bx6TJj1MVZUTKMTjeYUhQ/ry0EP3q4VyiqKwevVqVqz4kg0bfuLn\nn3dxxBFd6d27d1zeQ4cOHXj55RdYt24dO3fu5Mgjj+Swww5TDzuKouDxeHA6nZYWd0kGshcre2Yu\nl0utHwDjuefZip73LtIR4nn3er0EAgFLq/XFg5EXK48tFc8+YDlhm3gxU1wnECmHsrIy9u7dy/jx\n4+nZsycdO3bM0OqthW3QTZCoAdUif04weGDkaSytcGZy8Ua0b98eRdlIMFiF291Kvb6Ghs859tiO\n7Ny5k7vvfpB9+4bRvv1tOJ1F1NUtZ9asu+nZ8yVuvPFGAP7zn//wl788jd8/hIKCq1i37ismTfon\nH3+8hOeee4a8vLyY1+ZwOOjRowc9evQAjCu5zYRqrSxRahb5+oVnlo0qZvESDofVotSioiKAQ2K4\nirgmkWIC1B5trbBNc3nW4eADnehkgMbne/r06Vx33XUcf/zxeL1eOnbsyPz58+Paa5ojrgkTJvzF\nzA927do1pQuxIrIgggiFJxrqEhKHHo8Hn8+n5suLi4tj+myRR43FExYFRWVlZXz44XR27PgMt7sr\nDgfs3fsa8C8mTryGdevWMXv2t7Ru/TjhsOeXl6szDQ01bNkygyuvvJhAIMBtt91NVdUwnM7fsG1b\nDnV1PfD727J27essWjSfgQMH0KJFizjuUiPyy1xUVHTQ/REvf05ODh6PR93URF7S7/cTCATUvnnx\nT7Ygrl/UcYjrF1W/brdb9WxErYW4br/frxY0Ztt1C2QpU1HJL193bm6uenAREqI+n49gMNjkurPx\n2uHg6xdpCfGsixyz9lkH60w7TATx/LtcLoqKisjLy6NNmzZ06NCBdevW8e2337Ju3TqefvppVq1a\nhdvt5le/+lWml50SNm7caOrnsv9Il0aSlUMXXlddXR0ej4eSkpKY245i9dBF2La+vp527doxdeqT\nHHvsburrr2Lv3pEUFb3ExImXM27cOLZu3cr+/W7Wrt3Md9/9wHffrWPPnr14PIezf39j9f2WLVvY\nuHEXe/b8ip9+2oHPl0co1JVQ6GJCofasXr2f3/1uYhOlp1gQIj6i+M3MfHdxqi8qKqKkpISCggI1\n915XV0d1dbXaYx9tHG6mka8/mmCKCFkWFBRQXFxMYWEhHo+HcDhMQ0MDNTU11NbW4vV6VWNndcTv\nLNqkPFHpXlxcTHFxsZqu8nq91NbWUlNTQ319fZP6i2xAvn5t8ad41vPy8po86263W+2AEM+6mIGe\nTdcOTY25/PsPBoNMnTqVk046ierqapYuXcrNN9/M5s2bWbRoUYZXnXnskLsJkh1yFyHDREaexmLQ\n9fTYe/XqxXvvvcmqVavw+Xz06tWLNm3a4Pf7qaychde7F7e7HqezFz7fXjZv/omiohkMH94Tl8tF\nQ0MDu3ZtJxDYDnQFRJHWLBRlMzU1RSxY8DNnnDGMyZP/yIgRIw5a148//sj69etp06YNvXv3VttS\nRCVzIm1JZgrMrBiiTrQtK5ZCK6vm3o1Gn0ZDOykvUgW5iOZY4XeuJdYe+2S1BFoFcSjRXv+GDRsY\nPXo0F110Effffz9Op5MBAwYwYMAAJk+enHWHllRgG/QIyA99Ml4Aubgl3pGn8XynkR67x+PhhBNO\nUAcj1NTUMGfOHH76yUtBQW+83r8AF+NwtCMY/Bd+/zyuueZl9u7dyy23/JFAwAlMA44DHMAKYCJw\nOnAD4XADmzbN47bbJtOmTRtOOOEEoFG28f/+7//x4YcrqK934PEo9OnTmSlTJqtyscmUMdWqTekN\n2rCCkRPFf4FAIGltWWYKraxi5JKp/Kd3sBHGXe9gI0LYmSZRKddYhG2s8DvXYmTM165dy+jRo7n2\n2mu55557dNdrlWvIJLZBN0kiRWiyNrrI6SVjGlSk9ZjRY3c6nU2kXYPBID/++CNOZxcOO+wF9ux5\nktra51GUAHl5+RQX59C/f39ee+011q8PkZPzGOHwm4RCVwHtgfVAW+BBIASEyM09gbq6Hbz++luq\nQf/LXybz5puLKCg4h7ZtryYU2seKFQ9z6613MnXq07r58mRiRSMnK9+lSjBENmBaadZMGzn5MJOK\ntrRo3ruYBZApI5cqKVc9gZdIB5tMRmxEZEJ7mFm9ejVjxozh9ttv57bbbrMNdwRsg26SeA26rI0u\nNilhNBLFaD16euwidy+LM8gvhjByje0fO3A63bRtO4nWre8kHPaxZ8+THHbYhwSDQZYuXUlu7mA8\nnqPx+68mFLoR6Efj49QbOALYDlQTDoPHczJr137EihUreOKJJ5kxYwnh8JHU1S1l377FdOx4E2Vl\nd/Ltt9fz3XffccoppyTl/pjBrJFLZbgyU8p3egebTBi5dAumGKUlIqmYpdLIpVPKVe9gY3SYTWcq\nysiYr1q1ivPPP5/Jkydzww032MY8CrZBN0k8Bl0vdy027mSsx+x3yl45HKiA/f7771myZAmKojBg\nwAB69uxJeXk5f//7s+za9WfKyu7C7W5LXd2HOBzvMW7ceHw+Hy6XA0Wpp23bMrZseQ84DJgC3At8\nAfiBEqAat9uF3/8/1q9fx9ixN7Bvn49wuASXqz1u998IBv/Lli2P0b37MwQCTnbv3p3wvUkErZFL\n9cxzecBIJpXvYglRJ7M9LB2jT6MRS8Qm2UYuk1KuRodZ7cz3VGsdiDSDNjKxdOlSLrroIh555BEm\nTJhgG3MT2AY9Aok8QKJKWc5dJ/qZ2rVpK7Uj5ctlPXZFUXj88cd58cX3qKsroXEIyqtcfnkFf/jD\nRJ566mFuu+0ufv65nFDIQUGBwqWXDlOHIAwdehaLFr1AYeG5lJUVs3PndmAmcBYwB5gEXALUEg5P\no6FhAaFQJ4LBPwEbgQ8JhT6noWEseXlvEwotZdeuFygsVOjSpUtS7k8y0IYr9XS4E5keZtUBI2C+\nwCwRcZdMjD6NRqxpiUSMnNWkXGORZzWaMxALcppBWzMyf/58Lr/8cp599lnGjRtnqXfDytgG3SRm\nPXR55GlOTs5BLTeJ5OIjfWekfLm2F3nx4sU8++x7uFy30rHjWMDBvn2VvPTSg/TpcxzDhw/no49m\nsGTJEmpqaujTpw9HHnmk+n3nnHMOy5at4O23L6amRhwq/gh0xOm8gHD4HeAtcnPdFBQUEQ67aNFi\nCps3v044vBEYBgxHUabh812Dw3EYtbWzGDr0DHr16pXUe5NMzE4Pk42cHlYd+2pEJO893ipqq4w+\njUaqvHe5ZsKKUq6RuiUSmRIo0BpzIQyjKAqzZs3iN7/5Da+88gqjR4+29LthNaz1FFkYPY9Yi5mR\np3ILXCIPqjgYaHP0wtOLlC+fOXMWPl8vOnW6SP2zli3P5aefPuKDD+YwfPhw8vPzGTp06EHfu2XL\nFtasWUNBQS5OZwuKi6/F4TiN2tpNBIMv4XA8Q0lJLjk5DjyeYoJBB/X1XvLyvkFR1gEP4HQOQFG8\nKEo/wuFHcTjmM2BAP6ZM+b+seXn1WoXMtMXJIdZkVvKnE633bmawinyNVo5MRMJMiNqM9y6nGaw4\n+lYPI3lWvSmB0Q51RjUDiqLw/vvvc/PNN/P2228f0kNW4sU26BHQtq1F8qxF25d4SVOdCxNGu7q6\nWnd+uRgooxcK3bevBoejvc5ndmDPnm91v8/v93Pffffz7rtzqKtT2L9/Gw7HlXTvfiV5eXmEQr9i\n796j2bPnHAoKFBRlKOHwMGpqqvF632Pbtr/hcPTF6eyLoviAfTgcHpzOkyko+IyXX36J0tLSpN+n\ndGC2LU7MBlAUxRIh1mRgJi0hfkZcf6IaA1bBbIhaPtQJYw7WSTPESqTWuGiz7uVuBrlmQFEU3nzz\nTe68806mTZvGwIEDM3mJWYtt0E0SyaCLkacul4vi4uKIL2myPPRwOKx66Eb5cjnMLtO3by/mzZtJ\nMLgft7vRiIZCtYRCS+jXT/9Fev7553nttYXk599N27Znsn//2QSDPdm0aQtHHdUdl8tF69ZHsWeP\ni/r6w3C5fkddnQsoxensQzh8LYqyhpwcL6FQDeHwNtzuMIWFQY4++ihKSkrivhdWQ2+jF9KcAqHg\nZbU+4EQxs9E7HI3SteFwOCsNmh6RQtRygZn4WaFi2BwwK2zjdrtVZ0NrzKdOncpf//pXZs2axckn\nn5zJy8lqbIMehUiGXB556vF4KCwsjLoxywY9HkTuSWyOJSUluv3lkdZx/vnn85//zGL9+t9QWDgW\ncFFX9y5dugQYN27cQT8fDAZ5881puN2XUlp6Poqi/KLt/iM+3ynU1FRTWlqK37+ZYHAfLtdoGhoU\ncnKOwOnMJyenA/X1A1CU1wgGH8PjqaBt2zaUlQWoqlrOyJEDqampsUQvbLIRz08gEFDlOkV4PpXV\n41ZAzqsL8SKh3CeuPZGiQiujPdSJ64UDxXBWVCpMlEjeuywDvWjRIlasWMGwYcNYvnw5jz32GHPn\nzuX444/P4OqzH9ugm0Rr2MPhMLW1tWq7SToKm+R8udvtJhgMqp6OUb5cj7Zt2/Lii4/z1FPPMH/+\nP1AUhWHDTuK3v71HdwxhbW0t+/fXkZt7DNB4L8rKLmPbtr/90kM9CqfTy969T+Jy+aivX4uihAiF\nNpCT04pQyIuirKdRTe4lgsGPqa3tgsu1iVNO6cjVV1+Nx+NpEqq0qpJVLCiKgt/v15UxNer9hsSr\nx62EXPwlpxmihWmby/QwQD28iWdAa+TkyWnRCiqzDXFYFb9fsU9+9913PPHEE0yZMgW3282IESNY\nvXo1hx12GG3bts3wqrMX26CbRDbo8Y48FZ8DsXvo4XCYmpoaNV8uNgW9/nIzdOnShYceekAdWhEp\n519SUkK7dmVs3PgJxcVDACgtHYvf/wO7dz9GMDidujoHDQ3baGhog6L8ACwjHB6Jz7cBmAH8jNP5\nJoWFAFMJBmdxzTW/44Ybbmgy+jATwi6pwEwlu5nq8Wz2YOW2NG3xl1GYVq/IKhktUplCT8rV4XCk\ntT0sk8iteXI1/4033sjOnTtZvnw5ffr0YcWKFVx++eUATJo0icmTJ2dy2VmLbdCjIAy5eKG8Xi/1\n9fWm8uVGnwexGXS9/nKx4Qsv3ShfHg0zh5G6ujpOOOFXfPfd62zfnkeLFiMJBDYTDC7j1FP70LZt\nKdOmzSMYLANewOFYjKL8E3gL2A34cLvvxuHoQYsWrSguHsGOHRewb9++g+YYxyLsYlUvLl6xEKPq\n8Wz0YOXRn9Ha0ozCtKI1LNuuHcxLucbSHpZt3rtRa144HOaOO+7go48+Yu7cuXTu3BmA7du3M2fO\nHLp3757JZWc1tkGPkfr6+ibtYbESi0GP1F8uSHUu7uOPP+aOO/7C9u1egkEf9fUPUl39OG43lJXl\ns2uXh08/XU0odDRQBpwEnIjDMRpF+QB4DihHUc6guDiP4uKSXw5Jndi7d1/E75a9uEitMlbKQ4oq\n5nA4nFB/sVkP1krXLhADNsToy1jXpfd7lyMXVvfeE5Fy1WsP02uHtOq1C4xa80KhELfccgufffYZ\nCxcupEOHDurfad++PVdccUWmltwssA26CcRcaYD8/Hx1oEkqidZf7nQ6KS4uNuyDTUaB1c6dO7nt\ntnuoqjqDNm3+hNPZgs2bL6Wm5mNcrjOoqenG1q2zUZQALtepBIOfAXtQlDJcru6EQiNonMa2CkXx\nA4WAQjC4B4djBccdd7npteh5cWKj0+sBzsT0LNkrTWZ/sZlrh9RLdJoh0WlhWrQtgfK1W9GDNWrL\niodYrt1KkQutnK9YfyAQ4LrrruOHH35gwYIFtG7dOsMrbX7YBj0KgUCA6upq9f8n2l9uxkPX5ssj\n9ZdHU7FK5GWfPXs2u3e7aNduMk5nIQ0Nn1Nfvx54AJ+vL+Gwi3B4HPA7gsF1OBwlKMo9wG2EQm2A\nRTid1TidDYTDk9i//2wUxYXTOY3u3Qs5//zz47iDjUS7djFYJF3FZUIsxeVyUVBQkNKN1UwONt0D\nNtKlfqe9divNuU+1lGuka49V3CVVaOsmxHvg8/mYMGECu3fvZt68ebRo0SKt6zpUsA16FEToMC8v\nTy2ESwZGn2NGj10vX66nYpVomLKqqgqnsz1OZ6Nn/fPPMwkGWwPlBIPbCAb3AguAMLAMRbkOh+Nb\nFGUCUAfso3XrcbRqdRk7djzN/v0P4fU2cPnlFzBx4q2UlZUldhMjXLscnhY/k4rCOjlXmgmxlEg5\n2HS1xaV69KkRZgR9ID2Ri3RLuep575muOxDGXFs3UV9fz6WXXkogEGD27NkUFxendB2HMrZBj0JJ\nSYka5ob4+8cFkeQQzcwvN7sZ6xVYyS+7mfB0z549gXfw+TYQDLajocEPOGk01mHgURqL3k4HjgGm\nSfdnJ507/x9t294IQLduL7B795vk5j7Oo4/+LaVGz6iwLtkhWtmQaYdLZIpoI1EhuW1xVhowYka5\nLRWRCytIuZqtN0mV925kzGtqahg3bhyFhYW8//77FBQUJO07bQ7GNuhREEY1UUEYGW1Pu6iKFoMK\nRL5c9swT2Xi1L3u08LTYkAYPHkzv3v/kf/+7iUDgAqAYWEfjZLUAsB94gcZxqX5gFHAb7dr5CIc7\nUlIynHDYR1XVG+zdO5uGhq846qhC1q1bR48ePRK4g/Fdu9Z7TyREK3tk6R57aRYzbXGJRC6sMPrU\nCDPKbfLPxHuws+rEuFi6BuR3Ph7EoB1tumnfvn2cf/75dOzYkbfeesuS70hzwzVhwoS/mPnBrl27\npnQhVkWcdkWO0OPxJPzSNs4UbzSgIl8eCAQoLCxUC+6E4QVjCdd4ELKbbreb3NxcPB6Pemjx+/2q\nRGk4HMbtdjNkyJns2/cda9a8iNe74pc1rwI+BwYCvQAf0BbIIydnCxdddBwtWhSxbt377NjxNnv2\nfILP1xeH40RCoSCLFr3HKaf0pU2bNkm5pniuXXhycsGOuP5IOvjAL2I69aohyxZNdmHAPB4Pubm5\n6uFFbPRCjtbMAVJryKxkzPUQRi4nJwePx6MeXuTITTAYVCNxZt65VBVBJhvttYvwu4jeyO88xKZn\nIRtz2TPfvXs3o0ePpmfPnrz++uu2MU+QjRs3mvo526BHQRh0QK3eTXTzEpKXDoeDmpoaAIqLi5uE\nCmPZWBJBb5MXJ3oxROPss4fQo8cRLFv2OS1aPEFNTTcUZc4zwicAACAASURBVAXQFTgeaI3D0QrY\nRUHBZ5x4YiFTpvyVjRtX8OWXX+B03klJSQVduw6hffsL+fnnpdTVfcPw4ZmdphRpkxdeXDAYVCM0\nDoejSWjRypt4NGLZ5MW1i+cwEAg02cSz7R7oHWrFcy+uWxzs5N+9jNyaJxd/WR3toVa884DuwU77\nu5cR98DtdjdpT9y2bRujRo3i5JNPZurUqZY/7GUDZg26fadNksyQuzDaQr9czpfHIuGabIz6fwOB\nAAMHDmTYsI+YM+fPuN3H4fd3Aj4E+gADUJQ9OJ3fUlLyHQMH/oGysjKOP74P8+fX0bHjhU1e6qKi\ncpYte0K9TisQrahQ/IzwXq08wztWYgnRCo9eK2WbzcQy8zwnJ0d9JprDPTBKSUVLy8jGXL4HW7Zs\nYdSoUYwcOZJHH3202bwj2YJt0GNAm/uOBzEhLRgMpiRfnizEJi8fNO67bzKnnTaLN954m6VLPyUY\ndAF3Abk4HB0oLW1g5MjTGDZsmHrKh3q0DlwwWE1pqbXngMtFhUKHIBgMAgfaCq3S+5xsjA52fr9f\nff7D4bCqT25VcZN4iNQxIQ9YEfeoOWF0sNMqNYr5EW63u0lXx/r16xk9ejSXXnopf/3rX5vVO5Et\n2AY9CtqNKhGDLgyBoihquFLuL7eKMRfoFX5dfPHFzJ27kJKSfoRC4/B62xIKLcPtfp8LLxzOnXfe\nzkMPPcy0afOorq5n376dhEK3csQRj+NwOPF6f8TrfZ/RowdZ5jojISrZg8EgeXl5qpHT9j6nu+87\nXYgQragjETUXoVDooLa4TAn6pBLhvefk5Kh99sJQNTQ0qAWlcmtYc7l+vYOd1+tVD7bBYJDx48fT\nunVr+vXrx5QpU7jxxhu56667ms09yDZsgx4DiTykcn+57PnK/eVW2gyMip4+//xzli//ltatn6Gg\n4CRAAc5n+/YyvvhiHn/4w59YsmQ7eXm/wePpgts9naqqdwkEviA/vzdO57ecfHInrr766sxdnElk\nGVdtS5Ze73NzHIcaSZfeCoI+6cBIylVOyzTH372MqC0JBoNq7j0YDNKpUyfmzJnDK6+8gsvlYsGC\nBeTm5lJeXs6xxx7bbK4/W7ANegzEG3IXA11Evry+vj7mkafpROTH9HLF69atw+8voFWrE3/5k8Z1\nFxWdxdat/2Lr1r20aPEsBQUn/VIFPpCtW920bPkh48a14fjjh3L22WcfNJTFakSaFCZjpu87We1B\n6cZo9KnAbN1Btk3Kk4kk5Wo0TKe5jcKVxZNkvQURsXv33Xe57777KCkpYfbs2dx7773ccccdrFq1\nin79+mV6+YcUtkGPgvwCxmrQjfrLodFgiLnmVnnJzaietW7dGqeznkDgZzyeA7PTfb7vgQChUCsK\nC/s3KSJs2bKcYHA+l112GS1btmxy7VbMs8kVzLHIuJrp+86GwRpg/kAjE8nAJbv3OR3EIpojh6ch\n+T3/mUKOTuTm5jY5iC9ZsoTx48fzj3/8g8svvxyHw8HNN99MQ0MDixcv5vjjj8/gyg9NbIMeA7EY\n9Eh67C6XS9X+tkqITg6tRlI9GzhwIF26lPLjj5No3fpecnI6UVe3BJ/vZc44ow9Ll24kFKrC7W4c\nvNBYEbuZoqJc2rRpg8vl0s09WyH/KNqWklXBHE2tD6wzVEQmltGnRkRTLgNrH24SlXKN9XCT6Wdf\nj0hT4+bNm8eVV17J888/z9ixY5usPT8/n2HDMtuSeqhiG/QYMWPQRb4c0NVjd7vdFBcXNznFy+HZ\ndG/wkXLFWvLy8njssQf4/e//zMaN5xMK5ZCbG6C8vB+TJv2ZsWOv4Oef76Nt27txuVpRX/8ZXu9r\nXHbZ2RQWFjb5Tr3cc6Y8GK0nkmwZV6PKcasdbhIdfapHLG1xVjjcJFvKVeu9y5K02sONVbx3o1SD\noijMnDmT6667jtdee42KioqMr9XmAI4FCxaYcjnPOuusFC/FmgivDaC2tpZwOExJSYnhz2vz5XLb\nVygUUquGtcjFRUKpLB0bvPDGIDbpSr/fz5IlS6iqqqJnz5706tULh8PBihUrmDjxHrZt86EoReTk\nVHP66cfyj388bDiUQZt7FqI66drgIxV+pQPZexUVxJmI3CR79KkZtIcbWaUvExPT0i3lqp01IByG\nTKYmIhnz//znP9xyyy288847nH322Wld16HMwoULTf2cbdBNIHpP6+rqCAaDlJaWHvQz0fTYYyl+\nk71X4bmnwntN1cjPmpoaFixYwN69eznmmGM44YQTTH+2Njyb6g1e9sYKCgoyrmoV6XCTqqlZ6Rp9\nagb52ZdV+hLVXDdDMlINiRDp2U+X9y4fbuVonaIovP766/z5z39m2rRpnH766Slbg83BmDXodsjd\nBMIwG+XQw+EwtbW1BIPBg/Ll8VSyG1VOJys8meoNvLi4mDFjxsT1d7Xh2VS2hWk3cCsUaWkL67Th\n2XjG4EYiU6NPjYhVtS1ZkatUpBpiJZKwSzpGohoVASqKwosvvsh9993HrFmz6N+/f9K+0ya52AY9\nBvQMulG+XB65muikNKPWoHhyr9kwJUwmlrawWDa4VEUnko2Z3HO8876jtaVlGjOqbck43IlnwWpS\nrkayrHojURM93MnGXC4CVBSFxx9/nMcff5x58+bRp0+fZF6iTZKxDXoCxJsvjxf5BA/obnCRXvBs\nm5Clxch71W5wkULzZlrzrIq2sC7SvO9ohzsrjz41wmjOfSI9/5moG4iHVBYWGj0L4XCYBx98kJdf\nfpkFCxbQs2fPlF2fTXKw/ltsIYSHLrS9k5EvTwSjDU7Pe4NGqcrmNFhEu8EJz10771p7DwKBQEoq\n2dOJXs+7nveqV3dhxRnesZJo5bh8sMt03UA8mOmaMOO9a4257CxMmjSJadOmsWjRIrp165bW67OJ\nD9ugm0DOoQOG/eVWmZQmh+bl3KP4mWRMjLMaQrkqUu5VoBXIaA5E8l71pqVZPdUQK2Zzz8K4+f1+\n3f7qbEQbuZMPt5G8dyNjHgqFuOOOO1iwYAELFy7k8MMPz9i12cSGbdBjQOTERetasvPlyULWixdr\nE56cmdB8tqPNvYqCJ4HP5yMQCGSkLSodGIm6+P1+9TkV7ZjN/fqN6k4A9drlw3pzwOhwq/Xexax7\n2ZgHg0F+97vf8fnnn7Nw4ULat2+fyUuxiRHboJvE6/WqL0NRUVETsRjxYljJ2zESi9HLvVlFrS4V\niEp2kWoQQybEtTf3aWHiuRSGXfyO9VITqW4LywSy9yoXfrlcLjX10NyvX5uaEb93aNwPnnvuOZYu\nXcrZZ5/NggUL2LRpEwsWLKBVq1aZXLpNHNgG3QS1tbVq4UwgEGgyKc2Kw1VksRit0pVRYZUcmk5l\nz3M6MapejnT92mlh2ZhflpHb0rSDNSK1hWX7QBEtelKu6WyLswoiMuN0OikoKFDD7Rs3buSmm24C\noG/fvjz++OOUl5dz0kknZf07cChhC8uYoKamRg1bV1dXqxXtVjTmibRjyaFJoViWKcWuRIi3z97o\n+rM1NRGrAp6RYl22DRTRYpQr1vs5Kyj2pYpgMEhdXd1Bvfb19fVcfPHFOJ1OLrjgAubPn8+HH37I\nvn37uPXWW3n00UczvHIbW1gmieTn56sneUDNv1rJmMtGLN4WHKNhIkZV41a5dhnZiMVa8GRmmEq8\nPd/pJhZ9foGZrgmr6K2bJZaKfjOaB9kavTAy5tXV1YwbN46SkhLeeecd8vPzufbaawkGg6xYsYKW\nLVtmeOU2sWAbdJMoiqIWz8h9zFbIu4o2OmHEPB5PwuvRC81bYZBMJOIxYkaYHaZixc09ntGnWiKJ\nmsiFVVaO3iQi5aqXezYahWv16IUoCtWmnvbu3ct5551Hly5deOONN5pEcNxuN6eddlqmlmwTJ7ZB\nN4GoDhbTl0RbiJx3zVTeORQKUV9fn1KBkEiCLlaZFJYMI2ZEJEEfq40CTYUeuZ6oiVHPvxUOuJB8\nKVejcajpkGRNBCNjvmvXLsaMGcNxxx3HP//5z6wQFrKJjv1bjIKiKPTp04cuXbowcuRIRo0aRceO\nHcnLyzuoJUarVpZq4yZeVqfTqeb104EZrfV05l3lzTsdvdVWDU2nS488Ws+/trAw3Qe8VEu5plOS\nNRHEfdCm4H7++WfGjBnDwIEDeeqpp+yit2aEXRRngu3btzN9+nSmT5/O/PnzOeqoo6ioqKCiooLe\nvXurG7ZW0AFSU1QkKlW9Xq+l9Ke1g2TkUZCpMG5Wuw9Go0DTEb2wioRppMLCdBzwMn0f5D1ATIuD\n9NdeGN2HTZs2MWrUKM455xz+9re/WSaSYBMZe3xqChAFV3PnzqWyspKZM2eSn59PeXk5I0eO5Iwz\nzmgyO1jPuCX6Yhu1IVmNVBu3bLgP6agat9LoUy2RZn0n+4BnRSlXraiL9h1IVe2BMOba+/DDDz8w\nevRorrjiCiZPnmwb8yzCNuhpIBQKsXz5ciorK5k+fTrbtm1j6NChVFRUMGzYMEpLS5vov+sZNzk0\nH41sm5Qmo2fc4g1LygIh2XIfUmHcEqnoTzfRDniJGDf5UGPl+yCnp+QIXrI6R+RDjfaQu2bNGsaM\nGcNNN93En/70p4wfdmxiwzboaUZRFNauXasa91WrVnHKKacwcuRIKioq6Ny5s/oSxdPvLYvFZOtA\nDYGRcTPTNSD3FBcUFGRlMU8yohdWH30ajWQZNzlSky2HO2gawQsGg6okb7y1B5EONV9++SXnnHMO\nd999NzfffLNtzLMQ26BnEEVR2LVrFzNnzqSyspJ58+bRrVs3taiuT58+TfLusucqWuPkjU0uNGpO\nAzUgsnHTqrXJFdwFBQVZfaiRMTJuRqH55nCokTFj3PR+13KEIhsPNTJ6h3yz6ZlIxnzFihWMHTuW\nKVOm8Jvf/MY25lmKbdAtgvAgPvroIyorK5kxYwZut1vNu5955pnqC2i0sUHj5pafn99sjJgRRtEL\np9NJMBhsVuNf9TB6BuTIRUNDQ7M71MiYUewTxlyWcm0uRErPaNviIkUoPv74Yy6++GKeeOIJLr30\nUtuYZzG2QbcooVCITz/9VA3Nb9myhSFDhqh597KyMhwOB/v27WPBggUMHjxYzcNDdK+lOSEqhn0+\nn2rYskGtLlnIk9ICgYAavQDweDwZ73lPB0bGTZDtnnk0jJ4BccARhz6tMZ8zZw5XXXUVL7zwAhdc\ncEGzfkYOBWyDngUoisL69euZNm0a06dPZ8WKFfTv35/TTz+dt99+m6qqKlatWkX79u2TWlSWLcje\nh8fjwe12q96r7LlaTcwjFYieYjlaAYfeAUcWMxJkWtQonWgliQVff/0133//PcOHD2fVqlVcf/31\nvPHGG5SXlzfr+3GoYGu5ZwEOh4Pu3bszceJEbrvtNqqqqnjwwQd5+OGHadGiBcceeyzPPvsso0aN\nom/fvgeJmTTnEahGRV96M661gj5WlSKNB6N2rEhyvM31gBMKhdTfc2FhIYCuqFFzeg+0iOsT40/z\n8vJQFIWPP/6Y+++/X62ruOCCC2jVqlWzm/VuExnXhAkT/mLmB7t27ZrShdjA888/z+TJkxk+fDiz\nZ8+me/fufPXVV0yZMoUnnniCH374AYDDDz9cNe7CcxVzvoUUpwjNORyOrHuhhYyrkLPVC6mKjc3j\n8ZCbm6vmVcUBx+/3NwlPZts9gAMRCr/fT25ubpOeYjHnPCcnh9zcXNWACyPv9/ubhKiz8TmQkVXw\nhCqikKQV90CWJpbfA0VRsvYZ0CK3bBYWFqrv/+mnn07btm2prq7miCOOYOHChTz55JM888wzBAIB\nzjjjjEwv3SYBNm7caOrn7JC7Rairq+OEE07gnHPO4f7772+SHw+Hw6xcuVLNu2/YsIHBgwdTUVHB\n8OHDad26tbpZyTKc6VQqSxaynG08xW/RisqsMkgmGrGOPtX+Xa1aWTZ7rvFIucqCLs0lRWXU3aAo\nCs899xwPPPAAH3zwASeeeCLBYJBPPvmEmTNn0r17d379619nePU2iWDn0LOQ2tpaioqKIv6Moihs\n3LhRzbsvW7aME088Ue13P/LII5v0uxsplVlxUxMKV8mUcc3GA448NS7RCm6t1nq21R4kQ8pVrzUU\nsmcULhjPdFcUhUcffZSnn36a2bNnc9xxx2V4pTapwDbohwCKorB3714++OADKisrmTNnDocddhjl\n5eWMGjWKE088scmLr60WtsqErHTJl8ba750JYpnfHQ/xiBplglRJuWZSbz9ejIx5OBxmypQpvPba\na8ydO5cePXpkeKU2qcI26IcgXq+XRYsWMW3aNGbOnInP56O8vJzy8nIGDx6sejjyppZpry1T8qXp\nHiRjhmAwSF1dXdp67SOJGmXykJdOKdd06O0ngjjgacfhhsNh7r77bmbMmMHcuXM54ogjMrZGm9Rj\nG/RDnHA4zBdffKHm3detW8egQYOoqKhgxIgRtGnTxpQUbSo9Fjm0nMl+Yit4bekafWqEUWg+3boH\nmZRyjVR/kYn0hJExD4VCTJw4kcWLFzN37lw6deqUtjXZZAbboNuoKIrC5s2bqayspLKykqVLl3L8\n8cdTUVHByJEjOfroo1UDkq4RsKkOLSdCunv+Mz3yUw8zam3JXqfVpFz16i/SlZ4wMubBYJAbb7yR\nr7/+mtmzZ9O+ffuUfL+NtbANuo0uiqKwb98+Zs+ezfTp05k9ezZt2rRR8+79+/dvUj0baQRsvCFZ\n2Ru1ujZ9tEEyiYTm5dCyVUfAQnqKymTdAStKuUZLTyQzRSNSL9pojd/v55prrmHLli3MmjWLsrKy\npHyfjfWxDbqNKXw+H4sXL2batGnMmDGDuro6RowYQUVFBYMHD1arzZMxAlZRFPx+f9Ir2dNFLINk\nzHxWtow+lTFzD2JNTxgVfVmVSOmJRFM0Rsbc6/VyxRVXUFNTQ2VlJaWlpUm9JhtrYxt0m5gJh8N8\n9dVXqnFfs2YNZ511FiNHjqS8vJx27drFPQJWzo1a2RuNhXhD89k++lQm0fSElVMvZklWYZ2IXGkP\nu3V1dYwfPx6n08n7778ftbXVpvlhG3SbhFAUha1btzJ9+nQqKyv5+OOP6d27t5p379mzZ9QRsLJx\nFwYsm2ZWx4KZMbgOh6PZjT6ViTU9IY/DbS4T9GKZlCZjJJ5TXV3N2LFjKSsr4+233yY/Pz+t12Nj\nDWyDbpM0FEWhpqZGzbt/8MEHtGzZUs27n3rqqbp5d+0IWCHT2hw27kgYhWSdTifhcFg1YNnojZol\nWueAw+HA6/VmrKo/HUSalCYfdsXAGW1R5J49ezjvvPPo1q0br732WrM8CNuYwzboNinD7/ezdOlS\nNTS/d+9eRowYwciRIzn77LMpKirC4XCwZMkSvF4v/fv3V71TOLRGwEJjSNbr9TaZjmVFMZdUoheW\nBtT6i0z3e6cDPUlegcvlIj8/X30fdu7cyZgxY+jXrx8vvvhis4rk2MSObdBt0kI4HOabb75RjftX\nX33FwIEDOeqoo3j++ecZNGgQ7777ruqdHmojYKFpODU/P79JBCPbddZjRbTouVwudQysFUR90o3c\n4SCKTm+++Wa2bdvGmWeeyb///W+GDh3Kk08+eUgcem0iY9ag29PWbBLC4XDQrl07Bg4cyK9//Wuu\nuuoqPvjgA9544w169+6N3+9n586dtGjRgjZt2uhOSBNT4prDhDQZIV/q9XrxeDzk5+fjdDrVCIWY\nlAXNe0IYHCzlWlBQoE4LFO2P4XBYfQ5EukZMiWsu9wEO3AvRrijuRW1tLd9//z1vv/02u3btoqqq\nih9//BGn00mnTp1sL/0Qxuy0Ndug2yQNn8/HTTfdxHvvvcf999/PM888Q4cOHVixYgWTJ09m6tSp\nbNy4kdzcXDp27KiGW5vjCNhIo08FZkagNgfDppVy1RsDKw56Ho9HPegJ4+73+5vUIWTjPRAY3QuH\nw0FBQQFPPvkk119/PXfccQcA06dP56mnnqKoqIjTTz89w6u3yRT2+FSbtCPa3J5++mnGjh3b5L8F\nAgGWL1+uTonbtWsXw4cPp6KigqFDh1JcXNxsRsAmMvpUkA2DZMyQiJRrcxmFK4ikUb9mzRpGjx7N\nrbfeyh133NFEufGbb76hVatWdOjQIVNLt8kwdg7dJiPU1dVRWFgY8WcURWHNmjWqzvz//vc/Tjvt\nNHUEbMeOHSOOgLVy3j2Zo08FVhwkY4ZkSrmarRi30rMgE+lg88UXX3Deeedxzz33cNNNN1n2Gmwy\nh23QbbICRVHYsWMHM2bMoLKykvnz53PUUUdRUVFBRUUFvXv3Pqjf3YojYCE9IilWGCRjhlRLuRpF\nMKxYXBgpYvPJJ59w4YUX8uCDD/LrX//aMmu2sRa2QbfJOsTGN3fuXCorK5k5cyb5+fmUl5czcuRI\nzjjjDHUztNIIWEj/6FNBJJWyTEUw0i3larUpadq1GUUpFi1axCWXXMJTTz3FxRdfbBtzG0Nsg26T\n9YRCIZYvX66G5rdt28bQoUOpqKhg2LBhlJaWZnwELBxoS8u0SEoqB8mYxQpSrpmckiajKAp1dXUH\nRSkURWHOnDlcffXVvPTSS5x77rm2MbeJiG3QbZoViqKwdu1aKisrmTFjBitXruSUU05R8+6dO3dO\n+whYediMlUafQnIHyZjFilKuemIu6QjNRzLmlZWV/Pa3v+XNN9+kvLw86d9t0/ywDbpNs0VRFHbt\n2sXMmTOprKxk3rx5dOvWjZEjRzJq1Cj69OnTJO+eihGw2TL6VJDq4kJ5JK5VpVyNJHmTHZo3Sjko\nisLbb7/NxIkTee+99xg0aFDC32VzaGAbdJtDAlE9/NFHH6neu9vtVvPuZ555ptoelIwRsOJzsnH0\nqcDsIBmzGA0WsTqxTgw0+5lGxvyVV15h0qRJVFZWMmDAgKRfj03zxTboNockoVCITz/9VM27b9my\nhSFDhqh597KysrhHwIq/01xGn0LiXqtQwrNayiFWIh1yzEZyIhnzZ555hocffphZs2bRr1+/dFyS\nTTPCNug2hzyKorB+/XpVzGbFihX0799fDc137dq1Sd490ghYt9ut5kWb4+hTgdlDjlbKVU8JL1sx\nOuREqj8QxYDa+gFFUXjkkUd47rnn+PDDD+nVq1far8cm+7ENuo2NhKIoVFVV8cEHH1BZWcmcOXPo\n3Lmzatz79u3bxKMyGgErJDqbozHXEslrFf8tLy8Pj8fTbIy5HkaHHOG5A7rFgOFwmPvuu4+33nqL\nuXPncvTRR2fsGmyyG9ug29hEoKGhgQULFqh593A4zMiRIykvL2fQoEFqkZuiKKxatYpu3bo18coO\ntRGw8iHH7/erf26FXu90onfIEXg8Hurq6mjdujXhcJg//elPzJ49m7lz59qzMGwSwjboNjYmCYfD\nrFy5Us27b9iwgcGDB1NRUcHevXu56667mDJlCr/97W8P2tDB2lK0yURbDAhkrd5+MhCV/WK4ymef\nfcY555xD//79admyJevWrWPhwoV06tQp00u1yXLs8ak2NiZxOBx07NiRwYMHc/3113PxxRfj9Xp5\n5JFHePfddxk0aBBdunShTZs2lJWVHVIjYAVaKVcxIU/cB6fT2WwnpOkht+kVFRWRm5tLUVERnTp1\nYs2aNSxevJiqqioqKyvZsGEDbrebLl26HBJRDJvkY3baWvNPBNrYxIDD4aBTp058+eWXbNy4kUmT\nJtG9e3emT5/OQw89xGGHHUZ5eTmjRo3ixBNPVNvdtIVUQtDG6gNUzCBXbxcVFR2UYnA4HOroU239\ngQjPN4f7IBDGXNumV1paypIlS1AUhU2bNvG///3/9u4/Jur6jwP484PH3XEgRQMbuPihjqVIXUQM\naeVElB+n9APE1OiXaaXZj1VLM51lSyMpl5mTdJzmWi1xcXANQXPNHyQzdf1Yg03BX/zIHBScd58D\n7vP9o30+uwP0Cwrc587n488PMN6f0+3J5/36vF+v06isrMTevXthNptx+fLlW6I8Q97DLXeiPj76\n6COsWbMGZrMZCxcuVK47HA789NNPKC8vh9VqhSiKyM7ORnZ2NtLT0z2Obfn6CFjZzbRy9ZVBMkNx\nrTP3drsdhYWFsNlsqKioQGhoqPIzkiTh/PnziImJ8dayycexhk50gxwOB3799VekpKRc83tcLhdO\nnTql1N0bGhowY8YMmEwmZGZmYty4cT47AlY23K1c1ThIZijkMO975t5ms2H+/PnQarUoKyv7v+OD\niYaKgU40SuQnMIvFAovFgqNHj8JoNCp95uPj4/uddx9oBKyaxn6OdCtXNQySGQq5gU7fM/f//PMP\n8vPzERERgW+++UZ5WZBoODHQibxAkiR0dHSgqqoKFRUVqKqqQkREhFJ3T0lJ8RjUMRq9xYdqtFu5\nemOQzFA4HI4BG+hcuXIFjz76KOLj47F7926f7xo4HGw2G4qKilBXV4e6ujq0t7fDbDbjqaee8vbS\nfNpgA50vxRENI0EQEBYWhgULFmDBggUQRRGHDx9GeXk5lixZApvNhqysLOTk5GDmzJlKkxq9Xu8x\n393hcMDhcIzaCFiZN1q5CoKAMWPGKKHtvjXvcDgAeKdE4d4Nr+8Anra2NuTm5uKBBx5ASUnJLdFo\naDD+/vtvrF+/HjExMTAajYMOIhoe6tnTIvJDOp0OGRkZ2LJlC86cOYPq6mpMnDgRxcXFiI2Nxbx5\n81BaWorW1lYIgqAcfwoNDUVQUBDGjBkDURTR1dWFzs5O2O12jy3q4SJPj5O3lb3Zlz0gIABarRbB\nwcEIDQ2FwWDAmDFj0N3dDZvNhs7OTly9enVEPgeZ/HmIogi9Xu/xZH7x4kVkZWVh+vTp2LFjB8Pc\nTVRUFFpbW9HY2IiioqIR+/ehgfF/ItEoCQgIgNFohNFoxNq1a3Hx4kVUVFTAYrHgrbfeQmJiIkwm\nE3JycnD33XcPeBTMvVPbzY6AlbmPglVbK1f3l+aumvgqdAAACt1JREFUdzRwOEsU8gS/7u7uftP0\nGhsbMWfOHBQUFGDDhg2qqvOrQWBgIMaNG+ftZdyyGOhEXiAIAu666y4sW7YML730Ejo7O5W6e2Zm\nJsLCwpS6+7Rp05TQct+a7+npgd1uh91uH/IIWJl7eAUFBUGr1Y7gXd8c9+lnfT+HviWKGx1/er3P\no76+Hrm5uXj++eexdu1a1fzRQyRjoBN5mSAICA0NRUFBAQoKCuB0OnH06FGUl5dj+fLlaG9vV+ru\nGRkZCAkJUd6mvplQc2/l6oujYAMCApTatntLXqfTCVEUh3x64Hqfx++//47c3Fy8+eabeOONNxjm\npErcLyJSGa1WixkzZmDz5s1KP/DJkydjy5YtiIuLQ15eHnbs2IHm5mal7t633ux0OpV680B1d7n7\nW09PD4KDg30uzPuSt+YNBgPGjh2r3FNvby+uXr2Kf//9FzabDaIoekzPk10vzE+ePAmTyYQ1a9Yw\nzEnV+IROpGIBAQFITExEYmIiVq9ejZaWFqXuvmrVKkyePBkmkwkmkwlTpkzpV28eqAWrRqOBKIoA\nMGArV1/nvjUPeHbtG+j0gCAIHn3q3V9yq62tRUFBATZt2oRnnnmGYU6qxkAn8hGCICAqKgovvPAC\nli5dCpvNhv3798NiscBkMiEkJESZ756WlqZsNbuf8+57FEzu2OZvoe5OPhInb83Lf+TIR9Jk7i+/\nAcChQ4fw5JNPYtu2bZg/fz7DnFSPW+5EPkgQBISEhCAvLw+7du1Cc3Mzdu3aBb1ej1dffRWxsbF4\n7rnnUFZWhs7OTgQEBODUqVNYvHgxnE6nMiHN4XB4HInrO+Pb38iDZAwGA0JCQpSQFgQBHR0dmDBh\nAvLz87Fu3TosXLgQZrOZYU4+g4FO5AcCAwPx8MMPo7i4GPX19Thy5AjuvfdebNu2DRMmTMCsWbMw\nd+5ctLW1KW+Je/Oct7fJ42CB/8oOY8eOhU6nw/Lly9Ha2opNmzahs7MTGzduxIYNG/Dbb7/57WdB\n/oNb7kR+RhAEJCQkICEhAStXrsT27dvx8ssvIzY2FmfPnsWcOXOUuntiYuItMQLWncvlQldXFwDP\nCXK33347oqOj0dTUhH379qGjowMVFRX48MMPsXr1atTW1iI1NdWbS/cJW7duRUdHBy5dugQAsFgs\nuHDhAgDglVdewdixY725PL/GXu5Efmzr1q1YsWIFCgsL8eWXX6K7uxs1NTWwWCywWq3Q6/XIzs6G\nyWTCQw895HHu2l9GwLqTx8H2nSAnSRJKS0uxbt06VFZWegS3PDY3IyPDr981GC5xcXE4f/78gF9r\nbGxEdHT0KK/I93E4C9EtTpIkLFu2DMHBwSgqKur3dN3b24va2lplBGxLSwtmzZoFk8mE2bNn47bb\nbvP5EbDurhfmn3/+OT755BNUVVXhvvvu8/JKiTwx0IkILpcLgiAMqqlKfX09LBYLKisrceLECaSm\npiojYKOjo31uBKy7a812lyQJRUVF2LlzJ/bv34+EhAQvr5SoPwY6Ed0QSZJw+fJlWK1WWCwWHDhw\nAHFxcUrd3Wg0egSiGkfAuuvp6YHNZus3293lcuH999/Hd999h5qaGkyaNMmr6yS6FgY6Ed00ubf5\nwYMHlad3jUaD7Oxs5OTkYPr06R7nt93Pu8t199EeAeuuu7sbV69eHTDMV65cierqatTU1CAmJmbU\n1kQ0VAx0Ihp2vb29qKurU+ruFy5cwMyZM5W6+x133OGxNS/X3OU35uUWrfIT/EiGuxzmGo0GBoNB\n+V29vb147bXXcPz4cVRXVyMqKmrE1kA0HBjoRDSiJEnCmTNnUF5ejoqKChw/fhwpKSlKt7rY2FiP\ncHcfASuf6R6uEbB9OZ1O2O12BAYGesx27+7uxosvvoiGhgZUVVUhIiJi2H4n0UhhoBPRqJEkCVeu\nXMEPP/wAi8WC6upqREdHKy/VJSUlKUe+3FvR9vT0eByJu5ERsH1dK8xFUcSzzz6Lv/76C1arFWFh\nYTd/40SjgIFORF5jt9tx6NAhpe7ucrmUuvuMGTOg0+k86tlyuLsfibuRueaiKMLhcECr1UKv1ys/\nZ7fbsWjRIoiiCIvFwuYm5FMY6ESkCi6XCydOnFDq7mfPnkV6ejpMJhMyMzMRHh7e70icHPDykbjB\n1N0dDgdEUewX5l1dXSgoKEBQUBDKyspgMBhG7d6JhgMDnYhUR5IkNDU1KXX3Y8eOITk5Wdmanzhx\n4oB1956eHo8jce6taCVJUian6XQ6j6f/jo4O5OXlITIyEl9//TX0er3X7p3oRjHQiUjVJElCe3u7\nR909MjJSeakuOTl5UHV34L831/V6vccRuitXruCRRx7B5MmTYTabERgYOPo3STQMGOhE5FPknunl\n5eWorKyE0+lEdnY2srOzkZ6e7vGCm8vlgiiKcDqdys87nU58+umnyMnJQVxcHB577DGkpqZi+/bt\n7MFOPm2wge77o5OIyC/o9XpkZmbiiy++QFNTE6xWK6Kjo7Fx40bExsbiiSeewO7du9HW1obu7m4s\nXboU7777LvR6PQwGA86dO4c9e/YgMzMTSUlJcLlcMJlMcDgc3r41olHBQCci1QkICMD999+P9957\nD7/88gv++OMPzJ49G2VlZZgyZQpSUlJgsVgwdepUaLVaBAYGIjk5GQcPHkRaWhqSk5MhSRIef/xx\nhIeHY+7cubh48aK3b4toRDHQiUjVBEFATEwMVqxYge+//x5paWk4d+4cTCYT1q9fj6SkJLzzzjv4\n9ttvkZOTg6ysLPz444/4888/UV9fjw8++ACiKCI8PNzbt0I0olhDJyKf0NnZCZPJhJMnT8JisSA9\nPR2iKOLw4cMoLy/Hzp07sWjRIpSUlKhu2hvRzRhsDV0zsssgIhoeGo0G4eHhqKmpwbRp0wAAOp0O\nGRkZyMjIwMcffzzsLWSJfAkDnYh8QlBQEPbt23fNr/OMOd3qWEMnIiLyAwx0IiIiP8BAJyIi8gMM\ndFIVp9OJt99+G+PHj4fBYEBqaioOHDjg7WUREakeA51U5emnn8bmzZtRWFiIzz77DBqNBjk5OTh2\n7Ji3l0ZEpGo8h06qUVdXh9TUVBQXF+P1118H8N9866lTp+LOO+/EkSNHvLxCIqLRx17u5HP27t0L\njUaDJUuWKNd0Oh0WL16M2tpaXLp0yYurIyJSNwY6qcbp06cRHx+PkJAQj+spKSnK14mIaGAMdFKN\nlpYWREZG9rseGRkJSZLQ3NzshVUREfkGBjqpht1uh06n63dd7gBmt9tHe0lERD6DgU6qERQUBFEU\n+12X51kHBQWN9pKIiHwGA51UIzIyEi0tLf2uy9eioqJGe0lERD6DgU6qYTQa0dDQgK6uLo/rP//8\nMwRBgNFo9NLKiIjUj4FOqpGfn4+enh6UlJQo15xOJ8xmM1JTUzF+/Hgvro6ISN04PpVUIyUlBfPm\nzcOqVavQ1taGSZMmwWw249y5cygtLfX28oiIVI2BTqry1VdfYc2aNdizZw/a29txzz33wGq14sEH\nH/T20oiIVI2tX4mIiFSMrV+JiIhuIQx0IiIiP8BAJyIi8gMMdCIiIj/AQCciIvIDDHQiIiI/wEAn\nIiLyAwx0IiIiPzDoTnGDPdhOREREo49P6ERERH6AgU5EROQHGOhERER+gIFORETkBxjoREREfoCB\nTkRE5AcY6ERERH6AgU5EROQHGOhERER+4H8Sdfgrr+oF3QAAAABJRU5ErkJggg==\n",
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"338pt\" version=\"1.1\" viewBox=\"0 0 359 338\" width=\"359pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:100000;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 338.4 \n",
       "L 360 338.4 \n",
       "L 360 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:#bfbfbf;\"/>\n",
       "  </g>\n",
       "  <g id=\"patch_2\">\n",
       "   <path d=\"M 7.2 331.2 \n",
       "L 352.8 331.2 \n",
       "L 352.8 7.2 \n",
       "L 7.2 7.2 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "  </g>\n",
       "  <g id=\"pane3d_1\">\n",
       "   <g id=\"patch_3\">\n",
       "    <path d=\"M 53.01462 259.607418 \n",
       "L 151.927149 180.16996 \n",
       "L 150.28036 28.378014 \n",
       "L 45.85281 99.750937 \n",
       "\" style=\"fill:#f2f2f2;opacity:0.5;stroke:#f2f2f2;stroke-linejoin:miter;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"pane3d_2\">\n",
       "   <g id=\"patch_4\">\n",
       "    <path d=\"M 151.927149 180.16996 \n",
       "L 312.156145 224.580625 \n",
       "L 318.859934 68.211455 \n",
       "L 150.28036 28.378014 \n",
       "\" style=\"fill:#e6e6e6;opacity:0.5;stroke:#e6e6e6;stroke-linejoin:miter;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"pane3d_3\">\n",
       "   <g id=\"patch_5\">\n",
       "    <path d=\"M 53.01462 259.607418 \n",
       "L 221.597175 311.330875 \n",
       "L 312.156145 224.580625 \n",
       "L 151.927149 180.16996 \n",
       "\" style=\"fill:#ececec;opacity:0.5;stroke:#ececec;stroke-linejoin:miter;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axis3d_1\">\n",
       "   <g id=\"line2d_1\">\n",
       "    <path d=\"M 53.01462 259.607418 \n",
       "L 221.597175 311.330875 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-width:0.75;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_1\">\n",
       "    <!-- 0 -->\n",
       "    <defs>\n",
       "     <path d=\"M 31.78125 66.40625 \n",
       "Q 24.171875 66.40625 20.328125 58.90625 \n",
       "Q 16.5 51.421875 16.5 36.375 \n",
       "Q 16.5 21.390625 20.328125 13.890625 \n",
       "Q 24.171875 6.390625 31.78125 6.390625 \n",
       "Q 39.453125 6.390625 43.28125 13.890625 \n",
       "Q 47.125 21.390625 47.125 36.375 \n",
       "Q 47.125 51.421875 43.28125 58.90625 \n",
       "Q 39.453125 66.40625 31.78125 66.40625 \n",
       "M 31.78125 74.21875 \n",
       "Q 44.046875 74.21875 50.515625 64.515625 \n",
       "Q 56.984375 54.828125 56.984375 36.375 \n",
       "Q 56.984375 17.96875 50.515625 8.265625 \n",
       "Q 44.046875 -1.421875 31.78125 -1.421875 \n",
       "Q 19.53125 -1.421875 13.0625 8.265625 \n",
       "Q 6.59375 17.96875 6.59375 36.375 \n",
       "Q 6.59375 54.828125 13.0625 64.515625 \n",
       "Q 19.53125 74.21875 31.78125 74.21875 \n",
       "\" id=\"BitstreamVeraSans-Roman-30\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(116.610780547 324.606857811)scale(0.12 -0.12)\">\n",
       "     <use xlink:href=\"#BitstreamVeraSans-Roman-30\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"Line3DCollection_1\">\n",
       "    <path d=\"M 73.530439 265.901955 \n",
       "L 171.483883 185.590499 \n",
       "L 170.817005 33.230591 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 108.61639 276.666809 \n",
       "L 204.892601 194.850392 \n",
       "L 205.924955 41.526201 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 144.38815 287.64208 \n",
       "L 238.906336 204.277978 \n",
       "L 241.70146 49.979783 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 180.866026 298.833996 \n",
       "L 273.541674 213.877854 \n",
       "L 278.165802 58.595894 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 218.071132 310.249036 \n",
       "L 308.815811 223.654785 \n",
       "L 315.33801 67.379265 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "   </g>\n",
       "   <g id=\"xtick_1\">\n",
       "    <g id=\"line2d_2\">\n",
       "     <path d=\"M 74.374529 265.209891 \n",
       "L 71.839102 267.288673 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_2\">\n",
       "    <g id=\"line2d_3\">\n",
       "     <path d=\"M 109.446602 275.961288 \n",
       "L 106.952832 278.080517 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_3\">\n",
       "    <g id=\"line2d_4\">\n",
       "     <path d=\"M 145.203778 286.922705 \n",
       "L 142.753786 289.083573 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_4\">\n",
       "    <g id=\"line2d_5\">\n",
       "     <path d=\"M 181.666328 298.100355 \n",
       "L 179.262338 300.304102 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_5\">\n",
       "    <g id=\"line2d_6\">\n",
       "     <path d=\"M 218.855335 309.500701 \n",
       "L 216.499678 311.748617 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axis3d_2\">\n",
       "   <g id=\"line2d_7\">\n",
       "    <path d=\"M 312.156145 224.580625 \n",
       "L 221.597175 311.330875 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-width:0.75;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_2\">\n",
       "    <!-- 1 -->\n",
       "    <defs>\n",
       "     <path d=\"M 12.40625 8.296875 \n",
       "L 28.515625 8.296875 \n",
       "L 28.515625 63.921875 \n",
       "L 10.984375 60.40625 \n",
       "L 10.984375 69.390625 \n",
       "L 28.421875 72.90625 \n",
       "L 38.28125 72.90625 \n",
       "L 38.28125 8.296875 \n",
       "L 54.390625 8.296875 \n",
       "L 54.390625 0 \n",
       "L 12.40625 0 \n",
       "z\n",
       "\" id=\"BitstreamVeraSans-Roman-31\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(289.749044878 299.694047875)scale(0.12 -0.12)\">\n",
       "     <use xlink:href=\"#BitstreamVeraSans-Roman-31\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"Line3DCollection_2\">\n",
       "    <path d=\"M 48.1081 98.209518 \n",
       "L 55.142669 257.898365 \n",
       "L 223.551905 309.458356 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 61.487006 89.065459 \n",
       "L 67.774108 247.753954 \n",
       "L 235.148767 298.349235 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 74.609352 80.096752 \n",
       "L 80.175571 237.794238 \n",
       "L 246.524817 287.451637 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 87.482447 71.298399 \n",
       "L 92.353281 228.014219 \n",
       "L 257.686304 276.759579 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 100.113327 62.665593 \n",
       "L 104.313242 218.409077 \n",
       "L 268.63924 266.2673 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 112.508763 54.193704 \n",
       "L 116.06124 208.974164 \n",
       "L 279.389417 255.969253 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 124.675279 45.878276 \n",
       "L 127.602862 199.704993 \n",
       "L 289.942414 245.860093 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 136.619157 37.715014 \n",
       "L 138.943497 190.597236 \n",
       "L 300.303605 235.934672 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 148.346454 29.699778 \n",
       "L 150.088352 181.646713 \n",
       "L 310.478174 226.188024 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "   </g>\n",
       "   <g id=\"xtick_6\">\n",
       "    <g id=\"line2d_8\">\n",
       "     <path d=\"M 222.142249 309.026778 \n",
       "L 226.37438 310.322482 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_7\">\n",
       "    <g id=\"line2d_9\">\n",
       "     <path d=\"M 233.748366 297.925911 \n",
       "L 237.952679 299.196823 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_8\">\n",
       "    <g id=\"line2d_10\">\n",
       "     <path d=\"M 245.133577 287.036334 \n",
       "L 249.310359 288.283157 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_9\">\n",
       "    <g id=\"line2d_11\">\n",
       "     <path d=\"M 256.304129 276.35207 \n",
       "L 260.453667 277.575484 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_10\">\n",
       "    <g id=\"line2d_12\">\n",
       "     <path d=\"M 267.266036 265.867369 \n",
       "L 271.388613 267.068026 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_11\">\n",
       "    <g id=\"line2d_13\">\n",
       "     <path d=\"M 278.025091 255.57669 \n",
       "L 282.120989 256.755219 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_12\">\n",
       "    <g id=\"line2d_14\">\n",
       "     <path d=\"M 288.586871 245.474696 \n",
       "L 292.656372 246.631705 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_13\">\n",
       "    <g id=\"line2d_15\">\n",
       "     <path d=\"M 298.956754 235.556246 \n",
       "L 303.000136 236.692318 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_14\">\n",
       "    <g id=\"line2d_16\">\n",
       "     <path d=\"M 309.139922 225.816383 \n",
       "L 313.157462 226.93208 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axis3d_3\">\n",
       "   <g id=\"line2d_17\">\n",
       "    <path d=\"M 312.156145 224.580625 \n",
       "L 318.859934 68.211455 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-width:0.75;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_3\">\n",
       "    <!-- 2 -->\n",
       "    <defs>\n",
       "     <path d=\"M 19.1875 8.296875 \n",
       "L 53.609375 8.296875 \n",
       "L 53.609375 0 \n",
       "L 7.328125 0 \n",
       "L 7.328125 8.296875 \n",
       "Q 12.9375 14.109375 22.625 23.890625 \n",
       "Q 32.328125 33.6875 34.8125 36.53125 \n",
       "Q 39.546875 41.84375 41.421875 45.53125 \n",
       "Q 43.3125 49.21875 43.3125 52.78125 \n",
       "Q 43.3125 58.59375 39.234375 62.25 \n",
       "Q 35.15625 65.921875 28.609375 65.921875 \n",
       "Q 23.96875 65.921875 18.8125 64.3125 \n",
       "Q 13.671875 62.703125 7.8125 59.421875 \n",
       "L 7.8125 69.390625 \n",
       "Q 13.765625 71.78125 18.9375 73 \n",
       "Q 24.125 74.21875 28.421875 74.21875 \n",
       "Q 39.75 74.21875 46.484375 68.546875 \n",
       "Q 53.21875 62.890625 53.21875 53.421875 \n",
       "Q 53.21875 48.921875 51.53125 44.890625 \n",
       "Q 49.859375 40.875 45.40625 35.40625 \n",
       "Q 44.1875 33.984375 37.640625 27.21875 \n",
       "Q 31.109375 20.453125 19.1875 8.296875 \n",
       "\" id=\"BitstreamVeraSans-Roman-32\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(349.77774807 146.915694411)scale(0.12 -0.12)\">\n",
       "     <use xlink:href=\"#BitstreamVeraSans-Roman-32\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"Line3DCollection_3\">\n",
       "    <path d=\"M 312.28365 221.606507 \n",
       "L 151.89576 177.276724 \n",
       "L 52.878634 256.572102 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 312.968008 205.643528 \n",
       "L 151.727332 161.751965 \n",
       "L 52.1486 240.277261 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 313.659745 189.508416 \n",
       "L 151.557164 146.066818 \n",
       "L 51.410434 223.80089 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 314.358982 173.19837 \n",
       "L 151.385229 130.218786 \n",
       "L 50.663999 207.13994 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 315.065841 156.710531 \n",
       "L 151.211499 114.205317 \n",
       "L 49.909154 190.29129 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 315.780448 140.041976 \n",
       "L 151.035946 98.023808 \n",
       "L 49.145757 173.251751 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 316.502931 123.189716 \n",
       "L 150.858541 81.6716 \n",
       "L 48.373662 156.018061 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 317.23342 106.150698 \n",
       "L 150.679255 65.145977 \n",
       "L 47.592719 138.586882 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 317.972049 88.921801 \n",
       "L 150.498057 48.444166 \n",
       "L 46.802775 120.9548 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "    <path d=\"M 318.718956 71.499833 \n",
       "L 150.314917 31.563334 \n",
       "L 46.003674 103.118321 \n",
       "\" style=\"fill:none;stroke:#e6e6e6;\"/>\n",
       "   </g>\n",
       "   <g id=\"xtick_15\">\n",
       "    <g id=\"line2d_18\">\n",
       "     <path d=\"M 310.945442 221.236638 \n",
       "L 314.962849 222.347014 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_16\">\n",
       "    <g id=\"line2d_19\">\n",
       "     <path d=\"M 311.622388 205.277235 \n",
       "L 315.662062 206.376881 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_17\">\n",
       "    <g id=\"line2d_20\">\n",
       "     <path d=\"M 312.30663 189.145797 \n",
       "L 316.36882 190.234416 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_18\">\n",
       "    <g id=\"line2d_21\">\n",
       "     <path d=\"M 312.998289 172.839527 \n",
       "L 317.083245 173.916816 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_19\">\n",
       "    <g id=\"line2d_22\">\n",
       "     <path d=\"M 313.697484 156.355568 \n",
       "L 317.805465 157.421213 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_20\">\n",
       "    <g id=\"line2d_23\">\n",
       "     <path d=\"M 314.40434 139.690999 \n",
       "L 318.535605 140.74468 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_21\">\n",
       "    <g id=\"line2d_24\">\n",
       "     <path d=\"M 315.118984 122.842835 \n",
       "L 319.273798 123.884223 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_22\">\n",
       "    <g id=\"line2d_25\">\n",
       "     <path d=\"M 315.841544 105.808026 \n",
       "L 320.020178 106.836783 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_23\">\n",
       "    <g id=\"line2d_26\">\n",
       "     <path d=\"M 316.572154 88.583453 \n",
       "L 320.774882 89.599232 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"xtick_24\">\n",
       "    <g id=\"line2d_27\">\n",
       "     <path d=\"M 317.310948 71.165928 \n",
       "L 321.538049 72.168372 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"Path3DCollection_1\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 2.236068 \n",
       "C 0.593012 2.236068 1.161816 2.000462 1.581139 1.581139 \n",
       "C 2.000462 1.161816 2.236068 0.593012 2.236068 0 \n",
       "C 2.236068 -0.593012 2.000462 -1.161816 1.581139 -1.581139 \n",
       "C 1.161816 -2.000462 0.593012 -2.236068 0 -2.236068 \n",
       "C -0.593012 -2.236068 -1.161816 -2.000462 -1.581139 -1.581139 \n",
       "C -2.000462 -1.161816 -2.236068 -0.593012 -2.236068 0 \n",
       "C -2.236068 0.593012 -2.000462 1.161816 -1.581139 1.581139 \n",
       "C -1.161816 2.000462 -0.593012 2.236068 0 2.236068 \n",
       "z\n",
       "\" id=\"C0_0_3af6fb4475\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.392127278915;stroke:#000000;stroke-opacity:0.392127278915;\" x=\"157.096618502\" xlink:href=\"#C0_0_3af6fb4475\" y=\"134.753024433\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.533054753068;stroke:#000000;stroke-opacity:0.533054753068;\" x=\"188.192468251\" xlink:href=\"#C0_0_3af6fb4475\" y=\"141.658825277\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.657495988188;stroke:#000000;stroke-opacity:0.657495988188;\" x=\"209.749147685\" xlink:href=\"#C0_0_3af6fb4475\" y=\"156.690469719\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.329118771127;stroke:#000000;stroke-opacity:0.329118771127;\" x=\"154.191900173\" xlink:href=\"#C0_0_3af6fb4475\" y=\"156.784214423\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.445479643457;stroke:#000000;stroke-opacity:0.445479643457;\" x=\"147.290529918\" xlink:href=\"#C0_0_3af6fb4475\" y=\"130.216976738\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.323948455537;stroke:#000000;stroke-opacity:0.323948455537;\" x=\"170.863320342\" xlink:href=\"#C0_0_3af6fb4475\" y=\"152.957961088\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.567160517477;stroke:#000000;stroke-opacity:0.567160517477;\" x=\"120.134420811\" xlink:href=\"#C0_0_3af6fb4475\" y=\"192.203148731\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.323877919987;stroke:#000000;stroke-opacity:0.323877919987;\" x=\"167.365121772\" xlink:href=\"#C0_0_3af6fb4475\" y=\"155.628005137\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.511750277936;stroke:#000000;stroke-opacity:0.511750277936;\" x=\"154.992139409\" xlink:href=\"#C0_0_3af6fb4475\" y=\"177.213797184\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.484999658091;stroke:#000000;stroke-opacity:0.484999658091;\" x=\"122.744382383\" xlink:href=\"#C0_0_3af6fb4475\" y=\"164.444534335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.353746586389;stroke:#000000;stroke-opacity:0.353746586389;\" x=\"168.037032682\" xlink:href=\"#C0_0_3af6fb4475\" y=\"156.291223355\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.383626965565;stroke:#000000;stroke-opacity:0.383626965565;\" x=\"151.626447088\" xlink:href=\"#C0_0_3af6fb4475\" y=\"131.504324407\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.430570921661;stroke:#000000;stroke-opacity:0.430570921661;\" x=\"165.082078383\" xlink:href=\"#C0_0_3af6fb4475\" y=\"137.278412067\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.421381526297;stroke:#000000;stroke-opacity:0.421381526297;\" x=\"150.033879445\" xlink:href=\"#C0_0_3af6fb4475\" y=\"133.106497509\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.352774648625;stroke:#000000;stroke-opacity:0.352774648625;\" x=\"160.13383915\" xlink:href=\"#C0_0_3af6fb4475\" y=\"145.042691418\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.425634705279;stroke:#000000;stroke-opacity:0.425634705279;\" x=\"147.137158405\" xlink:href=\"#C0_0_3af6fb4475\" y=\"127.509278111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.460623506446;stroke:#000000;stroke-opacity:0.460623506446;\" x=\"150.251874559\" xlink:href=\"#C0_0_3af6fb4475\" y=\"123.296846061\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.516804197677;stroke:#000000;stroke-opacity:0.516804197677;\" x=\"175.288738233\" xlink:href=\"#C0_0_3af6fb4475\" y=\"130.797394744\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.514215152543;stroke:#000000;stroke-opacity:0.514215152543;\" x=\"146.783051035\" xlink:href=\"#C0_0_3af6fb4475\" y=\"121.188862518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.43444245302;stroke:#000000;stroke-opacity:0.43444245302;\" x=\"165.019243913\" xlink:href=\"#C0_0_3af6fb4475\" y=\"135.181598807\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.424286223894;stroke:#000000;stroke-opacity:0.424286223894;\" x=\"154.193994327\" xlink:href=\"#C0_0_3af6fb4475\" y=\"132.483672848\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.33778823146;stroke:#000000;stroke-opacity:0.33778823146;\" x=\"160.323570295\" xlink:href=\"#C0_0_3af6fb4475\" y=\"143.313529365\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.376453136376;stroke:#000000;stroke-opacity:0.376453136376;\" x=\"176.753545435\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.073267842\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.33956908797;stroke:#000000;stroke-opacity:0.33956908797;\" x=\"150.408393293\" xlink:href=\"#C0_0_3af6fb4475\" y=\"162.278299255\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.325254503225;stroke:#000000;stroke-opacity:0.325254503225;\" x=\"157.935610067\" xlink:href=\"#C0_0_3af6fb4475\" y=\"147.353931553\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.379165888949;stroke:#000000;stroke-opacity:0.379165888949;\" x=\"157.150480933\" xlink:href=\"#C0_0_3af6fb4475\" y=\"140.480334687\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.555614382998;stroke:#000000;stroke-opacity:0.555614382998;\" x=\"147.287449332\" xlink:href=\"#C0_0_3af6fb4475\" y=\"121.265379416\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.441059390472;stroke:#000000;stroke-opacity:0.441059390472;\" x=\"154.313775506\" xlink:href=\"#C0_0_3af6fb4475\" y=\"168.461953756\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.447951856954;stroke:#000000;stroke-opacity:0.447951856954;\" x=\"170.889651924\" xlink:href=\"#C0_0_3af6fb4475\" y=\"155.262737763\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.885192764734;stroke:#000000;stroke-opacity:0.885192764734;\" x=\"110.728974067\" xlink:href=\"#C0_0_3af6fb4475\" y=\"101.722573462\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.595605630098;stroke:#000000;stroke-opacity:0.595605630098;\" x=\"153.143025213\" xlink:href=\"#C0_0_3af6fb4475\" y=\"117.464615561\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.371507475505;stroke:#000000;stroke-opacity:0.371507475505;\" x=\"157.761350284\" xlink:href=\"#C0_0_3af6fb4475\" y=\"134.675113401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.39291720237;stroke:#000000;stroke-opacity:0.39291720237;\" x=\"146.682705717\" xlink:href=\"#C0_0_3af6fb4475\" y=\"137.049137826\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.383909028218;stroke:#000000;stroke-opacity:0.383909028218;\" x=\"161.642660461\" xlink:href=\"#C0_0_3af6fb4475\" y=\"132.634445191\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.510289236292;stroke:#000000;stroke-opacity:0.510289236292;\" x=\"155.949369783\" xlink:href=\"#C0_0_3af6fb4475\" y=\"125.04199611\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.500074098103;stroke:#000000;stroke-opacity:0.500074098103;\" x=\"144.570814201\" xlink:href=\"#C0_0_3af6fb4475\" y=\"120.869636377\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.387021508256;stroke:#000000;stroke-opacity:0.387021508256;\" x=\"175.215171994\" xlink:href=\"#C0_0_3af6fb4475\" y=\"143.388747092\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.349826415435;stroke:#000000;stroke-opacity:0.349826415435;\" x=\"150.881394524\" xlink:href=\"#C0_0_3af6fb4475\" y=\"146.418297885\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.902990004704;stroke:#000000;stroke-opacity:0.902990004704;\" x=\"206.124006356\" xlink:href=\"#C0_0_3af6fb4475\" y=\"184.91820164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.795552541384;stroke:#000000;stroke-opacity:0.795552541384;\" x=\"117.37187808\" xlink:href=\"#C0_0_3af6fb4475\" y=\"105.04148424\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.450952298233;stroke:#000000;stroke-opacity:0.450952298233;\" x=\"181.02096818\" xlink:href=\"#C0_0_3af6fb4475\" y=\"136.790103902\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.596129231631;stroke:#000000;stroke-opacity:0.596129231631;\" x=\"168.696227825\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.811178026\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.607873217595;stroke:#000000;stroke-opacity:0.607873217595;\" x=\"116.202474147\" xlink:href=\"#C0_0_3af6fb4475\" y=\"155.782433287\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.83782296045;stroke:#000000;stroke-opacity:0.83782296045;\" x=\"118.364213316\" xlink:href=\"#C0_0_3af6fb4475\" y=\"111.297004232\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.46459367203;stroke:#000000;stroke-opacity:0.46459367203;\" x=\"146.173988724\" xlink:href=\"#C0_0_3af6fb4475\" y=\"122.920547365\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.58854023332;stroke:#000000;stroke-opacity:0.58854023332;\" x=\"115.326330174\" xlink:href=\"#C0_0_3af6fb4475\" y=\"161.382215316\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.353453003565;stroke:#000000;stroke-opacity:0.353453003565;\" x=\"158.300564162\" xlink:href=\"#C0_0_3af6fb4475\" y=\"139.435565437\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.856697942793;stroke:#000000;stroke-opacity:0.856697942793;\" x=\"115.485063661\" xlink:href=\"#C0_0_3af6fb4475\" y=\"104.895410009\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.503110318314;stroke:#000000;stroke-opacity:0.503110318314;\" x=\"178.221108528\" xlink:href=\"#C0_0_3af6fb4475\" y=\"126.736356346\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.637778346959;stroke:#000000;stroke-opacity:0.637778346959;\" x=\"113.046088264\" xlink:href=\"#C0_0_3af6fb4475\" y=\"176.33657788\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.56140404309;stroke:#000000;stroke-opacity:0.56140404309;\" x=\"113.403718796\" xlink:href=\"#C0_0_3af6fb4475\" y=\"182.756326208\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.497438199128;stroke:#000000;stroke-opacity:0.497438199128;\" x=\"162.497928313\" xlink:href=\"#C0_0_3af6fb4475\" y=\"127.828091054\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.401665655098;stroke:#000000;stroke-opacity:0.401665655098;\" x=\"139.25684264\" xlink:href=\"#C0_0_3af6fb4475\" y=\"153.963504409\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.401271938555;stroke:#000000;stroke-opacity:0.401271938555;\" x=\"158.427637629\" xlink:href=\"#C0_0_3af6fb4475\" y=\"129.145049707\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.448173617677;stroke:#000000;stroke-opacity:0.448173617677;\" x=\"161.246119773\" xlink:href=\"#C0_0_3af6fb4475\" y=\"128.211832917\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.516984768541;stroke:#000000;stroke-opacity:0.516984768541;\" x=\"177.969798727\" xlink:href=\"#C0_0_3af6fb4475\" y=\"131.123349821\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.441384623022;stroke:#000000;stroke-opacity:0.441384623022;\" x=\"180.998174772\" xlink:href=\"#C0_0_3af6fb4475\" y=\"145.784381742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.826145686662;stroke:#000000;stroke-opacity:0.826145686662;\" x=\"117.030045911\" xlink:href=\"#C0_0_3af6fb4475\" y=\"102.665808142\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.417573202674;stroke:#000000;stroke-opacity:0.417573202674;\" x=\"136.572595053\" xlink:href=\"#C0_0_3af6fb4475\" y=\"178.724683768\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.416000315523;stroke:#000000;stroke-opacity:0.416000315523;\" x=\"162.879807096\" xlink:href=\"#C0_0_3af6fb4475\" y=\"131.023413373\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.49042035515;stroke:#000000;stroke-opacity:0.49042035515;\" x=\"128.736485457\" xlink:href=\"#C0_0_3af6fb4475\" y=\"162.946282977\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.380984242271;stroke:#000000;stroke-opacity:0.380984242271;\" x=\"157.53847414\" xlink:href=\"#C0_0_3af6fb4475\" y=\"143.303962804\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.426160677089;stroke:#000000;stroke-opacity:0.426160677089;\" x=\"133.763243161\" xlink:href=\"#C0_0_3af6fb4475\" y=\"155.135332125\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.76896645398;stroke:#000000;stroke-opacity:0.76896645398;\" x=\"118.119649545\" xlink:href=\"#C0_0_3af6fb4475\" y=\"120.874863474\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.881070030588;stroke:#000000;stroke-opacity:0.881070030588;\" x=\"105.454696273\" xlink:href=\"#C0_0_3af6fb4475\" y=\"107.669621044\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.341153262486;stroke:#000000;stroke-opacity:0.341153262486;\" x=\"152.772915779\" xlink:href=\"#C0_0_3af6fb4475\" y=\"164.012575412\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.301412773369;stroke:#000000;stroke-opacity:0.301412773369;\" x=\"163.221013161\" xlink:href=\"#C0_0_3af6fb4475\" y=\"147.979266171\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.312872371969;stroke:#000000;stroke-opacity:0.312872371969;\" x=\"165.391871085\" xlink:href=\"#C0_0_3af6fb4475\" y=\"163.009864686\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.652212689635;stroke:#000000;stroke-opacity:0.652212689635;\" x=\"198.250661833\" xlink:href=\"#C0_0_3af6fb4475\" y=\"138.088899565\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.772176067613;stroke:#000000;stroke-opacity:0.772176067613;\" x=\"110.174974269\" xlink:href=\"#C0_0_3af6fb4475\" y=\"236.093833073\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.789616002534;stroke:#000000;stroke-opacity:0.789616002534;\" x=\"104.112351135\" xlink:href=\"#C0_0_3af6fb4475\" y=\"242.633465397\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.804672206557;stroke:#000000;stroke-opacity:0.804672206557;\" x=\"104.369311578\" xlink:href=\"#C0_0_3af6fb4475\" y=\"241.916794815\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.786448677078;stroke:#000000;stroke-opacity:0.786448677078;\" x=\"102.338661619\" xlink:href=\"#C0_0_3af6fb4475\" y=\"241.948944936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.790161473263;stroke:#000000;stroke-opacity:0.790161473263;\" x=\"110.307061861\" xlink:href=\"#C0_0_3af6fb4475\" y=\"240.681783117\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.760981045104;stroke:#000000;stroke-opacity:0.760981045104;\" x=\"105.434068338\" xlink:href=\"#C0_0_3af6fb4475\" y=\"229.071859779\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.768019072714;stroke:#000000;stroke-opacity:0.768019072714;\" x=\"115.643258258\" xlink:href=\"#C0_0_3af6fb4475\" y=\"230.710808381\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.826958443746;stroke:#000000;stroke-opacity:0.826958443746;\" x=\"98.0877666069\" xlink:href=\"#C0_0_3af6fb4475\" y=\"229.407608407\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.812778778775;stroke:#000000;stroke-opacity:0.812778778775;\" x=\"100.239843593\" xlink:href=\"#C0_0_3af6fb4475\" y=\"240.006041391\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.78347415245;stroke:#000000;stroke-opacity:0.78347415245;\" x=\"107.242960261\" xlink:href=\"#C0_0_3af6fb4475\" y=\"229.948031923\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.579350238797;stroke:#000000;stroke-opacity:0.579350238797;\" x=\"118.117643816\" xlink:href=\"#C0_0_3af6fb4475\" y=\"201.538328175\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.527346192529;stroke:#000000;stroke-opacity:0.527346192529;\" x=\"121.356792213\" xlink:href=\"#C0_0_3af6fb4475\" y=\"188.756984707\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.819547983886;stroke:#000000;stroke-opacity:0.819547983886;\" x=\"93.0322062402\" xlink:href=\"#C0_0_3af6fb4475\" y=\"225.008179543\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.769053424064;stroke:#000000;stroke-opacity:0.769053424064;\" x=\"111.757431647\" xlink:href=\"#C0_0_3af6fb4475\" y=\"243.974312783\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.759390511039;stroke:#000000;stroke-opacity:0.759390511039;\" x=\"114.399433835\" xlink:href=\"#C0_0_3af6fb4475\" y=\"232.071473617\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.788722353424;stroke:#000000;stroke-opacity:0.788722353424;\" x=\"98.8040044387\" xlink:href=\"#C0_0_3af6fb4475\" y=\"227.003425833\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.77709718309;stroke:#000000;stroke-opacity:0.77709718309;\" x=\"106.913149102\" xlink:href=\"#C0_0_3af6fb4475\" y=\"228.43651244\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.940563511529;stroke:#000000;stroke-opacity:0.940563511529;\" x=\"112.351274063\" xlink:href=\"#C0_0_3af6fb4475\" y=\"107.085639414\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.786181292292;stroke:#000000;stroke-opacity:0.786181292292;\" x=\"193.883634572\" xlink:href=\"#C0_0_3af6fb4475\" y=\"176.445976697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.914689419234;stroke:#000000;stroke-opacity:0.914689419234;\" x=\"113.394552223\" xlink:href=\"#C0_0_3af6fb4475\" y=\"104.193692829\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.568991381431;stroke:#000000;stroke-opacity:0.568991381431;\" x=\"164.83912653\" xlink:href=\"#C0_0_3af6fb4475\" y=\"129.995301367\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.882355185213;stroke:#000000;stroke-opacity:0.882355185213;\" x=\"116.472225262\" xlink:href=\"#C0_0_3af6fb4475\" y=\"101.358644946\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.943864109686;stroke:#000000;stroke-opacity:0.943864109686;\" x=\"113.234622745\" xlink:href=\"#C0_0_3af6fb4475\" y=\"101.128075187\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.543364148472;stroke:#000000;stroke-opacity:0.543364148472;\" x=\"158.459763661\" xlink:href=\"#C0_0_3af6fb4475\" y=\"131.549594466\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.605813966337;stroke:#000000;stroke-opacity:0.605813966337;\" x=\"138.289107165\" xlink:href=\"#C0_0_3af6fb4475\" y=\"125.366032918\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.692863533482;stroke:#000000;stroke-opacity:0.692863533482;\" x=\"217.006968178\" xlink:href=\"#C0_0_3af6fb4475\" y=\"157.676688879\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.408667450495;stroke:#000000;stroke-opacity:0.408667450495;\" x=\"151.635852059\" xlink:href=\"#C0_0_3af6fb4475\" y=\"153.461291071\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.312328532191;stroke:#000000;stroke-opacity:0.312328532191;\" x=\"156.956055376\" xlink:href=\"#C0_0_3af6fb4475\" y=\"151.730583129\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.342305683306;stroke:#000000;stroke-opacity:0.342305683306;\" x=\"158.631889235\" xlink:href=\"#C0_0_3af6fb4475\" y=\"169.926610504\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.894991750599;stroke:#000000;stroke-opacity:0.894991750599;\" x=\"120.386941605\" xlink:href=\"#C0_0_3af6fb4475\" y=\"104.386855579\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.337697673899;stroke:#000000;stroke-opacity:0.337697673899;\" x=\"147.666282051\" xlink:href=\"#C0_0_3af6fb4475\" y=\"163.473652897\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.887656361277;stroke:#000000;stroke-opacity:0.887656361277;\" x=\"187.763604742\" xlink:href=\"#C0_0_3af6fb4475\" y=\"154.37006014\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.756733730669;stroke:#000000;stroke-opacity:0.756733730669;\" x=\"148.392990728\" xlink:href=\"#C0_0_3af6fb4475\" y=\"122.709053628\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.565635559487;stroke:#000000;stroke-opacity:0.565635559487;\" x=\"169.640707764\" xlink:href=\"#C0_0_3af6fb4475\" y=\"120.969834951\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.544629428413;stroke:#000000;stroke-opacity:0.544629428413;\" x=\"170.449557877\" xlink:href=\"#C0_0_3af6fb4475\" y=\"139.868077705\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.50663336027;stroke:#000000;stroke-opacity:0.50663336027;\" x=\"165.509009548\" xlink:href=\"#C0_0_3af6fb4475\" y=\"125.934321063\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.423217862201;stroke:#000000;stroke-opacity:0.423217862201;\" x=\"177.058241461\" xlink:href=\"#C0_0_3af6fb4475\" y=\"154.985610185\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.387435028976;stroke:#000000;stroke-opacity:0.387435028976;\" x=\"162.305446448\" xlink:href=\"#C0_0_3af6fb4475\" y=\"139.323832653\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.467358651902;stroke:#000000;stroke-opacity:0.467358651902;\" x=\"194.603158482\" xlink:href=\"#C0_0_3af6fb4475\" y=\"154.450379102\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.328386270158;stroke:#000000;stroke-opacity:0.328386270158;\" x=\"157.173753904\" xlink:href=\"#C0_0_3af6fb4475\" y=\"172.269951928\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.941516334955;stroke:#000000;stroke-opacity:0.941516334955;\" x=\"100.697674714\" xlink:href=\"#C0_0_3af6fb4475\" y=\"104.229065155\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.88856511885;stroke:#000000;stroke-opacity:0.88856511885;\" x=\"111.743947329\" xlink:href=\"#C0_0_3af6fb4475\" y=\"102.762754058\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.486871609251;stroke:#000000;stroke-opacity:0.486871609251;\" x=\"181.95982358\" xlink:href=\"#C0_0_3af6fb4475\" y=\"133.620635779\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.383950258032;stroke:#000000;stroke-opacity:0.383950258032;\" x=\"158.943162949\" xlink:href=\"#C0_0_3af6fb4475\" y=\"134.743543124\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.834906240029;stroke:#000000;stroke-opacity:0.834906240029;\" x=\"110.991244364\" xlink:href=\"#C0_0_3af6fb4475\" y=\"111.888584355\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.359819895434;stroke:#000000;stroke-opacity:0.359819895434;\" x=\"154.715656089\" xlink:href=\"#C0_0_3af6fb4475\" y=\"138.084004852\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.345533222475;stroke:#000000;stroke-opacity:0.345533222475;\" x=\"162.4987531\" xlink:href=\"#C0_0_3af6fb4475\" y=\"140.730368818\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.772935980603;stroke:#000000;stroke-opacity:0.772935980603;\" x=\"215.652405939\" xlink:href=\"#C0_0_3af6fb4475\" y=\"148.51296878\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.319792224795;stroke:#000000;stroke-opacity:0.319792224795;\" x=\"170.519430411\" xlink:href=\"#C0_0_3af6fb4475\" y=\"153.481647795\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.943538993977;stroke:#000000;stroke-opacity:0.943538993977;\" x=\"111.542140657\" xlink:href=\"#C0_0_3af6fb4475\" y=\"99.1490887779\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.885904844489;stroke:#000000;stroke-opacity:0.885904844489;\" x=\"129.399605126\" xlink:href=\"#C0_0_3af6fb4475\" y=\"102.497851472\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.322432065747;stroke:#000000;stroke-opacity:0.322432065747;\" x=\"171.408813051\" xlink:href=\"#C0_0_3af6fb4475\" y=\"146.205093602\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.73388758889;stroke:#000000;stroke-opacity:0.73388758889;\" x=\"244.090969987\" xlink:href=\"#C0_0_3af6fb4475\" y=\"167.527343241\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.370570654878;stroke:#000000;stroke-opacity:0.370570654878;\" x=\"176.262150552\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.498009054\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.630990957336;stroke:#000000;stroke-opacity:0.630990957336;\" x=\"196.469276654\" xlink:href=\"#C0_0_3af6fb4475\" y=\"139.11062097\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.375052169262;stroke:#000000;stroke-opacity:0.375052169262;\" x=\"170.626691664\" xlink:href=\"#C0_0_3af6fb4475\" y=\"150.260972828\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.400879123752;stroke:#000000;stroke-opacity:0.400879123752;\" x=\"163.323466196\" xlink:href=\"#C0_0_3af6fb4475\" y=\"127.260809817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.619372943117;stroke:#000000;stroke-opacity:0.619372943117;\" x=\"172.534506198\" xlink:href=\"#C0_0_3af6fb4475\" y=\"124.77893781\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.394003363791;stroke:#000000;stroke-opacity:0.394003363791;\" x=\"169.579497658\" xlink:href=\"#C0_0_3af6fb4475\" y=\"132.482330355\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.514787003428;stroke:#000000;stroke-opacity:0.514787003428;\" x=\"178.962121812\" xlink:href=\"#C0_0_3af6fb4475\" y=\"141.346873171\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.926360700951;stroke:#000000;stroke-opacity:0.926360700951;\" x=\"108.156696405\" xlink:href=\"#C0_0_3af6fb4475\" y=\"102.702522623\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.503910318244;stroke:#000000;stroke-opacity:0.503910318244;\" x=\"206.542238769\" xlink:href=\"#C0_0_3af6fb4475\" y=\"145.625005143\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.863868196153;stroke:#000000;stroke-opacity:0.863868196153;\" x=\"123.927608151\" xlink:href=\"#C0_0_3af6fb4475\" y=\"102.080761373\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.542048800773;stroke:#000000;stroke-opacity:0.542048800773;\" x=\"211.840032605\" xlink:href=\"#C0_0_3af6fb4475\" y=\"157.429039003\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.360431733317;stroke:#000000;stroke-opacity:0.360431733317;\" x=\"174.674717762\" xlink:href=\"#C0_0_3af6fb4475\" y=\"137.678560273\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.452440633339;stroke:#000000;stroke-opacity:0.452440633339;\" x=\"163.471844301\" xlink:href=\"#C0_0_3af6fb4475\" y=\"124.805545367\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.726619923378;stroke:#000000;stroke-opacity:0.726619923378;\" x=\"141.734916681\" xlink:href=\"#C0_0_3af6fb4475\" y=\"117.037125026\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.939075826464;stroke:#000000;stroke-opacity:0.939075826464;\" x=\"117.599175735\" xlink:href=\"#C0_0_3af6fb4475\" y=\"104.396685808\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.470222794418;stroke:#000000;stroke-opacity:0.470222794418;\" x=\"156.323367125\" xlink:href=\"#C0_0_3af6fb4475\" y=\"127.2916625\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.96403443533;stroke:#000000;stroke-opacity:0.96403443533;\" x=\"116.209081218\" xlink:href=\"#C0_0_3af6fb4475\" y=\"101.503589944\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.716263120567;stroke:#000000;stroke-opacity:0.716263120567;\" x=\"204.479437784\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.065128111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.663204108184;stroke:#000000;stroke-opacity:0.663204108184;\" x=\"194.374911066\" xlink:href=\"#C0_0_3af6fb4475\" y=\"152.824710913\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.345171917819;stroke:#000000;stroke-opacity:0.345171917819;\" x=\"162.449874929\" xlink:href=\"#C0_0_3af6fb4475\" y=\"139.592172429\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.526969451859;stroke:#000000;stroke-opacity:0.526969451859;\" x=\"130.21520811\" xlink:href=\"#C0_0_3af6fb4475\" y=\"177.656486133\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.534258947191;stroke:#000000;stroke-opacity:0.534258947191;\" x=\"128.990311742\" xlink:href=\"#C0_0_3af6fb4475\" y=\"183.301446424\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.712001883568;stroke:#000000;stroke-opacity:0.712001883568;\" x=\"159.37618086\" xlink:href=\"#C0_0_3af6fb4475\" y=\"173.406488182\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.444055866211;stroke:#000000;stroke-opacity:0.444055866211;\" x=\"158.010900733\" xlink:href=\"#C0_0_3af6fb4475\" y=\"125.945968951\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.432107077078;stroke:#000000;stroke-opacity:0.432107077078;\" x=\"180.303778841\" xlink:href=\"#C0_0_3af6fb4475\" y=\"136.28568939\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.621484852369;stroke:#000000;stroke-opacity:0.621484852369;\" x=\"141.821502636\" xlink:href=\"#C0_0_3af6fb4475\" y=\"173.985906311\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.354791796324;stroke:#000000;stroke-opacity:0.354791796324;\" x=\"143.884641235\" xlink:href=\"#C0_0_3af6fb4475\" y=\"167.463402931\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.575471627902;stroke:#000000;stroke-opacity:0.575471627902;\" x=\"130.973032446\" xlink:href=\"#C0_0_3af6fb4475\" y=\"158.110323842\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.553620413974;stroke:#000000;stroke-opacity:0.553620413974;\" x=\"168.233499121\" xlink:href=\"#C0_0_3af6fb4475\" y=\"169.193549289\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.60792921608;stroke:#000000;stroke-opacity:0.60792921608;\" x=\"173.217601528\" xlink:href=\"#C0_0_3af6fb4475\" y=\"120.411691868\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.625008423686;stroke:#000000;stroke-opacity:0.625008423686;\" x=\"169.633201833\" xlink:href=\"#C0_0_3af6fb4475\" y=\"120.248132215\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.721172548203;stroke:#000000;stroke-opacity:0.721172548203;\" x=\"203.265257839\" xlink:href=\"#C0_0_3af6fb4475\" y=\"153.280657974\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.767937208684;stroke:#000000;stroke-opacity:0.767937208684;\" x=\"210.805592338\" xlink:href=\"#C0_0_3af6fb4475\" y=\"149.448459136\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.872299061479;stroke:#000000;stroke-opacity:0.872299061479;\" x=\"107.514385105\" xlink:href=\"#C0_0_3af6fb4475\" y=\"106.818097225\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.396828085946;stroke:#000000;stroke-opacity:0.396828085946;\" x=\"142.085443686\" xlink:href=\"#C0_0_3af6fb4475\" y=\"168.196525062\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.34444545449;stroke:#000000;stroke-opacity:0.34444545449;\" x=\"170.41222339\" xlink:href=\"#C0_0_3af6fb4475\" y=\"143.256098286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.360802559109;stroke:#000000;stroke-opacity:0.360802559109;\" x=\"160.39300157\" xlink:href=\"#C0_0_3af6fb4475\" y=\"141.283870493\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.471592757131;stroke:#000000;stroke-opacity:0.471592757131;\" x=\"131.287360949\" xlink:href=\"#C0_0_3af6fb4475\" y=\"190.469944537\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.590381653591;stroke:#000000;stroke-opacity:0.590381653591;\" x=\"142.643505338\" xlink:href=\"#C0_0_3af6fb4475\" y=\"176.917777435\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.397318606615;stroke:#000000;stroke-opacity:0.397318606615;\" x=\"164.85121593\" xlink:href=\"#C0_0_3af6fb4475\" y=\"171.370567769\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.575825424325;stroke:#000000;stroke-opacity:0.575825424325;\" x=\"193.148074462\" xlink:href=\"#C0_0_3af6fb4475\" y=\"140.052226125\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.348498000481;stroke:#000000;stroke-opacity:0.348498000481;\" x=\"158.687022566\" xlink:href=\"#C0_0_3af6fb4475\" y=\"137.526808076\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.343143554717;stroke:#000000;stroke-opacity:0.343143554717;\" x=\"164.317364672\" xlink:href=\"#C0_0_3af6fb4475\" y=\"138.835607748\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.622079448561;stroke:#000000;stroke-opacity:0.622079448561;\" x=\"113.401340887\" xlink:href=\"#C0_0_3af6fb4475\" y=\"183.401870225\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.634813949177;stroke:#000000;stroke-opacity:0.634813949177;\" x=\"189.112138499\" xlink:href=\"#C0_0_3af6fb4475\" y=\"136.256710475\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.370967787216;stroke:#000000;stroke-opacity:0.370967787216;\" x=\"141.548561688\" xlink:href=\"#C0_0_3af6fb4475\" y=\"162.678646759\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.360309892059;stroke:#000000;stroke-opacity:0.360309892059;\" x=\"154.827994488\" xlink:href=\"#C0_0_3af6fb4475\" y=\"151.606234341\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.39387843049;stroke:#000000;stroke-opacity:0.39387843049;\" x=\"183.030996449\" xlink:href=\"#C0_0_3af6fb4475\" y=\"161.503441983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.329289235809;stroke:#000000;stroke-opacity:0.329289235809;\" x=\"156.622718718\" xlink:href=\"#C0_0_3af6fb4475\" y=\"161.94308907\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.309399366348;stroke:#000000;stroke-opacity:0.309399366348;\" x=\"156.244481766\" xlink:href=\"#C0_0_3af6fb4475\" y=\"154.9697594\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.5423582212;stroke:#000000;stroke-opacity:0.5423582212;\" x=\"205.352811223\" xlink:href=\"#C0_0_3af6fb4475\" y=\"165.586955446\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.327944845365;stroke:#000000;stroke-opacity:0.327944845365;\" x=\"161.101653673\" xlink:href=\"#C0_0_3af6fb4475\" y=\"165.57948464\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.345796676421;stroke:#000000;stroke-opacity:0.345796676421;\" x=\"160.76270233\" xlink:href=\"#C0_0_3af6fb4475\" y=\"156.742626282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.363574580644;stroke:#000000;stroke-opacity:0.363574580644;\" x=\"151.712638421\" xlink:href=\"#C0_0_3af6fb4475\" y=\"172.302486468\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.3;stroke:#000000;stroke-opacity:0.3;\" x=\"172.170802364\" xlink:href=\"#C0_0_3af6fb4475\" y=\"156.287761198\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.385967334091;stroke:#000000;stroke-opacity:0.385967334091;\" x=\"154.313993728\" xlink:href=\"#C0_0_3af6fb4475\" y=\"140.358130002\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.456061233307;stroke:#000000;stroke-opacity:0.456061233307;\" x=\"194.998333544\" xlink:href=\"#C0_0_3af6fb4475\" y=\"152.494695244\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.591458176315;stroke:#000000;stroke-opacity:0.591458176315;\" x=\"214.288417388\" xlink:href=\"#C0_0_3af6fb4475\" y=\"161.911934935\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.646230909963;stroke:#000000;stroke-opacity:0.646230909963;\" x=\"214.729094159\" xlink:href=\"#C0_0_3af6fb4475\" y=\"172.176226534\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.402766405662;stroke:#000000;stroke-opacity:0.402766405662;\" x=\"188.473907829\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.501412451\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.41845630313;stroke:#000000;stroke-opacity:0.41845630313;\" x=\"151.832650487\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.988852979\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.306649356262;stroke:#000000;stroke-opacity:0.306649356262;\" x=\"165.428161941\" xlink:href=\"#C0_0_3af6fb4475\" y=\"162.606357557\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.359388652876;stroke:#000000;stroke-opacity:0.359388652876;\" x=\"195.068094064\" xlink:href=\"#C0_0_3af6fb4475\" y=\"149.029929648\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.554465623922;stroke:#000000;stroke-opacity:0.554465623922;\" x=\"219.095002914\" xlink:href=\"#C0_0_3af6fb4475\" y=\"154.075977105\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.3417020685;stroke:#000000;stroke-opacity:0.3417020685;\" x=\"181.669103646\" xlink:href=\"#C0_0_3af6fb4475\" y=\"140.847751797\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.627759714614;stroke:#000000;stroke-opacity:0.627759714614;\" x=\"178.725742036\" xlink:href=\"#C0_0_3af6fb4475\" y=\"128.457845405\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.368285371483;stroke:#000000;stroke-opacity:0.368285371483;\" x=\"154.558915187\" xlink:href=\"#C0_0_3af6fb4475\" y=\"177.468698095\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.57514973968;stroke:#000000;stroke-opacity:0.57514973968;\" x=\"204.826973754\" xlink:href=\"#C0_0_3af6fb4475\" y=\"146.637081148\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.781905236922;stroke:#000000;stroke-opacity:0.781905236922;\" x=\"217.340577263\" xlink:href=\"#C0_0_3af6fb4475\" y=\"161.470756372\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.31871675876;stroke:#000000;stroke-opacity:0.31871675876;\" x=\"174.068132043\" xlink:href=\"#C0_0_3af6fb4475\" y=\"152.339176799\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.522502855677;stroke:#000000;stroke-opacity:0.522502855677;\" x=\"186.66715266\" xlink:href=\"#C0_0_3af6fb4475\" y=\"129.125226825\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.483855667566;stroke:#000000;stroke-opacity:0.483855667566;\" x=\"207.379898456\" xlink:href=\"#C0_0_3af6fb4475\" y=\"146.403662441\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.583677917023;stroke:#000000;stroke-opacity:0.583677917023;\" x=\"168.290505379\" xlink:href=\"#C0_0_3af6fb4475\" y=\"123.834344049\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.788179342219;stroke:#000000;stroke-opacity:0.788179342219;\" x=\"241.915923353\" xlink:href=\"#C0_0_3af6fb4475\" y=\"177.285816658\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.591332358894;stroke:#000000;stroke-opacity:0.591332358894;\" x=\"227.727825385\" xlink:href=\"#C0_0_3af6fb4475\" y=\"153.052645824\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.582293470851;stroke:#000000;stroke-opacity:0.582293470851;\" x=\"175.884568217\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.105231556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.599754655817;stroke:#000000;stroke-opacity:0.599754655817;\" x=\"214.397606549\" xlink:href=\"#C0_0_3af6fb4475\" y=\"147.490262951\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.395112946816;stroke:#000000;stroke-opacity:0.395112946816;\" x=\"141.594583166\" xlink:href=\"#C0_0_3af6fb4475\" y=\"180.713332235\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.680068324644;stroke:#000000;stroke-opacity:0.680068324644;\" x=\"221.780737839\" xlink:href=\"#C0_0_3af6fb4475\" y=\"159.63689151\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.522064385267;stroke:#000000;stroke-opacity:0.522064385267;\" x=\"186.017535487\" xlink:href=\"#C0_0_3af6fb4475\" y=\"133.154500814\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.681531900381;stroke:#000000;stroke-opacity:0.681531900381;\" x=\"175.440081089\" xlink:href=\"#C0_0_3af6fb4475\" y=\"123.480882766\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.968710086299;stroke:#000000;stroke-opacity:0.968710086299;\" x=\"233.452254196\" xlink:href=\"#C0_0_3af6fb4475\" y=\"183.601062813\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.461095431608;stroke:#000000;stroke-opacity:0.461095431608;\" x=\"183.231380329\" xlink:href=\"#C0_0_3af6fb4475\" y=\"148.038413946\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.394895261623;stroke:#000000;stroke-opacity:0.394895261623;\" x=\"159.282581194\" xlink:href=\"#C0_0_3af6fb4475\" y=\"145.248475592\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.44350103451;stroke:#000000;stroke-opacity:0.44350103451;\" x=\"191.666884945\" xlink:href=\"#C0_0_3af6fb4475\" y=\"141.929743281\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.4218970991;stroke:#000000;stroke-opacity:0.4218970991;\" x=\"176.064322475\" xlink:href=\"#C0_0_3af6fb4475\" y=\"134.356830194\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.619499337259;stroke:#000000;stroke-opacity:0.619499337259;\" x=\"210.550405641\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.932282947\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.850865416401;stroke:#000000;stroke-opacity:0.850865416401;\" x=\"217.610469284\" xlink:href=\"#C0_0_3af6fb4475\" y=\"166.901842979\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.403099271175;stroke:#000000;stroke-opacity:0.403099271175;\" x=\"159.69815036\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.550556086\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.613616119772;stroke:#000000;stroke-opacity:0.613616119772;\" x=\"198.13510817\" xlink:href=\"#C0_0_3af6fb4475\" y=\"142.061732421\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.965731002897;stroke:#000000;stroke-opacity:0.965731002897;\" x=\"235.799830316\" xlink:href=\"#C0_0_3af6fb4475\" y=\"190.915911868\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.741712017514;stroke:#000000;stroke-opacity:0.741712017514;\" x=\"232.507051845\" xlink:href=\"#C0_0_3af6fb4475\" y=\"160.524824664\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.53232727393;stroke:#000000;stroke-opacity:0.53232727393;\" x=\"209.17979569\" xlink:href=\"#C0_0_3af6fb4475\" y=\"160.031179487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.366855536627;stroke:#000000;stroke-opacity:0.366855536627;\" x=\"170.717202648\" xlink:href=\"#C0_0_3af6fb4475\" y=\"146.782755423\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.535580854849;stroke:#000000;stroke-opacity:0.535580854849;\" x=\"180.114636935\" xlink:href=\"#C0_0_3af6fb4475\" y=\"165.441491016\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.390969140802;stroke:#000000;stroke-opacity:0.390969140802;\" x=\"187.301690484\" xlink:href=\"#C0_0_3af6fb4475\" y=\"151.624023958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.583188840774;stroke:#000000;stroke-opacity:0.583188840774;\" x=\"174.122567581\" xlink:href=\"#C0_0_3af6fb4475\" y=\"149.146828749\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.510226962724;stroke:#000000;stroke-opacity:0.510226962724;\" x=\"189.413951845\" xlink:href=\"#C0_0_3af6fb4475\" y=\"148.891222331\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.321185471533;stroke:#000000;stroke-opacity:0.321185471533;\" x=\"159.48451227\" xlink:href=\"#C0_0_3af6fb4475\" y=\"166.09891866\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.491271985274;stroke:#000000;stroke-opacity:0.491271985274;\" x=\"167.377753463\" xlink:href=\"#C0_0_3af6fb4475\" y=\"168.474166256\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.452792899282;stroke:#000000;stroke-opacity:0.452792899282;\" x=\"170.474593975\" xlink:href=\"#C0_0_3af6fb4475\" y=\"133.449299827\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.542261159276;stroke:#000000;stroke-opacity:0.542261159276;\" x=\"147.33026795\" xlink:href=\"#C0_0_3af6fb4475\" y=\"186.14177079\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.389175581103;stroke:#000000;stroke-opacity:0.389175581103;\" x=\"155.92274017\" xlink:href=\"#C0_0_3af6fb4475\" y=\"140.794869998\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.588125323696;stroke:#000000;stroke-opacity:0.588125323696;\" x=\"148.395589072\" xlink:href=\"#C0_0_3af6fb4475\" y=\"149.918540764\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.574710261376;stroke:#000000;stroke-opacity:0.574710261376;\" x=\"206.313970877\" xlink:href=\"#C0_0_3af6fb4475\" y=\"147.238786494\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.308033286045;stroke:#000000;stroke-opacity:0.308033286045;\" x=\"168.96955285\" xlink:href=\"#C0_0_3af6fb4475\" y=\"158.597673853\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.52085615977;stroke:#000000;stroke-opacity:0.52085615977;\" x=\"147.255736408\" xlink:href=\"#C0_0_3af6fb4475\" y=\"159.556344116\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.453407976109;stroke:#000000;stroke-opacity:0.453407976109;\" x=\"170.296783722\" xlink:href=\"#C0_0_3af6fb4475\" y=\"142.467348696\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.938387520488;stroke:#000000;stroke-opacity:0.938387520488;\" x=\"114.557147976\" xlink:href=\"#C0_0_3af6fb4475\" y=\"104.86075519\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.675597843173;stroke:#000000;stroke-opacity:0.675597843173;\" x=\"117.495350424\" xlink:href=\"#C0_0_3af6fb4475\" y=\"166.840450175\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.762018739125;stroke:#000000;stroke-opacity:0.762018739125;\" x=\"104.384977889\" xlink:href=\"#C0_0_3af6fb4475\" y=\"230.319966927\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.819631270142;stroke:#000000;stroke-opacity:0.819631270142;\" x=\"106.783831588\" xlink:href=\"#C0_0_3af6fb4475\" y=\"230.919311648\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.809642494731;stroke:#000000;stroke-opacity:0.809642494731;\" x=\"112.674626618\" xlink:href=\"#C0_0_3af6fb4475\" y=\"232.835245664\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.788919892288;stroke:#000000;stroke-opacity:0.788919892288;\" x=\"92.8953376406\" xlink:href=\"#C0_0_3af6fb4475\" y=\"233.465314935\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.809006967914;stroke:#000000;stroke-opacity:0.809006967914;\" x=\"105.562373511\" xlink:href=\"#C0_0_3af6fb4475\" y=\"221.743299958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.791145270437;stroke:#000000;stroke-opacity:0.791145270437;\" x=\"105.945257329\" xlink:href=\"#C0_0_3af6fb4475\" y=\"239.630862786\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.495622409821;stroke:#000000;stroke-opacity:0.495622409821;\" x=\"180.856604832\" xlink:href=\"#C0_0_3af6fb4475\" y=\"140.022411721\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.801708121021;stroke:#000000;stroke-opacity:0.801708121021;\" x=\"201.745326744\" xlink:href=\"#C0_0_3af6fb4475\" y=\"155.952997363\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.883510728889;stroke:#000000;stroke-opacity:0.883510728889;\" x=\"199.038236859\" xlink:href=\"#C0_0_3af6fb4475\" y=\"158.765966538\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.376495604631;stroke:#000000;stroke-opacity:0.376495604631;\" x=\"160.028801995\" xlink:href=\"#C0_0_3af6fb4475\" y=\"156.672946176\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.365411991985;stroke:#000000;stroke-opacity:0.365411991985;\" x=\"158.644567207\" xlink:href=\"#C0_0_3af6fb4475\" y=\"161.645699295\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.367276148288;stroke:#000000;stroke-opacity:0.367276148288;\" x=\"165.114473853\" xlink:href=\"#C0_0_3af6fb4475\" y=\"165.476381013\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.416469240264;stroke:#000000;stroke-opacity:0.416469240264;\" x=\"194.794080164\" xlink:href=\"#C0_0_3af6fb4475\" y=\"153.950477946\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.401295087116;stroke:#000000;stroke-opacity:0.401295087116;\" x=\"180.333095613\" xlink:href=\"#C0_0_3af6fb4475\" y=\"151.585872921\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.364636992542;stroke:#000000;stroke-opacity:0.364636992542;\" x=\"167.196147849\" xlink:href=\"#C0_0_3af6fb4475\" y=\"141.74327489\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.887182699507;stroke:#000000;stroke-opacity:0.887182699507;\" x=\"223.425710563\" xlink:href=\"#C0_0_3af6fb4475\" y=\"175.994582063\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.341893977579;stroke:#000000;stroke-opacity:0.341893977579;\" x=\"169.363453561\" xlink:href=\"#C0_0_3af6fb4475\" y=\"142.470806248\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.31144034065;stroke:#000000;stroke-opacity:0.31144034065;\" x=\"167.970575496\" xlink:href=\"#C0_0_3af6fb4475\" y=\"149.305319247\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.522701167737;stroke:#000000;stroke-opacity:0.522701167737;\" x=\"152.081046198\" xlink:href=\"#C0_0_3af6fb4475\" y=\"120.53858841\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.781974410719;stroke:#000000;stroke-opacity:0.781974410719;\" x=\"214.148194551\" xlink:href=\"#C0_0_3af6fb4475\" y=\"145.760954091\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.343932633867;stroke:#000000;stroke-opacity:0.343932633867;\" x=\"170.807843148\" xlink:href=\"#C0_0_3af6fb4475\" y=\"139.754239754\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.796091980294;stroke:#000000;stroke-opacity:0.796091980294;\" x=\"188.011985303\" xlink:href=\"#C0_0_3af6fb4475\" y=\"164.327096653\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.924375336271;stroke:#000000;stroke-opacity:0.924375336271;\" x=\"119.844416652\" xlink:href=\"#C0_0_3af6fb4475\" y=\"105.878001984\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.341714972251;stroke:#000000;stroke-opacity:0.341714972251;\" x=\"168.033549961\" xlink:href=\"#C0_0_3af6fb4475\" y=\"151.654239286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.779420308738;stroke:#000000;stroke-opacity:0.779420308738;\" x=\"105.554906321\" xlink:href=\"#C0_0_3af6fb4475\" y=\"227.341266916\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.346757270883;stroke:#000000;stroke-opacity:0.346757270883;\" x=\"154.746731266\" xlink:href=\"#C0_0_3af6fb4475\" y=\"150.06145266\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.805646220295;stroke:#000000;stroke-opacity:0.805646220295;\" x=\"107.079579727\" xlink:href=\"#C0_0_3af6fb4475\" y=\"238.935215942\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.797761378145;stroke:#000000;stroke-opacity:0.797761378145;\" x=\"103.112356763\" xlink:href=\"#C0_0_3af6fb4475\" y=\"240.474476137\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.818010947291;stroke:#000000;stroke-opacity:0.818010947291;\" x=\"108.770809499\" xlink:href=\"#C0_0_3af6fb4475\" y=\"230.07659111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.81356051272;stroke:#000000;stroke-opacity:0.81356051272;\" x=\"102.030272845\" xlink:href=\"#C0_0_3af6fb4475\" y=\"235.749403737\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.799129751379;stroke:#000000;stroke-opacity:0.799129751379;\" x=\"108.941027731\" xlink:href=\"#C0_0_3af6fb4475\" y=\"247.460829546\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.77959095119;stroke:#000000;stroke-opacity:0.77959095119;\" x=\"108.210579097\" xlink:href=\"#C0_0_3af6fb4475\" y=\"238.016284062\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.825073309604;stroke:#000000;stroke-opacity:0.825073309604;\" x=\"112.271247767\" xlink:href=\"#C0_0_3af6fb4475\" y=\"237.795384281\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.79638452993;stroke:#000000;stroke-opacity:0.79638452993;\" x=\"104.939569767\" xlink:href=\"#C0_0_3af6fb4475\" y=\"234.31583506\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.747796605333;stroke:#000000;stroke-opacity:0.747796605333;\" x=\"113.900313261\" xlink:href=\"#C0_0_3af6fb4475\" y=\"233.912141677\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.807289989408;stroke:#000000;stroke-opacity:0.807289989408;\" x=\"119.802276351\" xlink:href=\"#C0_0_3af6fb4475\" y=\"222.13347159\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.627917358985;stroke:#000000;stroke-opacity:0.627917358985;\" x=\"226.833885397\" xlink:href=\"#C0_0_3af6fb4475\" y=\"163.376148562\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.552900211716;stroke:#000000;stroke-opacity:0.552900211716;\" x=\"211.972500827\" xlink:href=\"#C0_0_3af6fb4475\" y=\"158.941866753\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.804618348659;stroke:#000000;stroke-opacity:0.804618348659;\" x=\"125.857685735\" xlink:href=\"#C0_0_3af6fb4475\" y=\"205.145460004\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.820312786316;stroke:#000000;stroke-opacity:0.820312786316;\" x=\"169.488943052\" xlink:href=\"#C0_0_3af6fb4475\" y=\"180.894491033\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.890422518379;stroke:#000000;stroke-opacity:0.890422518379;\" x=\"241.995066607\" xlink:href=\"#C0_0_3af6fb4475\" y=\"181.048440532\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.464130220928;stroke:#000000;stroke-opacity:0.464130220928;\" x=\"168.12492074\" xlink:href=\"#C0_0_3af6fb4475\" y=\"146.852653235\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.866558811197;stroke:#000000;stroke-opacity:0.866558811197;\" x=\"160.448224021\" xlink:href=\"#C0_0_3af6fb4475\" y=\"183.63814469\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.885400590813;stroke:#000000;stroke-opacity:0.885400590813;\" x=\"201.127994773\" xlink:href=\"#C0_0_3af6fb4475\" y=\"177.712340628\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.69789135446;stroke:#000000;stroke-opacity:0.69789135446;\" x=\"147.688710443\" xlink:href=\"#C0_0_3af6fb4475\" y=\"112.602677448\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.692481087079;stroke:#000000;stroke-opacity:0.692481087079;\" x=\"192.449177298\" xlink:href=\"#C0_0_3af6fb4475\" y=\"148.771662901\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.60327454205;stroke:#000000;stroke-opacity:0.60327454205;\" x=\"223.427384142\" xlink:href=\"#C0_0_3af6fb4475\" y=\"170.474922265\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.404515926251;stroke:#000000;stroke-opacity:0.404515926251;\" x=\"158.911849769\" xlink:href=\"#C0_0_3af6fb4475\" y=\"177.92699605\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.910220623904;stroke:#000000;stroke-opacity:0.910220623904;\" x=\"118.654373475\" xlink:href=\"#C0_0_3af6fb4475\" y=\"98.2179791391\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.379861043582;stroke:#000000;stroke-opacity:0.379861043582;\" x=\"190.725003939\" xlink:href=\"#C0_0_3af6fb4475\" y=\"156.45575848\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.551926228318;stroke:#000000;stroke-opacity:0.551926228318;\" x=\"158.294189973\" xlink:href=\"#C0_0_3af6fb4475\" y=\"181.803344044\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.661864227865;stroke:#000000;stroke-opacity:0.661864227865;\" x=\"209.323913602\" xlink:href=\"#C0_0_3af6fb4475\" y=\"142.523143266\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.337308015776;stroke:#000000;stroke-opacity:0.337308015776;\" x=\"176.001715831\" xlink:href=\"#C0_0_3af6fb4475\" y=\"157.920582439\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.775863403105;stroke:#000000;stroke-opacity:0.775863403105;\" x=\"105.086928203\" xlink:href=\"#C0_0_3af6fb4475\" y=\"242.616261778\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.614476481328;stroke:#000000;stroke-opacity:0.614476481328;\" x=\"180.592305569\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.49952955\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.495349317324;stroke:#000000;stroke-opacity:0.495349317324;\" x=\"203.462153279\" xlink:href=\"#C0_0_3af6fb4475\" y=\"164.326219973\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.349885521165;stroke:#000000;stroke-opacity:0.349885521165;\" x=\"160.219511418\" xlink:href=\"#C0_0_3af6fb4475\" y=\"163.635373288\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.763678085925;stroke:#000000;stroke-opacity:0.763678085925;\" x=\"178.633114357\" xlink:href=\"#C0_0_3af6fb4475\" y=\"151.351861245\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.619006842581;stroke:#000000;stroke-opacity:0.619006842581;\" x=\"213.042887956\" xlink:href=\"#C0_0_3af6fb4475\" y=\"153.844503774\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.45974162992;stroke:#000000;stroke-opacity:0.45974162992;\" x=\"179.905830854\" xlink:href=\"#C0_0_3af6fb4475\" y=\"137.935579561\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.774192644479;stroke:#000000;stroke-opacity:0.774192644479;\" x=\"174.888532037\" xlink:href=\"#C0_0_3af6fb4475\" y=\"134.625448482\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.414311226498;stroke:#000000;stroke-opacity:0.414311226498;\" x=\"181.476845447\" xlink:href=\"#C0_0_3af6fb4475\" y=\"141.016157776\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.685223549145;stroke:#000000;stroke-opacity:0.685223549145;\" x=\"203.639204472\" xlink:href=\"#C0_0_3af6fb4475\" y=\"167.737717644\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.634731085009;stroke:#000000;stroke-opacity:0.634731085009;\" x=\"226.276892707\" xlink:href=\"#C0_0_3af6fb4475\" y=\"176.101295747\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.38986748539;stroke:#000000;stroke-opacity:0.38986748539;\" x=\"192.448493709\" xlink:href=\"#C0_0_3af6fb4475\" y=\"159.497998231\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.774085888368;stroke:#000000;stroke-opacity:0.774085888368;\" x=\"109.421430644\" xlink:href=\"#C0_0_3af6fb4475\" y=\"235.195843406\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.461657566523;stroke:#000000;stroke-opacity:0.461657566523;\" x=\"175.41322567\" xlink:href=\"#C0_0_3af6fb4475\" y=\"129.802049319\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.930988711968;stroke:#000000;stroke-opacity:0.930988711968;\" x=\"109.083147469\" xlink:href=\"#C0_0_3af6fb4475\" y=\"106.68038316\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.595337505006;stroke:#000000;stroke-opacity:0.595337505006;\" x=\"159.806946239\" xlink:href=\"#C0_0_3af6fb4475\" y=\"129.688126693\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.984242722231;stroke:#000000;stroke-opacity:0.984242722231;\" x=\"110.615059607\" xlink:href=\"#C0_0_3af6fb4475\" y=\"104.046834081\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.966332290434;stroke:#000000;stroke-opacity:0.966332290434;\" x=\"107.990453944\" xlink:href=\"#C0_0_3af6fb4475\" y=\"103.171085387\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.93230427374;stroke:#000000;stroke-opacity:0.93230427374;\" x=\"108.950851616\" xlink:href=\"#C0_0_3af6fb4475\" y=\"106.443135758\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.930596366763;stroke:#000000;stroke-opacity:0.930596366763;\" x=\"110.596038256\" xlink:href=\"#C0_0_3af6fb4475\" y=\"102.530304905\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.964160357298;stroke:#000000;stroke-opacity:0.964160357298;\" x=\"105.512812103\" xlink:href=\"#C0_0_3af6fb4475\" y=\"102.936189709\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.949259991976;stroke:#000000;stroke-opacity:0.949259991976;\" x=\"107.74308953\" xlink:href=\"#C0_0_3af6fb4475\" y=\"103.961983955\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.919195288063;stroke:#000000;stroke-opacity:0.919195288063;\" x=\"110.416366922\" xlink:href=\"#C0_0_3af6fb4475\" y=\"103.262437171\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.879597527766;stroke:#000000;stroke-opacity:0.879597527766;\" x=\"119.917659505\" xlink:href=\"#C0_0_3af6fb4475\" y=\"100.801448756\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.988137895465;stroke:#000000;stroke-opacity:0.988137895465;\" x=\"113.973909881\" xlink:href=\"#C0_0_3af6fb4475\" y=\"110.190875086\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.901188981263;stroke:#000000;stroke-opacity:0.901188981263;\" x=\"120.355068097\" xlink:href=\"#C0_0_3af6fb4475\" y=\"100.83880029\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.731193367139;stroke:#000000;stroke-opacity:0.731193367139;\" x=\"232.778572332\" xlink:href=\"#C0_0_3af6fb4475\" y=\"175.154986042\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.622475779359;stroke:#000000;stroke-opacity:0.622475779359;\" x=\"219.038700482\" xlink:href=\"#C0_0_3af6fb4475\" y=\"172.568019277\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.80369116037;stroke:#000000;stroke-opacity:0.80369116037;\" x=\"201.767169089\" xlink:href=\"#C0_0_3af6fb4475\" y=\"159.35875441\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.805286157061;stroke:#000000;stroke-opacity:0.805286157061;\" x=\"103.919014178\" xlink:href=\"#C0_0_3af6fb4475\" y=\"246.702034302\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.552293473364;stroke:#000000;stroke-opacity:0.552293473364;\" x=\"178.620115421\" xlink:href=\"#C0_0_3af6fb4475\" y=\"157.271754726\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.464449983831;stroke:#000000;stroke-opacity:0.464449983831;\" x=\"193.900471659\" xlink:href=\"#C0_0_3af6fb4475\" y=\"156.376442407\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.467129123907;stroke:#000000;stroke-opacity:0.467129123907;\" x=\"213.055642751\" xlink:href=\"#C0_0_3af6fb4475\" y=\"147.88013971\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.917491242946;stroke:#000000;stroke-opacity:0.917491242946;\" x=\"226.557151717\" xlink:href=\"#C0_0_3af6fb4475\" y=\"176.272665692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.487891571178;stroke:#000000;stroke-opacity:0.487891571178;\" x=\"180.457663238\" xlink:href=\"#C0_0_3af6fb4475\" y=\"136.081183528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.589666408278;stroke:#000000;stroke-opacity:0.589666408278;\" x=\"188.429381027\" xlink:href=\"#C0_0_3af6fb4475\" y=\"130.860172921\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.958185069944;stroke:#000000;stroke-opacity:0.958185069944;\" x=\"121.061388271\" xlink:href=\"#C0_0_3af6fb4475\" y=\"109.328231128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.703906663578;stroke:#000000;stroke-opacity:0.703906663578;\" x=\"213.838478191\" xlink:href=\"#C0_0_3af6fb4475\" y=\"155.395266172\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.946142686395;stroke:#000000;stroke-opacity:0.946142686395;\" x=\"108.311019272\" xlink:href=\"#C0_0_3af6fb4475\" y=\"100.843032115\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.569647873261;stroke:#000000;stroke-opacity:0.569647873261;\" x=\"227.737729551\" xlink:href=\"#C0_0_3af6fb4475\" y=\"159.007131132\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.612686984173;stroke:#000000;stroke-opacity:0.612686984173;\" x=\"215.186433941\" xlink:href=\"#C0_0_3af6fb4475\" y=\"152.464394637\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.668359175335;stroke:#000000;stroke-opacity:0.668359175335;\" x=\"202.027697833\" xlink:href=\"#C0_0_3af6fb4475\" y=\"142.537754627\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.804282354571;stroke:#000000;stroke-opacity:0.804282354571;\" x=\"238.7711934\" xlink:href=\"#C0_0_3af6fb4475\" y=\"174.621583545\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.675972860164;stroke:#000000;stroke-opacity:0.675972860164;\" x=\"228.920412054\" xlink:href=\"#C0_0_3af6fb4475\" y=\"164.299363262\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.794442202765;stroke:#000000;stroke-opacity:0.794442202765;\" x=\"234.655117782\" xlink:href=\"#C0_0_3af6fb4475\" y=\"164.88675862\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.48474984343;stroke:#000000;stroke-opacity:0.48474984343;\" x=\"187.336790459\" xlink:href=\"#C0_0_3af6fb4475\" y=\"138.696225843\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.670317255917;stroke:#000000;stroke-opacity:0.670317255917;\" x=\"216.485276974\" xlink:href=\"#C0_0_3af6fb4475\" y=\"170.741242614\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.696265131677;stroke:#000000;stroke-opacity:0.696265131677;\" x=\"227.063490838\" xlink:href=\"#C0_0_3af6fb4475\" y=\"162.055868558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.696618358912;stroke:#000000;stroke-opacity:0.696618358912;\" x=\"214.382804477\" xlink:href=\"#C0_0_3af6fb4475\" y=\"157.802433176\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.888292451181;stroke:#000000;stroke-opacity:0.888292451181;\" x=\"239.681267353\" xlink:href=\"#C0_0_3af6fb4475\" y=\"177.656788971\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.579647867949;stroke:#000000;stroke-opacity:0.579647867949;\" x=\"200.178051513\" xlink:href=\"#C0_0_3af6fb4475\" y=\"149.374173956\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.701301751547;stroke:#000000;stroke-opacity:0.701301751547;\" x=\"210.629230403\" xlink:href=\"#C0_0_3af6fb4475\" y=\"142.638645386\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.506601931701;stroke:#000000;stroke-opacity:0.506601931701;\" x=\"220.178393954\" xlink:href=\"#C0_0_3af6fb4475\" y=\"159.706043417\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.671642121824;stroke:#000000;stroke-opacity:0.671642121824;\" x=\"181.479295965\" xlink:href=\"#C0_0_3af6fb4475\" y=\"130.838972235\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.642652964519;stroke:#000000;stroke-opacity:0.642652964519;\" x=\"206.758887586\" xlink:href=\"#C0_0_3af6fb4475\" y=\"170.792060264\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.63582977751;stroke:#000000;stroke-opacity:0.63582977751;\" x=\"229.290368554\" xlink:href=\"#C0_0_3af6fb4475\" y=\"164.30978193\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.608498121162;stroke:#000000;stroke-opacity:0.608498121162;\" x=\"185.755762794\" xlink:href=\"#C0_0_3af6fb4475\" y=\"141.123461393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.719939839089;stroke:#000000;stroke-opacity:0.719939839089;\" x=\"248.663219272\" xlink:href=\"#C0_0_3af6fb4475\" y=\"170.931611245\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.775256127325;stroke:#000000;stroke-opacity:0.775256127325;\" x=\"226.062156413\" xlink:href=\"#C0_0_3af6fb4475\" y=\"167.744539756\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.62040980504;stroke:#000000;stroke-opacity:0.62040980504;\" x=\"202.961347757\" xlink:href=\"#C0_0_3af6fb4475\" y=\"151.118804162\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.728868964547;stroke:#000000;stroke-opacity:0.728868964547;\" x=\"209.35789789\" xlink:href=\"#C0_0_3af6fb4475\" y=\"160.269465271\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.776372076461;stroke:#000000;stroke-opacity:0.776372076461;\" x=\"217.985953188\" xlink:href=\"#C0_0_3af6fb4475\" y=\"150.074410171\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.823459529127;stroke:#000000;stroke-opacity:0.823459529127;\" x=\"203.465802688\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.833134742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.46748012017;stroke:#000000;stroke-opacity:0.46748012017;\" x=\"171.457735921\" xlink:href=\"#C0_0_3af6fb4475\" y=\"136.395853227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.5651792709;stroke:#000000;stroke-opacity:0.5651792709;\" x=\"222.818365757\" xlink:href=\"#C0_0_3af6fb4475\" y=\"163.106899864\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.636353630085;stroke:#000000;stroke-opacity:0.636353630085;\" x=\"236.398624453\" xlink:href=\"#C0_0_3af6fb4475\" y=\"166.023583375\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.763407404232;stroke:#000000;stroke-opacity:0.763407404232;\" x=\"225.350655017\" xlink:href=\"#C0_0_3af6fb4475\" y=\"169.165116602\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.625803559685;stroke:#000000;stroke-opacity:0.625803559685;\" x=\"223.828378572\" xlink:href=\"#C0_0_3af6fb4475\" y=\"166.172637022\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.948444985438;stroke:#000000;stroke-opacity:0.948444985438;\" x=\"202.131917324\" xlink:href=\"#C0_0_3af6fb4475\" y=\"170.81675037\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.949511394278;stroke:#000000;stroke-opacity:0.949511394278;\" x=\"226.852731896\" xlink:href=\"#C0_0_3af6fb4475\" y=\"183.339496655\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.983930451914;stroke:#000000;stroke-opacity:0.983930451914;\" x=\"235.731484093\" xlink:href=\"#C0_0_3af6fb4475\" y=\"192.486045183\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.783755762814;stroke:#000000;stroke-opacity:0.783755762814;\" x=\"185.332131348\" xlink:href=\"#C0_0_3af6fb4475\" y=\"158.813074742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.533858562528;stroke:#000000;stroke-opacity:0.533858562528;\" x=\"183.016816487\" xlink:href=\"#C0_0_3af6fb4475\" y=\"139.954392877\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.825095414682;stroke:#000000;stroke-opacity:0.825095414682;\" x=\"226.361765281\" xlink:href=\"#C0_0_3af6fb4475\" y=\"158.06051986\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.553082828144;stroke:#000000;stroke-opacity:0.553082828144;\" x=\"195.278382167\" xlink:href=\"#C0_0_3af6fb4475\" y=\"158.133498991\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.606260521926;stroke:#000000;stroke-opacity:0.606260521926;\" x=\"176.621067967\" xlink:href=\"#C0_0_3af6fb4475\" y=\"180.804925896\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.936439320971;stroke:#000000;stroke-opacity:0.936439320971;\" x=\"222.151547529\" xlink:href=\"#C0_0_3af6fb4475\" y=\"187.70942809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.835489984221;stroke:#000000;stroke-opacity:0.835489984221;\" x=\"228.209532012\" xlink:href=\"#C0_0_3af6fb4475\" y=\"167.471102531\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.857963118381;stroke:#000000;stroke-opacity:0.857963118381;\" x=\"183.146141465\" xlink:href=\"#C0_0_3af6fb4475\" y=\"176.609059221\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.639038714427;stroke:#000000;stroke-opacity:0.639038714427;\" x=\"177.127582802\" xlink:href=\"#C0_0_3af6fb4475\" y=\"151.686110634\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.867698769931;stroke:#000000;stroke-opacity:0.867698769931;\" x=\"238.472029651\" xlink:href=\"#C0_0_3af6fb4475\" y=\"173.015276244\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.59852923519;stroke:#000000;stroke-opacity:0.59852923519;\" x=\"230.410359989\" xlink:href=\"#C0_0_3af6fb4475\" y=\"169.433280725\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.781208986649;stroke:#000000;stroke-opacity:0.781208986649;\" x=\"221.249084252\" xlink:href=\"#C0_0_3af6fb4475\" y=\"162.910806143\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.957986130633;stroke:#000000;stroke-opacity:0.957986130633;\" x=\"196.465379085\" xlink:href=\"#C0_0_3af6fb4475\" y=\"194.73321031\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.782673928793;stroke:#000000;stroke-opacity:0.782673928793;\" x=\"198.472924038\" xlink:href=\"#C0_0_3af6fb4475\" y=\"163.062815175\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.955175915704;stroke:#000000;stroke-opacity:0.955175915704;\" x=\"196.719864066\" xlink:href=\"#C0_0_3af6fb4475\" y=\"181.667161172\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.981833578137;stroke:#000000;stroke-opacity:0.981833578137;\" x=\"184.56019717\" xlink:href=\"#C0_0_3af6fb4475\" y=\"179.511038825\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.730107848967;stroke:#000000;stroke-opacity:0.730107848967;\" x=\"184.16702911\" xlink:href=\"#C0_0_3af6fb4475\" y=\"155.62977856\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.946904803498;stroke:#000000;stroke-opacity:0.946904803498;\" x=\"205.816399399\" xlink:href=\"#C0_0_3af6fb4475\" y=\"195.140432851\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.890030015296;stroke:#000000;stroke-opacity:0.890030015296;\" x=\"196.33871082\" xlink:href=\"#C0_0_3af6fb4475\" y=\"171.77537141\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.928370753744;stroke:#000000;stroke-opacity:0.928370753744;\" x=\"189.192724621\" xlink:href=\"#C0_0_3af6fb4475\" y=\"199.634372206\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.940498946913;stroke:#000000;stroke-opacity:0.940498946913;\" x=\"192.779401706\" xlink:href=\"#C0_0_3af6fb4475\" y=\"200.564080231\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.93320706073;stroke:#000000;stroke-opacity:0.93320706073;\" x=\"182.233951271\" xlink:href=\"#C0_0_3af6fb4475\" y=\"147.744869915\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.907538054206;stroke:#000000;stroke-opacity:0.907538054206;\" x=\"197.020967395\" xlink:href=\"#C0_0_3af6fb4475\" y=\"196.71967208\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.939585839277;stroke:#000000;stroke-opacity:0.939585839277;\" x=\"192.148977334\" xlink:href=\"#C0_0_3af6fb4475\" y=\"200.493244625\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.918327066184;stroke:#000000;stroke-opacity:0.918327066184;\" x=\"130.525957143\" xlink:href=\"#C0_0_3af6fb4475\" y=\"104.001438939\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.982785218038;stroke:#000000;stroke-opacity:0.982785218038;\" x=\"121.15357209\" xlink:href=\"#C0_0_3af6fb4475\" y=\"106.921573828\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.785473955568;stroke:#000000;stroke-opacity:0.785473955568;\" x=\"146.268035117\" xlink:href=\"#C0_0_3af6fb4475\" y=\"110.537902282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.397545779387;stroke:#000000;stroke-opacity:0.397545779387;\" x=\"182.446840301\" xlink:href=\"#C0_0_3af6fb4475\" y=\"144.795361399\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.987540221797;stroke:#000000;stroke-opacity:0.987540221797;\" x=\"126.133944619\" xlink:href=\"#C0_0_3af6fb4475\" y=\"116.897300484\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.984815829782;stroke:#000000;stroke-opacity:0.984815829782;\" x=\"123.565366744\" xlink:href=\"#C0_0_3af6fb4475\" y=\"115.042980615\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.508514931884;stroke:#000000;stroke-opacity:0.508514931884;\" x=\"193.563630107\" xlink:href=\"#C0_0_3af6fb4475\" y=\"142.364927426\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.882387894084;stroke:#000000;stroke-opacity:0.882387894084;\" x=\"175.926394643\" xlink:href=\"#C0_0_3af6fb4475\" y=\"167.078168348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.718021972041;stroke:#000000;stroke-opacity:0.718021972041;\" x=\"232.987906504\" xlink:href=\"#C0_0_3af6fb4475\" y=\"166.846139002\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.680403179319;stroke:#000000;stroke-opacity:0.680403179319;\" x=\"212.562247989\" xlink:href=\"#C0_0_3af6fb4475\" y=\"149.548389209\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.876979689275;stroke:#000000;stroke-opacity:0.876979689275;\" x=\"220.215595625\" xlink:href=\"#C0_0_3af6fb4475\" y=\"176.291083761\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.933268673604;stroke:#000000;stroke-opacity:0.933268673604;\" x=\"226.737083537\" xlink:href=\"#C0_0_3af6fb4475\" y=\"187.08558707\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.571828542166;stroke:#000000;stroke-opacity:0.571828542166;\" x=\"188.139812968\" xlink:href=\"#C0_0_3af6fb4475\" y=\"166.802244439\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.634416160404;stroke:#000000;stroke-opacity:0.634416160404;\" x=\"234.419854736\" xlink:href=\"#C0_0_3af6fb4475\" y=\"164.407007004\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.578977791386;stroke:#000000;stroke-opacity:0.578977791386;\" x=\"232.749156911\" xlink:href=\"#C0_0_3af6fb4475\" y=\"168.207457715\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.676770239104;stroke:#000000;stroke-opacity:0.676770239104;\" x=\"222.899993204\" xlink:href=\"#C0_0_3af6fb4475\" y=\"168.203010208\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.740418696758;stroke:#000000;stroke-opacity:0.740418696758;\" x=\"239.478519574\" xlink:href=\"#C0_0_3af6fb4475\" y=\"172.376763161\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.599481271305;stroke:#000000;stroke-opacity:0.599481271305;\" x=\"214.050281841\" xlink:href=\"#C0_0_3af6fb4475\" y=\"163.902685721\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.910214155824;stroke:#000000;stroke-opacity:0.910214155824;\" x=\"189.525657635\" xlink:href=\"#C0_0_3af6fb4475\" y=\"170.062124098\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.977055087719;stroke:#000000;stroke-opacity:0.977055087719;\" x=\"235.813720382\" xlink:href=\"#C0_0_3af6fb4475\" y=\"191.832526844\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.810881498848;stroke:#000000;stroke-opacity:0.810881498848;\" x=\"220.957163693\" xlink:href=\"#C0_0_3af6fb4475\" y=\"171.03408013\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.873237109346;stroke:#000000;stroke-opacity:0.873237109346;\" x=\"237.759764811\" xlink:href=\"#C0_0_3af6fb4475\" y=\"171.078343108\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.726694586268;stroke:#000000;stroke-opacity:0.726694586268;\" x=\"248.946384811\" xlink:href=\"#C0_0_3af6fb4475\" y=\"174.68911246\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.697036429899;stroke:#000000;stroke-opacity:0.697036429899;\" x=\"247.718836442\" xlink:href=\"#C0_0_3af6fb4475\" y=\"172.111094818\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.876694882601;stroke:#000000;stroke-opacity:0.876694882601;\" x=\"191.895130278\" xlink:href=\"#C0_0_3af6fb4475\" y=\"146.848412469\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.879012012412;stroke:#000000;stroke-opacity:0.879012012412;\" x=\"240.330993218\" xlink:href=\"#C0_0_3af6fb4475\" y=\"172.466011003\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;stroke:#000000;\" x=\"236.70477577\" xlink:href=\"#C0_0_3af6fb4475\" y=\"188.785649661\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.81967590981;stroke:#000000;stroke-opacity:0.81967590981;\" x=\"222.074597426\" xlink:href=\"#C0_0_3af6fb4475\" y=\"175.123987977\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.842579398884;stroke:#000000;stroke-opacity:0.842579398884;\" x=\"239.58471865\" xlink:href=\"#C0_0_3af6fb4475\" y=\"176.366104986\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.974844830044;stroke:#000000;stroke-opacity:0.974844830044;\" x=\"238.171310206\" xlink:href=\"#C0_0_3af6fb4475\" y=\"189.655426464\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.99156914704;stroke:#000000;stroke-opacity:0.99156914704;\" x=\"236.962116265\" xlink:href=\"#C0_0_3af6fb4475\" y=\"192.838149353\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.947828701935;stroke:#000000;stroke-opacity:0.947828701935;\" x=\"222.496938301\" xlink:href=\"#C0_0_3af6fb4475\" y=\"175.987847341\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p1650f2cb30)\">\n",
       "     <use style=\"fill:#0000ff;fill-opacity:0.929141309754;stroke:#000000;stroke-opacity:0.929141309754;\" x=\"241.959698125\" xlink:href=\"#C0_0_3af6fb4475\" y=\"187.310996736\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p1650f2cb30\">\n",
       "   <rect height=\"324.0\" width=\"345.6\" x=\"7.2\" y=\"7.2\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<skbio._base.OrdinationResults at 0x10ca97f60>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "coordinates"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<div style='height:500px'></div>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "outputs": [],
   "source": [
    "from emperor import Emperor"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<script type=\"text/javascript\">\n",
       "\n",
       "if ($(\"#emperor-css\").length == 0){{\n",
       "    $(\"head\").append([\n",
       "\n",
       "        '<link id=\"emperor-css\" rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/css/emperor.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/jquery-ui.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/slick.grid.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/spectrum.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/chosen.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/jquery.contextMenu.min.css\">'\n",
       "    ]);\n",
       "}}\n",
       "</script>\n",
       "\n",
       "<div id='emperor-notebook-0x69a3643a' style=\"position: relative; width:100%; height:500px;\">\n",
       "  <div class='loading' style=\"position: absolute;top: 50%;left: 50%;margin-left: -229px; margin-top: -59px; z-index: 10000;height:118px;width:458px;padding:0px\"><img src='/nbextensions/emperor/support_files/img/emperor.png' alt='Emperor resources missing. Expected them to be found in /nbextensions/emperor/support_files'></div>\n",
       "</div>\n",
       "</div>\n",
       "\n",
       "<script type=\"text/javascript\">\n",
       "requirejs.config({\n",
       "// the left side is the module name, and the right side is the path\n",
       "// relative to the baseUrl attribute, do NOT include the .js extension\n",
       "'paths': {\n",
       "  /* jQuery */\n",
       "  'jquery': '/nbextensions/emperor/support_files/vendor/js/jquery-2.1.4.min',\n",
       "  'jqueryui': '/nbextensions/emperor/support_files/vendor/js/jquery-ui.min',\n",
       "  'jquery_drag': '/nbextensions/emperor/support_files/vendor/js/jquery.event.drag-2.2.min',\n",
       "\n",
       "  /* jQuery plugins */\n",
       "  'chosen': '/nbextensions/emperor/support_files/vendor/js/chosen.jquery.min',\n",
       "  'spectrum': '/nbextensions/emperor/support_files/vendor/js/spectrum.min',\n",
       "  'position': '/nbextensions/emperor/support_files/vendor/js/jquery.ui.position.min',\n",
       "  'contextmenu': '/nbextensions/emperor/support_files/vendor/js/jquery.contextMenu.min',\n",
       "\n",
       "  /* other libraries */\n",
       "  'underscore': '/nbextensions/emperor/support_files/vendor/js/underscore-min',\n",
       "  'chroma': '/nbextensions/emperor/support_files/vendor/js/chroma.min',\n",
       "  'filesaver': '/nbextensions/emperor/support_files/vendor/js/FileSaver.min',\n",
       "  'blob': '/nbextensions/emperor/support_files/vendor/js/Blob',\n",
       "  'd3': '/nbextensions/emperor/support_files/vendor/js/d3.min',\n",
       "\n",
       "\n",
       "  /* THREE.js and plugins */\n",
       "  'three': '/nbextensions/emperor/support_files/vendor/js/three.min',\n",
       "  'orbitcontrols': '/nbextensions/emperor/support_files/vendor/js/three.js-plugins/OrbitControls',\n",
       "\n",
       "  /* SlickGrid */\n",
       "  'slickcore': '/nbextensions/emperor/support_files/vendor/js/slick.core.min',\n",
       "  'slickgrid': '/nbextensions/emperor/support_files/vendor/js/slick.grid.min',\n",
       "  'slickformatters': '/nbextensions/emperor/support_files/vendor/js/slick.editors.min',\n",
       "  'slickeditors': '/nbextensions/emperor/support_files/vendor/js/slick.formatters.min',\n",
       "\n",
       "  /* Emperor's objects */\n",
       "  'util': '/nbextensions/emperor/support_files/js/util',\n",
       "  'model': '/nbextensions/emperor/support_files/js/model',\n",
       "  'view': '/nbextensions/emperor/support_files/js/view',\n",
       "  'controller': '/nbextensions/emperor/support_files/js/controller',\n",
       "  'draw': '/nbextensions/emperor/support_files/js/draw',\n",
       "  'scene3d': '/nbextensions/emperor/support_files/js/sceneplotview3d',\n",
       "  'viewcontroller': '/nbextensions/emperor/support_files/js/view-controller',\n",
       "  'colorviewcontroller': '/nbextensions/emperor/support_files/js/color-view-controller',\n",
       "  'visibilitycontroller': '/nbextensions/emperor/support_files/js/visibility-controller',\n",
       "  'scaleviewcontroller': '/nbextensions/emperor/support_files/js/scale-view-controller',\n",
       "  'shapecontroller': '/nbextensions/emperor/support_files/js/shape-controller',\n",
       "  'axescontroller': '/nbextensions/emperor/support_files/js/axes-controller',\n",
       "  'shape-editor': '/nbextensions/emperor/support_files/js/shape-editor',\n",
       "  'color-editor': '/nbextensions/emperor/support_files/js/color-editor',\n",
       "  'scale-editor': '/nbextensions/emperor/support_files/js/scale-editor',\n",
       "  'shapes': '/nbextensions/emperor/support_files/js/shapes'\n",
       "},\n",
       "/*\n",
       "   Libraries that are not AMD compatible need shim to declare their\n",
       "   dependencies.\n",
       " */\n",
       "'shim': {\n",
       "  'jquery_drag': {\n",
       "    'deps': ['jquery', 'jqueryui']\n",
       "  },\n",
       "  'chosen': {\n",
       "    'deps': ['jquery'],\n",
       "    'exports': 'jQuery.fn.chosen'\n",
       "  },\n",
       "  'contextmenu' : {\n",
       "    'deps': ['jquery', 'jqueryui', 'position']\n",
       "  },\n",
       "  'filesaver' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'orbitcontrols': {\n",
       "    'deps': ['three']\n",
       "  },\n",
       "'slickcore': ['jqueryui'],\n",
       "'slickgrid': ['slickcore', 'jquery_drag', 'slickformatters',\n",
       "              'slickeditors']\n",
       "}\n",
       "});\n",
       "\n",
       "requirejs(\n",
       "[\"jquery\", \"model\", \"controller\"],\n",
       "function($, model, EmperorController) {\n",
       "  var DecompositionModel = model.DecompositionModel;\n",
       "\n",
       "  var div = $('#emperor-notebook-0x69a3643a');\n",
       "\n",
       "  var ids = ['449.F21Plml', '449.M22Pinr', '449.M11Glns', '449.F33Fotr', '449.M54Plml', '449.F32Fotl', '449.M22Plml', '449.M24Fotl', '449.M21Glns', '449.M21Plmr', '449.M23Knee', '449.M43Plmr', '449.M63Plmr', '449.M64Plml', '449.F23Plml', '449.M44Plml', '449.F23Plmr', '449.M23Nost', '449.M54Plmr', '449.M53Plml', '449.M44Plmr', '449.M43Plml', '449.M63Plml', '449.M53Fotr', '449.F14Plmr', '449.F24Plmr', '449.M31Plmr', '449.F22Knel', '449.M31Knel', '449.F22Mout', '449.F21Navl', '449.F22Plmr', '449.F12Plmr', '449.F22Indr', '449.M31Plml', '449.M31Indr', '449.F12Indl', '449.F12Plml', '449.F21Fotl', '449.M32Mout', '449.F12Pinl', '449.F12Nstl', '449.M21Indr', '449.M21Mout', '449.M42Indl', '449.M21Indl', '449.F32Plmr', '449.F31Mout', '449.M42Frhd', '449.M21Nstr', '449.M21Plml', '449.F32Nstl', '449.M21Forl', '449.M42Plml', '449.F32Indr', '449.F32Nstr', '449.F32Forl', '449.F32Mout', '449.M22Forl', '449.F32Indl', '449.M22Indr', '449.F32Plml', '449.M21Forr', '449.M22Mout', '449.M42Mout', '449.M53Fotl', '449.F33Plml', '449.F31Fotr', '449.F32Nose', '449.M53Fcsw', '449.M14Fcsw', '449.M33Fcsw', '449.M13Fcsw', '449.M63Fcsw', '449.M24Fcsw', '449.M54Fcsw', '449.F21Fcsw', '449.F22Fcsw', '449.M31Fcsw', '449.M22Nstl', '449.M22Indl', '449.M21Fcsp', '449.F32Fcsw', '449.M41Fcsw', '449.M22Fcsw', '449.M42Fcsw', '449.M34Tong', '449.M44Urin', '449.M53Tong', '449.M43Frhd', '449.M54Tong', '449.F23Tong', '449.M53Uric', '449.M54Uric', '449.M23Ewax', '449.F23Knee', '449.F34Fotr', '449.M64Fotl', '449.M14Tong', '449.M44Knee', '449.F24Frhd', '449.M44Uric', '449.F23Nost', '449.M33Nost', '449.F24Nost', '449.M13Plml', '449.F21Indl', '449.M23Plml', '449.M54Fotr', '449.F21Tong', '449.F21Mout', '449.F21Hair', '449.F21Forr', '449.M31Mout', '449.F21Forl', '449.F12Indr', '449.M32Ewxl', '449.M31Fotr', '449.M31Tong', '449.M32Tong', '449.F22Forl', '449.M32Nose', '449.M32Plmr', '449.M31Hair', '449.F12Aptl', '449.F22Indl', '449.F21Nstl', '449.F21Plmr', '449.M32Pinl', '449.F22Tong', '449.M32Hair', '449.F12Mout', '449.F21Aptr', '449.F22Plml', '449.F22Hair', '449.M41Urin', '449.M41Tong', '449.M42Indr', '449.M42Tong', '449.M41Nose', '449.M42Urin', '449.M42Plmr', '449.M22Nose', '449.M22Nstr', '449.M21Ewxr', '449.M41Plml', '449.M42Hair', '449.M21Pinl', '449.M21Knel', '449.M21Frhd', '449.F31Ewxr', '449.M42Nose', '449.M42Pinl', '449.M41Navl', '449.M21Navl', '449.M41Mout', '449.M22Forr', '449.F31Indl', '449.M41Forr', '449.M22Hair', '449.M22Ewxl', '449.M41Kner', '449.M41Pinl', '449.F31Plmr', '449.M42Forl', '449.M21Nstl', '449.M41Frhd', '449.M22Plmr', '449.F33Plmr', '449.M23Fotr', '449.F33Fotl', '449.F34Fotl', '449.M11Fotl', '449.M22Fotr', '449.M42Knel', '449.M21Fotl', '449.M21Fotr', '449.F31Indr', '449.F14Fotr', '449.F13Fotr', '449.F12Fotl', '449.M42Glns', '449.M64Plmr', '449.M32Fotr', '449.F34Plmr', '449.M23Frhd', '449.F13Plml', '449.M43Nost', '449.M64Knee', '449.F33Nost', '449.F23Frhd', '449.F13Knee', '449.M44Frhd', '449.F34Frhd', '449.F14Frhd', '449.M44Ewax', '449.M63Frhd', '449.M24Nost', '449.F34Nost', '449.M24Knee', '449.M64Frhd', '449.F11Pinr', '449.M44Nost', '449.F13Ewax', '449.M31Pinl', '449.M32Forl', '449.F12Nose', '449.F12Pinr', '449.M31Pinr', '449.F22Pinl', '449.F21Knel', '449.M32Frhd', '449.F22Ewxr', '449.F12Navl', '449.F32Hair', '449.F31Forr', '449.M22Ewxr', '449.F31Knel', '449.M22Navl', '449.F31Frhd', '449.M22Fotl', '449.M21Aptr', '449.F32Forr', '449.M22Frhd', '449.M41Plmr', '449.M22Pinl', '449.M41Pinr', '449.F31Fotl', '449.M21Pinr', '449.M32Indr', '449.F13Tong', '449.M54Urin', '449.M23Fcsw', '449.F24Fcsw', '449.F23Fcsw', '449.M21Fcsw', '449.M22Fcsp', '449.M11Fcsp', '449.M32Nstl', '449.F31Labi', '449.F32Labi', '449.M34Plml', '449.M53Knee', '449.M64Fotr', '449.M14Fotr', '449.F24Plml', '449.M32Forr', '449.M12Ewxr', '449.M32Plml', '449.F32Kner', '449.M31Indl', '449.M31Nose', '449.M32Indl', '449.M21Ewxl', '449.M22Tong', '449.F31Forl', '449.F13Fcsw', '449.M41Knel', '449.F33Fcsw', '449.F34Fcsw', '449.M32Fcsp', '449.F22Fcsp', '449.F32Fcsp', '449.F31Fcsw', '449.M64Fcsw', '449.F31Fcsp', '449.M44Fcsw', '449.M41Fcsp', '449.F21Nstr', '449.M44Fotl', '449.F22Labi', '449.F21Labi', '449.F21Ewxr', '449.F21Kner', '449.F22Urin', '449.F21Urin', '449.M21Urin', '449.M22Urin', '449.M13Frhd', '449.M54Knee', '449.F33Tong', '449.M43Fotr', '449.M14Knee', '449.M42Pinr', '449.F32Fotr', '449.M43Fcsw', '449.M33Frhd', '449.M14Fotl', '449.M43Knee', '449.F12Labi', '449.F22Nose', '449.M12Frhd', '449.M41Nstl', '449.F32Navl', '449.M14Nost', '449.M14Frhd', '449.M12Fotl', '449.F14Fcsw', '449.M32Nstr', '449.M33Tong', '449.M53Urin', '449.M23Tong', '449.M43Tong', '449.M64Tong', '449.M13Tong', '449.M63Tong', '449.F24Tong', '449.M12Tong', '449.M11Tong', '449.M21Tong', '449.F31Tong', '449.F24Fotr', '449.F12Fotr', '449.M42Ewxr', '449.M32Fcsw', '449.M22Aptr', '449.M14Plml', '449.M24Frhd', '449.M34Ewax', '449.F32Frhd', '449.F12Nstr', '449.F14Tong', '449.M64Nost', '449.M44Tong', '449.F22Aptr', '449.M54Nost', '449.M63Nost', '449.M63Ewax', '449.M53Nost', '449.M64Ewax', '449.M34Nost', '449.M13Nost', '449.F13Nost', '449.M34Frhd', '449.M53Ewax', '449.M54Frhd', '449.F22Nstl', '449.F21Aptl', '449.F22Nstr', '449.M31Aptr', '449.M12Nstr', '449.M31Nstr', '449.M32Aptr', '449.M12Nose', '449.F11Nstr', '449.F11Navl', '449.F21Nose', '449.M31Frhd', '449.F32Aptr', '449.M42Aptl', '449.M42Aptr', '449.M41Aptl', '449.M41Aptr', '449.M32Ewxr', '449.F21Ewxl', '449.M43Ewax', '449.F11Urin', '449.F31Nose', '449.M42Ewxl', '449.M23Urin', '449.F32Ewxr', '449.F34Urin', '449.M24Ewax', '449.F12Urin', '449.M23Uric', '449.F22Pinr', '449.F22Fotl', '449.M11Ewxl', '449.F34Uric', '449.M63Urin', '449.F24Uric', '449.F24Urin', '449.M24Urin', '449.F33Uric', '449.F23Uric', '449.F23Urin', '449.F33Urin', '449.F24Ewax', '449.F31Urin', '449.F32Urin', '449.F32Tong', '449.F34Tong', '449.M14Urin', '449.M43Fotl', '449.F11Tong', '449.F12Tong', '449.F22Fotr', '449.F13Urin', '449.F21Frhd', '449.F22Frhd', '449.M41Ewxr', '449.F23Fotr', '449.M13Knee', '449.M53Frhd', '449.M12Aptl', '449.M41Fotr', '449.M41Fotl', '449.M42Fotl', '449.F14Urin', '449.M13Ewax', '449.F24Fotl', '449.F21Pinr', '449.M31Aptl', '449.M32Aptl', '449.M33Ewax', '449.M31Ewxl', '449.F23Ewax', '449.F14Ewax', '449.F23Fotl', '449.F22Ewxl', '449.M14Ewax', '449.M31Ewxr', '449.M41Ewxl'];\n",
       "  var coords = [[-0.179510749191, 0.160900395023, -0.00570965213906, 0.122334439759, -0.0162248971994], [0.0228821893392, 0.126741711638, 0.0271440495268, -0.127739135345, -0.0655730666726], [0.183540331585, 0.0665827817047, 0.0149981843457, -0.180790814272, 0.0458596183358], [-0.189814777693, 0.14904689444, -0.138100278244, 0.239620556398, 0.028344444461], [-0.205173121754, 0.114582609172, 0.0398627555447, 0.07244766104, -0.0215930402801], [-0.124142214882, 0.197819584616, -0.122694581868, 0.140928479817, 0.0352966233856], [-0.187907940136, -0.146385714479, -0.188015709799, -0.120087534466, -0.125465091547], [-0.135903719484, 0.184445879892, -0.135213773324, 0.188660177948, 0.0223723764383], [-0.0814511528085, -0.00568651768083, -0.145667548256, -0.133533259849, -0.00365061448338], [-0.248622159043, -0.0345018203617, -0.0996021905435, -0.0438472021975, -0.0983872474324], [-0.11929889346, 0.165384874511, -0.123996026279, 0.0279026748222, -0.0221042143156], [-0.211366228064, 0.160804510335, 0.00544133277503, 0.103479762106, 0.00322726986825], [-0.125025207581, 0.149060014638, 0.000378216123617, -0.0145717942932, 0.0173190252299], [-0.199810153428, 0.129956087796, 0.0154476269886, 0.0575242405483, -0.00926085731499], [-0.169396194858, 0.171588002571, -0.0720168003033, 0.0800154927305, 0.00707316305096], [-0.218021715122, 0.132329080564, 0.043308133209, 0.0716239484997, -0.0293826288586], [-0.195069136466, 0.125816416887, 0.0786829257692, 0.0663810202945, -0.0409093952209], [-0.0535424770638, 0.130586491484, 0.0695553358886, -0.135836133082, -0.00141589498417], [-0.189904019421, 0.0888923186883, 0.112458409843, 0.0403432544819, -0.0321117551796], [-0.12645811983, 0.150775475109, 0.0118595391129, 0.0280967908093, -0.0196118542186], [-0.181338561741, 0.13866410771, 0.0196675228915, 0.0245505236348, -0.00163710695978], [-0.177368488728, 0.185229217828, -0.0710370487845, 0.0733427454281, 0.0118119287304], [-0.0880449396977, 0.195632457877, -0.0563429887624, 0.0118467104699, 0.0194581123594], [-0.194435917003, 0.122461312712, -0.158732954881, 0.169371763413, 0.0329658639362], [-0.187812212119, 0.179540571482, -0.0958674373183, 0.127881348581, 0.0100379859399], [-0.177175665102, 0.157461540814, -0.0383623784397, 0.0810416615745, -0.00986792163051], [-0.170069696609, 0.0641762206453, 0.130514179372, -0.0384963444818, -0.0469977310492], [-0.125642219359, 0.054440582712, -0.138871060882, -0.0306514885899, 0.0657672567771], [-0.0685698748659, 0.114054794311, -0.0746527191097, -0.0706933440004, 0.0449494550806], [-0.211758804134, -0.172221161364, 0.358186760555, 0.0484334343703, -0.0472532821071], [-0.133258249828, 0.0600790203023, 0.165441318502, 0.00733831012149, -0.0206730968288], [-0.185611007071, 0.175726685163, -0.0149352183462, 0.0498301523534, 0.00677044039402], [-0.221424984433, 0.132393959865, -0.0160044973073, 0.108140655367, -0.00863649064777], [-0.166086814209, 0.180686633742, 0.000319788374454, 0.0573785006185, 0.00690758599678], [-0.147044673028, 0.103715434456, 0.0930254655136, -0.0406156715941, -0.0120435959408], [-0.205889555005, 0.0934723543447, 0.10760263905, -0.00326150687356, -0.0463021669488], [-0.0912100541004, 0.186826812547, -0.0482016928481, 0.0249971929338, -0.0176290876358], [-0.20926946971, 0.149867373833, -0.0798261488899, 0.109078315142, 0.02449669248], [0.296335834583, -0.143058020136, -0.00180926221968, 0.0984354651691, 0.00373318628023], [-0.21705766867, -0.11255902331, 0.306621647235, 0.0400651866289, -0.101275644257], [-0.0480918162255, 0.172564199148, 0.0120206680611, -0.0160121049963, -0.0696482346779], [-0.0307977025059, 0.0385085303593, 0.0412689973236, -0.0457931943199, -0.0371036746736], [-0.235244873321, -0.105589255443, -0.00330431171334, -0.0900944005, -0.134016443796], [-0.187342102369, -0.147446966471, 0.296311173965, 0.0124476343269, -0.0900283523995], [-0.211489447985, 0.11504434601, 0.0822123575561, 0.0351115383173, -0.0110157237501], [-0.239892229836, -0.107266191939, -0.0375876885733, -0.0911267820517, -0.143227451114], [-0.184662208949, 0.17865881082, -0.0455560135676, 0.0893210494134, 0.00612125547534], [-0.199730430885, -0.151900720092, 0.33250274357, 0.0339103803509, -0.0708251173258], [-0.0519580017921, 0.153945343881, 0.0820229642408, -0.103204918518, -0.0418704694851], [-0.209044781936, -0.171666599362, -0.0828888910585, -0.132147058686, -0.143773428809], [-0.231602250487, -0.138570589252, -0.147622961495, -0.0775305501218, -0.126219922145], [-0.120575459149, 0.120535729615, 0.0744533046732, -0.0801471519922, -0.0377332165471], [-0.227269725968, 0.0753971796869, -0.0904070033654, -0.00587876993606, -0.045774442447], [-0.177299047751, 0.169608272304, 0.0245466052116, 0.051737592675, -0.00147352320083], [-0.146276606525, 0.148003705384, 0.0503514989416, 0.0482982924457, -0.0211596781834], [-0.0415114797056, 0.135723743744, 0.0681421100411, -0.106512247735, -0.0309590912574], [-0.0401648965922, 0.160125795326, -0.0339368160306, -0.0269021603337, -0.0117783548373], [-0.208807709737, -0.126462281797, 0.329849067816, 0.0182013273653, -0.0671032565677], [-0.198911058061, 0.00846535783657, -0.197591376311, -0.0462190022607, -0.0810300962591], [-0.149130436304, 0.166281222394, 0.0226177972157, -0.0075601396217, 0.0127565965479], [-0.222435129332, -0.0215014963779, -0.0901721866274, -0.0334295065469, -0.112153199218], [-0.170912066707, 0.15133503927, -0.05063828922, 0.0783030507276, -0.0209253656128], [-0.238909942556, 0.0454020463427, -0.0842922237527, -0.0124631719874, -0.0675057972671], [-0.204602903554, -0.126805354826, 0.22474968717, -0.0334761077791, -0.111973783039], [-0.22774020745, -0.192765976063, 0.33020442755, 0.0790957870569, -0.0569387685214], [-0.181063680442, 0.123164401891, -0.166064512026, 0.20746816367, 0.0332876042428], [-0.174122194016, 0.205547806258, -0.110185479695, 0.178132029798, 0.0386154290351], [-0.139315247143, 0.171744626297, -0.175065768839, 0.224055936806, 0.0074122305135], [0.108457896135, 0.0821887053235, 0.0967696775384, -0.109461432051, -0.0653648612589], [-0.08786800301, -0.374702686024, -0.288979891408, -0.0352674660124, -0.0359606705665], [-0.0977493565075, -0.410466219006, -0.309904480124, -0.023787465516, -0.0227814886871], [-0.0914977770995, -0.416869132929, -0.299558432901, -0.0288813161848, -0.0106569715954], [-0.107340418762, -0.411153114676, -0.308355877504, -0.0201889609098, -0.0315881589378], [-0.0741251927186, -0.393598345488, -0.300935280323, -0.0543776042783, 0.0196507060286], [-0.12137588059, -0.364651767216, -0.262871302046, -0.101467000475, -0.0368265478771], [-0.073576399109, -0.349901543784, -0.266878369739, -0.0157281425088, -0.0332452885341], [-0.124745130135, -0.417939374379, -0.233393051898, -0.0309269886756, -0.0162712512983], [-0.107915579005, -0.426434216255, -0.287210493877, -0.0430984344589, -0.0137851413557], [-0.103489471565, -0.375140414315, -0.256180208606, -0.0806846501711, 0.0123706708183], [-0.179084163532, -0.176966320289, -0.224780772789, -0.126787040308, -0.122841591714], [-0.204156396956, -0.112860757269, -0.191183400354, -0.0853390317341, -0.128536047646], [-0.154504530619, -0.41595238138, -0.217275997198, -0.0525476712579, -0.0578031064134], [-0.0723270770104, -0.385206888765, -0.325702830426, -0.038956962193, -0.0556350823492], [-0.080528695103, -0.350415848492, -0.277049609313, -0.0761695869831, -0.00471274631444], [-0.140462327551, -0.390362507238, -0.240568154668, -0.0661838884321, -0.0662817519496], [-0.109415000232, -0.369269539114, -0.252430094803, -0.0487541928324, 0.000298320646725], [-0.175512934004, -0.21093378469, 0.356990482521, 0.0451577352783, -0.0214744759055], [0.191283664973, -0.0844013781301, -0.0160511362043, 0.189289927347, 0.11566009142], [-0.185447291517, -0.18831823125, 0.359249076454, 0.0682969786573, -0.0201736427353], [-0.0778733575687, 0.0767471280965, 0.0966083533763, -0.121531986755, 0.0116742498142], [-0.189582374406, -0.157652970851, 0.358642426587, 0.0571330154446, -0.0211210292337], [-0.178089107995, -0.199298570636, 0.384517646341, 0.0722902839931, -0.0130303457877], [-0.113775736855, 0.0754300153244, 0.0780522475809, -0.0599184526992, 0.0692788702612], [-0.181925210596, 0.00537173677046, 0.133993721679, -0.0473387125996, 0.0664746943893], [0.228984201268, 0.0587871300783, 0.0264057674623, -0.0841539183896, -0.168274221506], [-0.171098861586, 0.0996452915818, -0.0847750497061, 0.0488928323372, 0.118793083444], [-0.191817837226, 0.176562714432, -0.122524073858, 0.219609925038, 0.0367259262541], [-0.146960358061, 0.123319042351, -0.193110554729, 0.199437909898, 0.0376000294959], [-0.164321710532, -0.162553898398, 0.350397595584, 0.0434091661327, -0.00905261454535], [-0.205641586975, 0.115012814615, -0.165217176256, 0.0766447209946, 0.0153916918797], [0.175688150807, -0.113233814047, 0.126460802442, -0.0192234968597, -0.00467692941365], [-0.0802680683083, -0.0580660504047, 0.21134132811, 0.00743199447683, 0.0981611609187], [-0.0706072073019, 0.107666322374, 0.136295369751, -0.123233536005, 0.00143091637787], [-0.0508698087936, 0.0841183531111, 0.040582938845, -0.121317979441, 0.0123976867355], [-0.106269227092, 0.125289933793, 0.0872891455298, -0.0485829955696, -0.000215383694618], [-0.0525065935364, 0.143968406726, -0.0849838618063, -0.00868105956514, 0.0102465399891], [-0.152714896715, 0.166099106667, -0.0291231010698, 0.073932238418, 0.0013483432418], [0.0408714915749, 0.156105529334, -0.0618372613529, -0.0780533182285, -0.0437328038769], [-0.15617290378, 0.124053012514, -0.210831684405, 0.131534097121, 0.0405780308608], [-0.226795104523, -0.230548166019, 0.369937680031, 0.0713981939097, -0.023473175799], [-0.204833213993, -0.174044247091, 0.354959607112, 0.0649964254453, -0.0500793394453], [-0.0334124622565, 0.158391303776, 0.0430414001597, -0.0370969661889, 0.0191981521588], [-0.175001225856, 0.170267705755, -0.00946304273282, 0.0759411747647, -0.00206308802511], [-0.218499160144, -0.162684601997, 0.292473503718, 0.0613427351952, -0.0485827470372], [-0.19938453989, 0.169336608625, -0.0362691603906, 0.116486187376, 0.0402011900098], [-0.167984755557, 0.190486854177, -0.0553219693717, 0.122347893113, 0.007200626615], [0.241459184779, 0.025726044518, 0.102885134495, -0.052895822564, -0.0992320979714], [-0.126722124156, 0.198642609853, -0.127137193287, 0.0873996225741, 0.0437741662124], [-0.187709512169, -0.198826664643, 0.393087972258, 0.0591944559829, -0.0336876941215], [-0.1330872129, -0.134438378729, 0.355048543302, 0.00498511748383, -0.0689418461461], [-0.131516233037, 0.214063001389, -0.0918396936275, 0.0598876168879, 0.0622777698398], [0.37110237527, 0.0727465789931, 0.000467981760511, -0.0593256661322, -0.134570525925], [-0.0921013141194, 0.19743404343, -0.0610856462294, 0.0132153973052, 0.0144721864084], [0.0939128428507, 0.0892903597829, 0.0827407622102, -0.129712309833, 0.0194590523308], [-0.107045941238, 0.170027675022, -0.0857846680453, 0.0525274111819, 0.0315901307187], [-0.158697112685, 0.184672430118, 0.0331075514463, 0.0638268603259, 0.0148495919866], [-0.031275859196, 0.0730934681471, 0.142618996122, -0.123989483608, 0.00157855014837], [-0.127414335134, 0.192272933834, 0.0057028817355, 0.0329518870702, 0.024661963566], [-0.0246133701832, 0.118452895895, 0.0202631428013, -0.0559017723964, -0.016901169929], [-0.204196721427, -0.203162761778, 0.37054087256, 0.0605962607272, -0.0440678505117], [0.0953115650933, 0.177458918339, -0.00434083736699, -0.0519648844529, -0.055568756031], [-0.16530259832, -0.132505821227, 0.347912528565, 0.0147805972913, -0.0753184759649], [0.148733286965, 0.141103517702, -0.0409039672166, -0.165525436511, 0.000336240623249], [-0.11249376453, 0.214520063547, -0.0340936092314, 0.0506025473964, 0.0340687963785], [-0.139373204946, 0.157234090429, 0.0680160017215, 0.0281068843937, -0.0242573155722], [-0.127893299285, -0.0432095484942, 0.223942501274, 0.0185105094895, 0.0290924381238], [-0.157892847199, -0.193725956763, 0.368216157731, 0.0380060026075, -0.0391739367372], [-0.159418562135, 0.125055351159, 0.0646162587084, 0.028382002026, -0.0290718594958], [-0.157174706727, -0.205206639246, 0.390956549693, 0.0370612754395, -0.024424510403], [0.167336331298, 0.0446778206747, 0.0980289885289, -0.169360584807, 0.0713319702892], [0.115552076322, 0.037432293574, 0.0350326660918, -0.141543183838, 0.214019882071], [-0.169887498025, 0.192970052725, -0.0501815434804, 0.0917686064363, 0.0261187051258], [-0.180954357019, -0.0705122052842, -0.140515042958, -0.132439548942, -0.0988669384097], [-0.175657255001, -0.0891064899375, -0.162921016087, -0.134035701091, -0.10532950441], [0.0136406565862, -0.108971869605, -0.0357925834974, -0.132943004794, -0.0839162355978], [-0.165057488321, 0.148072194439, 0.0589406149951, 0.01758465021, -0.00256518240271], [-0.0597034991385, 0.18399634314, 0.00567828483964, -0.0227036870205, -0.0123641801267], [-0.0966157181682, -0.0944911319697, -0.0797355697943, -0.142734618416, -0.0332406862581], [-0.209365883637, 0.0874039293133, -0.175573613875, 0.0540376523238, -0.035027645872], [-0.182932428029, -0.0584905006433, -0.02866745304, -0.136978051286, -0.108253612198], [-0.0179945837717, 0.0141328372471, -0.0893261353077, -0.0648235290682, 0.0300328826171], [-0.038745892065, 0.090446694538, 0.157435809449, -0.134018036234, -0.0296257604229], [-0.0472596750547, 0.0724345665628, 0.165641932631, -0.0926692267189, -0.0631257375232], [0.176051852262, 0.0206777120479, 0.0587775251411, -0.14067744708, 0.137773151916], [0.220478692812, 0.0164411570694, 0.0965204906625, -0.129461060465, 0.0141835202638], [-0.223920869027, -0.181672208335, 0.330380159406, 0.0747472810286, -0.0491003222113], [-0.1979966958, 0.0553946898848, -0.158753070396, 0.00893606971, -0.0502849238869], [-0.130433436093, 0.20371527184, -0.06762039563, 0.0845355307368, -0.0169904290308], [-0.169860360082, 0.174806666346, -0.0507150870163, 0.0736590012783, 0.0178414132535], [-0.182942304521, -0.0606330039131, -0.225781552549, -0.0660377652578, -0.0797338980185], [-0.102123204127, -0.0799732371602, -0.107457181848, -0.159745025587, -0.095728306051], [-0.0944768052137, 0.0993753812114, -0.173249106352, -0.0200950387004, 0.0541672939724], [0.0591545413389, 0.114236926982, 0.0538088047611, -0.100663310946, 0.0416936030236], [-0.187687656766, 0.186665895044, -0.0389826378226, 0.0863070915965, 0.0149582285277], [-0.163611707835, 0.200039096797, -0.047605902198, 0.0535191012692, 0.012226021765], [-0.204929282034, -0.175876381295, -0.122184842471, -0.117203870225, -0.144359833301], [0.0604259651998, 0.0766458870858, 0.0973710149731, -0.156882179304, -0.0108200891645], [-0.218957326137, 0.0818533489414, -0.145504524108, 0.0775165749504, -0.0461507440764], [-0.180495935412, 0.141276334301, -0.0990353876262, 0.218975808905, 0.0196921227261], [-0.0300230212225, 0.162516656994, -0.129093091721, 0.163434387687, -0.0262866818845], [-0.172130365246, 0.14368792585, -0.162128375393, 0.22716536098, 0.014857487991], [-0.191838347812, 0.170106077918, -0.139099190306, 0.229335787304, 0.03446453572], [0.13180945512, 0.110052084976, -0.0780725713801, -0.0612526178119, 0.107840480065], [-0.148178537655, 0.147066722563, -0.179774912518, 0.218977856879, 0.000918550300187], [-0.153898580098, 0.153217192059, -0.129904728596, 0.0372350880615, 0.0353059485422], [-0.164863878969, 0.0893765547457, -0.193870220143, 0.110449817122, -0.014526257334], [-0.124161642002, 0.209338665559, -0.14980913568, 0.193334265302, 0.0177312872168], [-0.186767361752, 0.1470351733, -0.0346139387863, 0.00401953142258, -0.0208477632592], [0.0353959679549, 0.168145031035, -0.0580870859696, 0.126495596835, -0.0733488424583], [0.18429805397, 0.1066132104, -0.0386622193602, -0.0524978823266, 0.0762333429151], [0.220586493452, 0.0531715567071, -0.0601693414143, 0.0297133402921, -0.113619849022], [-0.0254330978624, 0.204051701465, -0.0459967725516, -0.0572416885411, 0.0774467046488], [-0.177376788162, 0.111357855826, -0.0409290470955, 0.0755131386132, -0.0118634491306], [-0.142378356775, 0.176676701395, -0.176196835436, 0.160134458964, 0.0431482135787], [-0.00859298185163, 0.237401909969, -0.0874697436672, 0.0184015051899, -0.0128183981928], [0.18023158357, 0.156091322202, -0.0198752732997, -0.0409090808079, -0.139161963899], [-0.0855934821988, 0.235761219651, -0.0576762268954, 0.0577852638485, 0.0102654369527], [0.00326333275678, 0.074037129696, 0.129612718319, -0.138193971594, 0.0337137195507], [-0.143474829622, 0.0820859808657, -0.215636676761, 0.0449476236183, 0.0263446991997], [0.117462591572, 0.126930428179, 0.0234872366765, -0.161232700808, 0.0485563048503], [0.268578235372, -0.00175023127873, 0.0489196197587, 0.0112018253748, -0.111387580211], [-0.113160597261, 0.209701438788, -0.122308255537, 0.0694494537065, 0.0733944058596], [-0.00452040804702, 0.155511194423, 0.0797849568198, -0.119678870094, 0.0319476327909], [0.091961126538, 0.190424900341, -0.0171399624385, -0.120223521564, 0.031823059848], [-0.0651839916875, 0.0877193306521, 0.131204123439, -0.0799671472025, -0.0106541869102], [0.394114238797, 0.0159060380801, -0.0189065848186, 0.0195439237596, -0.0486927263193], [0.229953108508, 0.15404006723, 0.00160096100362, -0.138767973555, -0.0102001999716], [-0.00666469588456, 0.0641786734048, 0.0382517168762, -0.135870235906, 0.0813040065002], [0.169028419547, 0.130863244622, 0.0307165002863, -0.126266307679, -0.0735676541697], [-0.184100767025, 0.0295487890357, -0.217680683525, -0.0042356600088, -0.0187207214654], [0.246892615868, 0.0730160590292, 0.0118353452824, 0.00144677824504, -0.169574962191], [-0.00215942421202, 0.146147460416, 0.0611220114176, -0.0823484399583, 0.058195356112], [0.00440239565561, 0.0440375461551, 0.175578905988, -0.166675103606, -0.0534607654807], [0.43075336528, -0.120484995245, 0.0327468478614, 0.225578682555, -0.149897230763], [-0.0193618111692, 0.147980486741, -0.0351919475367, -0.0820459424817, 0.0182043407027], [-0.154740776228, 0.14236353907, -0.0531484702086, 0.0409917476341, 0.0266982259829], [0.00177443714038, 0.190408630348, -0.0151476267332, -0.0390764332424, -0.0109197626407], [-0.0849393106267, 0.185037345138, 0.00989654900908, 0.0215129583689, -0.016897522579], [0.156930517476, 0.115375141979, 0.0512087857499, -0.0833246011561, -0.0215052369959], [0.302011380726, -0.052804029485, 0.0550295037071, 0.0592676348838, -0.034815551713], [-0.150360970186, 0.1395193334, -0.0460690729616, 0.0155508327035, 0.0758514652182], [0.0980318380119, 0.0976853345968, 0.0616080306738, -0.0659140475817, 0.0128210604567], [0.448636572787, -0.128078649237, -0.00064038850151, 0.308921690019, -0.0533745143653], [0.316172020477, 0.0569609961771, 0.0353679051894, -0.0638564300488, -0.0996248044121], [0.13694862737, 0.136041900822, -0.0572989453573, -0.0459534030121, -0.0456868781567], [-0.114802546603, 0.182675467558, -0.073471796896, 0.0665876215166, 0.0362922593914], [0.0206646500177, 0.0589992317542, -0.0805555918349, -0.167899697351, -0.060810828528], [-0.0258820420758, 0.194306832357, -0.084598387349, 0.00328861109605, 0.0651701579055], [-0.00719706974589, 0.0495594990349, 0.0156910977616, -0.156094870225, 0.0157715789543], [0.0284707020923, 0.128995085649, -0.0164322078696, -0.0583216319458, 0.0754532887057], [-0.157477786071, 0.146679903326, -0.185484523973, 0.190249913907, 0.0112639888124], [-0.0481946210167, 0.0521492961212, -0.115186319302, -0.0957909217902, 0.0303821352483], [-0.0974098866935, 0.154830480315, 0.0282842537532, -0.0300481342616, 0.0183843364115], [-0.0898791558841, -0.0593485007333, -0.172128213905, -0.168774730659, -0.0594519761925], [-0.177792232124, 0.147686828298, -0.035144008055, 0.0490359387632, 0.00819526037842], [-0.113991106693, -0.0113864126888, 0.0144129589486, -0.154105162214, -0.0744832623645], [0.12443701109, 0.129246454623, 0.0205452551622, -0.0858695721487, -0.016682261769], [-0.131652562274, 0.192109321891, -0.156749370114, 0.185703262666, 0.0362265661777], [-0.134278398943, 0.0076438846804, -0.060435961814, -0.0178683516774, -0.0643840490033], [-0.0859075081125, 0.135550098048, -0.0130506045041, -0.0610964532215, -0.00177967714886], [-0.170127885863, -0.200679117443, 0.365897068662, 0.0388307744147, -0.0102104396615], [-0.1866563183, -0.165152496597, -0.0226847492183, -0.0791274211709, -0.0565466476214], [-0.123740222359, -0.369977018987, -0.267966355534, -0.103943071872, -0.0307890589589], [-0.0895093008859, -0.398247834219, -0.243540787881, -0.033410151968, 0.00876477254971], [-0.0664839885959, -0.383852277837, -0.256752705913, -0.0132848815598, 0.0212405706955], [-0.156779129949, -0.415979098753, -0.269316284171, -0.0616340705259, -0.0686499686008], [-0.110703544331, -0.37678060614, -0.207642649502, -0.0638329336673, -0.0396234518406], [-0.0933290347829, -0.401427953775, -0.29577872361, -0.0345542132199, -0.0141438810793], [-0.0260740206096, 0.137358017335, 0.0175994460984, -0.0752059179818, 0.00222152030332], [0.203621087528, -0.0360730569689, 0.0822308966273, -0.141137828778, 0.213167281364], [0.226486293319, -0.0954777262222, 0.105242149814, -0.0760463655651, 0.25887228402], [-0.144035410688, 0.13213510948, -0.114924094129, 0.0275165246931, 0.0588182380236], [-0.148138182193, 0.125797964746, -0.143359201507, 0.0834712061019, 0.0225497627275], [-0.113954225956, 0.131178380248, -0.160299061485, 0.157489263408, 0.00316444819371], [0.0203179001585, 0.189907298197, -0.0833671179378, -0.00371876852357, -0.0199424462283], [-0.0519619383091, 0.17221732594, -0.0795515377638, 0.0748092445835, -0.0313287095484], [-0.137891471818, 0.186640266804, -0.0510524916988, 0.0516838435446, 0.0403361308198], [0.35090404994, -0.0795957637111, 0.0306624364581, 0.0795816241386, -0.0311533629123], [-0.137171926242, 0.204635777088, -0.0651612607082, 0.0947337917905, 0.0031503995896], [-0.147134889557, 0.206994009059, -0.111595025902, 0.1387603019, 0.0756549586315], [-0.164388714199, 0.0966188262244, 0.119223577682, -0.00271543228354, -0.0293011571728], [0.234982429582, 0.0225325139179, 0.119113986781, -0.0711385735031, -0.0910300175508], [-0.133639305364, 0.212197244345, -0.0515213595296, 0.0495318093115, 0.0156244608473], [0.154867604423, -0.078810802657, 0.0424270601761, -0.145319278262, -0.0520781470419], [-0.152709968297, -0.183478915305, 0.355751822077, -0.00456720523226, -0.0478460653698], [-0.130694218663, 0.182714022567, -0.108110048594, 0.0898555426664, -0.00207631445293], [-0.115569784146, -0.371318836413, -0.246444004866, -0.00524352664236, -0.0442869326027], [-0.188794582182, 0.152952302266, -0.0982760413334, 0.0705440682606, 0.0451986280529], [-0.0836171021231, -0.40569500169, -0.285811105199, -0.0442565589378, -0.0158637856462], [-0.10140693362, -0.412865732128, -0.296404783173, -0.0380241315291, -0.0366945479946], [-0.0829558850032, -0.39142439535, -0.240548451054, -0.0302566408416, -0.0126592594711], [-0.105595896897, -0.414608563232, -0.267888640696, -0.0223315661457, -0.0143337722049], [-0.0674950723429, -0.4149676844, -0.326906635098, -0.0265288881434, -0.0181746042604], [-0.0906115413288, -0.386901563504, -0.294052208107, -0.0413731744624, -0.0320523061155], [-0.0556054237395, -0.4031245009, -0.271577094049, -0.0269876291175, 0.0257917166935], [-0.102254864265, -0.395971233488, -0.269590314705, -0.0330167488839, -0.0301929392861], [-0.084948337409, -0.348605712844, -0.290799040107, -0.0856831676276, -0.00765192363577], [-0.0514228750693, -0.346069549421, -0.210179210647, -0.0346719328104, 0.0391767398931], [0.253557170953, 0.10852543799, -0.0286614910377, -0.097354397497, -0.0245437484925], [0.155535252286, 0.131535880232, -0.0428233327591, -0.0177671434295, 0.0542578659899], [-0.0490487386166, -0.297937174718, -0.135717586009, -0.0818681967424, 0.208310502813], [0.108081050899, -0.1654613243, -0.0206347102914, -0.0835146702874, 0.342399780574], [0.435446785825, -0.0519301024718, 0.00974723680236, 0.154725047262, -0.131812745446], [-0.0849770908494, 0.11498826676, -0.028318852441, -0.00591676613953, 0.0553582073047], [0.0917449872683, -0.216795243501, -0.0122509168781, -0.0502770441817, 0.353974180313], [0.260031258773, -0.129364068343, 0.0222863680147, 0.0185908276062, 0.392611231477], [-0.120369509849, -0.00424647870931, 0.231636055689, -0.00991174860207, 0.0614057459594], [0.113521570454, 0.0236156098046, 0.0663586088541, -0.106171692121, 0.112853948544], [0.239033848454, 0.101988083336, -0.0721816803133, -0.00264542061582, -0.00487827091921], [-0.108490430139, 0.0680695492652, -0.200197964932, -0.00815566599959, 0.0135746807425], [-0.173135930173, -0.162885506682, 0.383795068375, 0.00722349217628, -0.0513287935322], [-0.00908945350839, 0.199110632909, -0.112313237095, 0.0370694753944, 0.0497536350253], [-0.0446287216921, -0.0322867963055, -0.147730827583, -0.116687750014, 0.0574929890788], [0.164928321996, 0.0914448169178, 0.0809901392426, -0.114977378917, 0.0113194182571], [-0.0892497755752, 0.19044678175, -0.139474387372, 0.0885770545027, 0.0278976940442], [-0.0992760238706, -0.400666696399, -0.316379101699, -0.0434478284065, -0.0312477663039], [0.0268867237064, 0.0539092231089, 0.0509232030516, -0.1359357207, 0.0541511619745], [0.103440266937, 0.137871629412, -0.0942080839573, -0.109203322198, 0.0414240342049], [-0.145236785955, 0.135096021175, -0.160092690366, 0.0992556218893, 0.0408315624544], [0.0863861492479, -0.054044395874, 0.0861426437874, -0.161930985196, 0.229865353653], [0.179022292028, 0.103144917288, 0.0105382426556, -0.10762985674, 0.0306128872507], [-0.0477471358614, 0.162164981626, 0.0107726844246, -0.0787826705474, -0.0148805849408], [0.0531963652542, -0.0351021466592, 0.165505451964, -0.147766389543, 0.101218550556], [-0.0557085490622, 0.188235019664, -0.0244569554079, 0.00085677227399, 0.0398045356449], [0.182687096847, 0.0142786893397, -0.022377318875, -0.149429586221, 0.0581169911948], [0.270406703986, 0.0774770878139, -0.083225949768, 0.0400543501585, -0.0444855397031], [0.00662931981942, 0.19026044695, -0.121690486385, -0.0240537613994, 0.0216037753273], [-0.0914100489655, -0.375611284076, -0.284060098904, -0.0294959975021, -0.00330398878676], [-0.0772003226464, 0.167653056303, 0.0491554658534, -0.0385868372581, 0.0367528071237], [-0.193418813429, -0.211618108383, 0.354898003896, 0.0308476845292, 0.0130937633004], [-0.088894663191, 0.0501145773475, 0.109714019079, -0.0875973878029, 0.0860565147623], [-0.168765606454, -0.233387030017, 0.387892142854, 0.0392763754585, -0.00220493450599], [-0.187935117866, -0.227108410239, 0.384560846128, 0.0599718161687, 0.0267355327221], [-0.193727572573, -0.212182032511, 0.356475041881, 0.0450194152881, -0.000602631212391], [-0.192609621392, -0.200071067086, 0.373008357476, 0.0620335638407, 0.00828074717856], [-0.199310200281, -0.230653303464, 0.384720803974, 0.0554119631668, -0.0103595635052], [-0.194919720017, -0.219512093767, 0.374231175991, 0.0700953540849, 0.0285567281917], [-0.1970930682, -0.195383809843, 0.365178447115, 0.0571413068756, -0.0296014704169], [-0.177137929468, -0.147633784766, 0.359985849818, 0.017666448631, -0.0170033171617], [-0.145638619696, -0.240461123466, 0.362551347147, 0.0332410456447, 0.00988416450152], [-0.166448027159, -0.159223074701, 0.368581337214, 0.015899507005, -0.0613941038208], [0.332407699679, 0.0347374976335, -0.0349760773927, 0.0931194193513, -0.172775080173], [0.230384576549, 0.0763357803578, -0.0728458478674, 0.0356952504776, -0.0655218232675], [0.20883913173, -0.0439478149339, 0.0679183336811, -0.0422651112588, -0.0915192500873], [-0.0869750476649, -0.427652625135, -0.320590786586, -0.0385250000057, -0.0151936906134], [0.0102312464373, 0.0619881659054, -0.0354717767393, -0.141383961574, 0.0212055589183], [0.0392195824523, 0.152441008409, -0.0720720247498, -0.0444230400793, 0.00985220068821], [0.111759402183, 0.210620712923, -0.0316660562859, -0.0946623508443, -0.0231692298709], [0.375028622479, -0.0911411046868, 0.0426795400159, 0.104155367943, -0.04257104224], [-0.0362024499043, 0.149394713584, 0.0321896859954, 0.0667256746391, -0.0322957851966], [0.0324498740827, 0.113911332602, 0.101864622558, -0.0890370693519, -0.103608629907], [-0.12965357263, -0.206904585759, 0.35425998021, 0.0228956321731, -7.84066525324e-05], [0.216813470763, 0.0497755195894, 0.0416051861191, -0.144589593079, -0.0332084523358], [-0.197811924518, -0.210450517796, 0.386685668342, 0.0686829863213, 0.0288258812476], [0.229554547733, 0.155558800481, -0.0354256138055, -0.173324953975, -0.00542180905504], [0.183904327665, 0.114519675592, 0.0139482260766, -0.14678814832, -0.0341293527373], [0.136551414947, 0.0715769392027, 0.0837996656523, -0.169439818675, 0.0511002460563], [0.383298262405, 0.00481332978012, 0.000158426924161, -0.0266092782001, -0.0767577641785], [0.281634583248, 0.08166949853, -0.0110467802991, -0.133310312581, -0.0979703894417], [0.350021537503, 0.0211572074588, 0.0392028976471, -0.152113645582, -0.0261651481032], [-0.00435673577054, 0.161254647502, 0.0187278038784, -0.113603017284, 0.0153675890159], [0.235279513977, 0.0451873383521, -0.0426909844157, -0.199740269129, 0.0130745016707], [0.278425459033, 0.0697438343387, 0.0082018412125, -0.106053252871, -0.0603397509708], [0.21948368473, 0.0505644603505, 0.0275174134016, -0.0394380626989, 0.0407322187133], [0.420784882511, -0.0489387268218, 0.0238013945021, 0.0325261664158, -0.0471042564395], [0.102970354419, 0.108396285761, 0.0130504751238, -0.105155101114, 0.00141848850644], [0.185705668558, 0.0699299223437, 0.0978736126193, -0.20748318276, 0.0249349777014], [0.17371941901, 0.177146080632, -0.0677396305735, -0.0412663121249, -0.0930373746021], [0.0355963773284, 0.0484088348876, 0.138072320087, -0.188266184014, 0.0633691471646], [0.183583509122, 0.0407801836231, -0.0555400457579, -0.213480436737, 0.00484238893102], [0.268196569895, 0.107118130909, -0.0292912622276, -0.145935441024, -0.0514688202742], [0.0420572275932, 0.0756802030539, 0.0635817254441, -0.115251276325, 0.0858576169531], [0.38972631691, 0.0843187344732, -0.021083699178, -0.0820716955097, -0.103673515089], [0.310751684063, 0.00851646863849, 0.0179470693563, -0.124423056058, 0.0704147173699], [0.133100373571, 0.0857643134735, 0.023541609019, -0.0830533144321, -0.0102292266445], [0.213687501192, 0.0152538957866, 0.0308045232609, -0.138997608471, 0.215080076047], [0.25453137031, 0.0255788753689, 0.0974053094387, -0.125237343991, -0.0617318689236], [0.204673808742, -0.0232765510249, 0.141175608494, -0.130474746594, -0.00725374519244], [-0.0830899593611, 0.141689710487, 0.0214186507353, -0.0327280761757, 0.0218843163445], [0.212202743091, 0.139241830884, -0.056184409682, -0.0880619369704, 0.0403591553473], [0.300828640896, 0.118839297736, -0.0368178916938, -0.116615046995, -0.0307139409594], [0.305257104419, 0.0112632100858, 0.00632067908144, -0.116862985142, 0.0801215252072], [0.243641021775, 0.097635136105, -0.0423082205037, -0.13306959299, -0.00817835712366], [0.278433664299, -0.15005690203, 0.0801045167844, 0.0819481373453, 0.0344942590713], [0.396543094164, -0.12286178721, 0.0255373948193, 0.161363090014, -0.0692516946624], [0.456632293129, -0.141719674321, 0.00043898222997, 0.296058580067, -0.146118618911], [0.131837768885, -0.0663495157861, 0.0616032517453, -0.108180089455, 0.310851913521], [-0.00125616651111, 0.118308127439, 0.0353111308778, -0.0358496445852, -0.0101503061341], [0.317733832704, -0.00139975602104, 0.083050922225, 0.0427708972517, -0.10203831933], [0.0830369401493, 0.0963676950582, -0.0390460992784, -0.0743175061944, 0.194929089551], [0.0543032510015, -0.0230622983793, -0.117750359598, -0.12983206262, -0.00870249298396], [0.378097073071, -0.133818013768, 0.000603483095612, 0.182315679934, 0.191252500255], [0.341277831374, -0.0222103162588, 0.0457650914992, 0.10371934372, -0.244906634526], [0.173779647572, -0.149614701896, 0.0150973886249, -0.0707915486009, 0.4297225624], [0.0313816916485, 0.01687248672, 0.0293498605018, -0.112844346653, 0.262486516302], [0.402574397058, -0.0302728223534, 0.0353035742925, 0.0677807610199, -0.169792080596], [0.265637156586, 0.122309248482, -0.0696323564807, -0.0151189088917, -0.0763309146276], [0.286543842536, 0.00420636099678, 0.0421804871683, 0.0273923007578, -0.0643179970058], [0.288884972491, -0.21416343066, -0.0207945428494, 0.251939693019, 0.189611843725], [0.191919778565, -0.045909937704, 0.042148506508, 0.0283747151573, 0.224538538841], [0.272350448164, -0.186529006466, 0.0353506731131, 0.219426965673, 0.23245822641], [0.229356334777, -0.223105012737, 0.0563591399399, 0.258020246031, 0.245040451205], [0.102127117416, -0.0306424734876, 0.0521836206229, -0.0910775606457, 0.28308916147], [0.323917845565, -0.18889678652, -0.0274674273743, 0.346959409642, 0.163409097389], [0.234313266053, -0.130565598077, 0.0505564891908, -0.00503564850637, 0.355207912296], [0.254344424588, -0.222326514674, -0.0554483842518, 0.364961613828, 0.274004772579], [0.274750303384, -0.223431138814, -0.0541489919244, 0.385561829494, 0.199223530375], [0.161479238638, -0.13830861157, 0.175069749876, 0.0684702805847, 0.0393944038314], [0.275365112425, -0.188144259744, -0.0518736742277, 0.26503708168, 0.242494443756], [0.271728532467, -0.224109062908, -0.0542430213343, 0.381885197897, 0.240084383123], [-0.113417968645, -0.153693422598, 0.361570141643, -0.014720236651, -0.0587696993099], [-0.122446502567, -0.215953279587, 0.37471773422, 0.00871835010618, 0.00679389556888], [-0.0932325148546, -0.0554845519843, 0.277893463811, -0.0406874721261, 0.0877389041354], [-0.0534272007197, 0.193337728599, -0.0498035418981, 0.0426592749658, -0.00703427498583], [-0.0874931253759, -0.227560309846, 0.332949734489, 0.0242868997782, 0.034192579632], [-0.101464668289, -0.227826308043, 0.339966647978, 0.029076733199, 0.0288219680494], [0.0369856423431, 0.15254223011, 0.0127312352535, -0.0363948975454, 0.000871939957364], [0.140853517496, -0.160342173623, 0.0679952809602, -0.0914619117866, 0.366948746795], [0.317686231605, 0.0597529191674, -0.00358617356109, -0.0277285447268, -0.0318482693653], [0.194894017464, 0.0730304541955, 0.0574963059791, -0.126550873589, -0.00471823657505], [0.334305240286, -0.081051160366, 0.0248748302451, -0.0453181201338, 0.0249176749223], [0.395101226263, -0.121062080685, 0.00195656595555, 0.246567779522, -0.0881837630044], [0.0713629053459, 0.0515183179978, -0.0699721471804, -0.0660435538208, 0.125958759136], [0.289585495718, 0.118978539226, -0.0303753234466, -0.0554432744569, -0.120372501826], [0.266626337541, 0.142079197399, -0.0730741663053, -0.0485314327318, -0.0982300240404], [0.261546502546, 0.0602806221005, -0.0283029957508, 0.0608206741383, -0.142944408491], [0.360350082148, 0.0491637116576, -0.0183564977531, -0.0142855605272, 0.00192125858303], [0.188987545048, 0.0971015526591, -0.0440430153097, -0.0194707704069, -0.0468152469389], [0.211404362274, -0.153322492372, 0.0668998291139, -0.0308896332635, 0.342316486512], [0.453770404053, -0.136335480579, 0.000296170068617, 0.293775770095, -0.162169839067], [0.306669256352, -0.0301922171454, 0.0190689406366, 0.0847354819185, -0.137426182399], [0.399103206959, -0.0312430621704, 0.0463055286867, 0.135886685014, -0.142403902302], [0.398222683646, 0.0733166730477, -0.0349157365999, -0.0707620319614, -0.118151796159], [0.378997360724, 0.0939037785448, -0.0367380509788, -0.0537966506605, -0.142933560138], [0.179100732157, -0.0832746292489, 0.155039007986, 0.0495735894639, -0.0594094663987], [0.413584473544, -0.0318995309822, 0.042690126923, 0.117115004879, -0.216071546981], [0.461485309696, -0.141663937487, 0.0236325966034, 0.290241176323, -0.149545236976], [0.319795228131, -0.0410986073778, 0.00475882836592, 0.174115035744, -0.162305908872], [0.402584987113, -0.0196598389556, 0.00938205780319, 0.111633018464, -0.186733421847], [0.459940329389, -0.125854630123, 0.00887246303222, 0.259519794553, -0.164858326135], [0.464759013852, -0.144182325383, 0.00223406860158, 0.300819019316, -0.157920263677], [0.368677459555, -0.116742732815, 0.0571128180172, 0.029996967433, 0.061586167184], [0.456764913568, -0.0868741922781, -0.000848658352481, 0.231349295248, -0.229325094823]];\n",
       "  var pct_var = [12.2074583847, 10.0247517917, 8.60483771442, 3.9847746522699996, 2.91478655472];\n",
       "  var md_headers = ['SampleID', 'BarcodeSequence', 'LinkerPrimerSequence', 'barcode_read_group_tag', 'dna_extracted_prep', 'experiment_alias', 'experiment_center', 'experiment_design_description', 'experiment_title', 'key_seq', 'library_construction_protocol', 'linker', 'nucl_acid_amp', 'nucl_acid_ext', 'pcr_cond', 'pcr_primers', 'physical_specimen_remaining_prep', 'platform', 'pool_member_name', 'pool_proportion', 'primer_read_group_tag', 'region', 'run_alias', 'run_center', 'run_date', 'run_prefix', 'samp_collect_device_prep', 'samp_size', 'sample_center', 'sample_type_prep', 'sequencing_meth', 'study_center', 'study_ref', 'target_gene', 'target_subfragment', 'url', 'age', 'age_unit', 'altitude', 'anatomical_sample_site', 'anonymized_name', 'assigned_from_geo', 'body_habitat', 'body_product', 'body_site', 'collection_date', 'common_name', 'common_sample_site', 'country', 'depth', 'dna_extracted', 'elevation', 'env_biome', 'env_feature', 'env_matter', 'hiv', 'host_common_name', 'host_individual', 'host_subject_id', 'host_taxid', 'latitude', 'longitude', 'original_sample_site', 'physical_specimen_remaining', 'public', 'samp_collect_device', 'sample_type', 'sex', 'taxon_id', 'title', 'qiita_study_title', 'qiita_study_alias', 'qiita_owner', 'qiita_principal_investigator', 'Description'];\n",
       "  var metadata = [['449.F21Plml', 'ATGCACTGGCGA', 'CATGCTGCCTCCCGTAGGAGT', 'F21Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Plml', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left palm', 'sample193', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Pinr', 'CGCACATGTTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Pinr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Surface of right pinna', 'sample394', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M11Glns', 'ACACTAGATCCG', 'CATGCTGCCTCCCGTAGGAGT', 'M11Glns', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M11Glns', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Glans penis', 'sample065', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:glans penis', '2008-2009', 'human skin metagenome', 'glans penis', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'glans penis', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F33Fotr', 'GTATGTTGCTCA', 'CATGCTGCCTCCCGTAGGAGT', 'F33Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample454', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M54Plml', 'TAGCACACCTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M54Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Left palm', 'sample218', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Fotl', 'GCTTGCGAGACA', 'CATGCTGCCTCCCGTAGGAGT', 'F32Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Fotl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left foot surface', 'sample230', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Plml', 'CGATCGAGTGTT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Plml', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left palm', 'sample206', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M24Fotl', 'GCAGCCGAGTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M24Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M24Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left foot surface', 'sample240', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Glns', 'CAGTGCATATGC', 'CATGCTGCCTCCCGTAGGAGT', 'M21Glns', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Glns', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Glans penis', 'sample067', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:glans penis', '2008-2009', 'human skin metagenome', 'glans penis', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'glans penis', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Plmr', 'CATCTGTAGCGA', 'CATGCTGCCTCCCGTAGGAGT', 'M21Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Plmr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right palm', 'sample428', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M23Knee', 'GATCTATCCGAG', 'CATGCTGCCTCCCGTAGGAGT', 'M23Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Popliteal fossa', 'sample009', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M43Plmr', 'GTGAGGTCGCTA', 'CATGCTGCCTCCCGTAGGAGT', 'M43Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right palm', 'sample438', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M63Plmr', 'TAGTTGCGAGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M63Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M63Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Right palm', 'sample442', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M64Plml', 'TCACTATGGTCA', 'CATGCTGCCTCCCGTAGGAGT', 'M64Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Left palm', 'sample220', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F23Plml', 'GCATCGTCAACA', 'CATGCTGCCTCCCGTAGGAGT', 'F23Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left palm', 'sample195', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M44Plml', 'GTTGTATACTCG', 'CATGCTGCCTCCCGTAGGAGT', 'M44Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left palm', 'sample216', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F23Plmr', 'GCATATAGTCTC', 'CATGCTGCCTCCCGTAGGAGT', 'F23Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right palm', 'sample418', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M23Nost', 'GATCGTCCAGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M23Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample287', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M54Plmr', 'TAGATCCTCGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M54Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Right palm', 'sample441', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M53Plml', 'TACGGTATGTCT', 'CATGCTGCCTCCCGTAGGAGT', 'M53Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Left palm', 'sample217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M44Plmr', 'GTTGACGACAGC', 'CATGCTGCCTCCCGTAGGAGT', 'M44Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right palm', 'sample439', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M43Plml', 'GTGATAGTGCCG', 'CATGCTGCCTCCCGTAGGAGT', 'M43Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left palm', 'sample215', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M63Plml', 'TATACGCGCATT', 'CATGCTGCCTCCCGTAGGAGT', 'M63Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M63Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Left palm', 'sample219', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M53Fotr', 'TACTGCGACAGT', 'CATGCTGCCTCCCGTAGGAGT', 'M53Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Right foot surface', 'sample472', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F14Plmr', 'GAGCATTCTCTA', 'CATGCTGCCTCCCGTAGGAGT', 'F14Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F14Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right palm', 'sample415', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F24Plmr', 'GCGTATCTTGAT', 'CATGCTGCCTCCCGTAGGAGT', 'F24Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right palm', 'sample419', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Plmr', 'CGTTACTAGAGC', 'CATGCTGCCTCCCGTAGGAGT', 'M31Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Plmr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right palm', 'sample432', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Knel', 'ATGTGCACGACT', 'CATGCTGCCTCCCGTAGGAGT', 'F22Knel', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Knel', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left popliteal fossa', 'sample110', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'left back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Knel', 'CGCTAGAACGCA', 'CATGCTGCCTCCCGTAGGAGT', 'M31Knel', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Knel', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left popliteal fossa', 'sample117', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'left back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Mout', 'CACATTGTGAGC', 'CATGCTGCCTCCCGTAGGAGT', 'F22Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Mout', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Oral cavity', 'sample256', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F21Navl', 'ATGACCATCGTG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Navl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Navl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Umbilicus', 'sample269', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of abdomen', '2008-2009', 'human skin metagenome', 'navel', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'umbilicus', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Plmr', 'CACTCAACAGAC', 'CATGCTGCCTCCCGTAGGAGT', 'F22Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Plmr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right palm', 'sample417', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Plmr', 'ATACACGTGGCG', 'CATGCTGCCTCCCGTAGGAGT', 'F12Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Plmr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right palm', 'sample413', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Indr', 'CACAGTGGACGT', 'CATGCTGCCTCCCGTAGGAGT', 'F22Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Indr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right index finger', 'sample360', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Plml', 'CGTGTGATCAGG', 'CATGCTGCCTCCCGTAGGAGT', 'M31Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Plml', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left palm', 'sample209', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Indr', 'CGTCAGACGGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M31Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Indr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right index finger', 'sample367', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Indl', 'AGTGAGAGAAGC', 'CATGCTGCCTCCCGTAGGAGT', 'F12Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Indl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left index finger', 'sample136', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Plml', 'ATAATCTCGTCG', 'CATGCTGCCTCCCGTAGGAGT', 'F12Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Plml', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left palm', 'sample190', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Fotl', 'ATGGATACGCTC', 'CATGCTGCCTCCCGTAGGAGT', 'F21Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Fotl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left foot surface', 'sample225', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Mout', 'CTCCTACTGTCT', 'CATGCTGCCTCCCGTAGGAGT', 'M32Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Mout', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Oral cavity', 'sample264', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F12Pinl', 'ATACAGAGCTCC', 'CATGCTGCCTCCCGTAGGAGT', 'F12Pinl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Pinl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of left pinna', 'sample164', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'left outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'left lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Nstl', 'AGTTCAGACGCT', 'CATGCTGCCTCCCGTAGGAGT', 'F12Nstl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Nstl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample150', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'left nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'left naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M21Indr', 'CATAGACGTTCG', 'CATGCTGCCTCCCGTAGGAGT', 'M21Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Indr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right index finger', 'sample365', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Mout', 'CATAGCGAGTTC', 'CATGCTGCCTCCCGTAGGAGT', 'M21Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Mout', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Oral cavity', 'sample261', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M42Indl', 'GAGAGCTCTACG', 'CATGCTGCCTCCCGTAGGAGT', 'M42Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Indl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left index finger', 'sample148', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Indl', 'CATACCAGTAGC', 'CATGCTGCCTCCCGTAGGAGT', 'M21Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Indl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left index finger', 'sample143', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Plmr', 'GCTGTGTAGGAC', 'CATGCTGCCTCCCGTAGGAGT', 'F32Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Plmr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right palm', 'sample421', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F31Mout', 'GCACTCGTTAGA', 'CATGCTGCCTCCCGTAGGAGT', 'F31Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Mout', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Oral cavity', 'sample257', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M42Frhd', 'GAGAATACGTGA', 'CATGCTGCCTCCCGTAGGAGT', 'M42Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Frhd', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Forehead', 'sample058', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Nstr', 'CATCATGAGGCT', 'CATGCTGCCTCCCGTAGGAGT', 'M21Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Nstr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample379', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M21Plml', 'CATCGTATCAAC', 'CATGCTGCCTCCCGTAGGAGT', 'M21Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Plml', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left palm', 'sample205', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Nstl', 'GCTGCTGCAATA', 'CATGCTGCCTCCCGTAGGAGT', 'F32Nstl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Nstl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample154', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'left nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M21Forl', 'CAGTCACTAACG', 'CATGCTGCCTCCCGTAGGAGT', 'M21Forl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Forl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Surface of left arm', 'sample129', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'left forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left volar forearm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Plml', 'GAGTATGCAGCC', 'CATGCTGCCTCCCGTAGGAGT', 'M42Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Plml', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left palm', 'sample214', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Indr', 'GCTATCACGAGT', 'CATGCTGCCTCCCGTAGGAGT', 'F32Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Indr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right index finger', 'sample362', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Nstr', 'GCTGGTATCTGA', 'CATGCTGCCTCCCGTAGGAGT', 'F32Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Nstr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample376', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F32Forl', 'GCGTATCTTGAT', 'CATGCTGCCTCCCGTAGGAGT', 'F32Forl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Forl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of left arm', 'sample126', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'left forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left volar forearm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Mout', 'GCTCAGTGCAGA', 'CATGCTGCCTCCCGTAGGAGT', 'F32Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Mout', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Oral cavity', 'sample258', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M22Forl', 'CGAAGACTGCTG', 'CATGCTGCCTCCCGTAGGAGT', 'M22Forl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Forl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Surface of left arm', 'sample130', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'left forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left volar forearm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Indl', 'GCTAGTCTGAAC', 'CATGCTGCCTCCCGTAGGAGT', 'F32Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Indl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left index finger', 'sample140', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Indr', 'CGAGCAGCACAT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Indr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right index finger', 'sample366', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Plml', 'GCTGTAGTATGC', 'CATGCTGCCTCCCGTAGGAGT', 'F32Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Plml', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left palm', 'sample198', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Forr', 'CAGTCGAAGCTG', 'CATGCTGCCTCCCGTAGGAGT', 'M21Forr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Forr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Surface of right arm', 'sample351', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'right forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right volar forearm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Mout', 'CGAGGCTCAGTA', 'CATGCTGCCTCCCGTAGGAGT', 'M22Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Mout', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Oral cavity', 'sample262', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M42Mout', 'GAGCAGATGCCT', 'CATGCTGCCTCCCGTAGGAGT', 'M42Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Mout', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Oral cavity', 'sample266', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M53Fotl', 'TACTGGACGCGA', 'CATGCTGCCTCCCGTAGGAGT', 'M53Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Left foot surface', 'sample249', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F33Plml', 'GTATATCCGCAG', 'CATGCTGCCTCCCGTAGGAGT', 'F33Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left palm', 'sample199', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F31Fotr', 'GCATCGTCAACA', 'CATGCTGCCTCCCGTAGGAGT', 'F31Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Fotr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample452', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Nose', 'GCTGATGAGCTG', 'CATGCTGCCTCCCGTAGGAGT', 'F32Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Nose', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External nose', 'sample024', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M53Fcsw', 'TAGATAGCAGGA', 'CATGCTGCCTCCCGTAGGAGT', 'M53Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Feces', 'sample517', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M14Fcsw', 'GACTAGACCAGC', 'CATGCTGCCTCCCGTAGGAGT', 'M14Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample499', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M33Fcsw', 'GGTATACGCAGC', 'CATGCTGCCTCCCGTAGGAGT', 'M33Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M33Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample510', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M13Fcsw', 'GACAGCGTTGAC', 'CATGCTGCCTCCCGTAGGAGT', 'M13Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M13Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample498', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M63Fcsw', 'TCACAGATCCGA', 'CATGCTGCCTCCCGTAGGAGT', 'M63Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M63Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Feces', 'sample519', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M24Fcsw', 'GCATAGTAGCCG', 'CATGCTGCCTCCCGTAGGAGT', 'M24Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M24Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample505', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M54Fcsw', 'TAGTGTGCTTCA', 'CATGCTGCCTCCCGTAGGAGT', 'M54Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Feces', 'sample518', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F21Fcsw', 'ATCGATCTGTGG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Fcsw', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample483', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F22Fcsw', 'CAACTATCAGCT', 'CATGCTGCCTCCCGTAGGAGT', 'F22Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Fcsw', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample485', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M31Fcsw', 'CGTACAGTTATC', 'CATGCTGCCTCCCGTAGGAGT', 'M31Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Fcsw', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample507', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M22Nstl', 'CGATAGATCTTC', 'CATGCTGCCTCCCGTAGGAGT', 'M22Nstl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Nstl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample158', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'left nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M22Indl', 'CGAGAGTTACGC', 'CATGCTGCCTCCCGTAGGAGT', 'M22Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Indl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left index finger', 'sample144', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Fcsp', 'CAGGTGCTACTA', 'CATGCTGCCTCCCGTAGGAGT', 'M21Fcsp', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Fcsp', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample500', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F32Fcsw', 'GCGTACAACTGT', 'CATGCTGCCTCCCGTAGGAGT', 'F32Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Fcsw', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample491', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M41Fcsw', 'CTGTGGATCGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M41Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Fcsw', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample512', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M22Fcsw', 'CCTCTCGTGATC', 'CATGCTGCCTCCCGTAGGAGT', 'M22Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Fcsw', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample503', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M42Fcsw', 'GACTGCATCTTA', 'CATGCTGCCTCCCGTAGGAGT', 'M42Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Fcsw', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample514', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M34Tong', 'GTAGATGCTTCG', 'CATGCTGCCTCCCGTAGGAGT', 'M34Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M34Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample544', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M44Urin', 'TACATCACCACA', 'CATGCTGCCTCCCGTAGGAGT', 'M44Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample594', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M53Tong', 'TACTTCGCTCGC', 'CATGCTGCCTCCCGTAGGAGT', 'M53Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample549', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M43Frhd', 'GTGCAATCGACG', 'CATGCTGCCTCCCGTAGGAGT', 'M43Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Forehead', 'sample059', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M54Tong', 'TAGGTATCTCAC', 'CATGCTGCCTCCCGTAGGAGT', 'M54Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample550', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F23Tong', 'GCGAGATCCAGT', 'CATGCTGCCTCCCGTAGGAGT', 'F23Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample527', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M53Uric', 'TAGAGAGAGTGG', 'CATGCTGCCTCCCGTAGGAGT', 'M53Uric', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Uric', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Urine', 'sample595', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M54Uric', 'TAGTGCTGCGTA', 'CATGCTGCCTCCCGTAGGAGT', 'M54Uric', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Uric', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Urine', 'sample597', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M23Ewax', 'GATGATCGCCGA', 'CATGCTGCCTCCCGTAGGAGT', 'M23Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:External auditory canal', 'sample305', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F23Knee', 'GCCACTGATAGT', 'CATGCTGCCTCCCGTAGGAGT', 'F23Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Popliteal fossa', 'sample003', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F34Fotr', 'GTCTCATGTAGG', 'CATGCTGCCTCCCGTAGGAGT', 'F34Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample455', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M64Fotl', 'TCAGCCATGACA', 'CATGCTGCCTCCCGTAGGAGT', 'M64Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Left foot surface', 'sample252', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M14Tong', 'GACGCTAGTTCA', 'CATGCTGCCTCCCGTAGGAGT', 'M14Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample536', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M44Knee', 'TAAGCGCAGCAC', 'CATGCTGCCTCCCGTAGGAGT', 'M44Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Popliteal fossa', 'sample014', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F24Frhd', 'GCTAAGAGAGTA', 'CATGCTGCCTCCCGTAGGAGT', 'F24Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Forehead', 'sample040', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M44Uric', 'TACCGCTAGTAG', 'CATGCTGCCTCCCGTAGGAGT', 'M44Uric', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Uric', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample593', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F23Nost', 'GCATTGCGTGAG', 'CATGCTGCCTCCCGTAGGAGT', 'F23Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample281', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M33Nost', 'GCTTCATAGTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M33Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M33Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample289', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F24Nost', 'GCTAGATGCCAG', 'CATGCTGCCTCCCGTAGGAGT', 'F24Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample282', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M13Plml', 'CTTAGCACATCA', 'CATGCTGCCTCCCGTAGGAGT', 'M13Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M13Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left palm', 'sample203', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Indl', 'ATCTCTGGCATA', 'CATGCTGCCTCCCGTAGGAGT', 'F21Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Indl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left index finger', 'sample137', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M23Plml', 'GATCCGACACTA', 'CATGCTGCCTCCCGTAGGAGT', 'M23Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left palm', 'sample207', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M54Fotr', 'TAGCGGATCACG', 'CATGCTGCCTCCCGTAGGAGT', 'M54Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Right foot surface', 'sample473', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Tong', 'ATGGCGTGCACA', 'CATGCTGCCTCCCGTAGGAGT', 'F21Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Tong', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample525', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F21Mout', 'ATCTTAGACTGC', 'CATGCTGCCTCCCGTAGGAGT', 'F21Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Mout', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Oral cavity', 'sample255', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F21Hair', 'ATCTACTACACG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Hair', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Hair', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Head hair', 'sample075', 'False', 'UBERON:hair', 'UBERON:hair', 'UBERON:hair', '2008-2009', 'human hair metagenome', 'hair on head', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'hair on head', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human hair metagenome'], ['449.F21Forr', 'ATCGCTCGAGGA', 'CATGCTGCCTCCCGTAGGAGT', 'F21Forr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Forr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of right arm', 'sample345', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'right forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right volar forearm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Mout', 'CGTCGATCTCTC', 'CATGCTGCCTCCCGTAGGAGT', 'M31Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Mout', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Oral cavity', 'sample263', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F21Forl', 'ATCGCGGACGAT', 'CATGCTGCCTCCCGTAGGAGT', 'F21Forl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Forl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of left arm', 'sample123', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'left forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left volar forearm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Indr', 'AGTGCGATGCGT', 'CATGCTGCCTCCCGTAGGAGT', 'F12Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Indr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right index finger', 'sample358', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Ewxl', 'CTAGCGAACATC', 'CATGCTGCCTCCCGTAGGAGT', 'M32Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Ewxl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left external auditory canal', 'sample186', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M31Fotr', 'CTACACAAGCAC', 'CATGCTGCCTCCCGTAGGAGT', 'M31Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Fotr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample464', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Tong', 'CTACATCTAAGC', 'CATGCTGCCTCCCGTAGGAGT', 'M31Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Tong', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample541', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M32Tong', 'CTGAGCAGAGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M32Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Tong', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample542', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F22Forl', 'CAACTCATCGTA', 'CATGCTGCCTCCCGTAGGAGT', 'F22Forl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Forl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of left arm', 'sample124', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'left forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left volar forearm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Nose', 'CTCGATTAGATC', 'CATGCTGCCTCCCGTAGGAGT', 'M32Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Nose', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External nose', 'sample030', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Plmr', 'CTCTGAAGTCTA', 'CATGCTGCCTCCCGTAGGAGT', 'M32Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Plmr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right palm', 'sample433', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Hair', 'CGTCAACGATGT', 'CATGCTGCCTCCCGTAGGAGT', 'M31Hair', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Hair', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Head hair', 'sample083', 'False', 'UBERON:hair', 'UBERON:hair', 'UBERON:hair', '2008-2009', 'human hair metagenome', 'hair on head', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'hair on head', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human hair metagenome'], ['449.F12Aptl', 'AGCTGACTAGTC', 'CATGCTGCCTCCCGTAGGAGT', 'F12Aptl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Aptl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left axilla', 'sample094', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'left armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'left axilla', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Indl', 'CACAGCTCGAAT', 'CATGCTGCCTCCCGTAGGAGT', 'F22Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Indl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left index finger', 'sample138', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Nstl', 'ATGAGACTCCAC', 'CATGCTGCCTCCCGTAGGAGT', 'F21Nstl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Nstl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample151', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'left nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F21Plmr', 'ATGCAGCTCAGT', 'CATGCTGCCTCCCGTAGGAGT', 'F21Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Plmr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right palm', 'sample416', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Pinl', 'CTCTGCTAGCCT', 'CATGCTGCCTCCCGTAGGAGT', 'M32Pinl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Pinl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of left pinna', 'sample173', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'left outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Tong', 'CAGACTCGCAGA', 'CATGCTGCCTCCCGTAGGAGT', 'F22Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Tong', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample526', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M32Hair', 'CTCAGTATGCAG', 'CATGCTGCCTCCCGTAGGAGT', 'M32Hair', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Hair', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Head hair', 'sample084', 'False', 'UBERON:hair', 'UBERON:hair', 'UBERON:hair', '2008-2009', 'human hair metagenome', 'hair on head', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'hair on head', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human hair metagenome'], ['449.F12Mout', 'AGTGGATGCTCT', 'CATGCTGCCTCCCGTAGGAGT', 'F12Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Mout', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Oral cavity', 'sample254', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F21Aptr', 'ATATGCCAGTGC', 'CATGCTGCCTCCCGTAGGAGT', 'F21Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Aptr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right axilla', 'sample317', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Plml', 'CACTACTGTTGA', 'CATGCTGCCTCCCGTAGGAGT', 'F22Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Plml', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left palm', 'sample194', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Hair', 'CACACGTGAGCA', 'CATGCTGCCTCCCGTAGGAGT', 'F22Hair', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Hair', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Head hair', 'sample076', 'False', 'UBERON:hair', 'UBERON:hair', 'UBERON:hair', '2008-2009', 'human hair metagenome', 'hair on head', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'hair on head', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human hair metagenome'], ['449.M41Urin', 'GACGCAGTAGCT', 'CATGCTGCCTCCCGTAGGAGT', 'M41Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Urin', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample565', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M41Tong', 'GACGATATCGCG', 'CATGCTGCCTCCCGTAGGAGT', 'M41Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Tong', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample545', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M42Indr', 'GAGATGCCGACT', 'CATGCTGCCTCCCGTAGGAGT', 'M42Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Indr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right index finger', 'sample370', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Tong', 'GATAGTGCCACT', 'CATGCTGCCTCCCGTAGGAGT', 'M42Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Tong', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample546', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M41Nose', 'GAATGATGAGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M41Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Nose', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:External nose', 'sample031', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Urin', 'GATATGCGGCTG', 'CATGCTGCCTCCCGTAGGAGT', 'M42Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Urin', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample566', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M42Plmr', 'GAGTCTGAGTCT', 'CATGCTGCCTCCCGTAGGAGT', 'M42Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Plmr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right palm', 'sample437', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Nose', 'CGAGTTGTAGCG', 'CATGCTGCCTCCCGTAGGAGT', 'M22Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Nose', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:External nose', 'sample028', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Nstr', 'CGATATTCATCG', 'CATGCTGCCTCCCGTAGGAGT', 'M22Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Nstr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample380', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M21Ewxr', 'CAGCTAGAACGC', 'CATGCTGCCTCCCGTAGGAGT', 'M21Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Ewxr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right external auditory canal', 'sample406', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M41Plml', 'GACAGGAGATAG', 'CATGCTGCCTCCCGTAGGAGT', 'M41Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Plml', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left palm', 'sample213', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Hair', 'GAGAGAATGATC', 'CATGCTGCCTCCCGTAGGAGT', 'M42Hair', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Hair', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Head hair', 'sample086', 'False', 'UBERON:hair', 'UBERON:hair', 'UBERON:hair', '2008-2009', 'human hair metagenome', 'hair on head', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'hair on head', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human hair metagenome'], ['449.M21Pinl', 'CATGAGTGCTAC', 'CATGCTGCCTCCCGTAGGAGT', 'M21Pinl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Pinl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Surface of left pinna', 'sample170', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'left outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Knel', 'CAGCACTAAGCG', 'CATGCTGCCTCCCGTAGGAGT', 'M21Knel', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Knel', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left popliteal fossa', 'sample115', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'left back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Frhd', 'CAGTGATCCTAG', 'CATGCTGCCTCCCGTAGGAGT', 'M21Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Frhd', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Forehead', 'sample049', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F31Ewxr', 'GATCTCATAGGC', 'CATGCTGCCTCCCGTAGGAGT', 'F31Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Ewxr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right external auditory canal', 'sample403', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M42Nose', 'GAGCTGGCTGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M42Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Nose', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:External nose', 'sample032', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Pinl', 'GAGTGAGTACAA', 'CATGCTGCCTCCCGTAGGAGT', 'M42Pinl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Pinl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of left pinna', 'sample175', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'left outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Navl', 'GAAGTCTCGCAT', 'CATGCTGCCTCCCGTAGGAGT', 'M41Navl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Navl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Umbilicus', 'sample277', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of abdomen', '2008-2009', 'human skin metagenome', 'navel', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'umbilicus', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Navl', 'CATATACTCGCA', 'CATGCTGCCTCCCGTAGGAGT', 'M21Navl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Navl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Umbilicus', 'sample274', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of abdomen', '2008-2009', 'human skin metagenome', 'navel', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'umbilicus', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Mout', 'GAAGCTACTGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M41Mout', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Mout', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Oral cavity', 'sample265', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:mouth', '2008-2009', 'human oral metagenome', 'mouth', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'oral cavity', 'False', 'True', 'water_rinse', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M22Forr', 'CGAATCGACACT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Forr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Forr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Surface of right arm', 'sample352', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'right forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right volar forearm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F31Indl', 'GCAATAGCTGCT', 'CATGCTGCCTCCCGTAGGAGT', 'F31Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Indl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left index finger', 'sample139', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Forr', 'CTTAGCACATCA', 'CATGCTGCCTCCCGTAGGAGT', 'M41Forr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Forr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of right arm', 'sample355', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'right forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right volar forearm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Hair', 'CGACTTATGTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Hair', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Hair', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Head hair', 'sample082', 'False', 'UBERON:hair', 'UBERON:hair', 'UBERON:hair', '2008-2009', 'human hair metagenome', 'hair on head', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'hair on head', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human hair metagenome'], ['449.M22Ewxl', 'CCGACTGAGATG', 'CATGCTGCCTCCCGTAGGAGT', 'M22Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Ewxl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left external auditory canal', 'sample184', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M41Kner', 'CTGGCTGTATGA', 'CATGCTGCCTCCCGTAGGAGT', 'M41Kner', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Kner', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right popliteal fossa', 'sample341', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'right back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Pinl', 'GACATCGGCTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M41Pinl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Pinl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of left pinna', 'sample174', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'left outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F31Plmr', 'GCAGTATCACTG', 'CATGCTGCCTCCCGTAGGAGT', 'F31Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Plmr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right palm', 'sample420', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Forl', 'GACTGTCATGCA', 'CATGCTGCCTCCCGTAGGAGT', 'M42Forl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Forl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of left arm', 'sample134', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'left forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left volar forearm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Nstl', 'CATCAGCGTGTA', 'CATGCTGCCTCCCGTAGGAGT', 'M21Nstl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Nstl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample157', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'left nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M41Frhd', 'CTTGATGCGTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M41Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Frhd', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Forehead', 'sample057', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Plmr', 'CGATGCACCAGA', 'CATGCTGCCTCCCGTAGGAGT', 'M22Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Plmr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right palm', 'sample429', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F33Plmr', 'GTAGTGTCTAGC', 'CATGCTGCCTCCCGTAGGAGT', 'F33Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right palm', 'sample422', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M23Fotr', 'GATCTCATAGGC', 'CATGCTGCCTCCCGTAGGAGT', 'M23Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right foot surface', 'sample462', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F33Fotl', 'GTCAACGCGATG', 'CATGCTGCCTCCCGTAGGAGT', 'F33Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left foot surface', 'sample231', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F34Fotl', 'GTCTCTCTACGC', 'CATGCTGCCTCCCGTAGGAGT', 'F34Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left foot surface', 'sample232', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M11Fotl', 'ACCAGACGATGC', 'CATGCTGCCTCCCGTAGGAGT', 'M11Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M11Fotl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left foot surface', 'sample233', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Fotr', 'CGCAGACAGACT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Fotr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right foot surface', 'sample461', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Knel', 'GACTAACGTCAC', 'CATGCTGCCTCCCGTAGGAGT', 'M42Knel', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Knel', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left popliteal fossa', 'sample120', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'left back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Fotl', 'CATGGCTACACA', 'CATGCTGCCTCCCGTAGGAGT', 'M21Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Fotl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left foot surface', 'sample237', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Fotr', 'CATGTAATGCTC', 'CATGCTGCCTCCCGTAGGAGT', 'M21Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Fotr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right foot surface', 'sample460', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F31Indr', 'GCACATCGAGCA', 'CATGCTGCCTCCCGTAGGAGT', 'F31Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Indr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right index finger', 'sample361', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F14Fotr', 'GAGTCTGAGTCT', 'CATGCTGCCTCCCGTAGGAGT', 'F14Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F14Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample447', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F13Fotr', 'GACTTCAGTGTG', 'CATGCTGCCTCCCGTAGGAGT', 'F13Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F13Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample446', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Fotl', 'ATACTATTGCGC', 'CATGCTGCCTCCCGTAGGAGT', 'F12Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Fotl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left foot surface', 'sample222', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Glns', 'GAGACAGCTTGC', 'CATGCTGCCTCCCGTAGGAGT', 'M42Glns', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Glns', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Glans penis', 'sample072', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:glans penis', '2008-2009', 'human skin metagenome', 'glans penis', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'glans penis', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M64Plmr', 'TCACGATTAGCG', 'CATGCTGCCTCCCGTAGGAGT', 'M64Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Right palm', 'sample443', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Fotr', 'CTGAGATACGCG', 'CATGCTGCCTCCCGTAGGAGT', 'M32Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Fotr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample465', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F34Plmr', 'GTCGCTGTCTTC', 'CATGCTGCCTCCCGTAGGAGT', 'F34Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Plmr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right palm', 'sample423', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M23Frhd', 'GATCGCAGGTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M23Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Forehead', 'sample051', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F13Plml', 'GACTCGAATCGT', 'CATGCTGCCTCCCGTAGGAGT', 'F13Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F13Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left palm', 'sample191', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M43Nost', 'GTGCACATTATC', 'CATGCTGCCTCCCGTAGGAGT', 'M43Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample291', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M64Knee', 'TCAGACAGACCG', 'CATGCTGCCTCCCGTAGGAGT', 'M64Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Popliteal fossa', 'sample018', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F33Nost', 'GTATGACTGGCT', 'CATGCTGCCTCCCGTAGGAGT', 'F33Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample283', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F23Frhd', 'GCATGTGCATGT', 'CATGCTGCCTCCCGTAGGAGT', 'F23Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Forehead', 'sample039', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F13Knee', 'GACTGTCATGCA', 'CATGCTGCCTCCCGTAGGAGT', 'F13Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F13Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Popliteal fossa', 'sample001', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M44Frhd', 'TAACAGTCGCTG', 'CATGCTGCCTCCCGTAGGAGT', 'M44Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Forehead', 'sample060', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F34Frhd', 'GTCGTGTGTCAA', 'CATGCTGCCTCCCGTAGGAGT', 'F34Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Forehead', 'sample044', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F14Frhd', 'GAGGCTCATCAT', 'CATGCTGCCTCCCGTAGGAGT', 'F14Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F14Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Forehead', 'sample036', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M44Ewax', 'TACAGATGGCTC', 'CATGCTGCCTCCCGTAGGAGT', 'M44Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:External auditory canal', 'sample310', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M63Frhd', 'TATCAGGTGTGC', 'CATGCTGCCTCCCGTAGGAGT', 'M63Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M63Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Forehead', 'sample063', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M24Nost', 'GCACTCGTTAGA', 'CATGCTGCCTCCCGTAGGAGT', 'M24Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M24Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample288', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F34Nost', 'GTCTACACACAT', 'CATGCTGCCTCCCGTAGGAGT', 'F34Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample284', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M24Knee', 'GCACTGAGACGT', 'CATGCTGCCTCCCGTAGGAGT', 'M24Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M24Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Popliteal fossa', 'sample010', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M64Frhd', 'TCACTGGCAGTA', 'CATGCTGCCTCCCGTAGGAGT', 'M64Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Forehead', 'sample064', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F11Pinr', 'TGCGCGAATACT', 'CATGCTGCCTCCCGTAGGAGT', 'F11Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F11Pinr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of right pinna', 'sample385', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M44Nost', 'TAACTCTGATGC', 'CATGCTGCCTCCCGTAGGAGT', 'M44Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample292', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F13Ewax', 'GAGACAGCTTGC', 'CATGCTGCCTCCCGTAGGAGT', 'F13Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F13Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External auditory canal', 'sample297', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M31Pinl', 'CGTTATGTACAC', 'CATGCTGCCTCCCGTAGGAGT', 'M31Pinl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Pinl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of left pinna', 'sample172', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'left outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Forl', 'CTATCAGTGTAC', 'CATGCTGCCTCCCGTAGGAGT', 'M32Forl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Forl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of left arm', 'sample132', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'left forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left volar forearm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Nose', 'AGTGTTCGATCG', 'CATGCTGCCTCCCGTAGGAGT', 'F12Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Nose', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External nose', 'sample020', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Pinr', 'ATACGTCTTCGA', 'CATGCTGCCTCCCGTAGGAGT', 'F12Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Pinr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of right pinna', 'sample386', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Pinr', 'CGTTCGCATAGA', 'CATGCTGCCTCCCGTAGGAGT', 'M31Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Pinr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of right pinna', 'sample395', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Pinl', 'CACTCTGATTAG', 'CATGCTGCCTCCCGTAGGAGT', 'F22Pinl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Pinl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of left pinna', 'sample165', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'left outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Knel', 'ATCACGTAGCGG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Knel', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Knel', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left popliteal fossa', 'sample109', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'left back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Frhd', 'CTATGCTTGATG', 'CATGCTGCCTCCCGTAGGAGT', 'M32Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Frhd', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Forehead', 'sample054', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Ewxr', 'ATTCTGTGAGCG', 'CATGCTGCCTCCCGTAGGAGT', 'F22Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Ewxr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right external auditory canal', 'sample402', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F12Navl', 'AGTGTCACGGTG', 'CATGCTGCCTCCCGTAGGAGT', 'F12Navl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Navl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Umbilicus', 'sample268', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of abdomen', '2008-2009', 'human skin metagenome', 'navel', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'umbilicus', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Hair', 'GCTAGATGCCAG', 'CATGCTGCCTCCCGTAGGAGT', 'F32Hair', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Hair', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Head hair', 'sample078', 'False', 'UBERON:hair', 'UBERON:hair', 'UBERON:hair', '2008-2009', 'human hair metagenome', 'hair on head', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'hair on head', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human hair metagenome'], ['449.F31Forr', 'GATGTCGTGTCA', 'CATGCTGCCTCCCGTAGGAGT', 'F31Forr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Forr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of right arm', 'sample347', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'right forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right volar forearm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Ewxr', 'CCGATGTCAGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Ewxr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right external auditory canal', 'sample407', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F31Knel', 'GATCGCAGGTGT', 'CATGCTGCCTCCCGTAGGAGT', 'F31Knel', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Knel', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left popliteal fossa', 'sample111', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'left back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Navl', 'CGAGTCTAGTTG', 'CATGCTGCCTCCCGTAGGAGT', 'M22Navl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Navl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Umbilicus', 'sample275', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of abdomen', '2008-2009', 'human skin metagenome', 'navel', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'umbilicus', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F31Frhd', 'GATGTGAGCGCT', 'CATGCTGCCTCCCGTAGGAGT', 'F31Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Frhd', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Forehead', 'sample041', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Fotl', 'CGCACTCTAGAA', 'CATGCTGCCTCCCGTAGGAGT', 'M22Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Fotl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left foot surface', 'sample238', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Aptr', 'CAGATCGGATCG', 'CATGCTGCCTCCCGTAGGAGT', 'M21Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Aptr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right axilla', 'sample323', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Forr', 'GCGTTACACACA', 'CATGCTGCCTCCCGTAGGAGT', 'F32Forr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Forr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of right arm', 'sample348', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'right forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right volar forearm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Frhd', 'CGACAGCTGACA', 'CATGCTGCCTCCCGTAGGAGT', 'M22Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Frhd', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Forehead', 'sample050', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Plmr', 'GACAGTTACTGC', 'CATGCTGCCTCCCGTAGGAGT', 'M41Plmr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Plmr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right palm', 'sample436', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'right palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M22Pinl', 'CGATGTCGTCAA', 'CATGCTGCCTCCCGTAGGAGT', 'M22Pinl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Pinl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Surface of left pinna', 'sample171', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'left outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Pinr', 'GACCACTACGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M41Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Pinr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of right pinna', 'sample397', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F31Fotl', 'GCATATAGTCTC', 'CATGCTGCCTCCCGTAGGAGT', 'F31Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Fotl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left foot surface', 'sample229', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Pinr', 'CATGCAGACTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M21Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Pinr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Surface of right pinna', 'sample393', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Indr', 'CTCCACATGAGA', 'CATGCTGCCTCCCGTAGGAGT', 'M32Indr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Indr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right index finger', 'sample368', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'right index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F13Tong', 'GAGAGAATGATC', 'CATGCTGCCTCCCGTAGGAGT', 'F13Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F13Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample523', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M54Urin', 'TAGTCGTCTAGT', 'CATGCTGCCTCCCGTAGGAGT', 'M54Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Urine', 'sample598', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M23Fcsw', 'GATTAGCACTCT', 'CATGCTGCCTCCCGTAGGAGT', 'M23Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample504', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F24Fcsw', 'GCTGGTATCTGA', 'CATGCTGCCTCCCGTAGGAGT', 'F24Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample487', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F23Fcsw', 'GCGTACAACTGT', 'CATGCTGCCTCCCGTAGGAGT', 'F23Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample486', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M21Fcsw', 'CAGTACGATCTT', 'CATGCTGCCTCCCGTAGGAGT', 'M21Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Fcsw', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample501', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M22Fcsp', 'CCTAGTACTGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Fcsp', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Fcsp', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample502', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M11Fcsp', 'AATCGTGACTCG', 'CATGCTGCCTCCCGTAGGAGT', 'M11Fcsp', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M11Fcsp', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Feces', 'sample494', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M32Nstl', 'CTCGCACATATA', 'CATGCTGCCTCCCGTAGGAGT', 'M32Nstl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Nstl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample160', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'left nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F31Labi', 'GCACGACAACAC', 'CATGCTGCCTCCCGTAGGAGT', 'F31Labi', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Labi', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Frenulum of labia minora', 'sample091', 'False', 'UBERON:skin', 'UBERON:mucus', 'UBERON:labia minora', '2008-2009', 'human skin metagenome', 'labia minora', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:mucus', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'labia minora', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Labi', 'GCTATTCGACAT', 'CATGCTGCCTCCCGTAGGAGT', 'F32Labi', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Labi', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Frenulum of labia minora', 'sample092', 'False', 'UBERON:skin', 'UBERON:mucus', 'UBERON:labia minora', '2008-2009', 'human skin metagenome', 'labia minora', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:mucus', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'labia minora', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M34Plml', 'GGTCGTAGCGTA', 'CATGCTGCCTCCCGTAGGAGT', 'M34Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M34Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left palm', 'sample212', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M53Knee', 'TACTACATGGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M53Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Popliteal fossa', 'sample015', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M64Fotr', 'TCAGATCCGATG', 'CATGCTGCCTCCCGTAGGAGT', 'M64Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Right foot surface', 'sample475', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M14Fotr', 'GACGAGTCAGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M14Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right foot surface', 'sample459', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F24Plml', 'GCGTTACACACA', 'CATGCTGCCTCCCGTAGGAGT', 'F24Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left palm', 'sample196', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Forr', 'CTATCTAGCGAG', 'CATGCTGCCTCCCGTAGGAGT', 'M32Forr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Forr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of right arm', 'sample354', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'right forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right volar forearm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M12Ewxr', 'ACGCGATACTGG', 'CATGCTGCCTCCCGTAGGAGT', 'M12Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M12Ewxr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right external auditory canal', 'sample405', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M32Plml', 'CTCTCTACCTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M32Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Plml', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left palm', 'sample210', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Kner', 'GCGACTTGTGTA', 'CATGCTGCCTCCCGTAGGAGT', 'F32Kner', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Kner', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right popliteal fossa', 'sample334', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'right back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Indl', 'CGTCACGACTAA', 'CATGCTGCCTCCCGTAGGAGT', 'M31Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Indl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left index finger', 'sample145', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Nose', 'CGTGATCTCTCC', 'CATGCTGCCTCCCGTAGGAGT', 'M31Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Nose', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External nose', 'sample029', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Indl', 'CTCATGTACAGT', 'CATGCTGCCTCCCGTAGGAGT', 'M32Indl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Indl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left index finger', 'sample146', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of finger', '2008-2009', 'human skin metagenome', 'left index finger', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left palmar index finger', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M21Ewxl', 'CAGCGGTGACAT', 'CATGCTGCCTCCCGTAGGAGT', 'M21Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Ewxl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left external auditory canal', 'sample183', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M22Tong', 'CGCAGCGGTATA', 'CATGCTGCCTCCCGTAGGAGT', 'M22Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Tong', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample538', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F31Forl', 'GATGCATGACGC', 'CATGCTGCCTCCCGTAGGAGT', 'F31Forl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Forl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Surface of left arm', 'sample125', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of forearm', '2008-2009', 'human skin metagenome', 'left forearm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'left volar forearm', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F13Fcsw', 'GAGCAGATGCCT', 'CATGCTGCCTCCCGTAGGAGT', 'F13Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F13Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample480', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M41Knel', 'CTGGAGCATGAC', 'CATGCTGCCTCCCGTAGGAGT', 'M41Knel', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Knel', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left popliteal fossa', 'sample119', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'left back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F33Fcsw', 'GTCGACTCCTCT', 'CATGCTGCCTCCCGTAGGAGT', 'F33Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample492', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F34Fcsw', 'GTGACTGCGGAT', 'CATGCTGCCTCCCGTAGGAGT', 'F34Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample493', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M32Fcsp', 'CTAGTCAGCTGA', 'CATGCTGCCTCCCGTAGGAGT', 'M32Fcsp', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Fcsp', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample508', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F22Fcsp', 'CAACACGCACGA', 'CATGCTGCCTCCCGTAGGAGT', 'F22Fcsp', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Fcsp', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample484', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F32Fcsp', 'GCGGATGTGACT', 'CATGCTGCCTCCCGTAGGAGT', 'F32Fcsp', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Fcsp', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample490', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F31Fcsw', 'GATGATCGCCGA', 'CATGCTGCCTCCCGTAGGAGT', 'F31Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Fcsw', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample489', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M64Fcsw', 'TCAGTGACGTAC', 'CATGCTGCCTCCCGTAGGAGT', 'M64Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Feces', 'sample520', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F31Fcsp', 'GATCTTCAGTAC', 'CATGCTGCCTCCCGTAGGAGT', 'F31Fcsp', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Fcsp', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample488', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M44Fcsw', 'TACGATGACCAC', 'CATGCTGCCTCCCGTAGGAGT', 'M44Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample516', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M41Fcsp', 'CTGTGACATTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M41Fcsp', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Fcsp', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample511', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.F21Nstr', 'TCATCGCGATAT', 'CATGCTGCCTCCCGTAGGAGT', 'F21Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Nstr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample373', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M44Fotl', 'TACACGATCTAC', 'CATGCTGCCTCCCGTAGGAGT', 'M44Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left foot surface', 'sample248', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Labi', 'CACATCTAACAC', 'CATGCTGCCTCCCGTAGGAGT', 'F22Labi', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Labi', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Frenulum of labia minora', 'sample090', 'False', 'UBERON:skin', 'UBERON:mucus', 'UBERON:labia minora', '2008-2009', 'human skin metagenome', 'labia minora', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:mucus', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'labia minora', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Labi', 'ATCTGGTGCTAT', 'CATGCTGCCTCCCGTAGGAGT', 'F21Labi', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Labi', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Frenulum of labia minora', 'sample089', 'False', 'UBERON:skin', 'UBERON:mucus', 'UBERON:labia minora', '2008-2009', 'human skin metagenome', 'labia minora', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:mucus', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'labia minora', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Ewxr', 'ATCCGATCACAG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Ewxr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right external auditory canal', 'sample401', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F21Kner', 'ATCACTAGTCAC', 'CATGCTGCCTCCCGTAGGAGT', 'F21Kner', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Kner', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right popliteal fossa', 'sample331', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'right back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right popliteal fossa', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Urin', 'CAGAGGAGCTCT', 'CATGCTGCCTCCCGTAGGAGT', 'F22Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Urin', '0.006', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample556', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F21Urin', 'ATGGTCTACTAC', 'CATGCTGCCTCCCGTAGGAGT', 'F21Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Urin', '0.006', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample555', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M21Urin', 'CATTCGATGACT', 'CATGCTGCCTCCCGTAGGAGT', 'M21Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Urin', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Urine', 'sample563', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M22Urin', 'CGCATGAGGATC', 'CATGCTGCCTCCCGTAGGAGT', 'M22Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Urin', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Urine', 'sample564', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M13Frhd', 'CTTGATGCGTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M13Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M13Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Forehead', 'sample047', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M54Knee', 'TAGCGACATCTG', 'CATGCTGCCTCCCGTAGGAGT', 'M54Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Popliteal fossa', 'sample016', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F33Tong', 'GTCATATCGTAC', 'CATGCTGCCTCCCGTAGGAGT', 'F33Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample531', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M43Fotr', 'GTGTACCTATCA', 'CATGCTGCCTCCCGTAGGAGT', 'M43Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right foot surface', 'sample470', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M14Knee', 'GACCGAGCTATG', 'CATGCTGCCTCCCGTAGGAGT', 'M14Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Popliteal fossa', 'sample008', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Pinr', 'GAGTGGTAGAGA', 'CATGCTGCCTCCCGTAGGAGT', 'M42Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Pinr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of right pinna', 'sample398', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Fotr', 'GGACGTCACAGT', 'CATGCTGCCTCCCGTAGGAGT', 'F32Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Fotr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample453', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M43Fcsw', 'GTTCGCGTATAG', 'CATGCTGCCTCCCGTAGGAGT', 'M43Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Feces', 'sample515', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M33Frhd', 'GCTTACATCGAG', 'CATGCTGCCTCCCGTAGGAGT', 'M33Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M33Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Forehead', 'sample055', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M14Fotl', 'GACGATATCGCG', 'CATGCTGCCTCCCGTAGGAGT', 'M14Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left foot surface', 'sample236', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M43Knee', 'GTGGCGATACAC', 'CATGCTGCCTCCCGTAGGAGT', 'M43Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Popliteal fossa', 'sample013', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Labi', 'AGTTAGTGCGTC', 'CATGCTGCCTCCCGTAGGAGT', 'F12Labi', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Labi', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Frenulum of labia minora', 'sample088', 'False', 'UBERON:skin', 'UBERON:mucus', 'UBERON:labia minora', '2008-2009', 'human skin metagenome', 'labia minora', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:mucus', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'labia minora', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Nose', 'CACGGACTATAC', 'CATGCTGCCTCCCGTAGGAGT', 'F22Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Nose', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:External nose', 'sample022', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M12Frhd', 'ACGGTGAGTGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M12Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M12Frhd', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Forehead', 'sample046', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Nstl', 'GACACTCGAATC', 'CATGCTGCCTCCCGTAGGAGT', 'M41Nstl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Nstl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample161', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'left nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F32Navl', 'GCTCGCTACTTC', 'CATGCTGCCTCCCGTAGGAGT', 'F32Navl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Navl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Umbilicus', 'sample271', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of abdomen', '2008-2009', 'human skin metagenome', 'navel', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'umbilicus', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M14Nost', 'GACCACTACGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M14Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample286', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M14Frhd', 'GACATCGGCTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M14Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Forehead', 'sample048', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M12Fotl', 'ACTCTTCTAGAG', 'CATGCTGCCTCCCGTAGGAGT', 'M12Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M12Fotl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left foot surface', 'sample234', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F14Fcsw', 'GATATGCGGCTG', 'CATGCTGCCTCCCGTAGGAGT', 'F14Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F14Fcsw', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample481', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'female', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M32Nstr', 'CTCGTGGAGTAG', 'CATGCTGCCTCCCGTAGGAGT', 'M32Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Nstr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample382', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M33Tong', 'GGCGACATGTAC', 'CATGCTGCCTCCCGTAGGAGT', 'M33Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M33Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample543', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M53Urin', 'TAGACTGTACTC', 'CATGCTGCCTCCCGTAGGAGT', 'M53Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Urine', 'sample596', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M23Tong', 'GATGCATGACGC', 'CATGCTGCCTCCCGTAGGAGT', 'M23Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample539', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M43Tong', 'GTGTGTGTCAGG', 'CATGCTGCCTCCCGTAGGAGT', 'M43Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample547', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M64Tong', 'TCAGGACTGTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M64Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample552', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M13Tong', 'GAAGTCTCGCAT', 'CATGCTGCCTCCCGTAGGAGT', 'M13Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M13Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample535', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M63Tong', 'TATTCGTGTCAG', 'CATGCTGCCTCCCGTAGGAGT', 'M63Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M63Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample551', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F24Tong', 'GCTCGCTACTTC', 'CATGCTGCCTCCCGTAGGAGT', 'F24Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample528', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M12Tong', 'ACTGATCCTAGT', 'CATGCTGCCTCCCGTAGGAGT', 'M12Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M12Tong', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample534', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M11Tong', 'ACCGCAGAGTCA', 'CATGCTGCCTCCCGTAGGAGT', 'M11Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M11Tong', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample533', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M21Tong', 'CATGTCTCTCCG', 'CATGCTGCCTCCCGTAGGAGT', 'M21Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M21Tong', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample537', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F31Tong', 'GCATGTGCATGT', 'CATGCTGCCTCCCGTAGGAGT', 'F31Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Tong', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample529', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F24Fotr', 'GCTATCACGAGT', 'CATGCTGCCTCCCGTAGGAGT', 'F24Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right foot surface', 'sample451', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Fotr', 'ATACTCACTCAG', 'CATGCTGCCTCCCGTAGGAGT', 'F12Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Fotr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right foot surface', 'sample445', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Ewxr', 'GACTCGAATCGT', 'CATGCTGCCTCCCGTAGGAGT', 'M42Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Ewxr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right external auditory canal', 'sample411', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M32Fcsw', 'CTATAGTCGTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M32Fcsw', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Fcsw', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Feces', 'sample509', 'False', 'UBERON:feces', 'UBERON:feces', 'UBERON:feces', '2008-2009', 'human gut metagenome', 'stool', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:feces', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'stool', 'False', 'True', '', 'XXQIITAXX', 'male', '408170', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human gut metagenome'], ['449.M22Aptr', 'CCAGATGATCGT', 'CATGCTGCCTCCCGTAGGAGT', 'M22Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M22Aptr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Right axilla', 'sample324', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M14Plml', 'GACAGTTACTGC', 'CATGCTGCCTCCCGTAGGAGT', 'M14Plml', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Plml', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left palm', 'sample204', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of hand', '2008-2009', 'human skin metagenome', 'left palm', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'left palm', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M24Frhd', 'GCACGACAACAC', 'CATGCTGCCTCCCGTAGGAGT', 'M24Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M24Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Forehead', 'sample052', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M34Ewax', 'GTAGAGCTGTTC', 'CATGCTGCCTCCCGTAGGAGT', 'M34Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M34Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External auditory canal', 'sample308', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F32Frhd', 'GCTAAGAGAGTA', 'CATGCTGCCTCCCGTAGGAGT', 'F32Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Frhd', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Forehead', 'sample042', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F12Nstr', 'AGTTCTACGTCA', 'CATGCTGCCTCCCGTAGGAGT', 'F12Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Nstr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample372', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F14Tong', 'GATACGTCCTGA', 'CATGCTGCCTCCCGTAGGAGT', 'F14Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F14Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample524', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M64Nost', 'TCACTTCTCGCT', 'CATGCTGCCTCCCGTAGGAGT', 'M64Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Nares', 'sample296', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M44Tong', 'TACAGTCTCATG', 'CATGCTGCCTCCCGTAGGAGT', 'M44Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M44Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample548', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F22Aptr', 'ATGTCACCGTGA', 'CATGCTGCCTCCCGTAGGAGT', 'F22Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Aptr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right axilla', 'sample318', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M54Nost', 'TAGCCTCTCTGC', 'CATGCTGCCTCCCGTAGGAGT', 'M54Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Nares', 'sample294', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M63Nost', 'TATCGCGCGATA', 'CATGCTGCCTCCCGTAGGAGT', 'M63Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M63Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Nares', 'sample295', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M63Ewax', 'TATGGCACACAC', 'CATGCTGCCTCCCGTAGGAGT', 'M63Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M63Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:External auditory canal', 'sample313', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M53Nost', 'TACTAATCTGCG', 'CATGCTGCCTCCCGTAGGAGT', 'M53Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Nares', 'sample293', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M64Ewax', 'TCAGCTCAACTA', 'CATGCTGCCTCCCGTAGGAGT', 'M64Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M64Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:External auditory canal', 'sample314', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M34Nost', 'GTACAAGAGTGA', 'CATGCTGCCTCCCGTAGGAGT', 'M34Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M34Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample290', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M13Nost', 'CTTGTGTCGATA', 'CATGCTGCCTCCCGTAGGAGT', 'M13Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M13Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample285', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F13Nost', 'GACTGCATCTTA', 'CATGCTGCCTCCCGTAGGAGT', 'F13Nost', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F13Nost', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample279', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'nostrils', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'nares', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M34Frhd', 'GGTGCGTGTATG', 'CATGCTGCCTCCCGTAGGAGT', 'M34Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M34Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Forehead', 'sample056', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M53Ewax', 'TACTTACTGCAG', 'CATGCTGCCTCCCGTAGGAGT', 'M53Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:External auditory canal', 'sample311', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M54Frhd', 'TAGCATCGTGGT', 'CATGCTGCCTCCCGTAGGAGT', 'M54Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M54Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Forehead', 'sample062', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Nstl', 'CACGTCGATGGA', 'CATGCTGCCTCCCGTAGGAGT', 'F22Nstl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Nstl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample152', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'left nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F21Aptl', 'ATATCGCTACTG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Aptl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Aptl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left axilla', 'sample095', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'left armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left axilla', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Nstr', 'CACGTGACATGT', 'CATGCTGCCTCCCGTAGGAGT', 'F22Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Nstr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Nares', 'sample374', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M31Aptr', 'CGCGTAACTGTA', 'CATGCTGCCTCCCGTAGGAGT', 'M31Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Aptr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right axilla', 'sample325', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M12Nstr', 'ACTATTGTCACG', 'CATGCTGCCTCCCGTAGGAGT', 'M12Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M12Nstr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Nares', 'sample378', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M31Nstr', 'CGTGTACATCAG', 'CATGCTGCCTCCCGTAGGAGT', 'M31Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Nstr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample381', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.M32Aptr', 'CTACTGATATCG', 'CATGCTGCCTCCCGTAGGAGT', 'M32Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Aptr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right axilla', 'sample326', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M12Nose', 'ACTACGTGTGGT', 'CATGCTGCCTCCCGTAGGAGT', 'M12Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M12Nose', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:External nose', 'sample026', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F11Nstr', 'AGCATATGAGAG', 'CATGCTGCCTCCCGTAGGAGT', 'F11Nstr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F11Nstr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Nares', 'sample371', 'False', 'UBERON:nostril', 'UBERON:mucus', 'UBERON:nostril', '2008-2009', 'human nose metagenome', 'right nostril', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'right naris', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human nose metagenome'], ['449.F11Navl', 'AGCACGAGCCTA', 'CATGCTGCCTCCCGTAGGAGT', 'F11Navl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F11Navl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Umbilicus', 'sample267', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of abdomen', '2008-2009', 'human skin metagenome', 'navel', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'umbilicus', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Nose', 'ATGACTCATTCG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Nose', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:External nose', 'sample021', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Frhd', 'CGTATCTGCGAA', 'CATGCTGCCTCCCGTAGGAGT', 'M31Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Frhd', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Forehead', 'sample053', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F32Aptr', 'GCCAGAGTCGTA', 'CATGCTGCCTCCCGTAGGAGT', 'F32Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Aptr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right axilla', 'sample320', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Aptl', 'GACGCTAGTTCA', 'CATGCTGCCTCCCGTAGGAGT', 'M42Aptl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Aptl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left axilla', 'sample106', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'left armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Aptr', 'GACGTTGCACAG', 'CATGCTGCCTCCCGTAGGAGT', 'M42Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Aptr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right axilla', 'sample328', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Aptl', 'CTGCTGCGAAGA', 'CATGCTGCCTCCCGTAGGAGT', 'M41Aptl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Aptl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left axilla', 'sample105', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'left armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Aptr', 'CTGGACTCATAG', 'CATGCTGCCTCCCGTAGGAGT', 'M41Aptr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Aptr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right axilla', 'sample327', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'right armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Ewxr', 'CTAGGTCACTAG', 'CATGCTGCCTCCCGTAGGAGT', 'M32Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Ewxr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right external auditory canal', 'sample409', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F21Ewxl', 'ATCAGGCGTGTG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Ewxl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left external auditory canal', 'sample178', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M43Ewax', 'GTGTGCTATCAG', 'CATGCTGCCTCCCGTAGGAGT', 'M43Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:External auditory canal', 'sample309', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F11Urin', 'AGCTCTCAGAGG', 'CATGCTGCCTCCCGTAGGAGT', 'F11Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F11Urin', '0.006', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample553', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F31Nose', 'GCAGCACGTTGA', 'CATGCTGCCTCCCGTAGGAGT', 'F31Nose', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Nose', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External nose', 'sample023', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:nose', '2008-2009', 'human skin metagenome', 'external nose', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'external nose', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Ewxl', 'GACTCACTCAAT', 'CATGCTGCCTCCCGTAGGAGT', 'M42Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Ewxl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left external auditory canal', 'sample188', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M23Urin', 'GATGTCGTGTCA', 'CATGCTGCCTCCCGTAGGAGT', 'M23Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Urine', 'sample584', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F32Ewxr', 'GCGATATATCGC', 'CATGCTGCCTCCCGTAGGAGT', 'F32Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Ewxr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right external auditory canal', 'sample404', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F34Urin', 'GTCTTCGTCGCT', 'CATGCTGCCTCCCGTAGGAGT', 'F34Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample578', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M24Ewax', 'GCAGGATAGATA', 'CATGCTGCCTCCCGTAGGAGT', 'M24Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M24Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:External auditory canal', 'sample306', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F12Urin', 'ATAGGCGATCTC', 'CATGCTGCCTCCCGTAGGAGT', 'F12Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Urin', '0.006', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample554', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M23Uric', 'GATGTGAGCGCT', 'CATGCTGCCTCCCGTAGGAGT', 'M23Uric', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M23Uric', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Urine', 'sample583', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F22Pinr', 'CACTGGTATATC', 'CATGCTGCCTCCCGTAGGAGT', 'F22Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Pinr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of right pinna', 'sample388', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Fotl', 'CACTGTAGGACG', 'CATGCTGCCTCCCGTAGGAGT', 'F22Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Fotl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left foot surface', 'sample226', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M11Ewxl', 'AAGCTGCAGTCG', 'CATGCTGCCTCCCGTAGGAGT', 'M11Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M11Ewxl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left external auditory canal', 'sample181', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F34Uric', 'GTGACCTGATGT', 'CATGCTGCCTCCCGTAGGAGT', 'F34Uric', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Uric', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample577', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M63Urin', 'TCAACAGCATCG', 'CATGCTGCCTCCCGTAGGAGT', 'M63Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M63Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '', 'years', '0', 'FMA:Urine', 'sample600', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M6', 'M6', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F24Uric', 'GCTGCTGCAATA', 'CATGCTGCCTCCCGTAGGAGT', 'F24Uric', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Uric', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample573', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F24Urin', 'GCTGATGAGCTG', 'CATGCTGCCTCCCGTAGGAGT', 'F24Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample574', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M24Urin', 'GCAGTATCACTG', 'CATGCTGCCTCCCGTAGGAGT', 'M24Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M24Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Urine', 'sample586', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M2', 'M2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F33Uric', 'GTCCATAGCTAG', 'CATGCTGCCTCCCGTAGGAGT', 'F33Uric', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Uric', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample575', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F23Uric', 'GCGGATGTGACT', 'CATGCTGCCTCCCGTAGGAGT', 'F23Uric', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Uric', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample571', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F23Urin', 'GCGATATATCGC', 'CATGCTGCCTCCCGTAGGAGT', 'F23Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Urine', 'sample572', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F33Urin', 'GTCATTCACGAG', 'CATGCTGCCTCCCGTAGGAGT', 'F33Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F33Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample576', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F24Ewax', 'GCTCAGTGCAGA', 'CATGCTGCCTCCCGTAGGAGT', 'F24Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:External auditory canal', 'sample300', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F31Urin', 'GCATTGCGTGAG', 'CATGCTGCCTCCCGTAGGAGT', 'F31Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F31Urin', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample561', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F32Urin', 'GGCAGTGTATCG', 'CATGCTGCCTCCCGTAGGAGT', 'F32Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Urin', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample562', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F32Tong', 'GGATCGCAGATC', 'CATGCTGCCTCCCGTAGGAGT', 'F32Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F32Tong', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample530', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F34Tong', 'GTCTGGATAGCG', 'CATGCTGCCTCCCGTAGGAGT', 'F34Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F34Tong', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample532', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F3', 'F3', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.M14Urin', 'GACGTTGCACAG', 'CATGCTGCCTCCCGTAGGAGT', 'M14Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Urine', 'sample582', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M43Fotl', 'GTGTCTACATTG', 'CATGCTGCCTCCCGTAGGAGT', 'M43Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M43Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left foot surface', 'sample247', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F11Tong', 'AGCTCCATACAG', 'CATGCTGCCTCCCGTAGGAGT', 'F11Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F11Tong', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample521', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F12Tong', 'ATAGCTCCATAC', 'CATGCTGCCTCCCGTAGGAGT', 'F12Tong', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F12Tong', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Dorsal surface of tongue', 'sample522', 'False', 'UBERON:oral cavity', 'UBERON:saliva', 'UBERON:tongue', '2008-2009', 'human oral metagenome', 'tongue', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:saliva', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'dorsal surface of tongue', 'False', 'True', 'swab', 'XXQIITAXX', 'female', '447426', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human oral metagenome'], ['449.F22Fotr', 'CAGACATTGCGT', 'CATGCTGCCTCCCGTAGGAGT', 'F22Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Fotr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right foot surface', 'sample449', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F13Urin', 'GAGAGCTCTACG', 'CATGCTGCCTCCCGTAGGAGT', 'F13Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F13Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample568', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.F21Frhd', 'ATCGTACAACTC', 'CATGCTGCCTCCCGTAGGAGT', 'F21Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Frhd', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Forehead', 'sample037', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Frhd', 'CAAGTGAGAGAG', 'CATGCTGCCTCCCGTAGGAGT', 'F22Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Frhd', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Forehead', 'sample038', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Ewxr', 'CTGTCTCTCCTA', 'CATGCTGCCTCCCGTAGGAGT', 'M41Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Ewxr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right external auditory canal', 'sample410', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F23Fotr', 'GCCAGAGTCGTA', 'CATGCTGCCTCCCGTAGGAGT', 'F23Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Fotr', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right foot surface', 'sample450', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M13Knee', 'GAACATGATGAG', 'CATGCTGCCTCCCGTAGGAGT', 'M13Knee', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M13Knee', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Popliteal fossa', 'sample007', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of knee', '2008-2009', 'human skin metagenome', 'back of knees', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'popliteal fossae', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M53Frhd', 'TACGTGTACGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M53Frhd', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M53Frhd', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '60.0', 'years', '0', 'FMA:Forehead', 'sample061', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of head', '2008-2009', 'human skin metagenome', 'forehead', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M5', 'M5', '9606', '40.0149856', '-105.2705456', 'forehead', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M12Aptl', 'ACCTGTCTCTCT', 'CATGCTGCCTCCCGTAGGAGT', 'M12Aptl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M12Aptl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:Left axilla', 'sample100', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'left armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'left axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Fotr', 'GACGAGTCAGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M41Fotr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Fotr', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Right foot surface', 'sample468', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'right sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'right plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M41Fotl', 'GACCGAGCTATG', 'CATGCTGCCTCCCGTAGGAGT', 'M41Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Fotl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left foot surface', 'sample245', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M42Fotl', 'GATACGTCCTGA', 'CATGCTGCCTCCCGTAGGAGT', 'M42Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M42Fotl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left foot surface', 'sample246', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F14Urin', 'GATAGCTGTCTT', 'CATGCTGCCTCCCGTAGGAGT', 'F14Urin', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial variation in human body habitats across space and time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F14Urin', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '8/20/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Urine', 'sample570', 'False', 'UBERON:urine', 'UBERON:urine', 'UBERON:urine', '2008-2009', 'human urine metagenome', 'urine', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:urine', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'urine', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human urine metagenome'], ['449.M13Ewax', 'GAAGCTACTGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M13Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M13Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:External auditory canal', 'sample303', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F24Fotl', 'GCTATTCGACAT', 'CATGCTGCCTCCCGTAGGAGT', 'F24Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F24Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left foot surface', 'sample228', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F21Pinr', 'ATGCGTAGTGCG', 'CATGCTGCCTCCCGTAGGAGT', 'F21Pinr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F21Pinr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Surface of right pinna', 'sample387', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of outer ear', '2008-2009', 'human skin metagenome', 'right outer ear', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'right lateral pinna', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M31Aptl', 'CGCGATAGCAGT', 'CATGCTGCCTCCCGTAGGAGT', 'M31Aptl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Aptl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left axilla', 'sample103', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'left armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M32Aptl', 'CTACTACAGGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M32Aptl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M32Aptl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left axilla', 'sample104', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin of arm', '2008-2009', 'human skin metagenome', 'left armpit', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sweat', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left axilla', 'False', 'True', '', 'XXQIITAXX', 'male', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.M33Ewax', 'GGCAGTGTATCG', 'CATGCTGCCTCCCGTAGGAGT', 'M33Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M33Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External auditory canal', 'sample307', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M31Ewxl', 'CGGAGTGTCTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M31Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Ewxl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Left external auditory canal', 'sample185', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F23Ewax', 'GCGACTTGTGTA', 'CATGCTGCCTCCCGTAGGAGT', 'F23Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:External auditory canal', 'sample299', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F14Ewax', 'GAGTGGTAGAGA', 'CATGCTGCCTCCCGTAGGAGT', 'F14Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F14Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:External auditory canal', 'sample298', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F1', 'F1', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.F23Fotl', 'GCCTATACTACA', 'CATGCTGCCTCCCGTAGGAGT', 'F23Fotl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F23Fotl', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left foot surface', 'sample227', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:zone of skin of foot', '2008-2009', 'human skin metagenome', 'left sole of foot', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:sebum', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left plantar foot', 'False', 'True', '', 'XXQIITAXX', 'female', '539655', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human skin metagenome'], ['449.F22Ewxl', 'ATTATCGTGCAC', 'CATGCTGCCTCCCGTAGGAGT', 'F22Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'F22Ewxl', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left external auditory canal', 'sample179', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'F2', 'F2', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'female', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M14Ewax', 'GACGCAGTAGCT', 'CATGCTGCCTCCCGTAGGAGT', 'M14Ewax', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M14Ewax', '0.006', 'V2', '0', 'FKB0RMH', 'CGS-GL', '11/14/2008', 'FKB0RMH', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '36.0', 'years', '0', 'FMA:External auditory canal', 'sample304', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M1', 'M1', '9606', '40.0149856', '-105.2705456', 'external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M31Ewxr', 'CGGCGATGTACA', 'CATGCTGCCTCCCGTAGGAGT', 'M31Ewxr', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M31Ewxr', '0.005', 'V2', '0', 'FFLHOYS', 'CGS-GL', '8/18/2008', 'FFLHOYS', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '33.0', 'years', '0', 'FMA:Right external auditory canal', 'sample408', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:zone of skin of outer ear', '2008-2009', 'human metagenome', 'right outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M3', 'M3', '9606', '40.0149856', '-105.2705456', 'right external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome'], ['449.M41Ewxl', 'CTGTATCGTATG', 'CATGCTGCCTCCCGTAGGAGT', 'M41Ewxl', 'True', 'CostelloWholeBodySites', 'CCME', 'Healthy adults were recruited to donate samples on two consecutive days in June and again three months later on two consecutive days in September 2008. Volunteers were unrelated individuals of both sexes and most lived in Boulder, CO. Female subjects F1, F2, F3 and male subjects M1, M2, M3, M4, and M6 were 30-35 years old. Subject M5 was approximately 60 years old. The health status of the volunteers was self-reported. None of the volunteers indicated that they had been hospitalized in the previous 6 months. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0708.12). On the day of sampling, participants were instructed to bathe in the morning using their choice of personal hygiene products, but to avoid wearing deodorant or antiperspirant. The interval between subject bathing and the collection of most samples likely ranged from around several hours to half an hour. Participants were asked to avoid eating or drinking for one hour prior to oral sampling. Skin surfaces, hair on top of the head, and nostrils (nares) were sampled using autoclaved cotton-tipped swabs (Puritan) pre-moistened with a sterile solution of 0.15 M NaCl and 0.1% Tween 20 (S1, S2). Swabbing has been shown to be a suitable method for skin sample collection for microbial community analysis (S3). Dry cotton-tipped swabs were used to sample the external auditory canal [and earwax (cerumen) if present], the dorsal surface of the tongue, and stool. Stool samples have been shown to provide an adequately representative sample of the gut microbiota (S4). Stool was swabbed using two different methods for comparison: i) off of used bathroom tissue and ii) by stabbing the swab directly into the largest piece of feces (see Fig. S22 for comparison). We found that swabbing off of used bathroom tissue was adequate for obtaining material for microbial community analysis. Swabs were placed in sterile 15 ml plastic tubes and stored at -80C within 10 minutes of sampling. Volunteers sampled stool on the same days that the other samples were collected (within 24 hours). If needed, the subjects were provided dry ice for the transport of their stool samples back to the lab. Liquid samples were also collected. The oral cavity was rinsed by swishing 10 ml of sterile water for 10 seconds, and then spitting the water into a sterile 50 ml plastic tube (see Fig. S22 for comparison to dorsal tongue swabs). Liquids were stored at 4C for several hours before sterile filtration onto 0.2micro-m pore size cellulose-nitrate filters (Nalgene). Filtered samples were immediately placed at -80C. Volunteers sampled themselves while wearing sterile gloves after first having their palms and index fingers swabbed by another person wearing sterile gloves. Replicate samples from the right and left sides of the body were taken when possible. All but one subject had not taken antibiotics in the 6 months prior to the June sampling dates, and none took antibiotics during the study period. Figure S1 lists the sample types, dates, and subjects. Samples were stored at -80C for <1 week prior to DNA extraction. We performed a follow-up study aimed at addressing whether skin location influences bacterial community assembly. Our approach involved transplanting foreign bacteria from various donor sites (dorsal tongue, forehead, and volar forearm) onto disinfected recipient skin sites (forehead and volar forearm) both within and between individuals. All individuals were made aware of the nature of the experiment and gave written informed consent in accordance with the sampling protocol approved by the University of Colorado Human Research Committee (protocol 0109.23).', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'TCAG', 'Amplicon DNA concentrations were determined using the Quant-iT PicoGreen dsDNA reagent and kit (Invitrogen). Assays were carried out using 5-10 _l of cleaned PCR product in a total reaction volume of 200 _l in black, 96-well microtiter plates. Fluorescence was measured on a BioTek Synergy HT plate reader using the 480/520-nm excitation/emission filter pair. Following quantitation, cleaned amplicons were combined in equimolar ratios into a single tube. The final pool of DNA was precipitated on ice for 45 minutes following theaddition of 5 M NaCl (0.2 M final concentration) and 2 volumes of ice-cold 100% ethanol. The precipitated DNA was centrifuged at 7,800 x g for 40 minutes at 4C, and the resulting pellet was washed with an equal volume of ice-cold 70% ethanol and centrifuged again at 7,800 x g for 20 minutes at 4C. The supernatant was removed and the pellet was air dried for 10 minutes at room temperature and then resuspended in 100 micro-l of nuclease-free water (MO BIO). The final concentration of the pooled DNA was determined using a NanoDrop spectrophotometer (Thermo Fisher). Pyrosequencing was carried out using primer A on a 454 Life Sciences Genome Sequencer FLX instrument (Roche).', 'CA', 'PMID: 19892944', 'http://www.mobio.com/soil-dna-isolation/powersoil-dna-isolation-kit.html', 'initial denaturation: 94; annealing: 50_0.5; elongation: 72_1.5; final elongation: 72_10; 35', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:CATGCTGCCTCCCGTAGGAGT', 'False', 'FLX', 'M41Ewxl', '0.006', 'V2', '0', 'FFO92CG', 'CGS-GL', '8/20/2008', 'FFO92CG', 'swab with sterile saline', '1, swab', 'CCME', 'XXQIITAXX', 'pyrosequencing', 'CCME', 'costello_whole_body_sites', '16S rRNA', 'V2', 'http://www.scienceonline.org/cgi/content/abstract/1177486', '31.0', 'years', '0', 'FMA:Left external auditory canal', 'sample187', 'False', 'UBERON:external auditory canal', 'UBERON:cerumen', 'UBERON:ear canal', '2008-2009', 'human metagenome', 'left outer ear canal/earwax', 'GAZ:United States of America', '0', 'True', '1624.097656', 'ENVO:urban biome', 'ENVO:human-associated habitat', 'ENVO:ear wax', 'Negative', 'human', 'M4', 'M4', '9606', '40.0149856', '-105.2705456', 'left external auditory canal', 'False', 'True', '', 'XXQIITAXX', 'male', '646099', 'Bacterial Community Variation in Human Body Habitats Across Space and Time', 'Bacterial community variation in human body habitats across space and time', 'CostelloWholeBodySites', 'Gail Ackermann', 'Rob Knight', 'human metagenome']];\n",
       "  var axesNames = [0, 1, 2, 3, 4];\n",
       "\n",
       "  var dm, ec;\n",
       "\n",
       "  function init() {\n",
       "    // Initialize the DecompositionModel\n",
       "    dm = new DecompositionModel(name, ids, coords, pct_var,\n",
       "                                md_headers, metadata, axesNames);\n",
       "    // Initialize the EmperorController\n",
       "    ec = new EmperorController(dm, 'emperor-notebook-0x69a3643a');\n",
       "  }\n",
       "\n",
       "  function animate() {\n",
       "    requestAnimationFrame(animate);\n",
       "    ec.render();\n",
       "  }\n",
       "  $(window).resize(function() {\n",
       "    ec.resize(div.innerWidth(), div.innerHeight());\n",
       "  });\n",
       "\n",
       "  $(function(){\n",
       "    init();\n",
       "    animate();\n",
       "\n",
       "  });\n",
       "\n",
       "}); // END REQUIRE.JS block\n",
       "</script>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<emperor.core.Emperor at 0x1176da710>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Emperor(coordinates, metadata, remote=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "-"
    }
   },
   "source": [
    "<!-- RISE has some scrolling bugs, so we pre-allocate some space to display the emperor plot -->\n",
    "<div style='height:500px'></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Outline\n",
    "\n",
    "- ~~Background (why $\\beta$-diversity).~~\n",
    "\n",
    "- **What is Emperor.**\n",
    "\n",
    "- How can we use Emperor.\n",
    "    \n",
    "- Analyzing a use case."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# What is Emperor?\n",
    "\n",
    "- A Python 2/3 package that powers a JavaScript UI.\n",
    "    - https://github.com/biocore/emperor\n",
    "    \n",
    "- Originated in the context of Quantitative Insights Into Microbial Ecology http://2.qiime.org"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Scatter plot viewer.\n",
    "\n",
    "    - Visualize\n",
    "    - Interact\n",
    "    - Share"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Other applications"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Qiita\n",
    "    - https://qiita.ucsd.edu\n",
    "    ![qiita](https://raw.githubusercontent.com/biocore/qiita/master/qiita_pet/static/img/logo.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- American Gut (participant's results).\n",
    "    - http://americangut.org/\n",
    "    <img src='https://lh4.googleusercontent.com/88tkWe8K-u_ZUgO3GVIrnDSeJE65QGHOlCY_-WQ3mFxmmskkxEZGEmj6GxYilVe5f5gnEx2K_EdOZngTeY750V4rtNLt0n5rVA6pj1ow9oFHltwYlbQ' style='height:80px'></img>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Illumina's BaseSpace processing results for the QIIME app.\n",
    "- Metabolomic analysis, IPython notebook ..."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "    ... kinda"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Kinda?\n",
    "\n",
    "- https://github.com/jupyter/nbviewer/issues/316\n",
    "\n",
    "<img src='./images/glitch.gif' style='height:500px;'>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Nowadays"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Python API.\n",
    "    - Python 2 and 3.\n",
    "    - Integration with scikit-bio.\n",
    "    - Jupyter integration.\n",
    "    - Pandas 🐼 Integration."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- JavaScript API."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Command Line Interface (powered by QIIME 2)."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Outline\n",
    "\n",
    "- ~~Background (why $\\beta$-diversity).~~\n",
    "\n",
    "- ~~What is Emperor.~~\n",
    "\n",
    "- **How can we use Emperor**.\n",
    "    \n",
    "- Analyzing a use case."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Python"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- 1 main class `Emperor`, depends on `scikit-bio` and `pandas`.\n",
    "\n",
    "- Format Python data into JSON and display it using JavaScript."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from emperor import Emperor\n",
    "\n",
    "Emperor?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<div style='height:500px'></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# http://emperor.microbio.me/uno/\n",
    "\n",
    "![emperor](./images/main-doc.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# http://emperor.microbio.me/uno/\n",
    "\n",
    "![emperor](./images/python-doc.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# http://emperor.microbio.me/uno/\n",
    "\n",
    "![emperor](./images/python-doc-2.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# 🐼s integration - experimental"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "df = pd.read_csv('./tips.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>total_bill</th>\n",
       "      <th>tip</th>\n",
       "      <th>sex</th>\n",
       "      <th>smoker</th>\n",
       "      <th>day</th>\n",
       "      <th>time</th>\n",
       "      <th>size</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>16.99</td>\n",
       "      <td>1.01</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.34</td>\n",
       "      <td>1.66</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>21.01</td>\n",
       "      <td>3.50</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>23.68</td>\n",
       "      <td>3.31</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>24.59</td>\n",
       "      <td>3.61</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>25.29</td>\n",
       "      <td>4.71</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>8.77</td>\n",
       "      <td>2.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>26.88</td>\n",
       "      <td>3.12</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>15.04</td>\n",
       "      <td>1.96</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>14.78</td>\n",
       "      <td>3.23</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10.27</td>\n",
       "      <td>1.71</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>35.26</td>\n",
       "      <td>5.00</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>15.42</td>\n",
       "      <td>1.57</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>18.43</td>\n",
       "      <td>3.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>14.83</td>\n",
       "      <td>3.02</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>21.58</td>\n",
       "      <td>3.92</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>10.33</td>\n",
       "      <td>1.67</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>16.29</td>\n",
       "      <td>3.71</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>16.97</td>\n",
       "      <td>3.50</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sun</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>20.65</td>\n",
       "      <td>3.35</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>17.92</td>\n",
       "      <td>4.08</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>20.29</td>\n",
       "      <td>2.75</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>15.77</td>\n",
       "      <td>2.23</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>39.42</td>\n",
       "      <td>7.58</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>19.82</td>\n",
       "      <td>3.18</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>17.81</td>\n",
       "      <td>2.34</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>13.37</td>\n",
       "      <td>2.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>12.69</td>\n",
       "      <td>2.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>21.70</td>\n",
       "      <td>4.30</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>19.65</td>\n",
       "      <td>3.00</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>28.17</td>\n",
       "      <td>6.50</td>\n",
       "      <td>Female</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>12.90</td>\n",
       "      <td>1.10</td>\n",
       "      <td>Female</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>28.15</td>\n",
       "      <td>3.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>11.59</td>\n",
       "      <td>1.50</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>7.74</td>\n",
       "      <td>1.44</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>30.14</td>\n",
       "      <td>3.09</td>\n",
       "      <td>Female</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>12.16</td>\n",
       "      <td>2.20</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Fri</td>\n",
       "      <td>Lunch</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>13.42</td>\n",
       "      <td>3.48</td>\n",
       "      <td>Female</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Fri</td>\n",
       "      <td>Lunch</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>8.58</td>\n",
       "      <td>1.92</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Fri</td>\n",
       "      <td>Lunch</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>15.98</td>\n",
       "      <td>3.00</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Fri</td>\n",
       "      <td>Lunch</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>13.42</td>\n",
       "      <td>1.58</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Fri</td>\n",
       "      <td>Lunch</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>16.27</td>\n",
       "      <td>2.50</td>\n",
       "      <td>Female</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Fri</td>\n",
       "      <td>Lunch</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>10.09</td>\n",
       "      <td>2.00</td>\n",
       "      <td>Female</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Fri</td>\n",
       "      <td>Lunch</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>20.45</td>\n",
       "      <td>3.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>13.28</td>\n",
       "      <td>2.72</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>22.12</td>\n",
       "      <td>2.88</td>\n",
       "      <td>Female</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>24.01</td>\n",
       "      <td>2.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>15.69</td>\n",
       "      <td>3.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>11.61</td>\n",
       "      <td>3.39</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>10.77</td>\n",
       "      <td>1.47</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>15.53</td>\n",
       "      <td>3.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>10.07</td>\n",
       "      <td>1.25</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>12.60</td>\n",
       "      <td>1.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>32.83</td>\n",
       "      <td>1.17</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>35.83</td>\n",
       "      <td>4.67</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>29.03</td>\n",
       "      <td>5.92</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>27.18</td>\n",
       "      <td>2.00</td>\n",
       "      <td>Female</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>22.67</td>\n",
       "      <td>2.00</td>\n",
       "      <td>Male</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>17.82</td>\n",
       "      <td>1.75</td>\n",
       "      <td>Male</td>\n",
       "      <td>No</td>\n",
       "      <td>Sat</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>18.78</td>\n",
       "      <td>3.00</td>\n",
       "      <td>Female</td>\n",
       "      <td>No</td>\n",
       "      <td>Thur</td>\n",
       "      <td>Dinner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>244 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     total_bill   tip     sex smoker   day    time  size\n",
       "0         16.99  1.01  Female     No   Sun  Dinner     2\n",
       "1         10.34  1.66    Male     No   Sun  Dinner     3\n",
       "2         21.01  3.50    Male     No   Sun  Dinner     3\n",
       "3         23.68  3.31    Male     No   Sun  Dinner     2\n",
       "4         24.59  3.61  Female     No   Sun  Dinner     4\n",
       "5         25.29  4.71    Male     No   Sun  Dinner     4\n",
       "6          8.77  2.00    Male     No   Sun  Dinner     2\n",
       "7         26.88  3.12    Male     No   Sun  Dinner     4\n",
       "8         15.04  1.96    Male     No   Sun  Dinner     2\n",
       "9         14.78  3.23    Male     No   Sun  Dinner     2\n",
       "10        10.27  1.71    Male     No   Sun  Dinner     2\n",
       "11        35.26  5.00  Female     No   Sun  Dinner     4\n",
       "12        15.42  1.57    Male     No   Sun  Dinner     2\n",
       "13        18.43  3.00    Male     No   Sun  Dinner     4\n",
       "14        14.83  3.02  Female     No   Sun  Dinner     2\n",
       "15        21.58  3.92    Male     No   Sun  Dinner     2\n",
       "16        10.33  1.67  Female     No   Sun  Dinner     3\n",
       "17        16.29  3.71    Male     No   Sun  Dinner     3\n",
       "18        16.97  3.50  Female     No   Sun  Dinner     3\n",
       "19        20.65  3.35    Male     No   Sat  Dinner     3\n",
       "20        17.92  4.08    Male     No   Sat  Dinner     2\n",
       "21        20.29  2.75  Female     No   Sat  Dinner     2\n",
       "22        15.77  2.23  Female     No   Sat  Dinner     2\n",
       "23        39.42  7.58    Male     No   Sat  Dinner     4\n",
       "24        19.82  3.18    Male     No   Sat  Dinner     2\n",
       "25        17.81  2.34    Male     No   Sat  Dinner     4\n",
       "26        13.37  2.00    Male     No   Sat  Dinner     2\n",
       "27        12.69  2.00    Male     No   Sat  Dinner     2\n",
       "28        21.70  4.30    Male     No   Sat  Dinner     2\n",
       "29        19.65  3.00  Female     No   Sat  Dinner     2\n",
       "..          ...   ...     ...    ...   ...     ...   ...\n",
       "214       28.17  6.50  Female    Yes   Sat  Dinner     3\n",
       "215       12.90  1.10  Female    Yes   Sat  Dinner     2\n",
       "216       28.15  3.00    Male    Yes   Sat  Dinner     5\n",
       "217       11.59  1.50    Male    Yes   Sat  Dinner     2\n",
       "218        7.74  1.44    Male    Yes   Sat  Dinner     2\n",
       "219       30.14  3.09  Female    Yes   Sat  Dinner     4\n",
       "220       12.16  2.20    Male    Yes   Fri   Lunch     2\n",
       "221       13.42  3.48  Female    Yes   Fri   Lunch     2\n",
       "222        8.58  1.92    Male    Yes   Fri   Lunch     1\n",
       "223       15.98  3.00  Female     No   Fri   Lunch     3\n",
       "224       13.42  1.58    Male    Yes   Fri   Lunch     2\n",
       "225       16.27  2.50  Female    Yes   Fri   Lunch     2\n",
       "226       10.09  2.00  Female    Yes   Fri   Lunch     2\n",
       "227       20.45  3.00    Male     No   Sat  Dinner     4\n",
       "228       13.28  2.72    Male     No   Sat  Dinner     2\n",
       "229       22.12  2.88  Female    Yes   Sat  Dinner     2\n",
       "230       24.01  2.00    Male    Yes   Sat  Dinner     4\n",
       "231       15.69  3.00    Male    Yes   Sat  Dinner     3\n",
       "232       11.61  3.39    Male     No   Sat  Dinner     2\n",
       "233       10.77  1.47    Male     No   Sat  Dinner     2\n",
       "234       15.53  3.00    Male    Yes   Sat  Dinner     2\n",
       "235       10.07  1.25    Male     No   Sat  Dinner     2\n",
       "236       12.60  1.00    Male    Yes   Sat  Dinner     2\n",
       "237       32.83  1.17    Male    Yes   Sat  Dinner     2\n",
       "238       35.83  4.67  Female     No   Sat  Dinner     3\n",
       "239       29.03  5.92    Male     No   Sat  Dinner     3\n",
       "240       27.18  2.00  Female    Yes   Sat  Dinner     2\n",
       "241       22.67  2.00    Male    Yes   Sat  Dinner     2\n",
       "242       17.82  1.75    Male     No   Sat  Dinner     2\n",
       "243       18.78  3.00  Female     No  Thur  Dinner     2\n",
       "\n",
       "[244 rows x 7 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<div style='height:500px'></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# 🐼s integration - experimental"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- Coordinates are inferred from the numerical data, see additional `x`, `y` and `z` parameters."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<script type=\"text/javascript\">\n",
       "\n",
       "if ($(\"#emperor-css\").length == 0){{\n",
       "    $(\"head\").append([\n",
       "\n",
       "        '<link id=\"emperor-css\" rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/css/emperor.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/jquery-ui.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/slick.grid.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/spectrum.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/chosen.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"/nbextensions/emperor/support_files/vendor/css/jquery.contextMenu.min.css\">'\n",
       "    ]);\n",
       "}}\n",
       "</script>\n",
       "\n",
       "<div id='emperor-notebook-0xc5a5f7a5' style=\"position: relative; width:100%; height:500px;\">\n",
       "  <div class='loading' style=\"position: absolute;top: 50%;left: 50%;margin-left: -229px; margin-top: -59px; z-index: 10000;height:118px;width:458px;padding:0px\"><img src='/nbextensions/emperor/support_files/img/emperor.png' alt='Emperor resources missing. Expected them to be found in /nbextensions/emperor/support_files'></div>\n",
       "</div>\n",
       "</div>\n",
       "\n",
       "<script type=\"text/javascript\">\n",
       "requirejs.config({\n",
       "// the left side is the module name, and the right side is the path\n",
       "// relative to the baseUrl attribute, do NOT include the .js extension\n",
       "'paths': {\n",
       "  /* jQuery */\n",
       "  'jquery': '/nbextensions/emperor/support_files/vendor/js/jquery-2.1.4.min',\n",
       "  'jqueryui': '/nbextensions/emperor/support_files/vendor/js/jquery-ui.min',\n",
       "  'jquery_drag': '/nbextensions/emperor/support_files/vendor/js/jquery.event.drag-2.2.min',\n",
       "\n",
       "  /* jQuery plugins */\n",
       "  'chosen': '/nbextensions/emperor/support_files/vendor/js/chosen.jquery.min',\n",
       "  'spectrum': '/nbextensions/emperor/support_files/vendor/js/spectrum.min',\n",
       "  'position': '/nbextensions/emperor/support_files/vendor/js/jquery.ui.position.min',\n",
       "  'contextmenu': '/nbextensions/emperor/support_files/vendor/js/jquery.contextMenu.min',\n",
       "\n",
       "  /* other libraries */\n",
       "  'underscore': '/nbextensions/emperor/support_files/vendor/js/underscore-min',\n",
       "  'chroma': '/nbextensions/emperor/support_files/vendor/js/chroma.min',\n",
       "  'filesaver': '/nbextensions/emperor/support_files/vendor/js/FileSaver.min',\n",
       "  'blob': '/nbextensions/emperor/support_files/vendor/js/Blob',\n",
       "  'd3': '/nbextensions/emperor/support_files/vendor/js/d3.min',\n",
       "\n",
       "\n",
       "  /* THREE.js and plugins */\n",
       "  'three': '/nbextensions/emperor/support_files/vendor/js/three.min',\n",
       "  'orbitcontrols': '/nbextensions/emperor/support_files/vendor/js/three.js-plugins/OrbitControls',\n",
       "\n",
       "  /* SlickGrid */\n",
       "  'slickcore': '/nbextensions/emperor/support_files/vendor/js/slick.core.min',\n",
       "  'slickgrid': '/nbextensions/emperor/support_files/vendor/js/slick.grid.min',\n",
       "  'slickformatters': '/nbextensions/emperor/support_files/vendor/js/slick.editors.min',\n",
       "  'slickeditors': '/nbextensions/emperor/support_files/vendor/js/slick.formatters.min',\n",
       "\n",
       "  /* Emperor's objects */\n",
       "  'util': '/nbextensions/emperor/support_files/js/util',\n",
       "  'model': '/nbextensions/emperor/support_files/js/model',\n",
       "  'view': '/nbextensions/emperor/support_files/js/view',\n",
       "  'controller': '/nbextensions/emperor/support_files/js/controller',\n",
       "  'draw': '/nbextensions/emperor/support_files/js/draw',\n",
       "  'scene3d': '/nbextensions/emperor/support_files/js/sceneplotview3d',\n",
       "  'viewcontroller': '/nbextensions/emperor/support_files/js/view-controller',\n",
       "  'colorviewcontroller': '/nbextensions/emperor/support_files/js/color-view-controller',\n",
       "  'visibilitycontroller': '/nbextensions/emperor/support_files/js/visibility-controller',\n",
       "  'scaleviewcontroller': '/nbextensions/emperor/support_files/js/scale-view-controller',\n",
       "  'shapecontroller': '/nbextensions/emperor/support_files/js/shape-controller',\n",
       "  'axescontroller': '/nbextensions/emperor/support_files/js/axes-controller',\n",
       "  'shape-editor': '/nbextensions/emperor/support_files/js/shape-editor',\n",
       "  'color-editor': '/nbextensions/emperor/support_files/js/color-editor',\n",
       "  'scale-editor': '/nbextensions/emperor/support_files/js/scale-editor',\n",
       "  'shapes': '/nbextensions/emperor/support_files/js/shapes'\n",
       "},\n",
       "/*\n",
       "   Libraries that are not AMD compatible need shim to declare their\n",
       "   dependencies.\n",
       " */\n",
       "'shim': {\n",
       "  'jquery_drag': {\n",
       "    'deps': ['jquery', 'jqueryui']\n",
       "  },\n",
       "  'chosen': {\n",
       "    'deps': ['jquery'],\n",
       "    'exports': 'jQuery.fn.chosen'\n",
       "  },\n",
       "  'contextmenu' : {\n",
       "    'deps': ['jquery', 'jqueryui', 'position']\n",
       "  },\n",
       "  'filesaver' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'orbitcontrols': {\n",
       "    'deps': ['three']\n",
       "  },\n",
       "'slickcore': ['jqueryui'],\n",
       "'slickgrid': ['slickcore', 'jquery_drag', 'slickformatters',\n",
       "              'slickeditors']\n",
       "}\n",
       "});\n",
       "\n",
       "requirejs(\n",
       "[\"jquery\", \"model\", \"controller\"],\n",
       "function($, model, EmperorController) {\n",
       "  var DecompositionModel = model.DecompositionModel;\n",
       "\n",
       "  var div = $('#emperor-notebook-0xc5a5f7a5');\n",
       "\n",
       "  var ids = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31', '32', '33', '34', '35', '36', '37', '38', '39', '40', '41', '42', '43', '44', '45', '46', '47', '48', '49', '50', '51', '52', '53', '54', '55', '56', '57', '58', '59', '60', '61', '62', '63', '64', '65', '66', '67', '68', '69', '70', '71', '72', '73', '74', '75', '76', '77', '78', '79', '80', '81', '82', '83', '84', '85', '86', '87', '88', '89', '90', '91', '92', '93', '94', '95', '96', '97', '98', '99', '100', '101', '102', '103', '104', '105', '106', '107', '108', '109', '110', '111', '112', '113', '114', '115', '116', '117', '118', '119', '120', '121', '122', '123', '124', '125', '126', '127', '128', '129', '130', '131', '132', '133', '134', '135', '136', '137', '138', '139', '140', '141', '142', '143', '144', '145', '146', '147', '148', '149', '150', '151', '152', '153', '154', '155', '156', '157', '158', '159', '160', '161', '162', '163', '164', '165', '166', '167', '168', '169', '170', '171', '172', '173', '174', '175', '176', '177', '178', '179', '180', '181', '182', '183', '184', '185', '186', '187', '188', '189', '190', '191', '192', '193', '194', '195', '196', '197', '198', '199', '200', '201', '202', '203', '204', '205', '206', '207', '208', '209', '210', '211', '212', '213', '214', '215', '216', '217', '218', '219', '220', '221', '222', '223', '224', '225', '226', '227', '228', '229', '230', '231', '232', '233', '234', '235', '236', '237', '238', '239', '240', '241', '242', '243'];\n",
       "  var coords = [[0.334382995473332, 0.101, 0.3333333333333333], [0.2035032473922456, 0.16599999999999998, 0.5], [0.41350127927573316, 0.35, 0.5], [0.4660499901594174, 0.331, 0.3333333333333333], [0.48395985042314504, 0.361, 0.6666666666666666], [0.49773666601062777, 0.471, 0.6666666666666666], [0.17260381814603423, 0.2, 0.3333333333333333], [0.5290297185593387, 0.312, 0.6666666666666666], [0.29600472347962997, 0.196, 0.3333333333333333], [0.29088762054713635, 0.323, 0.3333333333333333], [0.20212556583349733, 0.17099999999999999, 0.3333333333333333], [0.6939578823066325, 0.5, 0.6666666666666666], [0.30348356622712064, 0.157, 0.3333333333333333], [0.3627238732532966, 0.3, 0.6666666666666666], [0.29187167880338516, 0.302, 0.3333333333333333], [0.42471954339696905, 0.392, 0.3333333333333333], [0.20330643574099586, 0.16699999999999998, 0.5], [0.3206061798858492, 0.371, 0.5], [0.33398937217083247, 0.35, 0.5], [0.4064160598307419, 0.335, 0.5], [0.3526864790395592, 0.40800000000000003, 0.3333333333333333], [0.3993308403857508, 0.275, 0.3333333333333333], [0.31037197402086203, 0.223, 0.3333333333333333], [0.7758315292265302, 0.758, 0.6666666666666666], [0.3900806927770124, 0.318, 0.3333333333333333], [0.3505215508758118, 0.23399999999999999, 0.6666666666666666], [0.26313717772092104, 0.2, 0.3333333333333333], [0.2497539854359378, 0.2, 0.3333333333333333], [0.4270812832119661, 0.43, 0.3333333333333333], [0.38673489470576655, 0.3, 0.3333333333333333], [0.18795512694351507, 0.145, 0.3333333333333333], [0.36114938004329855, 0.25, 0.6666666666666666], [0.2963983467821295, 0.3, 0.3333333333333333], [0.407203306435741, 0.24500000000000002, 0.6666666666666666], [0.3499311159220626, 0.327, 0.3333333333333333], [0.47352883290690806, 0.36, 0.5], [0.3209998031883487, 0.2, 0.5], [0.33320212556583345, 0.307, 0.5], [0.3678409761857902, 0.231, 0.5], [0.6154300334579806, 0.5, 0.5], [0.31568588860460534, 0.22400000000000003, 0.5], [0.34363314308207044, 0.254, 0.3333333333333333], [0.27435544184215704, 0.306, 0.3333333333333333], [0.19051367840976186, 0.132, 0.3333333333333333], [0.5983074197992521, 0.5599999999999999, 0.6666666666666666], [0.3599685101358, 0.3, 0.3333333333333333], [0.4375123007282031, 0.5, 0.3333333333333333], [0.6376697500492029, 0.6, 0.6666666666666666], [0.5618972643180477, 0.205, 0.5], [0.3550482188545562, 0.3, 0.3333333333333333], [0.24680181066719148, 0.25, 0.3333333333333333], [0.2025191891359968, 0.26, 0.3333333333333333], [0.6851013580003936, 0.52, 0.6666666666666666], [0.19563078134225545, 0.156, 0.3333333333333333], [0.5030505805943711, 0.434, 0.6666666666666666], [0.3835859082857705, 0.351, 0.3333333333333333], [0.7480810864003148, 0.3, 0.6666666666666666], [0.5197795709506002, 0.15, 0.3333333333333333], [0.22121629600472348, 0.176, 0.3333333333333333], [0.9500098405825625, 0.673, 0.6666666666666666], [0.3993308403857508, 0.321, 0.3333333333333333], [0.27179689037591026, 0.2, 0.3333333333333333], [0.21688643967722887, 0.198, 0.3333333333333333], [0.3599685101358, 0.376, 0.6666666666666666], [0.3461916945483172, 0.264, 0.5], [0.395197795709506, 0.315, 0.5], [0.3237551663058453, 0.24700000000000003, 0.3333333333333333], [0.06042117693367447, 0.1, 0.16666666666666666], [0.3981499704782523, 0.20099999999999998, 0.3333333333333333], [0.2954142885258807, 0.209, 0.3333333333333333], [0.23656760480220426, 0.197, 0.3333333333333333], [0.33595748868333003, 0.3, 0.5], [0.5286360952568392, 0.314, 0.3333333333333333], [0.4975398543593781, 0.5, 0.3333333333333333], [0.2899035622908876, 0.22000000000000003, 0.3333333333333333], [0.20684904546349142, 0.125, 0.3333333333333333], [0.3526864790395592, 0.308, 0.3333333333333333], [0.5353276913993308, 0.4, 0.6666666666666666], [0.44794331824444006, 0.3, 0.3333333333333333], [0.3402873450108246, 0.271, 0.3333333333333333], [0.38260185002952174, 0.3, 0.3333333333333333], [0.32788821098209014, 0.33999999999999997, 0.3333333333333333], [0.19818933280850226, 0.183, 0.16666666666666666], [0.643180476284196, 0.5, 0.3333333333333333], [0.31450501869710684, 0.20299999999999999, 0.3333333333333333], [0.6854949813028931, 0.517, 0.6666666666666666], [0.2564455815784294, 0.2, 0.3333333333333333], [0.3597716984845503, 0.4, 0.3333333333333333], [0.4863215902381421, 0.585, 0.3333333333333333], [0.4164534540444794, 0.3, 0.3333333333333333], [0.5701633536705373, 0.3, 0.3333333333333333], [0.44262940366069664, 0.35, 0.3333333333333333], [0.11316669946860854, 0.1, 0.3333333333333333], [0.3211966148395985, 0.43, 0.3333333333333333], [0.4477465065931903, 0.325, 0.3333333333333333], [0.7905924030702618, 0.47300000000000003, 0.6666666666666666], [0.5369021846093289, 0.4, 0.3333333333333333], [0.23676441645345403, 0.15, 0.3333333333333333], [0.41350127927573316, 0.3, 0.3333333333333333], [0.24522731745719348, 0.15, 0.3333333333333333], [0.22338122416847075, 0.25, 0.3333333333333333], [0.3026963196221216, 0.3, 0.3333333333333333], [0.8718756150364101, 0.25, 0.5], [0.44125172210194846, 0.348, 0.3333333333333333], [0.41172997441448533, 0.40800000000000003, 0.3333333333333333], [0.3023026963196221, 0.16399999999999998, 0.3333333333333333], [0.40326707341074586, 0.40599999999999997, 0.3333333333333333], [0.4961621728006298, 0.429, 0.3333333333333333], [0.35898445187955125, 0.376, 0.3333333333333333], [0.28163747293839797, 0.4, 0.3333333333333333], [0.2755363117496556, 0.3, 0.3333333333333333], [0.14268844715607162, 0.1, 0.16666666666666666], [0.7492619563078134, 0.4, 0.5], [0.47136390474316076, 0.255, 0.3333333333333333], [0.5060027553631175, 0.4, 0.5], [0.3406809683133241, 0.35, 0.3333333333333333], [0.5890572721905136, 0.507, 0.6666666666666666], [0.209604408580988, 0.15, 0.3333333333333333], [0.24463688250344418, 0.18, 0.3333333333333333], [0.47392245620940754, 0.292, 0.6666666666666666], [0.23007282031096238, 0.231, 0.3333333333333333], [0.26412123597716985, 0.16799999999999998, 0.3333333333333333], [0.28065341468214916, 0.25, 0.3333333333333333], [0.31391458374335757, 0.2, 0.3333333333333333], [0.24562094075969299, 0.252, 0.3333333333333333], [0.5864987207242669, 0.42000000000000004, 1.0], [0.16768352686479038, 0.148, 0.3333333333333333], [0.2857705176146428, 0.2, 0.3333333333333333], [0.22397165912222003, 0.2, 0.3333333333333333], [0.44912418815193855, 0.21800000000000003, 0.5], [0.37551663058453055, 0.15, 0.3333333333333333], [0.3989372170832513, 0.28300000000000003, 0.3333333333333333], [0.2198386144459752, 0.15, 0.3333333333333333], [0.24129108443219838, 0.2, 0.3333333333333333], [0.3593780751820508, 0.325, 0.3333333333333333], [0.16748671521354064, 0.125, 0.3333333333333333], [0.20330643574099586, 0.2, 0.3333333333333333], [0.27848848651840186, 0.2, 0.3333333333333333], [0.3148986419996064, 0.2, 0.3333333333333333], [0.25900413304467623, 0.275, 0.3333333333333333], [0.3438299547333202, 0.35, 0.3333333333333333], [0.6750639637866561, 0.67, 1.0], [0.8106671914977366, 0.5, 0.8333333333333334], [0.5323755166305845, 0.5, 1.0], [0.3233615430033458, 0.22999999999999998, 0.3333333333333333], [0.16433772879354455, 0.15, 0.3333333333333333], [0.36685691792954145, 0.136, 0.5], [0.23361543003345794, 0.16299999999999998, 0.3333333333333333], [0.19248179492225936, 0.173, 0.3333333333333333], [0.14780555008856522, 0.2, 0.3333333333333333], [0.2769139933084038, 0.25, 0.3333333333333333], [0.258413698090927, 0.2, 0.3333333333333333], [0.3396969100570754, 0.274, 0.5], [0.48317260381814603, 0.2, 0.6666666666666666], [0.3890966345207636, 0.2, 0.6666666666666666], [0.5874827789805156, 0.514, 0.8333333333333334], [0.9480417240700649, 0.5, 1.0], [0.49202912812438493, 0.375, 0.6666666666666666], [0.2635308010234206, 0.261, 0.3333333333333333], [0.3245424129108443, 0.2, 0.6666666666666666], [0.4231450501869711, 0.35, 0.6666666666666666], [0.24916355048218855, 0.25, 0.3333333333333333], [0.3190316866758512, 0.2, 0.5], [0.27179689037591026, 0.2, 0.3333333333333333], [0.34461720133831925, 0.3, 0.3333333333333333], [0.48258216886439675, 0.348, 0.5], [0.4085809879944893, 0.22400000000000003, 0.3333333333333333], [0.6240897461129699, 0.45, 0.6666666666666666], [0.20842353867348945, 0.161, 0.3333333333333333], [0.2092107852784885, 0.2, 0.3333333333333333], [1.0, 1.0, 0.5], [0.31115922062586104, 0.316, 0.3333333333333333], [0.14268844715607162, 0.515, 0.3333333333333333], [0.6268451092304664, 0.318, 0.3333333333333333], [0.3310371974020862, 0.4, 0.3333333333333333], [0.6475103326116906, 0.311, 0.3333333333333333], [0.35209604408580986, 0.2, 0.3333333333333333], [0.28498327100964377, 0.2, 0.3333333333333333], [0.1889391851997638, 0.4, 0.3333333333333333], [0.6815587482778981, 0.355, 0.3333333333333333], [0.6819523715803975, 0.368, 0.6666666666666666], [0.459161582365676, 0.5650000000000001, 0.3333333333333333], [0.8925408384176343, 0.35, 0.5], [0.45601259594568, 0.65, 0.6666666666666666], [0.7980712458177524, 0.3, 0.3333333333333333], [0.407203306435741, 0.5, 0.8333333333333334], [0.4113363511119858, 0.35, 0.5], [0.5994882897067506, 0.2, 0.8333333333333334], [0.3572131470183034, 0.35, 0.5], [0.4546349143869317, 0.4, 0.5], [0.308797480810864, 0.15, 0.3333333333333333], [0.3898838811257626, 0.41900000000000004, 0.3333333333333333], [0.5597323361543003, 0.256, 0.3333333333333333], [0.3046644361346192, 0.202, 0.3333333333333333], [0.32631371777209206, 0.4, 0.3333333333333333], [0.148789608344814, 0.144, 0.3333333333333333], [0.2035032473922456, 0.2, 0.3333333333333333], [0.8484550285376894, 0.5, 0.6666666666666666], [0.2558551466246802, 0.2, 0.3333333333333333], [0.2658925408384176, 0.2, 0.3333333333333333], [0.3682345994882897, 0.4, 0.5], [0.25073804369218655, 0.20099999999999998, 0.3333333333333333], [0.2558551466246802, 0.2, 0.3333333333333333], [0.32277110804959647, 0.25, 0.3333333333333333], [0.4040543200157449, 0.4, 0.6666666666666666], [0.32414878960834476, 0.323, 0.5], [0.5233221806730958, 0.341, 0.5], [0.762251525290297, 0.3, 0.6666666666666666], [0.47766187758315287, 0.20299999999999999, 0.3333333333333333], [0.2511316669946861, 0.223, 0.3333333333333333], [0.5916158236567605, 0.2, 0.5], [0.5095453650856131, 0.516, 0.6666666666666666], [0.951190710490061, 0.9, 0.6666666666666666], [0.26116906120842354, 0.25, 0.3333333333333333], [0.554418421570557, 0.65, 0.5], [0.2538870301121826, 0.11000000000000001, 0.3333333333333333], [0.5540247982680574, 0.3, 0.8333333333333334], [0.22810470379846487, 0.15, 0.3333333333333333], [0.1523322180673096, 0.144, 0.3333333333333333], [0.5931903168667585, 0.309, 0.6666666666666666], [0.23932296791970084, 0.22000000000000003, 0.3333333333333333], [0.26412123597716985, 0.348, 0.3333333333333333], [0.16886439677228893, 0.192, 0.16666666666666666], [0.31450501869710684, 0.3, 0.5], [0.26412123597716985, 0.158, 0.3333333333333333], [0.32021255658334974, 0.25, 0.3333333333333333], [0.19858295611100177, 0.2, 0.3333333333333333], [0.40247982680574684, 0.3, 0.6666666666666666], [0.2613658728596733, 0.272, 0.3333333333333333], [0.4353473725644558, 0.288, 0.3333333333333333], [0.4725447746506593, 0.2, 0.6666666666666666], [0.308797480810864, 0.3, 0.5], [0.22849832710096435, 0.339, 0.3333333333333333], [0.21196614839598502, 0.147, 0.3333333333333333], [0.3056484943908679, 0.3, 0.3333333333333333], [0.19818933280850226, 0.125, 0.3333333333333333], [0.24798268057469, 0.1, 0.3333333333333333], [0.6461326510529423, 0.11699999999999999, 0.3333333333333333], [0.7051761464278684, 0.46699999999999997, 0.5], [0.5713442235780358, 0.592, 0.5], [0.5349340680968313, 0.2, 0.3333333333333333], [0.4461720133831923, 0.2, 0.3333333333333333], [0.35071836252706157, 0.175, 0.3333333333333333], [0.36961228104703797, 0.3, 0.3333333333333333]];\n",
       "  var pct_var = [7925.293861397826, 191.44546380624726, 90.45908385616922];\n",
       "  var md_headers = ['#SampleID', 'total_bill', 'tip', 'sex', 'smoker', 'day', 'time', 'size'];\n",
       "  var metadata = [['0', '16.99', '1.01', 'Female', 'No', 'Sun', 'Dinner', '2'], ['1', '10.34', '1.66', 'Male', 'No', 'Sun', 'Dinner', '3'], ['2', '21.01', '3.5', 'Male', 'No', 'Sun', 'Dinner', '3'], ['3', '23.68', '3.31', 'Male', 'No', 'Sun', 'Dinner', '2'], ['4', '24.59', '3.61', 'Female', 'No', 'Sun', 'Dinner', '4'], ['5', '25.29', '4.71', 'Male', 'No', 'Sun', 'Dinner', '4'], ['6', '8.77', '2.0', 'Male', 'No', 'Sun', 'Dinner', '2'], ['7', '26.88', '3.12', 'Male', 'No', 'Sun', 'Dinner', '4'], ['8', '15.04', '1.96', 'Male', 'No', 'Sun', 'Dinner', '2'], ['9', '14.78', '3.23', 'Male', 'No', 'Sun', 'Dinner', '2'], ['10', '10.27', '1.71', 'Male', 'No', 'Sun', 'Dinner', '2'], ['11', '35.26', '5.0', 'Female', 'No', 'Sun', 'Dinner', '4'], ['12', '15.42', '1.57', 'Male', 'No', 'Sun', 'Dinner', '2'], ['13', '18.43', '3.0', 'Male', 'No', 'Sun', 'Dinner', '4'], ['14', '14.83', '3.02', 'Female', 'No', 'Sun', 'Dinner', '2'], ['15', '21.58', '3.92', 'Male', 'No', 'Sun', 'Dinner', '2'], ['16', '10.33', '1.67', 'Female', 'No', 'Sun', 'Dinner', '3'], ['17', '16.29', '3.71', 'Male', 'No', 'Sun', 'Dinner', '3'], ['18', '16.97', '3.5', 'Female', 'No', 'Sun', 'Dinner', '3'], ['19', '20.65', '3.35', 'Male', 'No', 'Sat', 'Dinner', '3'], ['20', '17.92', '4.08', 'Male', 'No', 'Sat', 'Dinner', '2'], ['21', '20.29', '2.75', 'Female', 'No', 'Sat', 'Dinner', '2'], ['22', '15.77', '2.23', 'Female', 'No', 'Sat', 'Dinner', '2'], ['23', '39.42', '7.58', 'Male', 'No', 'Sat', 'Dinner', '4'], ['24', '19.82', '3.18', 'Male', 'No', 'Sat', 'Dinner', '2'], ['25', '17.81', '2.34', 'Male', 'No', 'Sat', 'Dinner', '4'], ['26', '13.37', '2.0', 'Male', 'No', 'Sat', 'Dinner', '2'], ['27', '12.69', '2.0', 'Male', 'No', 'Sat', 'Dinner', '2'], ['28', '21.7', '4.3', 'Male', 'No', 'Sat', 'Dinner', '2'], ['29', '19.65', '3.0', 'Female', 'No', 'Sat', 'Dinner', '2'], ['30', '9.55', '1.45', 'Male', 'No', 'Sat', 'Dinner', '2'], ['31', '18.35', '2.5', 'Male', 'No', 'Sat', 'Dinner', '4'], ['32', '15.06', '3.0', 'Female', 'No', 'Sat', 'Dinner', '2'], ['33', '20.69', '2.45', 'Female', 'No', 'Sat', 'Dinner', '4'], ['34', '17.78', '3.27', 'Male', 'No', 'Sat', 'Dinner', '2'], ['35', '24.06', '3.6', 'Male', 'No', 'Sat', 'Dinner', '3'], ['36', '16.31', '2.0', 'Male', 'No', 'Sat', 'Dinner', '3'], ['37', '16.93', '3.07', 'Female', 'No', 'Sat', 'Dinner', '3'], ['38', '18.69', '2.31', 'Male', 'No', 'Sat', 'Dinner', '3'], ['39', '31.27', '5.0', 'Male', 'No', 'Sat', 'Dinner', '3'], ['40', '16.04', '2.24', 'Male', 'No', 'Sat', 'Dinner', '3'], ['41', '17.46', '2.54', 'Male', 'No', 'Sun', 'Dinner', '2'], ['42', '13.94', '3.06', 'Male', 'No', 'Sun', 'Dinner', '2'], ['43', '9.68', '1.32', 'Male', 'No', 'Sun', 'Dinner', '2'], ['44', '30.4', '5.6', 'Male', 'No', 'Sun', 'Dinner', '4'], ['45', '18.29', '3.0', 'Male', 'No', 'Sun', 'Dinner', '2'], ['46', '22.23', '5.0', 'Male', 'No', 'Sun', 'Dinner', '2'], ['47', '32.4', '6.0', 'Male', 'No', 'Sun', 'Dinner', '4'], ['48', '28.55', '2.05', 'Male', 'No', 'Sun', 'Dinner', '3'], ['49', '18.04', '3.0', 'Male', 'No', 'Sun', 'Dinner', '2'], ['50', '12.54', '2.5', 'Male', 'No', 'Sun', 'Dinner', '2'], ['51', '10.29', '2.6', 'Female', 'No', 'Sun', 'Dinner', '2'], ['52', '34.81', '5.2', 'Female', 'No', 'Sun', 'Dinner', '4'], ['53', '9.94', '1.56', 'Male', 'No', 'Sun', 'Dinner', '2'], ['54', '25.56', '4.34', 'Male', 'No', 'Sun', 'Dinner', '4'], ['55', '19.49', '3.51', 'Male', 'No', 'Sun', 'Dinner', '2'], ['56', '38.01', '3.0', 'Male', 'Yes', 'Sat', 'Dinner', '4'], ['57', '26.41', '1.5', 'Female', 'No', 'Sat', 'Dinner', '2'], ['58', '11.24', '1.76', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['59', '48.27', '6.73', 'Male', 'No', 'Sat', 'Dinner', '4'], ['60', '20.29', '3.21', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['61', '13.81', '2.0', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['62', '11.02', '1.98', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['63', '18.29', '3.76', 'Male', 'Yes', 'Sat', 'Dinner', '4'], ['64', '17.59', '2.64', 'Male', 'No', 'Sat', 'Dinner', '3'], ['65', '20.08', '3.15', 'Male', 'No', 'Sat', 'Dinner', '3'], ['66', '16.45', '2.47', 'Female', 'No', 'Sat', 'Dinner', '2'], ['67', '3.07', '1.0', 'Female', 'Yes', 'Sat', 'Dinner', '1'], ['68', '20.23', '2.01', 'Male', 'No', 'Sat', 'Dinner', '2'], ['69', '15.01', '2.09', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['70', '12.02', '1.97', 'Male', 'No', 'Sat', 'Dinner', '2'], ['71', '17.07', '3.0', 'Female', 'No', 'Sat', 'Dinner', '3'], ['72', '26.86', '3.14', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['73', '25.28', '5.0', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['74', '14.73', '2.2', 'Female', 'No', 'Sat', 'Dinner', '2'], ['75', '10.51', '1.25', 'Male', 'No', 'Sat', 'Dinner', '2'], ['76', '17.92', '3.08', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['77', '27.2', '4.0', 'Male', 'No', 'Thur', 'Lunch', '4'], ['78', '22.76', '3.0', 'Male', 'No', 'Thur', 'Lunch', '2'], ['79', '17.29', '2.71', 'Male', 'No', 'Thur', 'Lunch', '2'], ['80', '19.44', '3.0', 'Male', 'Yes', 'Thur', 'Lunch', '2'], ['81', '16.66', '3.4', 'Male', 'No', 'Thur', 'Lunch', '2'], ['82', '10.07', '1.83', 'Female', 'No', 'Thur', 'Lunch', '1'], ['83', '32.68', '5.0', 'Male', 'Yes', 'Thur', 'Lunch', '2'], ['84', '15.98', '2.03', 'Male', 'No', 'Thur', 'Lunch', '2'], ['85', '34.83', '5.17', 'Female', 'No', 'Thur', 'Lunch', '4'], ['86', '13.03', '2.0', 'Male', 'No', 'Thur', 'Lunch', '2'], ['87', '18.28', '4.0', 'Male', 'No', 'Thur', 'Lunch', '2'], ['88', '24.71', '5.85', 'Male', 'No', 'Thur', 'Lunch', '2'], ['89', '21.16', '3.0', 'Male', 'No', 'Thur', 'Lunch', '2'], ['90', '28.97', '3.0', 'Male', 'Yes', 'Fri', 'Dinner', '2'], ['91', '22.49', '3.5', 'Male', 'No', 'Fri', 'Dinner', '2'], ['92', '5.75', '1.0', 'Female', 'Yes', 'Fri', 'Dinner', '2'], ['93', '16.32', '4.3', 'Female', 'Yes', 'Fri', 'Dinner', '2'], ['94', '22.75', '3.25', 'Female', 'No', 'Fri', 'Dinner', '2'], ['95', '40.17', '4.73', 'Male', 'Yes', 'Fri', 'Dinner', '4'], ['96', '27.28', '4.0', 'Male', 'Yes', 'Fri', 'Dinner', '2'], ['97', '12.03', '1.5', 'Male', 'Yes', 'Fri', 'Dinner', '2'], ['98', '21.01', '3.0', 'Male', 'Yes', 'Fri', 'Dinner', '2'], ['99', '12.46', '1.5', 'Male', 'No', 'Fri', 'Dinner', '2'], ['100', '11.35', '2.5', 'Female', 'Yes', 'Fri', 'Dinner', '2'], ['101', '15.38', '3.0', 'Female', 'Yes', 'Fri', 'Dinner', '2'], ['102', '44.3', '2.5', 'Female', 'Yes', 'Sat', 'Dinner', '3'], ['103', '22.42', '3.48', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['104', '20.92', '4.08', 'Female', 'No', 'Sat', 'Dinner', '2'], ['105', '15.36', '1.64', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['106', '20.49', '4.06', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['107', '25.21', '4.29', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['108', '18.24', '3.76', 'Male', 'No', 'Sat', 'Dinner', '2'], ['109', '14.31', '4.0', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['110', '14.0', '3.0', 'Male', 'No', 'Sat', 'Dinner', '2'], ['111', '7.25', '1.0', 'Female', 'No', 'Sat', 'Dinner', '1'], ['112', '38.07', '4.0', 'Male', 'No', 'Sun', 'Dinner', '3'], ['113', '23.95', '2.55', 'Male', 'No', 'Sun', 'Dinner', '2'], ['114', '25.71', '4.0', 'Female', 'No', 'Sun', 'Dinner', '3'], ['115', '17.31', '3.5', 'Female', 'No', 'Sun', 'Dinner', '2'], ['116', '29.93', '5.07', 'Male', 'No', 'Sun', 'Dinner', '4'], ['117', '10.65', '1.5', 'Female', 'No', 'Thur', 'Lunch', '2'], ['118', '12.43', '1.8', 'Female', 'No', 'Thur', 'Lunch', '2'], ['119', '24.08', '2.92', 'Female', 'No', 'Thur', 'Lunch', '4'], ['120', '11.69', '2.31', 'Male', 'No', 'Thur', 'Lunch', '2'], ['121', '13.42', '1.68', 'Female', 'No', 'Thur', 'Lunch', '2'], ['122', '14.26', '2.5', 'Male', 'No', 'Thur', 'Lunch', '2'], ['123', '15.95', '2.0', 'Male', 'No', 'Thur', 'Lunch', '2'], ['124', '12.48', '2.52', 'Female', 'No', 'Thur', 'Lunch', '2'], ['125', '29.8', '4.2', 'Female', 'No', 'Thur', 'Lunch', '6'], ['126', '8.52', '1.48', 'Male', 'No', 'Thur', 'Lunch', '2'], ['127', '14.52', '2.0', 'Female', 'No', 'Thur', 'Lunch', '2'], ['128', '11.38', '2.0', 'Female', 'No', 'Thur', 'Lunch', '2'], ['129', '22.82', '2.18', 'Male', 'No', 'Thur', 'Lunch', '3'], ['130', '19.08', '1.5', 'Male', 'No', 'Thur', 'Lunch', '2'], ['131', '20.27', '2.83', 'Female', 'No', 'Thur', 'Lunch', '2'], ['132', '11.17', '1.5', 'Female', 'No', 'Thur', 'Lunch', '2'], ['133', '12.26', '2.0', 'Female', 'No', 'Thur', 'Lunch', '2'], ['134', '18.26', '3.25', 'Female', 'No', 'Thur', 'Lunch', '2'], ['135', '8.51', '1.25', 'Female', 'No', 'Thur', 'Lunch', '2'], ['136', '10.33', '2.0', 'Female', 'No', 'Thur', 'Lunch', '2'], ['137', '14.15', '2.0', 'Female', 'No', 'Thur', 'Lunch', '2'], ['138', '16.0', '2.0', 'Male', 'Yes', 'Thur', 'Lunch', '2'], ['139', '13.16', '2.75', 'Female', 'No', 'Thur', 'Lunch', '2'], ['140', '17.47', '3.5', 'Female', 'No', 'Thur', 'Lunch', '2'], ['141', '34.3', '6.7', 'Male', 'No', 'Thur', 'Lunch', '6'], ['142', '41.19', '5.0', 'Male', 'No', 'Thur', 'Lunch', '5'], ['143', '27.05', '5.0', 'Female', 'No', 'Thur', 'Lunch', '6'], ['144', '16.43', '2.3', 'Female', 'No', 'Thur', 'Lunch', '2'], ['145', '8.35', '1.5', 'Female', 'No', 'Thur', 'Lunch', '2'], ['146', '18.64', '1.36', 'Female', 'No', 'Thur', 'Lunch', '3'], ['147', '11.87', '1.63', 'Female', 'No', 'Thur', 'Lunch', '2'], ['148', '9.78', '1.73', 'Male', 'No', 'Thur', 'Lunch', '2'], ['149', '7.51', '2.0', 'Male', 'No', 'Thur', 'Lunch', '2'], ['150', '14.07', '2.5', 'Male', 'No', 'Sun', 'Dinner', '2'], ['151', '13.13', '2.0', 'Male', 'No', 'Sun', 'Dinner', '2'], ['152', '17.26', '2.74', 'Male', 'No', 'Sun', 'Dinner', '3'], ['153', '24.55', '2.0', 'Male', 'No', 'Sun', 'Dinner', '4'], ['154', '19.77', '2.0', 'Male', 'No', 'Sun', 'Dinner', '4'], ['155', '29.85', '5.14', 'Female', 'No', 'Sun', 'Dinner', '5'], ['156', '48.17', '5.0', 'Male', 'No', 'Sun', 'Dinner', '6'], ['157', '25.0', '3.75', 'Female', 'No', 'Sun', 'Dinner', '4'], ['158', '13.39', '2.61', 'Female', 'No', 'Sun', 'Dinner', '2'], ['159', '16.49', '2.0', 'Male', 'No', 'Sun', 'Dinner', '4'], ['160', '21.5', '3.5', 'Male', 'No', 'Sun', 'Dinner', '4'], ['161', '12.66', '2.5', 'Male', 'No', 'Sun', 'Dinner', '2'], ['162', '16.21', '2.0', 'Female', 'No', 'Sun', 'Dinner', '3'], ['163', '13.81', '2.0', 'Male', 'No', 'Sun', 'Dinner', '2'], ['164', '17.51', '3.0', 'Female', 'Yes', 'Sun', 'Dinner', '2'], ['165', '24.52', '3.48', 'Male', 'No', 'Sun', 'Dinner', '3'], ['166', '20.76', '2.24', 'Male', 'No', 'Sun', 'Dinner', '2'], ['167', '31.71', '4.5', 'Male', 'No', 'Sun', 'Dinner', '4'], ['168', '10.59', '1.61', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['169', '10.63', '2.0', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['170', '50.81', '10.0', 'Male', 'Yes', 'Sat', 'Dinner', '3'], ['171', '15.81', '3.16', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['172', '7.25', '5.15', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['173', '31.85', '3.18', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['174', '16.82', '4.0', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['175', '32.9', '3.11', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['176', '17.89', '2.0', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['177', '14.48', '2.0', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['178', '9.6', '4.0', 'Female', 'Yes', 'Sun', 'Dinner', '2'], ['179', '34.63', '3.55', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['180', '34.65', '3.68', 'Male', 'Yes', 'Sun', 'Dinner', '4'], ['181', '23.33', '5.65', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['182', '45.35', '3.5', 'Male', 'Yes', 'Sun', 'Dinner', '3'], ['183', '23.17', '6.5', 'Male', 'Yes', 'Sun', 'Dinner', '4'], ['184', '40.55', '3.0', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['185', '20.69', '5.0', 'Male', 'No', 'Sun', 'Dinner', '5'], ['186', '20.9', '3.5', 'Female', 'Yes', 'Sun', 'Dinner', '3'], ['187', '30.46', '2.0', 'Male', 'Yes', 'Sun', 'Dinner', '5'], ['188', '18.15', '3.5', 'Female', 'Yes', 'Sun', 'Dinner', '3'], ['189', '23.1', '4.0', 'Male', 'Yes', 'Sun', 'Dinner', '3'], ['190', '15.69', '1.5', 'Male', 'Yes', 'Sun', 'Dinner', '2'], ['191', '19.81', '4.19', 'Female', 'Yes', 'Thur', 'Lunch', '2'], ['192', '28.44', '2.56', 'Male', 'Yes', 'Thur', 'Lunch', '2'], ['193', '15.48', '2.02', 'Male', 'Yes', 'Thur', 'Lunch', '2'], ['194', '16.58', '4.0', 'Male', 'Yes', 'Thur', 'Lunch', '2'], ['195', '7.56', '1.44', 'Male', 'No', 'Thur', 'Lunch', '2'], ['196', '10.34', '2.0', 'Male', 'Yes', 'Thur', 'Lunch', '2'], ['197', '43.11', '5.0', 'Female', 'Yes', 'Thur', 'Lunch', '4'], ['198', '13.0', '2.0', 'Female', 'Yes', 'Thur', 'Lunch', '2'], ['199', '13.51', '2.0', 'Male', 'Yes', 'Thur', 'Lunch', '2'], ['200', '18.71', '4.0', 'Male', 'Yes', 'Thur', 'Lunch', '3'], ['201', '12.74', '2.01', 'Female', 'Yes', 'Thur', 'Lunch', '2'], ['202', '13.0', '2.0', 'Female', 'Yes', 'Thur', 'Lunch', '2'], ['203', '16.4', '2.5', 'Female', 'Yes', 'Thur', 'Lunch', '2'], ['204', '20.53', '4.0', 'Male', 'Yes', 'Thur', 'Lunch', '4'], ['205', '16.47', '3.23', 'Female', 'Yes', 'Thur', 'Lunch', '3'], ['206', '26.59', '3.41', 'Male', 'Yes', 'Sat', 'Dinner', '3'], ['207', '38.73', '3.0', 'Male', 'Yes', 'Sat', 'Dinner', '4'], ['208', '24.27', '2.03', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['209', '12.76', '2.23', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['210', '30.06', '2.0', 'Male', 'Yes', 'Sat', 'Dinner', '3'], ['211', '25.89', '5.16', 'Male', 'Yes', 'Sat', 'Dinner', '4'], ['212', '48.33', '9.0', 'Male', 'No', 'Sat', 'Dinner', '4'], ['213', '13.27', '2.5', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['214', '28.17', '6.5', 'Female', 'Yes', 'Sat', 'Dinner', '3'], ['215', '12.9', '1.1', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['216', '28.15', '3.0', 'Male', 'Yes', 'Sat', 'Dinner', '5'], ['217', '11.59', '1.5', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['218', '7.74', '1.44', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['219', '30.14', '3.09', 'Female', 'Yes', 'Sat', 'Dinner', '4'], ['220', '12.16', '2.2', 'Male', 'Yes', 'Fri', 'Lunch', '2'], ['221', '13.42', '3.48', 'Female', 'Yes', 'Fri', 'Lunch', '2'], ['222', '8.58', '1.92', 'Male', 'Yes', 'Fri', 'Lunch', '1'], ['223', '15.98', '3.0', 'Female', 'No', 'Fri', 'Lunch', '3'], ['224', '13.42', '1.58', 'Male', 'Yes', 'Fri', 'Lunch', '2'], ['225', '16.27', '2.5', 'Female', 'Yes', 'Fri', 'Lunch', '2'], ['226', '10.09', '2.0', 'Female', 'Yes', 'Fri', 'Lunch', '2'], ['227', '20.45', '3.0', 'Male', 'No', 'Sat', 'Dinner', '4'], ['228', '13.28', '2.72', 'Male', 'No', 'Sat', 'Dinner', '2'], ['229', '22.12', '2.88', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['230', '24.01', '2.0', 'Male', 'Yes', 'Sat', 'Dinner', '4'], ['231', '15.69', '3.0', 'Male', 'Yes', 'Sat', 'Dinner', '3'], ['232', '11.61', '3.39', 'Male', 'No', 'Sat', 'Dinner', '2'], ['233', '10.77', '1.47', 'Male', 'No', 'Sat', 'Dinner', '2'], ['234', '15.53', '3.0', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['235', '10.07', '1.25', 'Male', 'No', 'Sat', 'Dinner', '2'], ['236', '12.6', '1.0', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['237', '32.83', '1.17', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['238', '35.83', '4.67', 'Female', 'No', 'Sat', 'Dinner', '3'], ['239', '29.03', '5.92', 'Male', 'No', 'Sat', 'Dinner', '3'], ['240', '27.18', '2.0', 'Female', 'Yes', 'Sat', 'Dinner', '2'], ['241', '22.67', '2.0', 'Male', 'Yes', 'Sat', 'Dinner', '2'], ['242', '17.82', '1.75', 'Male', 'No', 'Sat', 'Dinner', '2'], ['243', '18.78', '3.0', 'Female', 'No', 'Thur', 'Dinner', '2']];\n",
       "  var axesNames = ['total_bill', 'tip', 'size'];\n",
       "\n",
       "  var dm, ec;\n",
       "\n",
       "  function init() {\n",
       "    // Initialize the DecompositionModel\n",
       "    dm = new DecompositionModel(name, ids, coords, pct_var,\n",
       "                                md_headers, metadata, axesNames);\n",
       "    // Initialize the EmperorController\n",
       "    ec = new EmperorController(dm, 'emperor-notebook-0xc5a5f7a5');\n",
       "  }\n",
       "\n",
       "  function animate() {\n",
       "    requestAnimationFrame(animate);\n",
       "    ec.render();\n",
       "  }\n",
       "  $(window).resize(function() {\n",
       "    ec.resize(div.innerWidth(), div.innerHeight());\n",
       "  });\n",
       "\n",
       "  $(function(){\n",
       "    init();\n",
       "    animate();\n",
       "\n",
       "  });\n",
       "\n",
       "}); // END REQUIRE.JS block\n",
       "</script>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<emperor.core.Emperor at 0x1176b74e0>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from emperor import scatterplot\n",
    "scatterplot(df, remote=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<div style='height:500px'></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Outline\n",
    "\n",
    "- ~~Background (why $\\beta$-diversity).~~\n",
    "\n",
    "- ~~What is Emperor.~~\n",
    "\n",
    "- **How can we use Emperor.**\n",
    "    \n",
    "- Analyzing a use case."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Emperor and Jupyter 📓\n",
    "\n",
    "- nbviewer\n",
    "    See the examples folder in our repo: https://github.com/biocore/emperor/tree/new-api/examples\n",
    "\n",
    "- Jupyter notebook\n",
    "\n",
    "- Standalone HTML plot\n",
    "    - Generate a standalone HTML file with the needed resources."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<div style='height:500px'></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# JavaScript"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Isn't this Sci**Py**."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "    SciJS?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Thoroughly unit tested."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Public API ready to be used:\n",
    "\n",
    "    http://emperor.microbio.me/uno/build/jsdoc/index.html"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# JavaScript"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "![JS-DOC](./images/js-doc.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# JavaScript"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- Integration with SAGE2:\n",
    "\n",
    "    http://sage2.sagecommons.org/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<div style='height:500px'></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# JavaScript"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- Integration with SAGE2:\n",
    "\n",
    "    http://sage2.sagecommons.org/\n",
    "\n",
    "![sage](./images/vroom.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# QIIME 2 integration"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- 1.5 hours to implement."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- In less than 80 lines of code we got a CLI, GUI and provenance tracking."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Consider QIIME 2 as a gateway to an expanded user base."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- CLI Provided through QIIME 2\n",
    "\n",
    "    https://github.com/qiime2/qiime2\n",
    "    \n",
    "    https://github.com/qiime2/q2-emperor/\n",
    "\n",
    "```bash\n",
    "qiime emperor plot --help\n",
    "```"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Outline\n",
    "\n",
    "- ~~Background (why $\\beta$-diversity).~~\n",
    "\n",
    "- ~~What is Emperor.~~\n",
    "\n",
    "- ~~How can we use Emperor.~~\n",
    "    \n",
    "- **Analyzing a use case.**"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Use case"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Let's leverage the technology we have available."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "- Create a small interface to:\n",
    "\n",
    "    - Subsample.\n",
    "    - Compute a distance matrix.\n",
    "        * Use all the cores in our machine.\n",
    "    - Visualize.\n",
    "    \n",
    "    - This was kinda possible through E-vident https://github.com/biocore/evident"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "outputs": [],
   "source": [
    "# biocore\n",
    "from emperor.qiime_backports.parse import parse_mapping_file\n",
    "from emperor import Emperor, nbinstall\n",
    "\n",
    "nbinstall()\n",
    "\n",
    "from skbio.stats.ordination import pcoa\n",
    "from skbio.diversity import beta_diversity\n",
    "from skbio import TreeNode\n",
    "from skbio.io.util import open_file\n",
    "\n",
    "from biom import load_table\n",
    "from biom.util import biom_open\n",
    "\n",
    "import qiime_default_reference\n",
    "\n",
    "# pydata/scipy\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "from scipy.spatial.distance import braycurtis, canberra\n",
    "from ipywidgets import interact\n",
    "from sklearn.metrics import pairwise_distances\n",
    "from functools import partial"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "outputs": [],
   "source": [
    "import warnings\n",
    "# don't try this at home\n",
    "warnings.filterwarnings(action='ignore', category=Warning)\n",
    "\n",
    "# -1 means all the processors available\n",
    "pw_dists = partial(pairwise_distances, n_jobs=-1)\n",
    "\n",
    "def load_mf(fn):\n",
    "    with open_file(fn) as f:\n",
    "        mapping_data, header, _ = parse_mapping_file(f)\n",
    "        _mapping_file = pd.DataFrame(mapping_data, columns=header)\n",
    "        _mapping_file.set_index('SampleID', inplace=True)\n",
    "    return _mapping_file"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Load the data (table and tree)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "mf = load_mf('keyboard/mapping-file.txt')\n",
    "bt = load_table('keyboard/otu-table.biom')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "tree = TreeNode.read(qiime_default_reference.get_reference_tree())\n",
    "\n",
    "for n in tree.traverse():\n",
    "    if n.length is None:\n",
    "        n.length = 0"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Interaction function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def evident(n, metric):\n",
    "    rarefied = bt.subsample(n)\n",
    "    data = np.array([rarefied.data(i) for i in rarefied.ids()], dtype='int64')\n",
    "    \n",
    "    # phylogenetic\n",
    "    if metric in ['unweighted_unifrac', 'weighted_unifrac']:\n",
    "        res = pcoa(beta_diversity(metric, data, rarefied.ids(),\n",
    "                                  otu_ids=rarefied.ids('observation'),\n",
    "                                  tree=tree, pairwise_func=pw_dists))\n",
    "    # non-phylogenetic\n",
    "    else:\n",
    "        res = pcoa(beta_diversity(metric, data, rarefied.ids(),\n",
    "                                  pairwise_func=pw_dists))\n",
    "    return Emperor(res, mf, remote=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<script type=\"text/javascript\">\n",
       "\n",
       "if ($(\"#emperor-css\").length == 0){{\n",
       "    $(\"head\").append([\n",
       "\n",
       "        '<link id=\"emperor-css\" rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/css/emperor.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/jquery-ui.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/slick.grid.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/spectrum.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/chosen.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/jquery.contextMenu.min.css\">'\n",
       "    ]);\n",
       "}}\n",
       "</script>\n",
       "\n",
       "<div id='emperor-notebook-0xc38db72e' style=\"position: relative; width:100%; height:500px;\">\n",
       "  <div class='loading' style=\"position: absolute;top: 50%;left: 50%;margin-left: -229px; margin-top: -59px; z-index: 10000;height:118px;width:458px;padding:0px\"><img src='https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/img/emperor.png' alt='Emperor resources missing. Expected them to be found in https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files'></div>\n",
       "</div>\n",
       "</div>\n",
       "\n",
       "<script type=\"text/javascript\">\n",
       "requirejs.config({\n",
       "// the left side is the module name, and the right side is the path\n",
       "// relative to the baseUrl attribute, do NOT include the .js extension\n",
       "'paths': {\n",
       "  /* jQuery */\n",
       "  'jquery': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery-2.1.4.min',\n",
       "  'jqueryui': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery-ui.min',\n",
       "  'jquery_drag': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.event.drag-2.2.min',\n",
       "\n",
       "  /* jQuery plugins */\n",
       "  'chosen': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/chosen.jquery.min',\n",
       "  'spectrum': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/spectrum.min',\n",
       "  'position': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.ui.position.min',\n",
       "  'contextmenu': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.contextMenu.min',\n",
       "\n",
       "  /* other libraries */\n",
       "  'underscore': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/underscore-min',\n",
       "  'chroma': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/chroma.min',\n",
       "  'filesaver': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/FileSaver.min',\n",
       "  'blob': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/Blob',\n",
       "  'd3': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/d3.min',\n",
       "\n",
       "\n",
       "  /* THREE.js and plugins */\n",
       "  'three': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.min',\n",
       "  'orbitcontrols': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/OrbitControls',\n",
       "\n",
       "  /* SlickGrid */\n",
       "  'slickcore': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.core.min',\n",
       "  'slickgrid': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.grid.min',\n",
       "  'slickformatters': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.editors.min',\n",
       "  'slickeditors': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.formatters.min',\n",
       "\n",
       "  /* Emperor's objects */\n",
       "  'util': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/util',\n",
       "  'model': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/model',\n",
       "  'view': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/view',\n",
       "  'controller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/controller',\n",
       "  'draw': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/draw',\n",
       "  'scene3d': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/sceneplotview3d',\n",
       "  'viewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/view-controller',\n",
       "  'colorviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/color-view-controller',\n",
       "  'visibilitycontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/visibility-controller',\n",
       "  'scaleviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/scale-view-controller',\n",
       "  'shapecontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shape-controller',\n",
       "  'axescontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/axes-controller',\n",
       "  'shape-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shape-editor',\n",
       "  'color-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/color-editor',\n",
       "  'scale-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/scale-editor',\n",
       "  'shapes': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shapes'\n",
       "},\n",
       "/*\n",
       "   Libraries that are not AMD compatible need shim to declare their\n",
       "   dependencies.\n",
       " */\n",
       "'shim': {\n",
       "  'jquery_drag': {\n",
       "    'deps': ['jquery', 'jqueryui']\n",
       "  },\n",
       "  'chosen': {\n",
       "    'deps': ['jquery'],\n",
       "    'exports': 'jQuery.fn.chosen'\n",
       "  },\n",
       "  'contextmenu' : {\n",
       "    'deps': ['jquery', 'jqueryui', 'position']\n",
       "  },\n",
       "  'filesaver' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'orbitcontrols': {\n",
       "    'deps': ['three']\n",
       "  },\n",
       "'slickcore': ['jqueryui'],\n",
       "'slickgrid': ['slickcore', 'jquery_drag', 'slickformatters',\n",
       "              'slickeditors']\n",
       "}\n",
       "});\n",
       "\n",
       "requirejs(\n",
       "[\"jquery\", \"model\", \"controller\"],\n",
       "function($, model, EmperorController) {\n",
       "  var DecompositionModel = model.DecompositionModel;\n",
       "\n",
       "  var div = $('#emperor-notebook-0xc38db72e');\n",
       "\n",
       "  var ids = ['232.M9Thmr217', '232.M3Rinr217', '232.M2Enter217', '232.M3Midr217', '232.M3Tkey217', '232.M2Okey217', '232.M9Hkey217', '232.M9Pinl217', '232.M9Midl217', '232.M9Rinl217', '232.M2Midr217', '232.M2Lsft217', '232.M9Mkey217', '232.M3Wkey217', '232.M9Gkey217', '232.M3Midl217', '232.M3Pinl217', '232.M9Vkey217', '232.M2Tkey217', '232.M9Dkey217', '232.M2Gkey217', '232.M3Lkey217', '232.M9Pkey217', '232.M2Ykey217', '232.M2Ikey217', '232.M9Midr217', '232.M9Rinr217', '232.M2Lkey217', '232.M2Vkey217', '232.M3Gkey217', '232.M9Indr217', '232.M3Ykey217', '232.M2Bkey217', '232.M3Pinr217', '232.M2Akey217', '232.M3Thmr217', '232.M2Skey217', '232.M3Rinl217'];\n",
       "  var coords = [[0.21082317684678245, -0.011729270321396059, 0.07648261103138583, -0.06472626907439936, 0.10107473617557027], [0.06452666285150635, 0.3501803743888372, -0.11259053525398556, -0.17401050281541014, -0.12731128224456892], [-0.36995186069497915, 0.07833256185821566, 0.1879046217929765, -0.12466637323629537, 0.25176710588622486], [0.15427944523121642, 0.22343487783597524, -0.05607151853394717, -0.011117198848393475, 0.11156772617293255], [0.2573240198016326, 0.014007945872554698, -0.056161642852417294, -0.03662943144235211, -0.06228514140087811], [-0.2700287936928274, 0.08518937959163413, 0.1662603922929612, 0.09676226142204662, -0.11328959868421509], [0.25734523380872876, -0.11385868650689271, 0.036945454758190895, 0.062138947006931566, -0.03554956332614192], [0.22609721598745913, -0.08966907144428689, 0.04739514718245031, -0.02440263713205035, 0.014023181104859788], [0.2768952019575235, -0.11054238200022659, 0.04091353489320524, -0.03645797004092059, 0.053360616278608025], [0.25595262588205675, -0.0877941773079497, 0.050530853487840144, -0.026257897730547652, 0.06273170591467549], [-0.5262301808924661, -0.135783388957506, -0.16622481095189626, -0.10681794074082229, 0.1197678521350139], [-0.286870789319698, 0.06122496302898579, 0.23975410144082493, -0.027318437147617253, -0.059819739842724], [0.24330665950536443, -0.10075999783074406, 0.0159171001725495, 0.023333994579695323, -0.03730207250774789], [0.2691403532794054, 0.0017916068555180575, -0.011608161317428705, 0.0027775051991313342, 0.050244514884892476], [0.24887922584561678, -0.11258978823846943, -0.004339878427831215, 0.035310266621801624, -0.0478044222230235], [0.24332344990457513, -0.002384592399404158, -0.032396154313634816, 0.009452267216313863, -0.006596293969871966], [0.13246337067261002, 0.21033447878300549, -0.06013266887938329, 0.03514443904054362, 0.09448969827245417], [0.23653596148146958, -0.11458459726395949, 0.035787013111871255, 0.021088577900465102, -0.024977825612387563], [-0.5301378336201343, -0.09170728702708765, -0.11936765136457368, -0.05862611962718643, -0.0436723748256655], [0.24014415949532625, -0.11076648403326222, 0.021915562528900246, 0.06791830351900689, -0.04770292599299448], [-0.5355442918473617, -0.05152927632557259, 0.1319202836591836, -0.0538703755142327, 0.002870583378425642], [0.2103623160457151, 0.11068002191407805, -0.0644916529563493, 0.052207789569134244, 0.022465589065906587], [0.2742832702771842, -0.09143288593327976, 0.034435016443474166, 0.0056494638622751996, 0.0287382005557633], [-0.32572303835890515, 0.07696403349213471, 0.17401678388737118, -0.03420124979083754, -0.11549061621522415], [-0.44905257963112, 0.08437642587871505, -0.05277235557631578, 0.28839382740062397, 0.07012134373826687], [0.258938436864333, -0.1114867630760053, 0.02062979792913688, -0.0029144768400706056, -0.02032899195260362], [0.240820545630616, -0.10164270400705734, 0.044739639952860163, -0.013304105489446447, -0.002494338757683312], [-0.3323886420908766, 0.13571761483659173, 0.10050050900924545, 0.2060642654488701, -0.006142209564833036], [-0.48435482694422416, -0.0038806293840811363, 0.0917639095488894, -0.013997363005751783, -0.056675642060725444], [0.26445862906153683, 0.038021083471936244, -0.039088858835817634, 0.030233469075147987, 0.01880300326884916], [0.24819346013645274, -0.11444832876741998, 0.0408043919349722, -0.0002749638137893078, -0.012914648097991704], [0.23048534713748675, 0.08148875466229809, -0.0870790404900401, -0.06070368645579805, -0.07082479638868157], [-0.5381998391962304, -0.08149768298436244, 0.003728379068020347, -0.05883326901379299, -0.024545090471501104], [0.16047284751480217, 0.22465520241932, -0.09534468798011791, 0.04502545100667157, 0.0368327240215615], [-0.5790028149297909, -0.07295202773035564, -0.12137823979642275, -0.07579615810598321, -0.04989904574014501], [0.21527739911812857, 0.11943351083602571, -0.09356004898100895, -0.07920629134071794, -0.08234002823292785], [-0.47625651558178406, -0.14508948304323654, -0.3581050177119676, 0.08203903132500373, -0.0011754015572142955], [0.28341299246286816, -0.0397033311432712, -0.03163217990317425, 0.020592857012751153, 0.010283468815750173]];\n",
       "  var pct_var = [57.76677409980787, 7.908201592271172, 6.507455218533565, 3.6518880627434247, 2.960429398562099];\n",
       "  var md_headers = ['SampleID', 'BarcodeSequence', 'LinkerPrimerSequence', 'center_name', 'center_project_name', 'emp_status', 'experiment_design_description', 'key_seq', 'library_construction_protocol', 'linker', 'platform', 'region', 'run_center', 'run_date', 'run_prefix', 'samp_size', 'sample_center', 'sequencing_meth', 'study_center', 'target_gene', 'target_subfragment', 'age', 'age_unit', 'altitude', 'anonymized_name', 'assigned_from_geo', 'body_habitat', 'body_product', 'body_site', 'collection_timestamp', 'country', 'depth', 'dna_extracted', 'elevation', 'env_biome', 'env_feature', 'env_matter', 'has_physical_specimen', 'host_subject_id', 'host_taxid', 'latitude', 'longitude', 'physical_specimen_remaining', 'project_name', 'required_sample_info_status', 'sample_type', 'sex', 'taxon_id', 'title', 'Description'];\n",
       "  var metadata = [['232.M9Thmr217', 'ACTACGTGTGGT', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '25', 'years', '0', 'M9Thmr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M9', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M3Rinr217', 'AGGTGTGATCGC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '33', 'years', '0', 'M3Rinr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M3', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M2Enter217', 'ACGGTGAGTGTC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Enter217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Ente', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Ente'], ['232.M3Midr217', 'AGGCTACACGAC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '33', 'years', '0', 'M3Midr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M3', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M3Tkey217', 'AGTCTACTCTGA', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M3Tkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M3', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Tkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Tkey'], ['232.M2Okey217', 'ACAGTTGCGCGA', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Okey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Okey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Okey'], ['232.M9Hkey217', 'AGAGTAGCTAAG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M9Hkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M9', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Hkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Hkey'], ['232.M9Pinl217', 'ACTACAGCCTAT', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '25', 'years', '0', 'M9Pinl217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M9', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M9Midl217', 'ACGTGCCGTAGA', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '25', 'years', '0', 'M9Midl217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M9', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M9Rinl217', 'ACGTTAGCACAC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '25', 'years', '0', 'M9Rinl217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M9', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M2Midr217', 'ACACACTATGGC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '36', 'years', '0', 'M2Midr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M2', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M2Lsft217', 'ACGGATCGTCAG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Lsft217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Left_shift', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Left_shift'], ['232.M9Mkey217', 'AGCATATGAGAG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M9Mkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M9', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Mkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Mkey'], ['232.M3Wkey217', 'AGTAGTATCCTC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M3Wkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M3', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Wkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Wkey'], ['232.M9Gkey217', 'AGAGCAAGAGCA', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M9Gkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M9', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Gkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Gkey'], ['232.M3Midl217', 'AGCTCCATACAG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '33', 'years', '0', 'M3Midl217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M3', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M3Pinl217', 'AGCTGACTAGTC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '33', 'years', '0', 'M3Pinl217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M3', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M9Vkey217', 'AGCACGAGCCTA', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M9Vkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M9', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Vkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Vkey'], ['232.M2Tkey217', 'ACAGAGTCGGCT', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Tkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Tkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Tkey'], ['232.M9Dkey217', 'AGACTGCGTACT', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M9Dkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M9', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Dkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Dkey'], ['232.M2Gkey217', 'ACCAGACGATGC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Gkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Gkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Gkey'], ['232.M3Lkey217', 'ATACTATTGCGC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M3Lkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M3', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Lkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Lkey'], ['232.M9Pkey217', 'AGAACACGTCTC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M9Pkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M9', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Pkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Pkey'], ['232.M2Ykey217', 'ACAGCAGTGGTC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Ykey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Ykey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Ykey'], ['232.M2Ikey217', 'ACAGTGCTTCAT', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Ikey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Ikey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Ikey'], ['232.M9Midr217', 'ACTATTGTCACG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '25', 'years', '0', 'M9Midr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M9', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M9Rinr217', 'ACTCACGGTATG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '25', 'years', '0', 'M9Rinr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M9', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M2Lkey217', 'ACCTGTCTCTCT', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Lkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Lkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Lkey'], ['232.M2Vkey217', 'ACGCAACTGCTA', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Vkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Vkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Vkey'], ['232.M3Gkey217', 'ATAATCTCGTCG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M3Gkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M3', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Gkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Gkey'], ['232.M9Indr217', 'ACTAGCTCCATA', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '25', 'years', '0', 'M9Indr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M9', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M3Ykey217', 'AGTCTCGCATAT', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M3Ykey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M3', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Ykey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Ykey'], ['232.M2Bkey217', 'ACGCGATACTGG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Bkey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Bkey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Bkey'], ['232.M3Pinr217', 'AGTACGCTCGAG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '33', 'years', '0', 'M3Pinr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M3', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M2Akey217', 'ACATGATCGTTC', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Akey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Akey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Akey'], ['232.M3Thmr217', 'AGCTTGACAGCT', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '33', 'years', '0', 'M3Thmr217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M3', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip'], ['232.M2Skey217', 'ACATGTCACGTG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', 'unknown', 'years', '0', 'M2Skey217', 'False', 'unknown', 'unknown', 'unknown', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:surface', 'ENVO:surface', 'ENVO:surface', 'True', 'M2', '36244', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'Skey', 'male', '408169', 'Forensic_identification_using_skin_bacterial_communities', 'Skey'], ['232.M3Rinl217', 'AGCTCTCAGAGG', 'CATGCTGCCTCCCGTAGGAGT', 'CCME', 'Forensic_identification_using_skin_bacterial_communities', 'EMP', 'Forensic_identification_using_skin_bacterial_communities', 'TCAG', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300gnt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', 'CA', 'FLX', '0', 'CCME', '8/14/08', 'FFCKVMW', '1, swab', 'CCME', 'pyrosequencing', 'CCME', '16S rRNA', 'V2', '33', 'years', '0', 'M3Rinl217', 'False', 'UBERON:skin', 'UBERON:sebum', 'UBERON:skin', '7/15/08', 'GAZ:United States of America', '0', 'True', '1624', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'ENVO:human-associated habitat', 'True', 'M3', '9606', '40.0083', '-105.2705', 'False', 'fierer_forensic_keyboard', 'completed', 'finger_tip', 'male', '539655', 'Forensic_identification_using_skin_bacterial_communities', 'finger_tip']];\n",
       "  var axesNames = ['PC1', 'PC2', 'PC3', 'PC4', 'PC5'];\n",
       "\n",
       "  var dm, ec;\n",
       "\n",
       "  function init() {\n",
       "    // Initialize the DecompositionModel\n",
       "    dm = new DecompositionModel(name, ids, coords, pct_var,\n",
       "                                md_headers, metadata, axesNames);\n",
       "    // Initialize the EmperorController\n",
       "    ec = new EmperorController(dm, 'emperor-notebook-0xc38db72e');\n",
       "  }\n",
       "\n",
       "  function animate() {\n",
       "    requestAnimationFrame(animate);\n",
       "    ec.render();\n",
       "  }\n",
       "  $(window).resize(function() {\n",
       "    ec.resize(div.innerWidth(), div.innerHeight());\n",
       "  });\n",
       "\n",
       "  $(function(){\n",
       "    init();\n",
       "    animate();\n",
       "\n",
       "  });\n",
       "\n",
       "}); // END REQUIRE.JS block\n",
       "</script>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<emperor.core.Emperor at 0x118882278>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "interact(evident, n=(200, 2000, 50),\n",
    "         metric=['unweighted_unifrac', 'weighted_unifrac', 'braycurtis', 'euclidean'],\n",
    "         __manual=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<div style='height:500px'></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Summarizing"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "Ready to be used!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "👍\n"
     ]
    }
   ],
   "source": [
    "print(b'\\xF0\\x9F\\x91\\x8D'.decode('utf-8'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "## version 1.0 $\\beta$ is out!"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "```bash\n",
    "pip install jupyter\n",
    "pip install emperor --pre\n",
    "\n",
    "\n",
    "conda install -c biocore emperor jupyter\n",
    "```"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Acknowledments"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "-"
    }
   },
   "source": [
    "- Thanks to all our users (**cited 99 times since 2013**)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "outputs": [],
   "source": [
    "from knightlab.members import current\n",
    "from knightlab.members import past\n",
    "from caporasolab.members import current as c_current\n",
    "\n",
    "__credits__ = ['Antonio Gonzalez', 'Joshua Shorenstein', 'Jamie Morton',\n",
    "               'Jose Navas', 'Rob Knight'] + current + past + c_current"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'We are hiring, contact robknight@ucsd.edu'"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "one_more_thing()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "fragment"
    }
   },
   "source": [
    "<table style='border:none; width:100%;' cellspacing=\"0\" cellpadding=\"0\">\n",
    "<tr style='border:none;'>\n",
    "<td style='border:none;'>\n",
    "<img src=\"https://upload.wikimedia.org/wikipedia/commons/f/f6/UCSD_logo.png\" alt=\"ucsd\" style=\"height: 50px;\"/>\n",
    "</td>\n",
    "<td style='border:none;'>\n",
    "<img src=\"./images/knight-logo.png\" alt=\"knight-lab\" style=\"height: 100px;\"/>\n",
    "</td>\n",
    "</tr>\n",
    "</table>"
   ]
  }
 ],
 "metadata": {
  "celltoolbar": "Slideshow",
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.1"
  },
  "widgets": {
   "state": {
    "17677a6304674f019c1ee7d53f747793": {
     "views": []
    },
    "2d01207ced7f414484f16f56e1ba830e": {
     "views": []
    },
    "3538694b46744935ae560c76ceb8c1ed": {
     "views": []
    },
    "6d68824a4ae94551844e3566bec2f449": {
     "views": []
    },
    "ae42cd6d165947cd9085cca86055cedc": {
     "views": []
    },
    "f47a2fe4ecb04d6892d275e10c00c878": {
     "views": []
    },
    "f7d8683df9304a15bc6403d42363c1e0": {
     "views": [
      {
       "cell_index": 83
      }
     ]
    },
    "fe964521953048ce80e0910bba898c47": {
     "views": []
    }
   },
   "version": "1.1.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
